diff --git a/assets/js/components/product.vue b/assets/js/components/product.vue
index e2cec9c2..e721440f 100644
--- a/assets/js/components/product.vue
+++ b/assets/js/components/product.vue
@@ -73,7 +73,7 @@
 </template>
 
 <script>
-import { addItemToCart, getCart, getCartTotalItems } from '@/services/cart-service.js';
+import { addItemToCart } from '@/services/cart-service.js';
 import { getProduct } from '@/services/products-service';
 import ColorSelector from '@/components/color-selector';
 import Loading from '@/components/loading';
@@ -95,7 +95,6 @@ export default {
     },
     data() {
         return {
-            cart: { items: [] },
             product: {
                 name: '', image: '', price: 0, colors: [],
             },
@@ -128,16 +127,6 @@ export default {
 
         this.product = response.data;
     },
-    async mounted() {
-        try {
-            this.cart = await getCart();
-        } catch (e) {
-            return;
-        }
-
-        document.getElementById('js-shopping-cart-items')
-            .innerHTML = getCartTotalItems(this.cart).toString();
-    },
     methods: {
         async addToCart() {
             if (this.product.colors.length && this.selectedColorId === null) {
diff --git a/assets/js/pages/products.vue b/assets/js/pages/products.vue
index 391599f0..073532ce 100644
--- a/assets/js/pages/products.vue
+++ b/assets/js/pages/products.vue
@@ -26,6 +26,7 @@ import Product from '@/components/product';
 import Sidebar from '@/components/sidebar';
 import { getCurrentCategoryId, getCurrentProductId } from '@/services/page-context';
 import { fetchCategories } from '@/services/categories-service';
+import { getCart, getCartTotalItems } from '@/services/cart-service';
 
 export default {
     name: 'Products',
@@ -36,9 +37,10 @@ export default {
     },
     data() {
         return {
-            sidebarCollapsed: false,
+            cart: { items: [] },
             categories: [],
             currentCategoryId: getCurrentCategoryId(),
+            sidebarCollapsed: false,
         };
     },
     computed: {
@@ -70,6 +72,16 @@ export default {
 
         this.categories = response.data['hydra:member'];
     },
+    async mounted() {
+        try {
+            this.cart = await getCart();
+        } catch (e) {
+            return;
+        }
+
+        document.getElementById('js-shopping-cart-items')
+            .innerHTML = getCartTotalItems(this.cart).toString();
+    },
     methods: {
         toggleSidebarCollapsed() {
             this.sidebarCollapsed = !this.sidebarCollapsed;
