diff --git a/assets/components/product.vue b/assets/components/product.vue
index 00688d41f..1d8186a59 100644
--- a/assets/components/product.vue
+++ b/assets/components/product.vue
@@ -91,6 +91,10 @@ export default {
             type: String,
             default: null,
         },
+        cart: {
+            type: Object,
+            default: null,
+        },
     },
     data() {
         return {
diff --git a/assets/pages/products.vue b/assets/pages/products.vue
index 6ad70b20c..1dcc90b02 100644
--- a/assets/pages/products.vue
+++ b/assets/pages/products.vue
@@ -60,6 +60,7 @@ export default {
             return this.currentProductId !== null
                 ? {
                     currentProductId: this.currentProductId,
+                    cart: this.cart,
                 }
                 : {
                     currentCategoryId: this.currentCategoryId,
