diff --git a/assets/js/components/product.vue b/assets/js/components/product.vue
index 3ffd3f78..0f855fd1 100644
--- a/assets/js/components/product.vue
+++ b/assets/js/components/product.vue
@@ -93,6 +93,7 @@ export default {
     },
     data() {
         return {
+            cart: { items: [] },
             product: {
                 name: '', image: '', price: 0, colors: [],
             },
@@ -125,6 +126,13 @@ export default {
 
         this.product = response.data;
     },
+    async mounted() {
+        try {
+            this.cart = await getCart();
+        } catch (e) {
+            // do nothing
+        }
+    },
     methods: {
         addToCart() {
             // TODO
