diff --git a/assets/components/product-show.vue b/assets/components/product-show.vue
index ed0606a08..e2ea9e774 100644
--- a/assets/components/product-show.vue
+++ b/assets/components/product-show.vue
@@ -55,6 +55,9 @@
                             @click="addToCart"
                         >
                             Add to Cart
+                            <i
+                                class="fas fa-spinner fa-spin"
+                                v-show="addToCartLoading" />
                         </button>
                     </div>
                 </div>
@@ -87,6 +90,7 @@ export default {
     data() {
         return {
             cart: null,
+            addToCartLoading: false,
             product: null,
             loading: true,
         };
@@ -112,12 +116,14 @@ export default {
         }
     },
     methods: {
-        addToCart() {
-            addItemToCart(this.cart, {
+        async addToCart() {
+            this.addToCartLoading = true;
+            await addItemToCart(this.cart, {
                 product: this.product['@id'],
                 color: null,
                 quantity: 1,
             });
+            this.addToCartLoading = false;
         },
     },
 };
