diff --git a/assets/js/services/cart.js b/assets/js/services/cart.js
index 1537ac5..6788fec 100644
--- a/assets/js/services/cart.js
+++ b/assets/js/services/cart.js
@@ -59,6 +59,15 @@ const cart = {
         cart.saveStorage([]);
     },
 
+    /**
+     * Returns the total number of items in our shopping cart
+     *
+     * @return {number}
+     */
+    totalItems() {
+        return cart.getStorage().reduce((acc, item) => (acc + item.qty), 0);
+    },
+
     /**
      * Initializes the local storage for our shopping cart
      */
