diff --git a/assets/components/product-show.vue b/assets/components/product-show.vue
index bc1de5aea..af93a54d0 100644
--- a/assets/components/product-show.vue
+++ b/assets/components/product-show.vue
@@ -62,6 +62,10 @@
                                 v-show="addToCartLoading"
                                 class="fas fa-spinner fa-spin"
                             />
+                            <i
+                                v-show="addToCartSuccess"
+                                class="fas fa-check"
+                            />
                         </button>
                     </div>
                 </div>
@@ -95,6 +99,7 @@ export default {
         return {
             cart: null,
             addToCartLoading: false,
+            addToCartSuccess: false,
             product: null,
             loading: true,
         };
@@ -122,12 +127,14 @@ export default {
     methods: {
         async addToCart() {
             this.addToCartLoading = true;
+            this.addToCartSuccess = false;
             await addItemToCart(this.cart, {
                 product: this.product['@id'],
                 color: null,
                 quantity: 1,
             });
             this.addToCartLoading = false;
+            this.addToCartSuccess = true;
         },
     },
 };
