diff --git a/assets/components/product-show.vue b/assets/components/product-show.vue
index 544cc149a..66b06ae9f 100644
--- a/assets/components/product-show.vue
+++ b/assets/components/product-show.vue
@@ -55,6 +55,7 @@
                         <button
                             class="btn btn-info btn-sm"
                             :disabled="cart === null"
+                            @click="addToCart"
                         >
                             Add to Cart
                         </button>
@@ -66,7 +67,7 @@
 </template>
 
 <script>
-import { fetchCart } from '@/services/cart-service.js';
+import { fetchCart, addItemToCart } from '@/services/cart-service.js';
 import formatPrice from '@/helpers/format-price';
 import { fetchOneProduct } from '@/services/products-service';
 import ColorSelector from '@/components/color-selector';
@@ -113,6 +114,15 @@ export default {
             this.loading = false;
         }
     },
+    methods: {
+        addToCart() {
+            addItemToCart(this.cart, {
+                product: this.product['@id'],
+                color: null,
+                quantity: 1,
+            });
+        },
+    },
 };
 </script>
 
