diff --git a/assets/js/components/product.vue b/assets/js/components/product.vue
index 060a7b86..679f7322 100644
--- a/assets/js/components/product.vue
+++ b/assets/js/components/product.vue
@@ -95,6 +95,7 @@ export default {
     },
     data() {
         return {
+            cart: { items: [] },
             product: {
                 name: '', image: '', price: 0, colors: [],
             },
@@ -127,6 +128,13 @@ export default {
 
         this.product = response.data;
     },
+    async mounted() {
+        try {
+            this.cart = await getCart();
+        } catch (e) {
+            // do nothing
+        }
+    },
     methods: {
         addToCart() {
             // TODO
