diff --git a/assets/js/components/product.vue b/assets/js/components/product.vue
index 7a3f8953..dae86825 100644
--- a/assets/js/components/product.vue
+++ b/assets/js/components/product.vue
@@ -60,6 +60,7 @@
 </template>
 
 <script>
+import cartService from '@/services/cart.js';
 import formatPrice from '@/helpers/format-price';
 import productsService from '@/services/products';
 import ColorSelector from '@/components/color-selector';
