diff --git a/assets/js/components/product.vue b/assets/js/components/product.vue
index 5e475d3d..b127fae2 100644
--- a/assets/js/components/product.vue
+++ b/assets/js/components/product.vue
@@ -71,7 +71,7 @@
 </template>
 
 <script>
-import { addItemToCart, getCart } from '@/services/cart-service.js';
+import { addItemToCart, getCart, getCartTotalItems } from '@/services/cart-service.js';
 import formatPrice from '@/helpers/format-price';
 import { getProduct } from '@/services/products-service';
 import ColorSelector from '@/components/color-selector';
@@ -130,8 +130,11 @@ export default {
         try {
             this.cart = await getCart();
         } catch (e) {
-            // do nothing
+            return;
         }
+
+        document.getElementById('js-shopping-cart-items')
+            .innerHTML = getCartTotalItems(this.cart).toString();
     },
     methods: {
         async addToCart() {
