diff --git a/assets/components/product-show.vue b/assets/components/product-show.vue
index d0e8e7e19..c482205a0 100644
--- a/assets/components/product-show.vue
+++ b/assets/components/product-show.vue
@@ -115,6 +115,22 @@ export default {
             // do nothing
         }
     },
+    methods: {
+        async addToCart() {
+            if (this.product.colors.length && this.selectedColorId === null) {
+                alert('Please, select a product color first!');
+                return;
+            }
+
+            await addItemToCart(this.cart, {
+                product: this.product['@id'],
+                color: this.selectedColorId,
+                quantity: this.qty,
+            });
+
+            window.location = '/';
+        },
+    },
 };
 </script>
 
