diff --git a/assets/js/components/product.vue b/assets/js/components/product.vue
index 6bf0d27c..7e81067c 100644
--- a/assets/js/components/product.vue
+++ b/assets/js/components/product.vue
@@ -67,8 +67,6 @@ import ColorSelector from '@/components/color-selector';
 import Loading from '@/components/loading';
 import TitleComponent from '@/components/title';
 
-window.cartService = cartService;
-
 export default {
     name: 'Product',
     components: {
@@ -114,7 +112,14 @@ export default {
     },
     methods: {
         addToCart() {
-            // TODO
+            if (this.product.colors.length && this.selectedColorId === null) {
+                alert('Please, select a product color first!');
+                return;
+            }
+
+            cartService.addItem(this.product['@id'], this.selectedColorId, this.qty);
+
+            window.location = '/';
         },
 
         /**
