diff --git a/assets/js/components/product.vue b/assets/js/components/product.vue
index 9e6e8bbc..c465593d 100644
--- a/assets/js/components/product.vue
+++ b/assets/js/components/product.vue
@@ -59,6 +59,7 @@
 
                         <button
                             class="btn btn-info btn-sm"
+                            :disabled="cart === null"
                             @click="addToCart"
                         >
                             Add to Cart
