diff --git a/templates/base.html.twig b/templates/base.html.twig
index 4c09c43..7a32d29 100644
--- a/templates/base.html.twig
+++ b/templates/base.html.twig
@@ -34,7 +34,7 @@
                     </li>
                     <li class="nav-item">
                         <a class="nav-link" href="#">
-                            Shopping Cart (<span>0</span>)
+                            Shopping Cart (<span id="shopping-cart-items">0</span>)
                         </a>
                     </li>
                 </ul>
