diff --git a/assets/components/product-show.vue b/assets/components/product-show.vue
index c5ef8d15c..8fa298075 100644
--- a/assets/components/product-show.vue
+++ b/assets/components/product-show.vue
@@ -37,7 +37,9 @@
 
                 <div class="p-3 row align-items-center">
                     <div class="col-3" v-if="product.colors.length !== 0">
-                        <color-selector />
+                        <color-selector
+                            @color-selected="updateSelectedColor"
+                        />
                     </div>
 
                     <div class="col-2">
@@ -95,6 +97,7 @@ export default {
         return {
             cart: null,
             quantity: 1,
+            selectedColorId: null,
             addToCartLoading: false,
             addToCartSuccess: false,
             product: null,
@@ -123,16 +126,25 @@ export default {
     },
     methods: {
         async addToCart() {
+            if (this.product.colors.length && this.selectedColorId === null) {
+                alert('Please select a color first!');
+                return;
+            }
+
             this.addToCartLoading = true;
             this.addToCartSuccess = false;
             this.cart = await addItemToCart(this.cart, {
                 product: this.product['@id'],
-                color: null,
+                color: this.selectedColorId,
                 quantity: this.quantity,
             });
             this.addToCartLoading = false;
             this.addToCartSuccess = true;
         },
+
+        updateSelectedColor(iri) {
+            this.selectedColorId = iri;
+        },
     },
 };
 </script>
