diff --git a/assets/components/product-show.vue b/assets/components/product-show.vue
index dfba8ccd3..641365b59 100644
--- a/assets/components/product-show.vue
+++ b/assets/components/product-show.vue
@@ -45,6 +45,7 @@
 
                     <div class="col-2">
                         <input
+                            v-model.number="quantity"
                             class="form-control"
                             type="number"
                             min="1"
@@ -98,6 +99,7 @@ export default {
     data() {
         return {
             cart: null,
+            quantity: 1,
             addToCartLoading: false,
             addToCartSuccess: false,
             product: null,
@@ -131,7 +133,7 @@ export default {
             await addItemToCart(this.cart, {
                 product: this.product['@id'],
                 color: null,
-                quantity: 1,
+                quantity: this.quantity,
             });
             this.addToCartLoading = false;
             this.addToCartSuccess = true;
