diff --git a/assets/components/product-show.vue b/assets/components/product-show.vue
index dba286046..04ca89058 100644
--- a/assets/components/product-show.vue
+++ b/assets/components/product-show.vue
@@ -86,6 +86,7 @@ export default {
     },
     data() {
         return {
+            cart: null,
             product: null,
             loading: true,
         };
@@ -106,6 +107,13 @@ export default {
             this.loading = false;
         }
     },
+    async mounted() {
+        try {
+            this.cart = await getCart();
+        } catch (e) {
+            // do nothing
+        }
+    },
 };
 </script>
 
