diff --git a/assets/components/product-show.vue b/assets/components/product-show.vue
index ed324fe72..bc1de5aea 100644
--- a/assets/components/product-show.vue
+++ b/assets/components/product-show.vue
@@ -58,6 +58,10 @@
                             @click="addToCart"
                         >
                             Add to Cart
+                            <i
+                                v-show="addToCartLoading"
+                                class="fas fa-spinner fa-spin"
+                            />
                         </button>
                     </div>
                 </div>
@@ -90,6 +94,7 @@ export default {
     data() {
         return {
             cart: null,
+            addToCartLoading: false,
             product: null,
             loading: true,
         };
@@ -115,12 +120,14 @@ export default {
         }
     },
     methods: {
-        addToCart() {
-            addItemToCart(this.cart, {
+        async addToCart() {
+            this.addToCartLoading = true;
+            await addItemToCart(this.cart, {
                 product: this.product['@id'],
                 color: null,
                 quantity: 1,
             });
+            this.addToCartLoading = false;
         },
     },
 };
