diff --git a/assets/components/product-show.vue b/assets/components/product-show.vue
index 922354b5..a82bea80 100644
--- a/assets/components/product-show.vue
+++ b/assets/components/product-show.vue
@@ -54,6 +54,10 @@
                                 @click="addToCart"
                             >
                                 Add to Cart
+                                <i
+                                    v-show="addToCartLoading"
+                                    class="fas fa-spinner fa-spin"
+                                />
                             </button>
                         </div>
                     </div>
@@ -87,6 +91,7 @@ export default {
     data() {
         return {
             cart: null,
+            addToCartLoading: false,
             product: null,
             loading: true,
         };
@@ -112,12 +117,14 @@ export default {
         }
     },
     methods: {
-        addToCart() {
-            addItemToCart(this.cart, {
+        async addToCart() {
+            this.addToCartLoading = true;
+            await addItemToCart(this.cart, {
                 product: this.product['@id'],
                 color: null,
                 quantity: 1,
             });
+            this.addToCartLoading = false;
         },
     },
 };
