diff --git a/templates/base.html.twig b/templates/base.html.twig
index 53b0297d..62c38d12 100644
--- a/templates/base.html.twig
+++ b/templates/base.html.twig
@@ -41,7 +41,7 @@
                     {% endif %}
                     <li class="nav-item">
                         <a class="nav-link" href="#">
-                            Shopping Cart (<span>0</span>)
+                            Shopping Cart (<span id="js-shopping-cart-items">0</span>)
                         </a>
                     </li>
                 </ul>
