diff --git a/assets/js/components/product.vue b/assets/js/components/product.vue
index a7b92fbe..00688d41 100644
--- a/assets/js/components/product.vue
+++ b/assets/js/components/product.vue
@@ -72,7 +72,7 @@
 </template>
 
 <script>
-import { addItemToCart, getCart, getCartTotalItems } from '@/services/cart-service.js';
+import { addItemToCart } from '@/services/cart-service.js';
 import formatPrice from '@/helpers/format-price';
 import { getProduct } from '@/services/products-service';
 import ColorSelector from '@/components/color-selector';
@@ -94,7 +94,6 @@ export default {
     },
     data() {
         return {
-            cart: null,
             product: {
                 name: '', image: '', price: 0, colors: [],
             },
@@ -127,16 +126,6 @@ export default {
 
         this.product = response.data;
     },
-    async mounted() {
-        try {
-            this.cart = await getCart();
-        } catch (e) {
-            return;
-        }
-
-        document.getElementById('js-shopping-cart-items')
-            .innerHTML = getCartTotalItems(this.cart).toString();
-    },
     methods: {
         async addToCart() {
             if (this.product.colors.length && this.selectedColorId === null) {
diff --git a/assets/js/pages/products.vue b/assets/js/pages/products.vue
index 391599f0..6ad70b20 100644
--- a/assets/js/pages/products.vue
+++ b/assets/js/pages/products.vue
@@ -26,6 +26,7 @@ import Product from '@/components/product';
 import Sidebar from '@/components/sidebar';
 import { getCurrentCategoryId, getCurrentProductId } from '@/services/page-context';
 import { fetchCategories } from '@/services/categories-service';
+import { getCart, getCartTotalItems } from '@/services/cart-service';
 
 export default {
     name: 'Products',
@@ -36,6 +37,7 @@ export default {
     },
     data() {
         return {
+            cart: null,
             sidebarCollapsed: false,
             categories: [],
             currentCategoryId: getCurrentCategoryId(),
@@ -70,6 +72,16 @@ export default {
 
         this.categories = response.data['hydra:member'];
     },
+    async mounted() {
+        try {
+            this.cart = await getCart();
+        } catch (e) {
+            return;
+        }
+
+        document.getElementById('js-shopping-cart-items')
+            .innerHTML = getCartTotalItems(this.cart).toString();
+    },
     methods: {
         toggleSidebarCollapsed() {
             this.sidebarCollapsed = !this.sidebarCollapsed;
