diff --git a/assets/components/product-show.vue b/assets/components/product-show.vue
index c482205a0..60316416b 100644
--- a/assets/components/product-show.vue
+++ b/assets/components/product-show.vue
@@ -65,7 +65,7 @@
 </template>
 
 <script>
-import { addItemToCart, getCart } from '@/services/cart-service.js';
+import { addItemToCart, getCart, getCartTotalItems } from '@/services/cart-service.js';
 import formatPrice from '@/helpers/format-price';
 import { fetchOneProduct } from '@/services/products-service';
 import ColorSelector from '@/components/color-selector';
@@ -112,8 +112,11 @@ export default {
         try {
             this.cart = await getCart();
         } catch (e) {
-            // do nothing
+            return;
         }
+
+        document.getElementById('js-shopping-cart-items')
+            .innerHTML = getCartTotalItems(this.cart).toString();
     },
     methods: {
         async addToCart() {
