diff --git a/assets/components/product-show.vue b/assets/components/product-show.vue
index ac8a1419..63a6913d 100644
--- a/assets/components/product-show.vue
+++ b/assets/components/product-show.vue
@@ -50,6 +50,7 @@
 
                             <button
                                 class="btn btn-info btn-sm"
+                                :disabled="cart === null"
                             >
                                 Add to Cart
                             </button>
