diff --git a/assets/components/product-show.vue b/assets/components/product-show.vue
index ba068de17..4a8952023 100644
--- a/assets/components/product-show.vue
+++ b/assets/components/product-show.vue
@@ -87,6 +87,7 @@ export default {
     },
     data() {
         return {
+            cart: null,
             product: null,
             loading: true,
         };
@@ -101,6 +102,10 @@ export default {
         },
     },
     async created() {
+        fetchCart().then((cart) => {
+            this.cart = cart;
+        });
+
         try {
             this.product = (await fetchOneProduct(this.productId)).data;
         } finally {
