diff --git a/assets/js/components/product.vue b/assets/js/components/product.vue
index fd00ca9a..5499f76e 100644
--- a/assets/js/components/product.vue
+++ b/assets/js/components/product.vue
@@ -60,12 +60,15 @@
 </template>
 
 <script>
+import cartService from '@/services/cart.js';
 import ColorSelector from '@/components/color-selector';
 import { getProduct } from '@/services/products-service';
 import Loading from '@/components/loading';
 import TitleComponent from '@/components/title';
 import formatPrice from '@/helpers/format-price';
 
+window.cartService = cartService;
+
 export default {
     name: 'Product',
     components: {
