diff --git a/assets/components/product-show.vue b/assets/components/product-show.vue
index a131b948f..f41af964f 100644
--- a/assets/components/product-show.vue
+++ b/assets/components/product-show.vue
@@ -54,6 +54,7 @@
                     <div class="col">
                         <button
                             class="btn btn-info btn-sm"
+                            :disabled="cart === null"
                         >
                             Add to Cart
                         </button>
