diff --git a/assets/js/components/product.vue b/assets/js/components/product.vue
index e721440f..ab799e47 100644
--- a/assets/js/components/product.vue
+++ b/assets/js/components/product.vue
@@ -92,6 +92,10 @@ export default {
             type: String,
             default: null,
         },
+        cart: {
+            type: Object,
+            required: true,
+        },
     },
     data() {
         return {
diff --git a/assets/js/pages/products.vue b/assets/js/pages/products.vue
index 073532ce..a8d7553b 100644
--- a/assets/js/pages/products.vue
+++ b/assets/js/pages/products.vue
@@ -60,6 +60,7 @@ export default {
             return this.currentProductId !== null
                 ? {
                     currentProductId: this.currentProductId,
+                    cart: this.cart,
                 }
                 : {
                     currentCategoryId: this.currentCategoryId,
