diff --git a/assets/components/product-show.vue b/assets/components/product-show.vue
index 641365b59..d802ddbf1 100644
--- a/assets/components/product-show.vue
+++ b/assets/components/product-show.vue
@@ -40,7 +40,9 @@
                         v-if="product.colors.length !== 0"
                         class="col-3"
                     >
-                        <color-selector />
+                        <color-selector
+                            @color-selected="updateSelectedColor"
+                        />
                     </div>
 
                     <div class="col-2">
@@ -100,6 +102,7 @@ export default {
         return {
             cart: null,
             quantity: 1,
+            selectedColorId: null,
             addToCartLoading: false,
             addToCartSuccess: false,
             product: null,
@@ -128,16 +131,25 @@ export default {
     },
     methods: {
         async addToCart() {
+            if (this.product.colors.length && this.selectedColorId === null) {
+                alert('Please select a color first!');
+                return;
+            }
+
             this.addToCartLoading = true;
             this.addToCartSuccess = false;
             await addItemToCart(this.cart, {
                 product: this.product['@id'],
-                color: null,
+                color: this.selectedColorId,
                 quantity: this.quantity,
             });
             this.addToCartLoading = false;
             this.addToCartSuccess = true;
         },
+
+        updateSelectedColor(iri) {
+            this.selectedColorId = iri;
+        },
     },
 };
 </script>
