diff --git a/assets/components/product-show.vue b/assets/components/product-show.vue
index ff6fa1097..a8881baf2 100644
--- a/assets/components/product-show.vue
+++ b/assets/components/product-show.vue
@@ -84,6 +84,10 @@ export default {
             type: String,
             required: true,
         },
+        cart: {
+            type: Object,
+            default: null,
+        },
     },
     data() {
         return {
diff --git a/assets/pages/products.vue b/assets/pages/products.vue
index efed492eb..0493b3aee 100644
--- a/assets/pages/products.vue
+++ b/assets/pages/products.vue
@@ -59,6 +59,7 @@ export default {
         currentProps() {
             return this.currentComponent === ProductShow ? {
                 productId: this.currentProductId,
+                cart: this.cart,
             } : {
                 currentCategoryId: this.currentCategoryId,
                 categories: this.categories,
