diff --git a/assets/components/product-show.vue b/assets/components/product-show.vue
index 63a6913d..922354b5 100644
--- a/assets/components/product-show.vue
+++ b/assets/components/product-show.vue
@@ -51,6 +51,7 @@
                             <button
                                 class="btn btn-info btn-sm"
                                 :disabled="cart === null"
+                                @click="addToCart"
                             >
                                 Add to Cart
                             </button>
@@ -63,7 +64,7 @@
 </template>
 
 <script>
-import { fetchCart } from '@/services/cart-service.js';
+import { fetchCart, addItemToCart } from '@/services/cart-service.js';
 import formatPrice from '@/helpers/format-price';
 import { fetchOneProduct } from '@/services/products-service';
 import ColorSelector from '@/components/color-selector';
@@ -110,6 +111,15 @@ export default {
             this.loading = false;
         }
     },
+    methods: {
+        addToCart() {
+            addItemToCart(this.cart, {
+                product: this.product['@id'],
+                color: null,
+                quantity: 1,
+            });
+        },
+    },
 };
 </script>
 
