diff --git a/assets/js/components/product.vue b/assets/js/components/product.vue
index c6a4d8d2..93b3c7e3 100644
--- a/assets/js/components/product.vue
+++ b/assets/js/components/product.vue
@@ -73,12 +73,15 @@
 </template>
 
 <script>
+import cartService from '@/services/cart-service.js';
 import { getProduct } from '@/services/products-service';
 import ColorSelector from '@/components/color-selector';
 import Loading from '@/components/loading';
 import TitleComponent from '@/components/title';
 import formatPrice from '@/helpers/format-price';
 
+window.cartService = cartService;
+
 export default {
     name: 'Product',
     components: {
