diff --git a/assets/components/product-show.vue b/assets/components/product-show.vue
index 4a94bdc09..eb9352fff 100644
--- a/assets/components/product-show.vue
+++ b/assets/components/product-show.vue
@@ -40,6 +40,7 @@
                         <div class="d-flex align-items-center justify-content-center">
                             <color-selector
                                 v-if="product.colors.length !== 0"
+                                @color-selected="updateSelectedColor"
                             />
 
                             <input
@@ -97,6 +98,7 @@ export default {
         return {
             cart: null,
             quantity: 1,
+            selectedColorId: null,
             addToCartLoading: false,
             addToCartSuccess: false,
             product: null,
@@ -125,16 +127,25 @@ export default {
     },
     methods: {
         async addToCart() {
+            if (this.product.colors.length && this.selectedColorId === null) {
+                alert('Please select a color first!');
+                return;
+            }
+
             this.addToCartLoading = true;
             this.addToCartSuccess = false;
             await addItemToCart(this.cart, {
                 product: this.product['@id'],
-                color: null,
+                color: this.selectedColorId,
                 quantity: this.quantity,
             });
             this.addToCartLoading = false;
             this.addToCartSuccess = true;
         },
+
+        updateSelectedColor(iri) {
+            this.selectedColorId = iri;
+        },
     },
 };
 </script>
