diff --git a/assets/components/product-show.vue b/assets/components/product-show.vue
index 8631a95c4..0a6f6bbc5 100644
--- a/assets/components/product-show.vue
+++ b/assets/components/product-show.vue
@@ -65,6 +65,7 @@
 </template>
 
 <script>
+import { getCart } from '@/services/cart-service.js';
 import formatPrice from '@/helpers/format-price';
 import { fetchOneProduct } from '@/services/products-service';
 import ColorSelector from '@/components/color-selector';
