diff --git a/assets/js/components/product.vue b/assets/js/components/product.vue
index c6a4d8d2..36ab7a5f 100644
--- a/assets/js/components/product.vue
+++ b/assets/js/components/product.vue
@@ -73,6 +73,7 @@
 </template>
 
 <script>
+import { addItemToCart, getCart } from '@/services/cart-service.js';
 import { getProduct } from '@/services/products-service';
 import ColorSelector from '@/components/color-selector';
 import Loading from '@/components/loading';
