diff --git a/assets/js/components/product.vue b/assets/js/components/product.vue
index cbb3dda7..e2cec9c2 100644
--- a/assets/js/components/product.vue
+++ b/assets/js/components/product.vue
@@ -73,7 +73,7 @@
 </template>
 
 <script>
-import { addItemToCart, getCart } from '@/services/cart-service.js';
+import { addItemToCart, getCart, getCartTotalItems } from '@/services/cart-service.js';
 import { getProduct } from '@/services/products-service';
 import ColorSelector from '@/components/color-selector';
 import Loading from '@/components/loading';
@@ -132,8 +132,11 @@ export default {
         try {
             this.cart = await getCart();
         } catch (e) {
-            // do nothing
+            return;
         }
+
+        document.getElementById('js-shopping-cart-items')
+            .innerHTML = getCartTotalItems(this.cart).toString();
     },
     methods: {
         async addToCart() {
