diff --git a/assets/js/components/product.vue b/assets/js/components/product.vue
index 15ef8dc0..24ded68c 100644
--- a/assets/js/components/product.vue
+++ b/assets/js/components/product.vue
@@ -62,6 +62,7 @@
 import { getProduct } from '@/services/products-service';
 import Loading from '@/components/loading';
 import TitleComponent from '@/components/title';
+import formatPrice from '@/helpers/format-price';
 
 export default {
     name: 'Product',
@@ -85,6 +86,15 @@ export default {
             loading: true,
         };
     },
+    computed: {
+        /**
+         * Returns a formatted price for the product
+         * @returns {string}
+         */
+        price() {
+            return formatPrice(this.product.price);
+        },
+    },
     async created() {
         let response;
 
