diff --git a/assets/js/components/product.vue b/assets/js/components/product.vue
index 24cfd99f..21c7f716 100644
--- a/assets/js/components/product.vue
+++ b/assets/js/components/product.vue
@@ -1,12 +1,21 @@
 <template>
-    <div>
-        // Product goes here
+    <div class="row">
+        <div :class="$style['top-bar']">
+            <title-component />
+        </div>
+
+        TODO: Show product here!
     </div>
 </template>
 
 <script>
+import TitleComponent from '@/components/title';
+
 export default {
     name: 'Product',
+    components: {
+        TitleComponent,
+    },
     props: {
         currentProductId: {
             type: Number,
@@ -19,3 +28,7 @@ export default {
     },
 };
 </script>
+
+<style lang="scss" module>
+@import '~styles/top-bar.scss';
+</style>
