diff --git a/assets/js/services/products.js b/assets/js/services/products.js
index 944dfcaf..2b936e1f 100644
--- a/assets/js/services/products.js
+++ b/assets/js/services/products.js
@@ -16,6 +16,19 @@ const products = {
             params: generateListingParams(category, searchTerm),
         });
     },
+
+    /**
+     * Gets a product from the database according to the product id
+     *
+     * @param {number} id
+     * @return {Promise}
+     */
+    getProduct(id) {
+        return axios({
+            method: 'get',
+            url: `/api/products/${id}`,
+        });
+    },
 };
 
 export default products;
