diff --git a/assets/js/components/product.vue b/assets/js/components/product.vue
index f00aada..1d5744b 100644
--- a/assets/js/components/product.vue
+++ b/assets/js/components/product.vue
@@ -1,12 +1,22 @@
 <template>
-    <div>
-        // Product goes here
+    <div class="row">
+        <div :class="$style['top-bar']">
+            <title-component />
+        </div>
+
+        TODO: Show product here!
     </div>
 </template>
 
 <script>
+import productsService from '@/services/products';
+import TitleComponent from '@/components/title';
+
 export default {
     name: 'Product',
+    components: {
+        TitleComponent,
+    },
     props: {
         currentCategoryId: {
             type: Number,
@@ -21,5 +31,19 @@ export default {
             default: () => ([]),
         },
     },
+    data: () => ({
+        product: null,
+        loading: true,
+    }),
+    async created() {
+        try {
+            const response = await productsService.getProduct(this.currentProductId);
+
+            this.product = response.data;
+            this.loading = false;
+        } catch (e) {
+            this.loading = false;
+        }
+    },
 };
 </script>
