diff --git a/assets/js/components/product.vue b/assets/js/components/product.vue
index b927d209..3f874853 100644
--- a/assets/js/components/product.vue
+++ b/assets/js/components/product.vue
@@ -10,7 +10,14 @@
             <div
                 :class="[$style.product, 'col-11', 'row']"
             >
-                <div class="col-8">
+                <div class="col-12 p-3">
+                    <loading v-show="loading" />
+                </div>
+
+                <div
+                    v-show="!loading"
+                    class="col-8"
+                >
                     <img
                         class="d-block"
                         :src="product.image"
@@ -27,7 +34,10 @@
                     </div>
                 </div>
 
-                <div class="col-4">
+                <div
+                    v-show="!loading"
+                    class="col-4"
+                >
                     <div class="p-3 row">
                         <p v-text="product.description" />
 
@@ -49,11 +59,13 @@
 
 <script>
 import { getProduct } from '@/services/products-service';
+import Loading from '@/components/loading';
 import TitleComponent from '@/components/title';
 
 export default {
     name: 'Product',
     components: {
+        Loading,
         TitleComponent,
     },
     props: {
