diff --git a/assets/services/products-service.js b/assets/services/products-service.js
index 62681e633..63078a73f 100644
--- a/assets/services/products-service.js
+++ b/assets/services/products-service.js
@@ -19,3 +19,13 @@ export function fetchProducts(categoryIri, searchTerm) {
         params,
     });
 }
+
+/**
+ * Gets a product from the API according to the IRI.
+ *
+ * @param {string} iri
+ * @return {Promise}
+ */
+export function fetchOneProduct(iri) {
+    return axios.get(iri);
+}
