diff --git a/assets/js/components/product.vue b/assets/js/components/product.vue
index de6a3b04..ea1dbf8b 100644
--- a/assets/js/components/product.vue
+++ b/assets/js/components/product.vue
@@ -72,6 +72,16 @@ export default {
         product: { name: '', image: '', price: 0 },
         loading: true,
     }),
+    computed: {
+        /**
+         * Returns a formatted price for the product
+         * @return {string}
+         */
+        price() {
+            return (this.product.price / 100)
+                .toLocaleString('en-US', { minimumFractionDigits: 2 });
+        },
+    },
     async created() {
         try {
             const response = await getProduct(this.currentProductId);
