diff --git a/assets/js/components/product.vue b/assets/js/components/product.vue
index 1a1f8e89..3d1b422a 100644
--- a/assets/js/components/product.vue
+++ b/assets/js/components/product.vue
@@ -79,8 +79,7 @@ export default {
          * @return {string}
          */
         price() {
-            return (this.product.price / 100)
-                .toLocaleString('en-US', { minimumFractionDigits: 2 });
+            return formatPrice(this.product.price);
         },
     },
     async created() {
