diff --git a/assets/js/components/product.vue b/assets/js/components/product.vue
index dbfd65af..f0a7953a 100644
--- a/assets/js/components/product.vue
+++ b/assets/js/components/product.vue
@@ -1,12 +1,21 @@
 <template>
-    <div>
-        // Product goes here
+    <div class="row">
+        <div class="col-3">
+            <title-component />
+        </div>
+
+        TODO: Show product here!
     </div>
 </template>
 
 <script>
+import TitleComponent from '@/components/title';
+
 export default {
     name: 'Product',
+    components: {
+        TitleComponent,
+    },
     props: {
         currentProductId: {
             type: String,
