diff --git a/assets/js/services/products-service.js b/assets/js/services/products-service.js
index 62681e63..a4544239 100644
--- a/assets/js/services/products-service.js
+++ b/assets/js/services/products-service.js
@@ -19,3 +19,16 @@ export function fetchProducts(categoryIri, searchTerm) {
         params,
     });
 }
+
+/**
+ * Gets a product from the database according to the productIri
+ *
+ * @param {string} productIri
+ * @return {Promise}
+ */
+export function getProduct(productIri) {
+    return axios({
+        method: 'get',
+        url: productIri,
+    });
+}
