diff --git a/assets/js/components/product.vue b/assets/js/components/product.vue
index 5901819..c38dd6f 100644
--- a/assets/js/components/product.vue
+++ b/assets/js/components/product.vue
@@ -72,6 +72,16 @@ export default {
         product: null,
         loading: true,
     }),
+    computed: {
+        /**
+         * Returns a formatted price for the product
+         * @returns {string}
+         */
+        price() {
+            const strPrice = this.item.price.toString();
+            return `${strPrice.substr(0, strPrice.length - 2)}.${strPrice.substr(-2)}`;
+        },
+    },
     async created() {
         try {
             const response = await productsService.getProduct(this.currentProductId);
