diff --git a/assets/js/components/product.vue b/assets/js/components/product.vue
index 56a8b80c..a2832fb9 100644
--- a/assets/js/components/product.vue
+++ b/assets/js/components/product.vue
@@ -45,6 +45,7 @@
                     <div class="p-3">
                         <button
                             class="btn btn-info btn-sm"
+                            @click="addToCart"
                         >
                             Add to Cart
                         </button>
@@ -97,6 +98,11 @@ export default {
 
         this.product = response.data;
     },
+    methods: {
+        addToCart() {
+            // TODO
+        },
+    },
 };
 </script>
 
