diff --git a/assets/js/components/catalog.vue b/assets/js/components/catalog.vue
index b6cbb91..503c206 100644
--- a/assets/js/components/catalog.vue
+++ b/assets/js/components/catalog.vue
@@ -49,6 +49,10 @@ export default {
         },
     },
     data: () => ({
+        category: {
+            id: null,
+            name: 'All Products',
+        },
         products: [],
         searchTerm: '',
         loading: true,
