diff --git a/assets/js/components/product.vue b/assets/js/components/product.vue
index 21c7f716..13111a31 100644
--- a/assets/js/components/product.vue
+++ b/assets/js/components/product.vue
@@ -9,6 +9,7 @@
 </template>
 
 <script>
+import { getProduct } from '@/services/products-service';
 import TitleComponent from '@/components/title';
 
 export default {
@@ -26,6 +27,19 @@ export default {
             default: () => ([]),
         },
     },
+    data: () => ({
+        product: null,
+        loading: true,
+    }),
+    async created() {
+        try {
+            const response = await getProduct(this.currentProductId);
+            this.product = response.data;
+            this.loading = false;
+        } catch (e) {
+            this.loading = false;
+        }
+    },
 };
 </script>
 
