diff --git a/assets/js/components/catalog.vue b/assets/js/components/catalog.vue
index cbb3b375..e798d26a 100644
--- a/assets/js/components/catalog.vue
+++ b/assets/js/components/catalog.vue
@@ -55,6 +55,15 @@ export default {
             legend: 'Shipping takes 10-13 weeks, and products probably won\'t work',
         };
     },
+    computed: {
+        categoryName() {
+            if (this.currentCategoryId === null) {
+                return 'All Products';
+            }
+
+            return this.categories.find((cat) => (cat.id === this.currentCategoryId)).name;
+        },
+    },
     created() {
         this.getProducts('');
     },
