diff --git a/assets/js/components/product.vue b/assets/js/components/product.vue
index cd3a1ff1..d7e7f762 100644
--- a/assets/js/components/product.vue
+++ b/assets/js/components/product.vue
@@ -36,6 +36,7 @@
                     <div :class="$style['actions-wrapper']">
                         <button
                             class="btn btn-info btn-sm"
+                            @click="addToCart"
                         >
                             Add to Cart
                         </button>
@@ -86,6 +87,11 @@ export default {
 
         this.product = response.data;
     },
+    methods: {
+        addToCart() {
+            // TODO
+        },
+    },
 };
 </script>
 
