diff --git a/assets/js/components/product.vue b/assets/js/components/product.vue
index 75fe8a74..acbff5fd 100644
--- a/assets/js/components/product.vue
+++ b/assets/js/components/product.vue
@@ -125,6 +125,10 @@ export default {
         this.product = response.data;
     },
     methods: {
+        addToCart() {
+            // TODO
+        },
+
         /**
          * Update the selectedColorId index accordingly
          *
