diff --git a/assets/js/components/product.vue b/assets/js/components/product.vue
index 0ea253c9..f796c1e0 100644
--- a/assets/js/components/product.vue
+++ b/assets/js/components/product.vue
@@ -71,7 +71,9 @@ export default {
     },
     data() {
         return {
-            product: { name: '', image: '', price: 0 },
+            product: {
+                name: '', image: '', price: 0, colors: [],
+            },
             loading: true,
         };
     },
