diff --git a/assets/js/components/product.vue b/assets/js/components/product.vue
index 8b8e326c..ecb9558a 100644
--- a/assets/js/components/product.vue
+++ b/assets/js/components/product.vue
@@ -1,10 +1,8 @@
 <template>
     <div class="row">
         <div class="col-12">
-            <title-component />
+            <title-component :text="product.name" />
         </div>
-
-        I'd <3 to see a Product here!
     </div>
 </template>
 
