diff --git a/assets/components/product-show.vue b/assets/components/product-show.vue
index 1784d21e2..c108724e1 100644
--- a/assets/components/product-show.vue
+++ b/assets/components/product-show.vue
@@ -5,6 +5,59 @@
         <div v-if="product">
             <title-component :text="product.name" />
         </div>
+
+        <div
+            v-if="product"
+            :class="$style.product"
+            class="row"
+        >
+            <div class="col-4 pt-3">
+                <img
+                    class="d-block"
+                    :src="product.image"
+                    :alt="product.name"
+                >
+
+                <div class="p-2">
+                    <small>brought to you by </small>
+
+                    <small
+                        class="d-inline"
+                        v-text="product.brand"
+                    />
+                </div>
+            </div>
+
+            <div class="col-8">
+                <div class="p-3">
+                    <p v-text="product.description" />
+
+                    <p>Price: <strong>${{ price }}</strong></p>
+                </div>
+
+                <div class="p-3 row align-items-center">
+                    <div class="col-3" v-if="product.colors.length !== 0">
+                        <color-selector />
+                    </div>
+
+                    <div class="col-2">
+                        <input
+                            class="form-control"
+                            type="number"
+                            min="1"
+                        >
+                    </div>
+
+                    <div class="col">
+                        <button
+                            class="btn btn-info btn-sm"
+                        >
+                            Add to Cart
+                        </button>
+                    </div>
+                </div>
+            </div>
+        </div>
     </div>
 </template>
 
@@ -40,3 +93,16 @@ export default {
     },
 };
 </script>
+
+<style lang="scss" module>
+@import '~styles/components/light-component';
+
+.product {
+    @include light-component;
+
+    img {
+        max-width:100%;
+        max-height:100%;
+    }
+}
+</style>
