diff --git a/assets/components/product-show.vue b/assets/components/product-show.vue
index ef2e83afa..2db7eca98 100644
--- a/assets/components/product-show.vue
+++ b/assets/components/product-show.vue
@@ -1,10 +1,12 @@
 <template>
     <div>
-        I'd ❤️ to see Product {{ productId }} here!
+        <h1>{{ product.name }}</h1>
     </div>
 </template>
 
 <script>
+import { fetchOneProduct } from '@/services/products-service';
+
 export default {
     name: 'ProductShow',
     props: {
@@ -13,5 +15,18 @@ export default {
             required: true,
         },
     },
+    data() {
+        return {
+            product: null,
+            loading: true,
+        };
+    },
+    async created() {
+        try {
+            this.product = (await fetchOneProduct(this.productId)).data;
+        } finally {
+            this.loading = false;
+        }
+    },
 };
 </script>
