diff --git a/assets/components/product-show.vue b/assets/components/product-show.vue
index 1070c7dd3..01565d8d8 100644
--- a/assets/components/product-show.vue
+++ b/assets/components/product-show.vue
@@ -64,6 +64,7 @@
 </template>
 
 <script>
+import formatPrice from '@/helpers/format-price';
 import { fetchOneProduct } from '@/services/products-service';
 import ColorSelector from '@/components/color-selector';
 import Loading from '@/components/loading';
@@ -88,6 +89,15 @@ export default {
             loading: true,
         };
     },
+    computed: {
+        /**
+         * Returns a formatted price for the product
+         * @returns {string}
+         */
+        price() {
+            return formatPrice(this.product.price);
+        },
+    },
     async created() {
         try {
             this.product = (await fetchOneProduct(this.productId)).data;
