diff --git a/assets/js/components/product.vue b/assets/js/components/product.vue
index 7ba34a1..501cd50 100644
--- a/assets/js/components/product.vue
+++ b/assets/js/components/product.vue
@@ -1,7 +1,7 @@
 <template>
     <div class="row">
         <div :class="$style['top-bar']">
-            <title-component />
+            <title-component :text="product.name" />
         </div>
 
         TODO: Show product here!
