diff --git a/assets/js/components/product.vue b/assets/js/components/product.vue
index 6941d9e0..ea66bd43 100644
--- a/assets/js/components/product.vue
+++ b/assets/js/components/product.vue
@@ -1,10 +1,13 @@
 <template>
     <div>
+        <h1>{{ product.name }}</h1>
         I'd <3 to see a Product here!
     </div>
 </template>
 
 <script>
+import { getProduct } from '@/services/products-service';
+
 export default {
     name: 'Product',
     props: {
@@ -13,5 +16,26 @@ export default {
             default: null,
         },
     },
+    data() {
+        return {
+            product: null,
+            loading: true,
+        };
+    },
+    async created() {
+        let response;
+
+        try {
+            response = await getProduct(this.currentProductId);
+
+            this.loading = false;
+        } catch (e) {
+            this.loading = false;
+
+            return;
+        }
+
+        this.product = response.data;
+    },
 };
 </script>
