diff --git a/assets/js/components/product.vue b/assets/js/components/product.vue
index 93537900..d43af972 100644
--- a/assets/js/components/product.vue
+++ b/assets/js/components/product.vue
@@ -5,6 +5,8 @@
         </div>
 
         <div :class="$style.content">
+            <loading v-show="loading" />
+
             <div
                 v-show="!loading"
                 :class="$style.product"
@@ -46,11 +48,13 @@
 
 <script>
 import { getProduct } from '@/services/products-service';
+import Loading from '@/components/loading';
 import TitleComponent from '@/components/title';
 
 export default {
     name: 'Product',
     components: {
+        Loading,
         TitleComponent,
     },
     props: {
