diff --git a/assets/js/components/product.vue b/assets/js/components/product.vue
index fb855c33..71d91b6c 100644
--- a/assets/js/components/product.vue
+++ b/assets/js/components/product.vue
@@ -71,6 +71,7 @@
 </template>
 
 <script>
+import formatPrice from '@/helpers/format-price';
 import { getProduct } from '@/services/products-service';
 import ColorSelector from '@/components/color-selector';
 import Loading from '@/components/loading';
@@ -95,6 +96,15 @@ export default {
             loading: true,
         };
     },
+    computed: {
+        /**
+         * Returns a formatted price for the product
+         * @returns {string}
+         */
+        price() {
+            return formatPrice(this.product.price);
+        },
+    },
     async created() {
         let response;
 
