diff --git a/assets/js/services/products-service.js b/assets/js/services/products-service.js
index 62681e63..0a12a0dc 100644
--- a/assets/js/services/products-service.js
+++ b/assets/js/services/products-service.js
@@ -19,3 +19,16 @@ export function fetchProducts(categoryIri, searchTerm) {
         params,
     });
 }
+
+/**
+ * Gets a product from the database according to the product id
+ *
+ * @param {number} id
+ * @return {Promise}
+ */
+export function getProduct(id) {
+    return axios({
+        method: 'get',
+        url: `/api/products/${id}`,
+    });
+}
