diff --git a/assets/js/components/product.vue b/assets/js/components/product.vue
index c38dd6f..776a177 100644
--- a/assets/js/components/product.vue
+++ b/assets/js/components/product.vue
@@ -78,7 +78,7 @@ export default {
          * @returns {string}
          */
         price() {
-            const strPrice = this.item.price.toString();
+            const strPrice = this.product.price.toString();
             return `${strPrice.substr(0, strPrice.length - 2)}.${strPrice.substr(-2)}`;
         },
     },
