diff --git a/assets/js/components/product.vue b/assets/js/components/product.vue
index 9329a0db..4bab0e2f 100644
--- a/assets/js/components/product.vue
+++ b/assets/js/components/product.vue
@@ -1,7 +1,7 @@
 <template>
     <div class="row">
         <div class="col-3">
-            <title-component />
+            <title-component :text="product.name" />
         </div>
 
         TODO: Show product here!
@@ -29,7 +29,7 @@ export default {
     },
     data() {
         return {
-            product: null,
+            product: { name: '' },
             loading: true,
         };
     },
