diff --git a/assets/js/components/product.vue b/assets/js/components/product.vue
index 3d1b422a..f93a28a9 100644
--- a/assets/js/components/product.vue
+++ b/assets/js/components/product.vue
@@ -70,7 +70,9 @@ export default {
         },
     },
     data: () => ({
-        product: { name: '', image: '', price: 0 },
+        product: {
+            name: '', image: '', price: 0, colors: [],
+        },
         loading: true,
     }),
     computed: {
