diff --git a/assets/js/components/product.vue b/assets/js/components/product.vue
index 2513f201..57d56f6a 100644
--- a/assets/js/components/product.vue
+++ b/assets/js/components/product.vue
@@ -1,7 +1,7 @@
 <template>
     <div class="row">
         <div :class="$style['top-bar']">
-            <title-component />
+            <title-component :text="product.name" />
         </div>
 
         TODO: Show product here!
@@ -28,7 +28,7 @@ export default {
         },
     },
     data: () => ({
-        product: null,
+        product: { name: '' },
         loading: true,
     }),
     async created() {
