diff --git a/assets/components/product-show.vue b/assets/components/product-show.vue
index 527edc5d2..1784d21e2 100644
--- a/assets/components/product-show.vue
+++ b/assets/components/product-show.vue
@@ -3,7 +3,7 @@
         <loading v-if="loading" />
 
         <div v-if="product">
-            <title-component />
+            <title-component :text="product.name" />
         </div>
     </div>
 </template>
