diff --git a/assets/js/components/product.vue b/assets/js/components/product.vue
index 7803aba..77a5c5d 100644
--- a/assets/js/components/product.vue
+++ b/assets/js/components/product.vue
@@ -30,7 +30,7 @@
                     <div :class="$style.description">
                         <p v-text="product.description" />
 
-                        <p>Price: <strong>$ {{ product.price }}</strong></p>
+                        <p>Price: <strong>$ {{ price }}</strong></p>
                     </div>
 
                     <div :class="$style['actions-wrapper']">
