diff --git a/assets/js/components/product.vue b/assets/js/components/product.vue
index 3f874853..15ef8dc0 100644
--- a/assets/js/components/product.vue
+++ b/assets/js/components/product.vue
@@ -47,6 +47,7 @@
                     <div class="p-3 row align-items-center">
                         <button
                             class="btn btn-info btn-sm"
+                            @click="addToCart"
                         >
                             Add to Cart
                         </button>
@@ -99,6 +100,11 @@ export default {
 
         this.product = response.data;
     },
+    methods: {
+        addToCart() {
+            // TODO
+        },
+    },
 };
 </script>
 
