diff --git a/assets/js/components/title.vue b/assets/js/components/title.vue
index 8537f6f..6787140 100644
--- a/assets/js/components/title.vue
+++ b/assets/js/components/title.vue
@@ -15,15 +15,6 @@ export default {
             required: true,
         },
     },
-    computed: {
-        categoryName() {
-            if (this.currentCategoryId === null) {
-                return 'All Products';
-            }
-
-            return this.categories.find((cat) => (cat.id === this.currentCategoryId)).name;
-        },
-    },
 };
 </script>
 
