diff --git a/assets/js/components/product.vue b/assets/js/components/product.vue
index 71d91b6c..8b546691 100644
--- a/assets/js/components/product.vue
+++ b/assets/js/components/product.vue
@@ -92,7 +92,9 @@ export default {
     },
     data() {
         return {
-            product: { name: '', image: '', price: 0 },
+            product: {
+                name: '', image: '', price: 0, colors: [],
+            },
             loading: true,
         };
     },
