diff --git a/assets/js/components/catalog.vue b/assets/js/components/catalog.vue
index a3a2b2b..554f92c 100644
--- a/assets/js/components/catalog.vue
+++ b/assets/js/components/catalog.vue
@@ -2,11 +2,7 @@
     <div>
         <div class="row">
             <div :class="$style['top-bar']">
-                <title-component
-                    :current-category-id="currentCategoryId"
-                    :categories="categories"
-                    :class="$style.title"
-                />
+                <title-component :text="category.name" />
 
                 <search-bar @search-products="onSearchProducts" />
             </div>
