diff --git a/assets/js/components/product.vue b/assets/js/components/product.vue
index d58c33e3..f479d07b 100644
--- a/assets/js/components/product.vue
+++ b/assets/js/components/product.vue
@@ -5,7 +5,40 @@
 </template>
 
 <script>
+import { getProduct } from '@/services/products-service';
+
 export default {
     name: 'Product',
+    props: {
+        currentProductId: {
+            type: String,
+            default: null,
+        },
+        categories: {
+            type: Array,
+            default: () => ([]),
+        },
+    },
+    data() {
+        return {
+            product: null,
+            loading: true,
+        };
+    },
+    async created() {
+        let response;
+
+        try {
+            response = await getProduct(this.currentProductId);
+
+            this.loading = false;
+        } catch (e) {
+            this.loading = false;
+
+            return;
+        }
+
+        this.product = response.data;
+    },
 };
 </script>
