diff --git a/assets/js/components/product.vue b/assets/js/components/product.vue
index d7e7f762..4f9971b3 100644
--- a/assets/js/components/product.vue
+++ b/assets/js/components/product.vue
@@ -51,6 +51,7 @@
 import { getProduct } from '@/services/products-service';
 import Loading from '@/components/loading';
 import TitleComponent from '@/components/title';
+import formatPrice from '@/helpers/format-price';
 
 export default {
     name: 'Product',
@@ -72,6 +73,15 @@ export default {
         product: { name: '', image: '', price: 0 },
         loading: true,
     }),
+    computed: {
+        /**
+         * Returns a formatted price for the product
+         * @returns {string}
+         */
+        price() {
+            return formatPrice(this.product.price);
+        },
+    },
     async created() {
         let response;
 
