diff --git a/assets/js/components/catalog.vue b/assets/js/components/catalog.vue
index 83b863a..bc8ac8f 100644
--- a/assets/js/components/catalog.vue
+++ b/assets/js/components/catalog.vue
@@ -52,6 +52,10 @@ export default {
         },
     },
     data: () => ({
+        category: {
+            id: null,
+            name: 'All Products',
+        },
         products: [],
         searchTerm: '',
         loading: true,
