diff --git a/assets/components/title.vue b/assets/components/title.vue
index bf266bc99..6787140ea 100644
--- a/assets/components/title.vue
+++ b/assets/components/title.vue
@@ -15,17 +15,6 @@ export default {
             required: true,
         },
     },
-    computed: {
-        categoryName() {
-            if (this.currentCategoryId === null) {
-                return 'All Products';
-            }
-
-            const category = this.categories.find((cat) => (cat['@id'] === this.currentCategoryId));
-
-            return category ? category.name : '';
-        },
-    },
 };
 </script>
 
