diff --git a/assets/js/components/product.vue b/assets/js/components/product.vue
index ecfa3460..de6a3b04 100644
--- a/assets/js/components/product.vue
+++ b/assets/js/components/product.vue
@@ -36,6 +36,7 @@
                     <div :class="$style['actions-wrapper']">
                         <button
                             class="btn btn-info btn-sm"
+                            @click="addToCart"
                         >
                             Add to Cart
                         </button>
@@ -80,6 +81,11 @@ export default {
             this.loading = false;
         }
     },
+    methods: {
+        addToCart() {
+            // TODO
+        },
+    },
 };
 </script>
 
