diff --git a/assets/js/components/product.vue b/assets/js/components/product.vue
index f0a7953a..9329a0db 100644
--- a/assets/js/components/product.vue
+++ b/assets/js/components/product.vue
@@ -9,6 +9,7 @@
 </template>
 
 <script>
+import { getProduct } from '@/services/products-service';
 import TitleComponent from '@/components/title';
 
 export default {
@@ -26,5 +27,26 @@ export default {
             default: () => ([]),
         },
     },
+    data() {
+        return {
+            product: null,
+            loading: true,
+        };
+    },
+    async created() {
+        let response;
+
+        try {
+            response = await getProduct(this.currentProductId);
+
+            this.loading = false;
+        } catch (e) {
+            this.loading = false;
+
+            return;
+        }
+
+        this.product = response.data;
+    },
 };
 </script>
