diff --git a/assets/js/components/catalog.vue b/assets/js/components/catalog.vue
index 66c74d00..997cc1a5 100644
--- a/assets/js/components/catalog.vue
+++ b/assets/js/components/catalog.vue
@@ -56,21 +56,8 @@ export default {
             legend: 'Shipping takes 10-13 weeks, and products probably won\'t work',
         };
     },
-    async created() {
-        this.loading = true;
-
-        let response;
-        try {
-            response = await fetchProducts(this.currentCategoryId);
-
-            this.loading = false;
-        } catch (e) {
-            this.loading = false;
-
-            return;
-        }
-
-        this.products = response.data['hydra:member'];
+    created() {
+        this.fetchProducts('');
     },
     methods: {
         onSearchProducts(event) {
