diff --git a/assets/js/components/catalog.vue b/assets/js/components/catalog.vue
index ff4ee124..475cbf84 100644
--- a/assets/js/components/catalog.vue
+++ b/assets/js/components/catalog.vue
@@ -36,6 +36,7 @@ export default {
     data() {
         return {
             products: [],
+            loading: false,
             legend: 'Shipping takes 10-13 weeks, and products probably won\'t work',
         };
     },
