diff --git a/assets/js/components/search-bar.vue b/assets/js/components/search-bar.vue
index 1e29847c..210b898a 100644
--- a/assets/js/components/search-bar.vue
+++ b/assets/js/components/search-bar.vue
@@ -15,11 +15,19 @@ export default {
     data() {
         return {
             searchTerm: '',
+            searchTimeout: null,
         };
     },
     watch: {
         searchTerm(newTerm) {
-            this.$emit('search-products', { term: newTerm });
+            if (this.searchTimeout !== null) {
+                window.clearTimeout(this.searchTimeout);
+                this.searchTimeout = null;
+            }
+
+            this.searchTimeout = window.setTimeout(() => {
+                this.$emit('search-products', { term: newTerm });
+            }, 200);
         },
     },
 };
