diff --git a/assets/js/components/sidebar.vue b/assets/js/components/sidebar.vue
index 43bfc700..102e1324 100644
--- a/assets/js/components/sidebar.vue
+++ b/assets/js/components/sidebar.vue
@@ -42,9 +42,13 @@
 
 <script>
 import axios from 'axios';
+import Loading from '@/components/loading';
 
 export default {
     name: 'Sidebar',
+    components: {
+        Loading: Loading,
+    },
     props: {
         collapsed: {
             type: Boolean,
