diff --git a/assets/js/components/catalog.vue b/assets/js/components/catalog.vue
index b45a58b..44a4553 100644
--- a/assets/js/components/catalog.vue
+++ b/assets/js/components/catalog.vue
@@ -44,13 +44,17 @@ export default {
         this.products = [];
         this.loading = true;
 
-        const response = await axios({
-            method: 'get',
-            url,
-        });
+        try {
+            const response = await axios({
+                method: 'get',
+                url,
+            });
 
-        this.loading = false;
-        this.products = response.data['hydra:member'];
+            this.loading = false;
+            this.products = response.data['hydra:member'];
+        } catch (e) {
+            this.loading = false;
+        }
     },
 };
 </script>
