diff --git a/assets/js/components/catalog.vue b/assets/js/components/catalog.vue
index bd2f81d..5f53a16 100644
--- a/assets/js/components/catalog.vue
+++ b/assets/js/components/catalog.vue
@@ -11,7 +11,7 @@
             </div>
 
             <product-list
-                :products="filteredProducts"
+                :products="products"
                 :loading="loading"
             />
         </div>
