diff --git a/assets/js/components/catalog.vue b/assets/js/components/catalog.vue
index 1852830..94696b5 100644
--- a/assets/js/components/catalog.vue
+++ b/assets/js/components/catalog.vue
@@ -35,6 +35,7 @@ export default {
     },
     data: () => ({
         products: [],
+        loading: true,
         legend: 'Shipping takes 10-12 weeks, and products probably won\'t work',
     }),
     async created() {
