diff --git a/assets/js/components/product-list/index.vue b/assets/js/components/product-list/index.vue
index 57b69d609..3b703409e 100644
--- a/assets/js/components/product-list/index.vue
+++ b/assets/js/components/product-list/index.vue
@@ -26,6 +26,10 @@ export default {
         ProductCard,
     },
     props: {
+        loading: {
+            type: Boolean,
+            required: true,
+        },
         products: {
             type: Array,
             required: true,
