diff --git a/assets/js/components/catalog.vue b/assets/js/components/catalog.vue
index d9926dfb..fb7c786e 100644
--- a/assets/js/components/catalog.vue
+++ b/assets/js/components/catalog.vue
@@ -55,22 +55,8 @@ export default {
         loading: true,
         legend: 'Shipping takes 10-12 weeks, and products probably won\'t work',
     }),
-    async created() {
-        const url = this.currentCategoryId
-            ? `/api/products?category=${this.currentCategoryId}`
-            : '/api/products';
-
-        this.products = [];
-        this.loading = true;
-
-        try {
-            const response = await axios.get(url);
-
-            this.loading = false;
-            this.products = response.data['hydra:member'];
-        } catch (e) {
-            this.loading = false;
-        }
+    created() {
+        this.fetchProducts('');
     },
     methods: {
         onSearchProducts(event) {
