diff --git a/assets/js/components/catalog.vue b/assets/js/components/catalog.vue
index def83531..505ed3fd 100644
--- a/assets/js/components/catalog.vue
+++ b/assets/js/components/catalog.vue
@@ -48,16 +48,20 @@ export default {
 
         this.loading = true;
 
+        let response;
         try {
-            const response = await axios.get('/api/products', {
+            response = await axios.get('/api/products', {
                 params,
             });
 
             this.loading = false;
-            this.products = response.data['hydra:member'];
         } catch (e) {
             this.loading = false;
+
+            return;
         }
+
+        this.products = response.data['hydra:member'];
     },
 };
 </script>
