diff --git a/assets/js/components/catalog.vue b/assets/js/components/catalog.vue
index b466ae26..78039fa0 100644
--- a/assets/js/components/catalog.vue
+++ b/assets/js/components/catalog.vue
@@ -8,7 +8,10 @@
             </div>
         </div>
 
-        <product-list :products="products" />
+        <product-list
+            :products="products"
+            :loading="loading"
+        />
 
         <div class="row">
             <legend-component :title="legend" />
