diff --git a/assets/js/components/catalog.vue b/assets/js/components/catalog.vue
index e97b31be..5bef666d 100644
--- a/assets/js/components/catalog.vue
+++ b/assets/js/components/catalog.vue
@@ -45,8 +45,11 @@ export default {
             ? `/api/products?category=${this.currentCategoryId}`
             : '/api/products';
 
+        this.loading = true;
+
         const response = await axios.get(url);
 
+        this.loading = false;
         this.products = response.data['hydra:member'];
     },
 };
