diff --git a/assets/js/components/catalog.vue b/assets/js/components/catalog.vue
index 44a4553..391b077 100644
--- a/assets/js/components/catalog.vue
+++ b/assets/js/components/catalog.vue
@@ -6,7 +6,10 @@
             </h2>
         </div>
 
-        <product-list :products="products" />
+        <product-list
+            :products="products"
+            :loading="loading"
+        />
 
         <div class="row">
             <legend-component :title="legend" />
