diff --git a/assets/js/components/catalog.vue b/assets/js/components/catalog.vue
index c080c53c..def83531 100644
--- a/assets/js/components/catalog.vue
+++ b/assets/js/components/catalog.vue
@@ -48,12 +48,16 @@ export default {
 
         this.loading = true;
 
-        const response = await axios.get('/api/products', {
-            params,
-        });
+        try {
+            const response = await axios.get('/api/products', {
+                params,
+            });
 
-        this.loading = false;
-        this.products = response.data['hydra:member'];
+            this.loading = false;
+            this.products = response.data['hydra:member'];
+        } catch (e) {
+            this.loading = false;
+        }
     },
 };
 </script>
