diff --git a/assets/js/components/catalog.vue b/assets/js/components/catalog.vue
index ff1d54cf..9219509f 100644
--- a/assets/js/components/catalog.vue
+++ b/assets/js/components/catalog.vue
@@ -58,6 +58,15 @@ export default {
         this.fetchProducts('');
     },
     methods: {
+        /**
+         * Handles a change in the searchTerm provided by the search bar and fetches new products
+         *
+         * @param {string} term
+         */
+        onSearchProducts({ term }) {
+            this.fetchProducts(term);
+        },
+
         /**
          * Fetches products from the database according to current category and search term
          *
