diff --git a/assets/js/components/catalog.vue b/assets/js/components/catalog.vue
index c109d91d..5bd317fa 100644
--- a/assets/js/components/catalog.vue
+++ b/assets/js/components/catalog.vue
@@ -45,9 +45,11 @@ export default {
             : '/api/products';
 
         this.products = [];
+        this.loading = true;
 
         const response = await axios.get(url);
 
+        this.loading = false;
         this.products = response.data['hydra:member'];
     },
 };
