diff --git a/assets/js/components/product-list/index.vue b/assets/js/components/product-list/index.vue
index 3b703409..5bba0cd0 100644
--- a/assets/js/components/product-list/index.vue
+++ b/assets/js/components/product-list/index.vue
@@ -2,13 +2,13 @@
     <div class="row">
         <div class="col-12">
             <div class="mt-4">
-                <loading v-show="products.length === 0" />
+                <loading v-show="loading" />
             </div>
         </div>
 
         <product-card
             v-for="product in products"
-            v-show="products.length > 0"
+            v-show="!loading"
             :key="product['@id']"
             :product="product"
         />
