diff --git a/assets/js/components/catalog.vue b/assets/js/components/catalog.vue
index 4cea3f09..14522172 100644
--- a/assets/js/components/catalog.vue
+++ b/assets/js/components/catalog.vue
@@ -60,8 +60,13 @@ export default {
         this.fetchProducts('');
     },
     methods: {
-        onSearchProducts(event) {
-            this.fetchProducts(event.term);
+        /**
+         * Handles a change in the searchTerm provided by the search bar and fetches new products
+         *
+         * @param {string} term
+         */
+        onSearchProducts({ term }) {
+            this.fetchProducts(term);
         },
 
         /**
