diff --git a/assets/js/components/product-list/index.vue b/assets/js/components/product-list/index.vue
index bdf964a..70de4c0 100644
--- a/assets/js/components/product-list/index.vue
+++ b/assets/js/components/product-list/index.vue
@@ -1,10 +1,17 @@
 <template>
     <div :class="$style.component">
-        <loading v-show="products.length === 0" />
+        <loading v-show="loading" />
+
+        <h5
+            v-show="!loading && products.length === 0"
+            class="mt-4 ml-4"
+        >
+            Whoopsie Daisy, no products found!
+        </h5>
 
         <product-card
             v-for="product in products"
-            v-show="products.length > 0"
+            v-show="!loading"
             :key="product['@id']"
             :item="product"
         />
