diff --git a/assets/js/components/sidebar.vue b/assets/js/components/sidebar.vue
index 30792a9..b857900 100644
--- a/assets/js/components/sidebar.vue
+++ b/assets/js/components/sidebar.vue
@@ -40,9 +40,13 @@
 
 <script>
 import axios from 'axios';
+import LoadingComponent from '@/components/loading';
 
 export default {
     name: 'Sidebar',
+    components: {
+        LoadingComponent,
+    },
     props: {
         collapsed: {
             type: Boolean,
