diff --git a/assets/js/services/cart.js b/assets/js/services/cart.js
new file mode 100644
index 0000000..bf1dcbf
--- /dev/null
+++ b/assets/js/services/cart.js
@@ -0,0 +1,69 @@
+const cart = {
+    /**
+     * Gets the list of all shopping cart items in an array of objects containing:
+     * - productId
+     * - colorId
+     * - qty
+     *
+     * @return {object}
+     */
+    getItems() {
+        // TODO: implement
+        return {};
+    },
+
+    /**
+     * Adds a product into the cart, providing a product id, an optional color id and the quantity
+     *
+     * @param {string} productId
+     * @param {string|null} colorId
+     * @param {number} qty
+     */
+    addItem(productId, colorId, qty) {
+        // TODO: implement
+    },
+
+    /**
+     * Removes an item from the shopping cart identified by productId and an optional colorId
+     *
+     * @param {string} productId
+     * @param {string|null} colorId
+     */
+    removeItem(productId, colorId) {
+        // TODO: implement
+    },
+
+    /**
+     * Clears the shopping cart
+     */
+    clear() {
+        // TODO: implement
+    },
+
+    /**
+     * Initializes the local storage for our shopping cart
+     */
+    initializeStorage() {
+        // TODO: implement
+    },
+
+    /**
+     * Gets the local storage as an object
+     *
+     * @returns {array}
+     */
+    getStorage() {
+        // TODO: implement
+    },
+
+    /**
+     * Saves the local storage in its string format
+     *
+     * @param {array} storage
+     */
+    saveStorage(storage) {
+        // TODO: implement
+    },
+};
+
+export default cart;
