diff --git a/assets/components/product-show/cart-add-controls.vue b/assets/components/product-show/cart-add-controls.vue
index 263e3e377..22d84c786 100644
--- a/assets/components/product-show/cart-add-controls.vue
+++ b/assets/components/product-show/cart-add-controls.vue
@@ -66,6 +66,12 @@ export default {
         updateSelectedColor(iri) {
             this.selectedColorId = iri;
         },
+        addToCart() {
+            this.$emit('add-to-cart', {
+                quantity: this.quantity,
+                selectedColorId: this.selectedColorId,
+            });
+        },
     },
 };
 </script>
diff --git a/assets/components/product-show/index.vue b/assets/components/product-show/index.vue
index 87cf5a547..972701dbf 100644
--- a/assets/components/product-show/index.vue
+++ b/assets/components/product-show/index.vue
@@ -43,6 +43,7 @@
                             :allow-add-to-cart="cart !== null"
                             :add-to-cart-loading="addToCartLoading"
                             :add-to-cart-success="addToCartSuccess"
+                            @add-to-cart="addToCart"
                         />
                     </div>
                 </div>
@@ -96,8 +97,8 @@ export default {
         }
     },
     methods: {
-        addToCart() {
-            this.addProductToCart(this.product, this.selectedColorId, this.quantity);
+        addToCart({ quantity, selectedColorId }) {
+            this.addProductToCart(this.product, selectedColorId, quantity);
         },
     },
 };
