diff --git a/assets/components/product-show/cart-add-controls.vue b/assets/components/product-show/cart-add-controls.vue
index 5bec865d9..6b458b25e 100644
--- a/assets/components/product-show/cart-add-controls.vue
+++ b/assets/components/product-show/cart-add-controls.vue
@@ -1,5 +1,5 @@
 <template>
-    <div class="d-flex align-items-center justify-content-center">
+    <div :class="[$style.component, 'd-flex align-items-center', 'justify-content-center']">
         <color-selector
             v-if="product.colors.length !== 0"
             @color-selected="updateSelectedColor"
@@ -40,3 +40,13 @@ export default {
     },
 };
 </script>
+
+<style lang="scss" module>
+@import '~styles/components/light-component.scss';
+
+.component :global {
+    input {
+        width: 60px;
+    }
+}
+</style>
diff --git a/assets/components/product-show/index.vue b/assets/components/product-show/index.vue
index 912e7a33f..0117959a9 100644
--- a/assets/components/product-show/index.vue
+++ b/assets/components/product-show/index.vue
@@ -111,9 +111,5 @@ export default {
         max-width:100%;
         max-height:100%;
     }
-
-    input {
-        width: 60px;
-    }
 }
 </style>
