diff --git a/assets/components/product-show/cart-add-controls.vue b/assets/components/product-show/cart-add-controls.vue
index 22d84c78..84cf5737 100644
--- a/assets/components/product-show/cart-add-controls.vue
+++ b/assets/components/product-show/cart-add-controls.vue
@@ -17,7 +17,7 @@
             :disabled="!allowAddToCart"
             @click="addToCart"
         >
-            Add to Cart
+            {{ addButtonText }}
             <i
                 v-show="addToCartLoading"
                 class="fas fa-spinner fa-spin"
@@ -43,6 +43,10 @@ export default {
             type: Object,
             required: true,
         },
+        addButtonText: {
+            type: String,
+            default: 'Add to Cart',
+        },
         allowAddToCart: {
             type: Boolean,
             required: true,
diff --git a/assets/components/shopping-cart/cart-sidebar.vue b/assets/components/shopping-cart/cart-sidebar.vue
index 5ec4b4ac..76ed19cd 100644
--- a/assets/components/shopping-cart/cart-sidebar.vue
+++ b/assets/components/shopping-cart/cart-sidebar.vue
@@ -22,6 +22,7 @@
                 :add-to-cart-loading="false"
                 :add-to-cart-success="false"
                 :allow-add-to-cart="false"
+                add-button-text="+"
             />
         </div>
     </div>
