diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index 552b66ceb..38b559dea 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -93,7 +93,7 @@ export default {
                 return;
             }
 
-            this.featuredProduct = featuredProductsResponse.data['hydra:member'][0];
+            [this.featuredProduct] = featuredProductsResponse.data['hydra:member'];
         },
     },
 };
