diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index 4338d196c..9254f770e 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -99,7 +99,7 @@ export default {
                 return;
             }
 
-            this.featuredProduct = featuredProductsResponse.data['hydra:member'][0];
+            [this.featuredProduct] = featuredProductsResponse.data['hydra:member'];
         },
     },
 };
