diff --git a/assets/components/product-show/cart-add-controls.vue b/assets/components/product-show/cart-add-controls.vue
index 50a22168..1cbd19a4 100644
--- a/assets/components/product-show/cart-add-controls.vue
+++ b/assets/components/product-show/cart-add-controls.vue
@@ -56,6 +56,12 @@ export default {
             required: true,
         },
     },
+    data() {
+        return {
+            quantity: 1,
+            selectedColorId: null,
+        };
+    },
 };
 </script>
 
diff --git a/assets/components/product-show/index.vue b/assets/components/product-show/index.vue
index 0117959a..4659c55d 100644
--- a/assets/components/product-show/index.vue
+++ b/assets/components/product-show/index.vue
@@ -67,8 +67,6 @@ export default {
     },
     data() {
         return {
-            quantity: 1,
-            selectedColorId: null,
             product: null,
             loading: true,
         };
