diff --git a/assets/js/components/product.vue b/assets/js/components/product.vue
index 6baf642..07c5608 100644
--- a/assets/js/components/product.vue
+++ b/assets/js/components/product.vue
@@ -70,7 +70,7 @@ export default {
         },
     },
     data: () => ({
-        product: { name: '', image: '', price: 0 },
+        product: { name: '', image: '', price: 0, colors: [] },
         loading: true,
     }),
     computed: {
