diff --git a/assets/js/components/catalog.vue b/assets/js/components/catalog.vue
index 2108d218..68eb7558 100644
--- a/assets/js/components/catalog.vue
+++ b/assets/js/components/catalog.vue
@@ -27,15 +27,25 @@ export default {
         LegendComponent,
         ProductList,
     },
+    props: {
+        currentCategoryId: {
+            type: String,
+            default: null,
+        },
+    },
     data: () => ({
         /** @type {Product[]} */
         products: [],
         legend: 'Shipping takes 10-12 weeks, and products probably won\'t work',
     }),
     async created() {
+        const url = this.currentCategoryId
+            ? `/api/products?category=${this.currentCategoryId}`
+            : '/api/products';
+
         this.products = [];
 
-        const response = await axios.get('/api/products');
+        const response = await axios.get(url);
 
         this.products = response.data['hydra:member'];
     },
