diff --git a/templates/product/index.html.twig b/templates/product/index.html.twig
index 32a165ce..3906b779 100644
--- a/templates/product/index.html.twig
+++ b/templates/product/index.html.twig
@@ -13,5 +13,13 @@
 {% block javascripts %}
     {{ parent() }}
 
+    <script>
+        {% if currentCategoryId is defined %}
+            window.currentCategoryId = '{{ currentCategoryId|e('js') }}';
+        {% else %}
+            window.currentCategoryId = null;
+        {% endif %}
+    </script>
+
     {{ encore_entry_script_tags('products') }}
 {% endblock %}
