diff --git a/assets/js/pages/products.vue b/assets/js/pages/products.vue
index b1d899e9..5a37f1b0 100644
--- a/assets/js/pages/products.vue
+++ b/assets/js/pages/products.vue
@@ -10,7 +10,7 @@
             </aside>
 
             <div :class="contentClass">
-                <catalog />
+                <catalog :current-category-id="currentCategoryId" />
             </div>
         </div>
     </div>
