diff --git a/templates/product/index.html.twig b/templates/product/index.html.twig
index a4ea49a..b437c50 100644
--- a/templates/product/index.html.twig
+++ b/templates/product/index.html.twig
@@ -11,5 +11,10 @@
 
 {% block javascripts %}
     {{ parent() }}
+
+    <script>
+        window.currentCategoryId = {{ currentCategoryId|default('null') }};
+    </script>
+
     {{ encore_entry_script_tags('products') }}
 {% endblock %}
