diff --git a/templates/product/index.html.twig b/templates/product/index.html.twig
index 32a165ce..d66a8d5a 100644
--- a/templates/product/index.html.twig
+++ b/templates/product/index.html.twig
@@ -13,5 +13,9 @@
 {% block javascripts %}
     {{ parent() }}
 
+    <script>
+        window.currentCategoryId = {{ currentCategoryId|default('null') }};
+    </script>
+
     {{ encore_entry_script_tags('products') }}
 {% endblock %}
