diff --git a/assets/js/components/catalog.vue b/assets/js/components/catalog.vue
index afa924d5..b88e3eba 100644
--- a/assets/js/components/catalog.vue
+++ b/assets/js/components/catalog.vue
@@ -27,6 +27,12 @@ export default {
         LegendComponent,
         ProductList,
     },
+    props: {
+        currentCategoryId: {
+            type: String,
+            default: null,
+        },
+    },
     data() {
         return {
             /** @type {Product[]} */
@@ -35,9 +41,13 @@ export default {
         };
     },
     async created() {
+        const url = this.currentCategoryId
+            ? `/api/products?category=${this.currentCategoryId}`
+            : '/api/products';
+
         this.products = [];
 
-        const response = await axios.get('/api/products');
+        const response = await axios.get(url);
 
         this.products = response.data['hydra:member'];
     },
