diff --git a/src/Controller/ProductController.php b/src/Controller/ProductController.php
index efae5f2..f2f5f40 100644
--- a/src/Controller/ProductController.php
+++ b/src/Controller/ProductController.php
@@ -27,7 +27,9 @@ class ProductController extends AbstractController
      */
     public function showCategory(int $id)
     {
-        return $this->render('product/index.html.twig');
+        return $this->render('product/index.html.twig', [
+            'currentCategoryId' => $id,
+        ]);
     }
 
     /**
