diff --git a/src/Controller/ProductController.php b/src/Controller/ProductController.php
index b2c263c..1c4b453 100644
--- a/src/Controller/ProductController.php
+++ b/src/Controller/ProductController.php
@@ -22,7 +22,9 @@ class ProductController extends AbstractController
      */
     public function showCategory(int $id): Response
     {
-        return $this->render('product/index.html.twig');
+        return $this->render('product/index.html.twig', [
+            'currentCategoryId' => $id,
+        ]);
     }
 
     /**
