diff --git a/assets/js/pages/products.vue b/assets/js/pages/products.vue
index 7fbab7fd..799e24c2 100644
--- a/assets/js/pages/products.vue
+++ b/assets/js/pages/products.vue
@@ -9,7 +9,7 @@
             </aside>
 
             <div :class="contentClass">
-                <catalog />
+                <catalog :current-category-id="currentCategoryId" />
             </div>
         </div>
     </div>
