diff --git a/assets/js/products.vue b/assets/js/products.vue
index 0f8cbf7..fa6df16 100644
--- a/assets/js/products.vue
+++ b/assets/js/products.vue
@@ -25,6 +25,12 @@ export default {
         CatalogComponent,
         SidebarComponent,
     },
+    props: {
+        currentCategoryId: {
+            type: Number,
+            default: null,
+        },
+    },
     data: () => ({
         sidebarCollapsed: false,
     }),
