diff --git a/assets/js/pages/products.vue b/assets/js/pages/products.vue
index 3090872e..7fbab7fd 100644
--- a/assets/js/pages/products.vue
+++ b/assets/js/pages/products.vue
@@ -25,6 +25,12 @@ export default {
         Catalog,
         Sidebar,
     },
+    props: {
+        currentCategoryId: {
+            type: Number,
+            default: null,
+        },
+    },
     data() {
         return {
             sidebarCollapsed: false,
