diff --git a/assets/js/pages/checkout.vue b/assets/js/pages/checkout.vue
index 1ca7aec8..9146616e 100644
--- a/assets/js/pages/checkout.vue
+++ b/assets/js/pages/checkout.vue
@@ -10,6 +10,22 @@
                     </div>
 
                     <div :class="$style.content">
+                        <form>
+                            <div class="form-row">
+                                <label
+                                    for="customerName"
+                                    class="col-form-label"
+                                >
+                                    Name:
+                                </label>
+                                <input
+                                    id="customerName"
+                                    v-model="form.customerName"
+                                    type="text"
+                                    class="form-control"
+                                >
+                            </div>
+                        </form>
                     </div>
                 </div>
             </div>
