diff --git a/assets/js/components/checkout/index.vue b/assets/js/components/checkout/index.vue
index a38bb458..dc0de4fc 100644
--- a/assets/js/components/checkout/index.vue
+++ b/assets/js/components/checkout/index.vue
@@ -26,6 +26,7 @@ export default {
                 customerPhone: '',
                 purchaseItems: [],
             },
+            validationErrors: {},
         };
     },
 };
