diff --git a/assets/components/checkout/index.vue b/assets/components/checkout/index.vue
index 015f985c8..a38bb4587 100644
--- a/assets/components/checkout/index.vue
+++ b/assets/components/checkout/index.vue
@@ -9,6 +9,12 @@
 <script>
 export default {
     name: 'CheckoutForm',
+    props: {
+        cart: {
+            type: Object,
+            required: true,
+        },
+    },
     data() {
         return {
             form: {
diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index fc3ecd05c..1e422c0c3 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -36,6 +36,7 @@
 
                     <checkout-form
                         v-else
+                        :cart="cart"
                     />
                 </transition>
 
