diff --git a/assets/components/checkout/index.vue b/assets/components/checkout/index.vue
index e5da69f0..adf09ff5 100644
--- a/assets/components/checkout/index.vue
+++ b/assets/components/checkout/index.vue
@@ -13,8 +13,17 @@
                         id="customerName"
                         v-model.trim="form.customerName"
                         type="text"
-                        class="form-control"
+                        :class="{
+                            'is-invalid': !isFieldValid('customerName'),
+                            'form-control': true,
+                        }"
                     >
+                    <span
+                        v-show="!isFieldValid('customerName')"
+                        class="invalid-feedback"
+                    >
+                        {{ validationErrors.customerName }}
+                    </span>
                 </div>
             </form>
         </div>
@@ -37,5 +46,10 @@ export default {
             validationErrors: {},
         };
     },
+    methods: {
+        isFieldValid(fieldName) {
+            return !(fieldName in this.validationErrors);
+        },
+    },
 };
 </script>
