diff --git a/assets/js/pages/checkout.vue b/assets/js/pages/checkout.vue
index 9146616e..f968e1a5 100644
--- a/assets/js/pages/checkout.vue
+++ b/assets/js/pages/checkout.vue
@@ -20,7 +20,7 @@
                                 </label>
                                 <input
                                     id="customerName"
-                                    v-model="form.customerName"
+                                    v-model.trim="form.customerName"
                                     type="text"
                                     class="form-control"
                                 >
