diff --git a/assets/js/pages/checkout.vue b/assets/js/pages/checkout.vue
index 3a95786f..5fed7a23 100644
--- a/assets/js/pages/checkout.vue
+++ b/assets/js/pages/checkout.vue
@@ -27,6 +27,19 @@ export default {
     components: {
         TitleComponent,
     },
+    data() {
+        return {
+            form: {
+                customerName: '',
+                customerEmail: '',
+                customerAddress: '',
+                customerZip: '',
+                customerCity: '',
+                customerPhone: '',
+                purchaseItems: [],
+            },
+        };
+    },
 };
 </script>
 
