diff --git a/assets/js/pages/checkout.vue b/assets/js/pages/checkout.vue
index f5b978e8..6f09638d 100644
--- a/assets/js/pages/checkout.vue
+++ b/assets/js/pages/checkout.vue
@@ -19,6 +19,41 @@
                                 label="Name:"
                                 :error-message="validationErrors.customerName"
                             />
+
+                            <form-input
+                                id="customerName"
+                                v-model="form.customerName"
+                                label="Name:"
+                                :error-message="validationErrors.customerName"
+                            />
+
+                            <form-input
+                                id="customerName"
+                                v-model="form.customerName"
+                                label="Name:"
+                                :error-message="validationErrors.customerName"
+                            />
+
+                            <form-input
+                                id="customerName"
+                                v-model="form.customerName"
+                                label="Name:"
+                                :error-message="validationErrors.customerName"
+                            />
+
+                            <form-input
+                                id="customerName"
+                                v-model="form.customerName"
+                                label="Name:"
+                                :error-message="validationErrors.customerName"
+                            />
+
+                            <form-input
+                                id="customerName"
+                                v-model="form.customerName"
+                                label="Name:"
+                                :error-message="validationErrors.customerName"
+                            />
                         </form>
                     </div>
                 </div>
