diff --git a/assets/js/pages/checkout.vue b/assets/js/pages/checkout.vue
index 6e009c2c..43ea7baa 100644
--- a/assets/js/pages/checkout.vue
+++ b/assets/js/pages/checkout.vue
@@ -18,6 +18,7 @@
 </template>
 
 <script>
+import cartService from '@/services/cart';
 import TitleComponent from '@/components/title';
 
 export default {
@@ -36,6 +37,15 @@ export default {
             purchaseItems: [],
         },
     }),
+    async created() {
+        const itemsInCart = cartService.getItems();
+
+        this.form.purchaseItems = itemsInCart.map((item) => ({
+            product: item.productId,
+            color: item.colorId,
+            quantity: item.qty,
+        }));
+    },
 };
 </script>
 
