diff --git a/assets/js/pages/checkout.vue b/assets/js/pages/checkout.vue
index 7ce6216e..a3b1f45a 100644
--- a/assets/js/pages/checkout.vue
+++ b/assets/js/pages/checkout.vue
@@ -22,6 +22,7 @@
                                 id="customerEmail"
                                 v-model="form.customerEmail"
                                 label="Email:"
+                                type="email"
                                 :error-message="validationErrors.customerEmail"
                             />
 
@@ -50,6 +51,7 @@
                                 id="customerPhone"
                                 v-model="form.customerPhone"
                                 label="Phone Number:"
+                                type="tel"
                                 :error-message="validationErrors.customerPhone"
                             />
                         </form>
