diff --git a/assets/js/components/checkout/index.vue b/assets/js/components/checkout/index.vue
index 015f985c..e1df1ee2 100644
--- a/assets/js/components/checkout/index.vue
+++ b/assets/js/components/checkout/index.vue
@@ -9,6 +9,12 @@
 <script>
 export default {
     name: 'CheckoutForm',
+    props: {
+        cart: {
+            type: Object,
+            required: true,
+        },
+    },
     data() {
         return {
             form: {
@@ -22,5 +28,8 @@ export default {
             },
         };
     },
+    created() {
+        this.form.purchaseItems = this.cart.items;
+    },
 };
 </script>
diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index 426de8a6..2e6f43b6 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -43,6 +43,7 @@
 
                         <checkout-form
                             v-else
+                            :cart="cart"
                         />
                     </transition>
 
