diff --git a/templates/checkout/index.html.twig b/templates/checkout/index.html.twig
index 742c21c2..ce9e814b 100644
--- a/templates/checkout/index.html.twig
+++ b/templates/checkout/index.html.twig
@@ -6,10 +6,10 @@
 
 {% block stylesheets %}
     {{ parent() }}
-    {{ encore_entry_link_tags('shopping-cart') }}
+    {{ encore_entry_link_tags('checkout') }}
 {% endblock %}
 
 {% block javascripts %}
     {{ parent() }}
-    {{ encore_entry_script_tags('shopping-cart') }}
+    {{ encore_entry_script_tags('checkout') }}
 {% endblock %}
