diff --git a/assets/components/checkout/index.vue b/assets/components/checkout/index.vue
index 744475736..e5da69f0d 100644
--- a/assets/components/checkout/index.vue
+++ b/assets/components/checkout/index.vue
@@ -11,7 +11,7 @@
                     </label>
                     <input
                         id="customerName"
-                        v-model="form.customerName"
+                        v-model.trim="form.customerName"
                         type="text"
                         class="form-control"
                     >
