diff --git a/assets/js/components/checkout/form-input.vue b/assets/js/components/checkout/form-input.vue
new file mode 100644
index 0000000..5063fdf
--- /dev/null
+++ b/assets/js/components/checkout/form-input.vue
@@ -0,0 +1,36 @@
+<template>
+    <div class="form-row">
+        <label
+            for="customerName"
+            class="col-form-label"
+        >
+            Name:
+        </label>
+        <input
+            id="customerName"
+            v-model.trim="form.customerName"
+            type="text"
+            :class="{
+                'is-invalid': !isFieldValid('customerName'),
+                'form-control': true,
+            }"
+        >
+        <span
+            v-show="isFieldValid('customerName')"
+            class="invalid-feedback"
+        >
+            {{ validation.customerName }}
+        </span>
+    </div>
+</template>
+
+<script>
+export default {
+    name: 'FormInput',
+    methods: {
+        isFieldValid(fieldName) {
+            return (this.validation[fieldName] === null);
+        },
+    },
+};
+</script>
diff --git a/assets/js/pages/checkout.vue b/assets/js/pages/checkout.vue
index 244837a..bcdff86 100644
--- a/assets/js/pages/checkout.vue
+++ b/assets/js/pages/checkout.vue
@@ -11,29 +11,7 @@
 
                     <div :class="$style.content">
                         <form>
-                            <div class="form-row">
-                                <label
-                                    for="customerName"
-                                    class="col-form-label"
-                                >
-                                    Name:
-                                </label>
-                                <input
-                                    id="customerName"
-                                    v-model.trim="form.customerName"
-                                    type="text"
-                                    :class="{
-                                        'is-invalid': !form.customerName,
-                                        'form-control': true,
-                                    }"
-                                >
-                                <span
-                                    v-show="isFieldValid('customerName')"
-                                    class="invalid-feedback"
-                                >
-                                    {{ validation.customerName }}
-                                </span>
-                            </div>
+
                         </form>
                     </div>
                 </div>
@@ -79,11 +57,6 @@ export default {
             quantity: item.qty,
         }));
     },
-    methods: {
-        isFieldValid(fieldName) {
-            return (this.validation[fieldName] === null);
-        },
-    },
 };
 </script>
 
