diff --git a/assets/js/pages/checkout.vue b/assets/js/pages/checkout.vue
index 9709a82c..f5b978e8 100644
--- a/assets/js/pages/checkout.vue
+++ b/assets/js/pages/checkout.vue
@@ -13,7 +13,12 @@
                 <div class="row">
                     <div :class="[$style.content, 'col-11']">
                         <form>
-
+                            <form-input
+                                id="customerName"
+                                v-model="form.customerName"
+                                label="Name:"
+                                :error-message="validationErrors.customerName"
+                            />
                         </form>
                     </div>
                 </div>
@@ -24,11 +29,13 @@
 
 <script>
 import cartService from '@/services/cart-service';
+import FormInput from '@/components/checkout/form-input';
 import TitleComponent from '@/components/title';
 
 export default {
     name: 'Checkout',
     components: {
+        FormInput,
         TitleComponent,
     },
     data() {
