diff --git a/assets/js/components/checkout/index.vue b/assets/js/components/checkout/index.vue
index e1df1ee2..4495f648 100644
--- a/assets/js/components/checkout/index.vue
+++ b/assets/js/components/checkout/index.vue
@@ -26,6 +26,7 @@ export default {
                 customerPhone: '',
                 purchaseItems: [],
             },
+            validationErrors: {},
         };
     },
     created() {
