diff --git a/assets/js/components/checkout/index.vue b/assets/js/components/checkout/index.vue
index 90934cbb..13ea7e11 100644
--- a/assets/js/components/checkout/index.vue
+++ b/assets/js/components/checkout/index.vue
@@ -13,8 +13,17 @@
                         id="customerName"
                         v-model.trim="form.customerName"
                         type="text"
-                        class="form-control"
+                        :class="{
+                            'is-invalid': !isFieldValid('customerName'),
+                            'form-control': true,
+                        }"
                     >
+                    <span
+                        v-show="!isFieldValid('customerName')"
+                        class="invalid-feedback"
+                    >
+                        {{ validationErrors.customerName }}
+                    </span>
                 </div>
             </form>
         </div>
@@ -47,5 +56,10 @@ export default {
     created() {
         this.form.purchaseItems = this.cart.items;
     },
+    methods: {
+        isFieldValid(fieldName) {
+            return (typeof this.validationErrors[fieldName] === 'undefined');
+        },
+    },
 };
 </script>
