diff --git a/assets/js/components/checkout/index.vue b/assets/js/components/checkout/index.vue
index 015f985c..a38bb458 100644
--- a/assets/js/components/checkout/index.vue
+++ b/assets/js/components/checkout/index.vue
@@ -9,6 +9,12 @@
 <script>
 export default {
     name: 'CheckoutForm',
+    props: {
+        cart: {
+            type: Object,
+            required: true,
+        },
+    },
     data() {
         return {
             form: {
diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index 02b99983..94da63ce 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -44,6 +44,7 @@
 
                         <checkout-form
                             v-else
+                            :cart="cart"
                         />
                     </transition>
 
