diff --git a/assets/js/pages/checkout.vue b/assets/js/pages/checkout.vue
index 13dea2b2..56827de0 100644
--- a/assets/js/pages/checkout.vue
+++ b/assets/js/pages/checkout.vue
@@ -24,6 +24,7 @@
                                 id="customerEmail"
                                 v-model="form.customerEmail"
                                 label="Email:"
+                                type="email"
                                 :error-message="validationErrors.customerEmail"
                             />
 
@@ -52,6 +53,7 @@
                                 id="customerPhone"
                                 v-model="form.customerPhone"
                                 label="Phone Number:"
+                                type="tel"
                                 :error-message="validationErrors.customerPhone"
                             />
                         </form>
