diff --git a/assets/js/pages/checkout.vue b/assets/js/pages/checkout.vue
index 313ec73..6e009c2 100644
--- a/assets/js/pages/checkout.vue
+++ b/assets/js/pages/checkout.vue
@@ -25,6 +25,17 @@ export default {
     components: {
         TitleComponent,
     },
+    data: () => ({
+        form: {
+            customerName: '',
+            customerEmail: '',
+            customerAddress: '',
+            customerZip: '',
+            customerCity: '',
+            customerPhone: '',
+            purchaseItems: [],
+        },
+    }),
 };
 </script>
 
