diff --git a/assets/js/pages/checkout.vue b/assets/js/pages/checkout.vue
index 49ddeded..6c5e86b7 100644
--- a/assets/js/pages/checkout.vue
+++ b/assets/js/pages/checkout.vue
@@ -39,6 +39,14 @@ export default {
                 customerPhone: '',
                 purchaseItems: [],
             },
+            validationErrors: {
+                customerName: null,
+                customerEmail: null,
+                customerAddress: null,
+                customerZip: null,
+                customerCity: null,
+                customerPhone: null,
+            },
         };
     },
     async created() {
