diff --git a/assets/components/checkout/index.vue b/assets/components/checkout/index.vue
index 8aa32543..127f8567 100644
--- a/assets/components/checkout/index.vue
+++ b/assets/components/checkout/index.vue
@@ -9,5 +9,17 @@
 <script>
 export default {
     name: 'CheckoutForm',
+    data() {
+        return {
+            form: {
+                customerName: '',
+                customerEmail: '',
+                customerAddress: '',
+                customerZip: '',
+                customerCity: '',
+                customerPhone: '',
+            },
+        };
+    },
 };
 </script>
