diff --git a/templates/checkout/index.html.twig b/templates/checkout/index.html.twig
new file mode 100644
index 00000000..742c21c2
--- /dev/null
+++ b/templates/checkout/index.html.twig
@@ -0,0 +1,15 @@
+{% extends 'base.html.twig' %}
+
+{% block body %}
+    <div id="app"></div>
+{% endblock %}
+
+{% block stylesheets %}
+    {{ parent() }}
+    {{ encore_entry_link_tags('shopping-cart') }}
+{% endblock %}
+
+{% block javascripts %}
+    {{ parent() }}
+    {{ encore_entry_script_tags('shopping-cart') }}
+{% endblock %}
