diff --git a/assets/js/pages/checkout.vue b/assets/js/pages/checkout.vue
index 6c5e86b7..393f9c24 100644
--- a/assets/js/pages/checkout.vue
+++ b/assets/js/pages/checkout.vue
@@ -12,6 +12,22 @@
 
                 <div class="row">
                     <div :class="$style.content">
+                        <form>
+                            <div class="form-row">
+                                <label
+                                    for="customerName"
+                                    class="col-form-label"
+                                >
+                                    Name:
+                                </label>
+                                <input
+                                    id="customerName"
+                                    v-model="form.customerName"
+                                    type="text"
+                                    class="form-control"
+                                >
+                            </div>
+                        </form>
                     </div>
                 </div>
             </div>
