diff --git a/assets/js/pages/checkout.vue b/assets/js/pages/checkout.vue
index 5fed7a23..49ddeded 100644
--- a/assets/js/pages/checkout.vue
+++ b/assets/js/pages/checkout.vue
@@ -20,6 +20,7 @@
 </template>
 
 <script>
+import cartService from '@/services/cart';
 import TitleComponent from '@/components/title';
 
 export default {
@@ -40,6 +41,15 @@ export default {
             },
         };
     },
+    async created() {
+        const itemsInCart = cartService.getItems();
+
+        this.form.purchaseItems = itemsInCart.map((item) => ({
+            product: item.productId,
+            color: item.colorId,
+            quantity: item.qty,
+        }));
+    },
 };
 </script>
 
