diff --git a/assets/js/pages/products.vue b/assets/js/pages/products.vue
index cd8533f7..5e0be519 100644
--- a/assets/js/pages/products.vue
+++ b/assets/js/pages/products.vue
@@ -46,7 +46,7 @@
                 </div>
 
                 <div class="row">
-                    <legend-component title="{{ legend }}" />
+                    <legend-component v-bind:title="legend" />
                 </div>
             </div>
         </div>
