diff --git a/assets/js/pages/products.vue b/assets/js/pages/products.vue
index 57b8a114..82b90502 100644
--- a/assets/js/pages/products.vue
+++ b/assets/js/pages/products.vue
@@ -46,9 +46,7 @@
                 </div>
 
                 <div class="row">
-                    <span class="p-3">
-                        {{ legend }}
-                    </span>
+                    <legend-component />
                 </div>
             </div>
         </div>
@@ -56,8 +54,13 @@
 </template>
 
 <script>
+import LegendComponent from '../components/legend';
+
 export default {
     name: 'Products',
+    components: {
+        LegendComponent,
+    },
     data() {
         return {
             legend: 'Shipping takes 10-13 weeks, and products probably won\'t work',
