diff --git a/assets/js/components/legend.vue b/assets/js/components/legend.vue
index 96696bc6..9bdd07ba 100644
--- a/assets/js/components/legend.vue
+++ b/assets/js/components/legend.vue
@@ -1,11 +1,12 @@
 <template>
     <span class="p-3">
-        Shipping takes 10-12 weeks, and products probably won't work
+        {{ title }}
     </span>
 </template>
 
 <script>
 export default {
     name: 'Legend',
+    props: ['title'],
 };
 </script>
diff --git a/assets/js/pages/products.vue b/assets/js/pages/products.vue
index 82b90502..6f1736fb 100644
--- a/assets/js/pages/products.vue
+++ b/assets/js/pages/products.vue
@@ -46,7 +46,7 @@
                 </div>
 
                 <div class="row">
-                    <legend-component />
+                    <legend-component title="TODO PUT LEGEND HERE" />
                 </div>
             </div>
         </div>
