diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index 3b39c9a6..fbcd09c3 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -59,6 +59,7 @@
 
                             <button
                                 class="btn btn-info btn-sm"
+                                :disabled="items.length === 0"
                             >
                                 Checkout!
                             </button>
