diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index a684b773..9e24064a 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -52,9 +52,25 @@
                             />
 
                             <div class="row p-3">
-                                <span class="col-12">
+                                <span class="col-3">
                                     Total: <strong>${{ totalPrice }}</strong>
                                 </span>
+
+                                <span
+                                    v-show="!loading"
+                                    class="col">
+                                    <button
+                                        class="btn btn-info btn-sm"
+                                    >
+                                        Clear Shopping Cart
+                                    </button>
+
+                                    <button
+                                        class="btn btn-info btn-sm"
+                                    >
+                                        Checkout!
+                                    </button>
+                                </span>
                             </div>
                         </div>
                     </div>
