diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index 9e24064a..36cb0a2a 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -61,6 +61,7 @@
                                     class="col">
                                     <button
                                         class="btn btn-info btn-sm"
+                                        @click="clearCart"
                                     >
                                         Clear Shopping Cart
                                     </button>
@@ -171,6 +172,16 @@ export default {
             this.loading = false;
         },
     },
+    methods: {
+        /**
+         * Clears the shopping cart and goes to the home page
+         */
+        clearCart() {
+            cartService.clear();
+
+            window.location = '/';
+        },
+    },
 };
 </script>
 
