diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index 91474541..0a6a2286 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -59,6 +59,7 @@
 
                                     <button
                                         class="btn btn-info btn-sm"
+                                        :disabled="items.length === 0"
                                     >
                                         Checkout!
                                     </button>
