diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index 36cb0a2a..34fa20ba 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -68,6 +68,7 @@
 
                                     <button
                                         class="btn btn-info btn-sm"
+                                        :disabled="items.length === 0"
                                     >
                                         Checkout!
                                     </button>
