diff --git a/assets/js/components/shopping-cart/actions.vue b/assets/js/components/shopping-cart/actions.vue
new file mode 100644
index 00000000..e33ea98e
--- /dev/null
+++ b/assets/js/components/shopping-cart/actions.vue
@@ -0,0 +1,34 @@
+<template>
+    <div
+        :class="$style.component"
+    >
+        <button
+            class="btn btn-info btn-sm"
+        >
+            Clear Shopping Cart
+        </button>
+
+        <button
+            class="btn btn-info btn-sm"
+        >
+            Check Out!
+        </button>
+    </div>
+</template>
+
+<script>
+export default {
+    name: 'CheckoutActions',
+};
+</script>
+
+<style lang="scss" module>
+.component {
+    display: flex;
+    justify-content: flex-end;
+
+    button {
+        margin-right: 20px;
+    }
+}
+</style>
