diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index d2e96932..7fb72702 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -57,6 +57,7 @@
 
                             <button
                                 class="btn btn-info btn-sm"
+                                :disabled="items.length === 0"
                             >
                                 Checkout!
                             </button>
