diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index 98be7037..ea3072c5 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -53,6 +53,7 @@
                             <button
                                 class="btn btn-info btn-sm"
                                 :disabled="items.length === 0"
+                                @click="checkout"
                             >
                                 Checkout!
                             </button>
@@ -153,6 +154,13 @@ export default {
 
             window.location = '/';
         },
+
+        /**
+         * Proceeds to check-out
+         */
+        checkout() {
+            window.location = '/checkout';
+        },
     },
 };
 </script>
