diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index 1c7772c7..581cc277 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -40,6 +40,7 @@
                                 <button
                                     class="btn btn-info btn-sm"
                                     :disabled="items.length === 0"
+                                    @click="checkout"
                                 >
                                     Checkout!
                                 </button>
@@ -136,6 +137,13 @@ export default {
 
             window.location = '/';
         },
+
+        /**
+         * Proceeds to check-out
+         */
+        checkout() {
+            window.location = '/checkout';
+        },
     },
 };
 </script>
