diff --git a/assets/js/components/shopping-cart/actions.vue b/assets/js/components/shopping-cart/actions.vue
index e33ea98e..3106dbf8 100644
--- a/assets/js/components/shopping-cart/actions.vue
+++ b/assets/js/components/shopping-cart/actions.vue
@@ -4,6 +4,7 @@
     >
         <button
             class="btn btn-info btn-sm"
+            @click="clearCart"
         >
             Clear Shopping Cart
         </button>
@@ -17,8 +18,20 @@
 </template>
 
 <script>
+import cartService from '@/services/cart';
+
 export default {
     name: 'CheckoutActions',
+    methods: {
+        /**
+         * Clears the shopping cart and goes to the home page
+         */
+        clearCart() {
+            cartService.clear();
+
+            window.location = '/';
+        },
+    },
 };
 </script>
 
