diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index 1af5ba2a..f1b9f7f1 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -26,16 +26,13 @@
                         />
 
                         <div class="row p-3">
-                            <span class="col-3">
-                                Total: <strong>${{ totalPrice }}</strong>
-                            </span>
-
                             <span
                                 v-show="!loading"
                                 class="col"
                             >
                                 <button
                                     class="btn btn-info btn-sm"
+                                    @click="clearCart"
                                 >
                                     Clear Shopping Cart
                                 </button>
@@ -57,6 +54,7 @@
 <script>
 import { getProductsById } from '@/services/products-service';
 import { getColors } from '@/services/colors-service';
+import { clearCart } from '@/services/cart-service';
 import Loading from '@/components/loading';
 import TitleComponent from '@/components/title';
 import ShoppingCartList from '@/components/shopping-cart';
@@ -128,6 +126,16 @@ export default {
             this.loading = false;
         },
     },
+    methods: {
+        /**
+         * Clears the shopping cart and goes to the home page
+         */
+        clearCart() {
+            clearCart();
+
+            window.location = '/';
+        },
+    },
 };
 </script>
 
