diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index c115d0d3..97fe4010 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -41,6 +41,8 @@
                                 <span>Total: <strong>${{ totalPrice }}</strong></span>
                             </div>
                         </div>
+
+                        <shopping-cart-actions v-show="!loading" />
                     </div>
                 </div>
             </div>
@@ -54,6 +56,7 @@ import colorsService from '@/services/colors';
 import formatPrice from '@/helpers/format-price';
 import productsService from '@/services/products';
 import Loading from '@/components/loading';
+import ShoppingCartActions from '@/components/shopping-cart/actions';
 import ShoppingCartItem from '@/components/shopping-cart/cart-item';
 import TitleComponent from '@/components/title';
 
@@ -61,6 +64,7 @@ export default {
     name: 'ShoppingCart',
     components: {
         Loading,
+        ShoppingCartActions,
         ShoppingCartItem,
         TitleComponent,
     },
