diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index 9e24064a..dc87f1bc 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -61,6 +61,7 @@
                                     class="col">
                                     <button
                                         class="btn btn-info btn-sm"
+                                        @click="clearCart"
                                     >
                                         Clear Shopping Cart
                                     </button>
@@ -83,6 +84,7 @@
 <script>
 import { getProductsById } from '@/services/products-service';
 import { getColors } from '@/services/colors-service';
+import { clearCart } from '@/services/cart-service';
 import formatPrice from '@/helpers/format-price';
 import Loading from '@/components/loading';
 import TitleComponent from '@/components/title';
@@ -171,6 +173,16 @@ export default {
             this.loading = false;
         },
     },
+    methods: {
+        /**
+         * Clears the shopping cart and goes to the home page
+         */
+        clearCart() {
+            clearCart();
+
+            window.location = '/';
+        },
+    },
 };
 </script>
 
