diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index c5d346a0..c3157b9d 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -52,6 +52,7 @@
                                     class="col">
                                     <button
                                         class="btn btn-info btn-sm"
+                                        @click="clearCart"
                                     >
                                         Clear Shopping Cart
                                     </button>
@@ -161,6 +162,16 @@ export default {
 
         this.loading = false;
     },
+    methods: {
+        /**
+         * Clears the shopping cart and goes to the home page
+         */
+        clearCart() {
+            cartService.clear();
+
+            window.location = '/';
+        },
+    },
 };
 </script>
 
