diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index 57d9d76f..717abb1d 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -60,6 +60,7 @@
                                     <button
                                         class="btn btn-info btn-sm"
                                         :disabled="items.length === 0"
+                                        @click="checkout"
                                     >
                                         Checkout!
                                     </button>
@@ -172,6 +173,13 @@ export default {
 
             window.location = '/';
         },
+
+        /**
+         * Proceeds to check-out
+         */
+        checkout() {
+            window.location = '/checkout';
+        },
     },
 };
 </script>
