diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index 334de6df..3b39c9a6 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -52,6 +52,7 @@
                         <div v-show="!loading">
                             <button
                                 class="btn btn-info btn-sm"
+                                @click="clearCart"
                             >
                                 Clear Shopping Cart
                             </button>
@@ -157,6 +158,16 @@ export default {
 
         this.loading = false;
     },
+    methods: {
+        /**
+         * Clears the shopping cart and goes to the home page
+         */
+        clearCart() {
+            cartService.clear();
+
+            window.location = '/';
+        },
+    },
 };
 </script>
 
