diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index fafa7f11..334de6df 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -48,6 +48,20 @@
                                 </span>
                             </div>
                         </div>
+
+                        <div v-show="!loading">
+                            <button
+                                class="btn btn-info btn-sm"
+                            >
+                                Clear Shopping Cart
+                            </button>
+
+                            <button
+                                class="btn btn-info btn-sm"
+                            >
+                                Checkout!
+                            </button>
+                        </div>
                     </div>
                 </div>
             </div>
