diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index 93958f9a..1af5ba2a 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -24,6 +24,29 @@
                             :items="items"
                             :cart="cart"
                         />
+
+                        <div class="row p-3">
+                            <span class="col-3">
+                                Total: <strong>${{ totalPrice }}</strong>
+                            </span>
+
+                            <span
+                                v-show="!loading"
+                                class="col"
+                            >
+                                <button
+                                    class="btn btn-info btn-sm"
+                                >
+                                    Clear Shopping Cart
+                                </button>
+
+                                <button
+                                    class="btn btn-info btn-sm"
+                                >
+                                    Checkout!
+                                </button>
+                            </span>
+                        </div>
                     </div>
                 </div>
             </div>
