diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index e4ce3665..3d824be2 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -19,12 +19,16 @@
                             <loading />
                         </div>
 
-                        <component
-                            :is="currentComponent"
-                            v-show="!loading"
-                            :items="items"
-                            :cart="cart"
-                        />
+                        <transition
+                            name="fade"
+                        >
+                            <component
+                                :is="currentComponent"
+                                v-show="!loading"
+                                :items="items"
+                                :cart="cart"
+                            />
+                        </transition>
 
                         <div
                             v-show="!loading && items.length"
@@ -155,5 +159,13 @@ export default {
     .content {
         @include light-component;
     }
+
+    .fade-enter-active, .fade-leave-active {
+        transition: opacity .2s;
+    }
+
+    .fade-enter, .fade-leave-to {
+        opacity: 0;
+    }
 }
 </style>
