diff --git a/assets/components/checkout/form-input.vue b/assets/components/checkout/form-input.vue
index 22f78c99a..aa9303332 100644
--- a/assets/components/checkout/form-input.vue
+++ b/assets/components/checkout/form-input.vue
@@ -15,6 +15,7 @@
                 'form-control': true,
             }"
             @input="$emit('input', $event.target.value)"
+            @blur="$emit('blur', $event)"
         >
         <span
             v-show="!isValid"
diff --git a/assets/components/checkout/index.vue b/assets/components/checkout/index.vue
index 66c351f03..2773d7c44 100644
--- a/assets/components/checkout/index.vue
+++ b/assets/components/checkout/index.vue
@@ -14,6 +14,7 @@
                         v-model="form.customerName"
                         class="col"
                         v-bind="getFieldProps('customerName', 'Name:')"
+                        @blur="validateForm"
                     />
 
                     <form-input
@@ -21,12 +22,14 @@
                         class="col"
                         type="email"
                         v-bind="getFieldProps('customerEmail', 'Email:')"
+                        @blur="validateForm"
                     />
                 </div>
 
                 <form-input
                     v-model="form.customerAddress"
                     v-bind="getFieldProps('customerAddress', 'Address:')"
+                    @blur="validateForm"
                 />
 
                 <div class="form-row">
@@ -34,12 +37,14 @@
                         v-model="form.customerZip"
                         class="col"
                         v-bind="getFieldProps('customerZip', 'Zip Code:')"
+                        @blur="validateForm"
                     />
 
                     <form-input
                         v-model="form.customerCity"
                         class="col"
                         v-bind="getFieldProps('customerCity', 'City:')"
+                        @blur="validateForm"
                     />
 
                     <form-input
@@ -47,6 +52,7 @@
                         class="col"
                         type="tel"
                         v-bind="getFieldProps('customerPhone', 'Phone Number:')"
+                        @blur="validateForm"
                     />
                 </div>
 
