diff --git a/assets/components/checkout/index.vue b/assets/components/checkout/index.vue
index d97e0732c..06575825a 100644
--- a/assets/components/checkout/index.vue
+++ b/assets/components/checkout/index.vue
@@ -146,8 +146,6 @@ export default {
             }
         },
         validateForm(event) {
-            console.log(this.validationErrors);
-
             const validationMessages = {
                 customerName: 'Please, enter your full name!',
                 customerEmail: 'Please, enter your email address!',
