diff --git a/assets/components/checkout/index.vue b/assets/components/checkout/index.vue
index 52f9cf25c..761c4dc4f 100644
--- a/assets/components/checkout/index.vue
+++ b/assets/components/checkout/index.vue
@@ -148,6 +148,16 @@ export default {
                 this.loading = false;
             }
         },
+        validateForm(event) {
+            const validationMessages = {
+                customerName: 'Please, enter your full name!',
+                customerEmail: 'Please, enter your email address!',
+                customerAddress: 'Please, enter your street address!',
+                customerZip: 'Please, enter your ZIP code!',
+                customerCity: 'Please, enter your city!',
+                customerPhone: 'Please, provide a phone number!',
+            };
+        },
     },
 };
 </script>
