diff --git a/assets/components/checkout/index.vue b/assets/components/checkout/index.vue
index 3b635d541..c52159eeb 100644
--- a/assets/components/checkout/index.vue
+++ b/assets/components/checkout/index.vue
@@ -145,7 +145,7 @@ export default {
                 this.loading = false;
             }
         },
-        validateField() {
+        validateField(event) {
             const validationMessages = {
                 customerName: 'Please, enter your full name!',
                 customerEmail: 'Please, enter your email address!',
@@ -154,6 +154,14 @@ export default {
                 customerCity: 'Please, enter your city!',
                 customerPhone: 'Please, provide a phone number!',
             };
+
+            const validationField = event.target.id;
+
+            if (!this.form[validationField]) {
+                this.validationErrors[validationField] = validationMessages[validationField];
+            } else {
+                delete this.validationErrors[validationField];
+            }
         },
     },
 };
