diff --git a/assets/components/checkout/form-input.vue b/assets/components/checkout/form-input.vue
index 02866233e..f8b29d00c 100644
--- a/assets/components/checkout/form-input.vue
+++ b/assets/components/checkout/form-input.vue
@@ -15,6 +15,7 @@
             }"
             :value="value"
             @input="$emit('input', $event.target.value)"
+            @blur="$emit('blur', $event)"
         >
         <span
             v-show="!isValid"
diff --git a/assets/components/checkout/index.vue b/assets/components/checkout/index.vue
index cecd8e272..b0c398379 100644
--- a/assets/components/checkout/index.vue
+++ b/assets/components/checkout/index.vue
@@ -12,33 +12,39 @@
                 <form-input
                     v-model="form.customerName"
                     v-bind="getFieldProps('customerName', 'Name:')"
+                    @blur="validateForm"
                 />
 
                 <form-input
                     v-model="form.customerEmail"
                     type="email"
                     v-bind="getFieldProps('customerEmail', 'Email:')"
+                    @blur="validateForm"
                 />
 
                 <form-input
                     v-model="form.customerAddress"
                     v-bind="getFieldProps('customerAddress', 'Address:')"
+                    @blur="validateForm"
                 />
 
                 <form-input
                     v-model="form.customerZip"
                     v-bind="getFieldProps('customerZip', 'Zip Code:')"
+                    @blur="validateForm"
                 />
 
                 <form-input
                     v-model="form.customerCity"
                     v-bind="getFieldProps('customerCity', 'City:')"
+                    @blur="validateForm"
                 />
 
                 <form-input
                     v-model="form.customerPhone"
                     type="tel"
                     v-bind="getFieldProps('customerPhone', 'Phone Number:')"
+                    @blur="validateForm"
                 />
 
                 <div class="row p-3 justify-content-end align-items-center">
