diff --git a/assets/js/components/checkout/index.vue b/assets/js/components/checkout/index.vue
index 0517ce8f..07c2d558 100644
--- a/assets/js/components/checkout/index.vue
+++ b/assets/js/components/checkout/index.vue
@@ -1,7 +1,7 @@
 <template>
     <div class="row p-3">
         <div class="col-12">
-            <form @submit="onSubmit">
+            <form @submit.prevent="onSubmit">
                 <div
                     v-show="serverError"
                     class="alert alert-danger m-3"
@@ -121,7 +121,6 @@ export default {
             };
         },
         async onSubmit(event) {
-            event.preventDefault();
             this.loading = true;
             this.serverError = false;
             this.validationErrors = {};
