diff --git a/assets/js/pages/checkout.vue b/assets/js/pages/checkout.vue
index 102d9485..f8def629 100644
--- a/assets/js/pages/checkout.vue
+++ b/assets/js/pages/checkout.vue
@@ -180,6 +180,14 @@ export default {
                 customerCity: 'Please, enter your City!',
                 customerPhone: 'Please, provide a phone number!',
             };
+
+            const validationField = event.target.id;
+
+            if (!this.form[validationField]) {
+                this.validationErrors[validationField] = validationMessages[validationField];
+            } else {
+                this.validationErrors[validationField] = null;
+            }
         },
     },
 };
