diff --git a/assets/components/checkout/index.vue b/assets/components/checkout/index.vue
index 18c4b96a3..d0df6885d 100644
--- a/assets/components/checkout/index.vue
+++ b/assets/components/checkout/index.vue
@@ -126,7 +126,9 @@ export default {
         async onSubmit() {
             this.loading = true;
             this.serverError = false;
-            this.validationErrors = {};
+            this.validationErrors = {
+                customerName: null
+            };
 
             try {
                 const response = await createOrder({
