diff --git a/assets/js/components/checkout/index.vue b/assets/js/components/checkout/index.vue
index b0c39837..3b069e89 100644
--- a/assets/js/components/checkout/index.vue
+++ b/assets/js/components/checkout/index.vue
@@ -3,7 +3,7 @@
         <div class="col-12">
             <form @submit.prevent="onSubmit">
                 <div
-                    v-show="serverError"
+                    v-show="formError"
                     class="alert alert-danger m-3"
                 >
                     Oops, there's been an error sending your data! Please, try again!
@@ -72,6 +72,7 @@
 import FormInput from '@/components/checkout/form-input';
 import Loading from '@/components/loading';
 import { createOrder } from '@/services/checkout-service';
+import { clearCart } from '@/services/cart-service';
 
 export default {
     name: 'CheckoutForm',
@@ -124,6 +125,10 @@ export default {
 
             try {
                 const response = await createOrder(this.form);
+                await clearCart();
+
+                const id = Number(response.data['@id'].split('/').pop());
+                window.location = `/confirmation/${id}`;
             } catch (error) {
                 const { response } = error;
 
