diff --git a/assets/js/components/checkout/form-input.vue b/assets/js/components/checkout/form-input.vue
index b35c2cdc..46fe3a4f 100644
--- a/assets/js/components/checkout/form-input.vue
+++ b/assets/js/components/checkout/form-input.vue
@@ -15,6 +15,7 @@
             }"
             :value="value"
             @input="$emit('input', $event.target.value)"
+            @blur="$emit('blur', $event)"
         >
         <span
             v-show="!isValid"
diff --git a/assets/js/pages/checkout.vue b/assets/js/pages/checkout.vue
index 4c1aa54e..60127d39 100644
--- a/assets/js/pages/checkout.vue
+++ b/assets/js/pages/checkout.vue
@@ -23,6 +23,7 @@
                                 v-model="form.customerName"
                                 label="Name:"
                                 :error-message="validationErrors.customerName"
+                                @blur="validateForm"
                             />
 
                             <form-input
@@ -31,6 +32,7 @@
                                 label="Email:"
                                 type="email"
                                 :error-message="validationErrors.customerEmail"
+                                @blur="validateForm"
                             />
 
                             <form-input
@@ -38,6 +40,7 @@
                                 v-model="form.customerAddress"
                                 label="Address:"
                                 :error-message="validationErrors.customerAddress"
+                                @blur="validateForm"
                             />
 
                             <form-input
@@ -45,6 +48,7 @@
                                 v-model="form.customerZip"
                                 label="Zip Code:"
                                 :error-message="validationErrors.customerZip"
+                                @blur="validateForm"
                             />
 
                             <form-input
@@ -52,6 +56,7 @@
                                 v-model="form.customerCity"
                                 label="City:"
                                 :error-message="validationErrors.customerCity"
+                                @blur="validateForm"
                             />
 
                             <form-input
@@ -60,6 +65,7 @@
                                 label="Phone Number:"
                                 type="tel"
                                 :error-message="validationErrors.customerPhone"
+                                @blur="validateForm"
                             />
 
                             <div :class="$style['form-action']">
