diff --git a/assets/components/checkout/index.vue b/assets/components/checkout/index.vue
index b0284558b..b07c4f389 100644
--- a/assets/components/checkout/index.vue
+++ b/assets/components/checkout/index.vue
@@ -162,6 +162,8 @@ export default {
             }
         },
         validateForm(event) {
+            console.log(this.validationErrors);
+
             const validationMessages = {
                 customerName: 'Please, enter your full name!',
                 customerEmail: 'Please, enter your email address!',
