diff --git a/assets/components/checkout/index.vue b/assets/components/checkout/index.vue
index 87758c66f..618659405 100644
--- a/assets/components/checkout/index.vue
+++ b/assets/components/checkout/index.vue
@@ -96,7 +96,14 @@ export default {
                 customerCity: '',
                 customerPhone: '',
             },
-            validationErrors: {},
+            validationErrors: {
+                customerName: null,
+                customerEmail: null,
+                customerAddress: null,
+                customerZip: null,
+                customerCity: null,
+                customerPhone: null,
+            },
             loading: false,
             serverError: false,
         };
