diff --git a/assets/components/checkout/index.vue b/assets/components/checkout/index.vue
index 9108d8bcf..cecd8e272 100644
--- a/assets/components/checkout/index.vue
+++ b/assets/components/checkout/index.vue
@@ -141,6 +141,14 @@ export default {
                 customerCity: 'Please, enter your City!',
                 customerPhone: 'Please, provide a phone number!',
             };
+
+            const validationField = event.target.id;
+
+            if (!this.form[validationField]) {
+                this.validationErrors[validationField] = validationMessages[validationField];
+            } else {
+                delete this.validationErrors[validationField];
+            }
         },
     },
 };
