diff --git a/assets/components/checkout/index.vue b/assets/components/checkout/index.vue
index 97d5f56fb..a170ccd10 100644
--- a/assets/components/checkout/index.vue
+++ b/assets/components/checkout/index.vue
@@ -174,7 +174,7 @@ export default {
             if (!this.form[validationField]) {
                 this.validationErrors[validationField] = validationMessages[validationField];
             } else {
-                delete this.validationErrors[validationField];
+                this.validationErrors[validationField] = null;
             }
         },
     },
