diff --git a/assets/components/checkout/index.vue b/assets/components/checkout/index.vue
index 318fc1f9e..ed178ddae 100644
--- a/assets/components/checkout/index.vue
+++ b/assets/components/checkout/index.vue
@@ -148,6 +148,14 @@ export default {
                 customerCity: 'Please, enter your city!',
                 customerPhone: 'Please, provide a phone number!',
             };
+
+            const validationField = event.target.id;
+
+            if (!this.form[validationField]) {
+                this.validationErrors[validationField] = validationMessages[validationField];
+            } else {
+                delete this.validationErrors[validationField];
+            }
         },
     },
 };
