diff --git a/assets/js/components/checkout/form-input.vue b/assets/js/components/checkout/form-input.vue
index 30a17f64..cf729da3 100644
--- a/assets/js/components/checkout/form-input.vue
+++ b/assets/js/components/checkout/form-input.vue
@@ -14,6 +14,7 @@
                 'form-control': true,
             }"
             @input="$emit('input', $event.target.value)"
+            @blur="$emit('blur', $event)"
         >
         <span
             v-show="!valid"
diff --git a/assets/js/pages/checkout.vue b/assets/js/pages/checkout.vue
index 2c5d4212..6c2259c8 100644
--- a/assets/js/pages/checkout.vue
+++ b/assets/js/pages/checkout.vue
@@ -23,6 +23,7 @@
                                 v-model="form.customerName"
                                 label="Name:"
                                 :error-message="validation.customerName"
+                                @blur="validateForm"
                             />
 
                             <form-input
@@ -30,6 +31,7 @@
                                 v-model="form.customerEmail"
                                 label="Email:"
                                 :error-message="validation.customerEmail"
+                                @blur="validateForm"
                             />
 
                             <form-input
@@ -37,6 +39,7 @@
                                 v-model="form.customerAddress"
                                 label="Address:"
                                 :error-message="validation.customerAddress"
+                                @blur="validateForm"
                             />
 
                             <form-input
@@ -44,6 +47,7 @@
                                 v-model="form.customerZip"
                                 label="Zip Code:"
                                 :error-message="validation.customerZip"
+                                @blur="validateForm"
                             />
 
                             <form-input
@@ -51,6 +55,7 @@
                                 v-model="form.customerCity"
                                 label="City:"
                                 :error-message="validation.customerCity"
+                                @blur="validateForm"
                             />
 
                             <form-input
@@ -58,6 +63,7 @@
                                 v-model="form.customerPhone"
                                 label="Phone Number:"
                                 :error-message="validation.customerPhone"
+                                @blur="validateForm"
                             />
 
                             <div :class="$style['form-action']">
