diff --git a/assets/js/components/checkout/index.vue b/assets/js/components/checkout/index.vue
index 5d4e947f..b1785bc6 100644
--- a/assets/js/components/checkout/index.vue
+++ b/assets/js/components/checkout/index.vue
@@ -117,7 +117,7 @@ export default {
         async onSubmit(event) {
             event.preventDefault();
             this.loading = true;
-            this.formError = false;
+            this.serverError = false;
             this.validationErrors = {};
 
             try {
@@ -137,6 +137,16 @@ export default {
                 this.loading = false;
             }
         },
+        validateForm(event) {
+            const validationMessages = {
+                customerName: 'Please, enter your full name!',
+                customerEmail: 'Please, enter your email address!',
+                customerAddress: 'Please, enter your street address!',
+                customerZip: 'Please, enter your ZIP code!',
+                customerCity: 'Please, enter your City!',
+                customerPhone: 'Please, provide a phone number!',
+            };
+        },
     },
 };
 </script>
