diff --git a/assets/components/checkout/index.vue b/assets/components/checkout/index.vue
index cdcc2d32f..f49010c4b 100644
--- a/assets/components/checkout/index.vue
+++ b/assets/components/checkout/index.vue
@@ -152,8 +152,6 @@ export default {
             }
         },
         validateField(event) {
-            console.log(this.validationErrors);
-
             const validationMessages = {
                 customerName: 'Please, enter your full name!',
                 customerEmail: 'Please, enter your email address!',
