diff --git a/assets/components/checkout/index.vue b/assets/components/checkout/index.vue
index afab21d13..064ded6d9 100644
--- a/assets/components/checkout/index.vue
+++ b/assets/components/checkout/index.vue
@@ -159,6 +159,8 @@ export default {
             }
         },
         validateField(event) {
+            console.log(this.validationErrors);
+
             const validationMessages = {
                 customerName: 'Please, enter your full name!',
                 customerEmail: 'Please, enter your email address!',
