diff --git a/assets/components/checkout/index.vue b/assets/components/checkout/index.vue
index b07c4f389..80920e747 100644
--- a/assets/components/checkout/index.vue
+++ b/assets/components/checkout/index.vue
@@ -135,7 +135,9 @@ export default {
         async onSubmit() {
             this.loading = true;
             this.serverError = false;
-            this.validationErrors = {};
+            this.validationErrors = {
+                customerName: null
+            };
 
             try {
                 const response = await createOrder({
