diff --git a/assets/components/checkout/index.vue b/assets/components/checkout/index.vue
index 357d4e2ec..40f62ec85 100644
--- a/assets/components/checkout/index.vue
+++ b/assets/components/checkout/index.vue
@@ -146,7 +146,7 @@ export default {
                 this.loading = false;
             }
         },
-        validateForm() {
+        validateForm(event) {
             const validationMessages = {
                 customerName: 'Please, enter your full name!',
                 customerEmail: 'Please, enter your email address!',
@@ -155,6 +155,14 @@ export default {
                 customerCity: 'Please, enter your city!',
                 customerPhone: 'Please, provide a phone number!',
             };
+
+            const validationField = event.target.id;
+
+            if (!this.form[validationField]) {
+                this.validationErrors[validationField] = validationMessages[validationField];
+            } else {
+                delete this.validationErrors[validationField];
+            }
         },
     },
 };
