diff --git a/assets/js/pages/checkout.vue b/assets/js/pages/checkout.vue
index d2c0572e..fdf1144e 100644
--- a/assets/js/pages/checkout.vue
+++ b/assets/js/pages/checkout.vue
@@ -108,14 +108,7 @@ export default {
                 customerPhone: '',
                 purchaseItems: [],
             },
-            validationErrors: {
-                customerName: null,
-                customerEmail: null,
-                customerAddress: null,
-                customerZip: null,
-                customerCity: null,
-                customerPhone: null,
-            },
+            validationErrors: this.initializeValidationFields(),
             loading: false,
             serverError: false,
         };
@@ -139,7 +132,7 @@ export default {
             event.preventDefault();
             this.loading = true;
             this.serverError = false;
-            this.resetValidationFields();
+            this.validation = this.initializeValidationFields();
 
             try {
                 const response = await createOrder(this.form);
@@ -160,10 +153,12 @@ export default {
         },
 
         /**
-         * Resets our validation fields back to null
+         * Returns the initial validation fields set to null
+         *
+         * @return {object}
          */
-        resetValidationFields() {
-            this.validation = {
+        initializeValidationFields() {
+            return {
                 customerName: null,
                 customerEmail: null,
                 customerAddress: null,
