diff --git a/assets/components/checkout/form-input.vue b/assets/components/checkout/form-input.vue
index 635f55b9f..40fbf317f 100644
--- a/assets/components/checkout/form-input.vue
+++ b/assets/components/checkout/form-input.vue
@@ -16,6 +16,7 @@
                 'form-control': true,
             }"
             @input="$emit('input', $event.target.value)"
+            @blur="$emit('blur', $event)"
         >
         <span
             v-show="!isValid"
diff --git a/assets/components/checkout/index.vue b/assets/components/checkout/index.vue
index 40f62ec85..b636b7868 100644
--- a/assets/components/checkout/index.vue
+++ b/assets/components/checkout/index.vue
@@ -15,6 +15,7 @@
                         v-model="form.customerName"
                         class="col"
                         v-bind="getFieldProps('customerName', 'Name:')"
+                        @blur="validateForm"
                     />
 
                     <form-input
@@ -22,12 +23,14 @@
                         class="col"
                         type="email"
                         v-bind="getFieldProps('customerEmail', 'Email:')"
+                        @blur="validateForm"
                     />
                 </div>
 
                 <form-input
                     v-model="form.customerAddress"
                     v-bind="getFieldProps('customerAddress', 'Address:')"
+                    @blur="validateForm"
                 />
 
                 <div class="form-row">
@@ -35,12 +38,14 @@
                         v-model="form.customerZip"
                         class="col"
                         v-bind="getFieldProps('customerZip', 'Zip Code:')"
+                        @blur="validateForm"
                     />
 
                     <form-input
                         v-model="form.customerCity"
                         class="col"
                         v-bind="getFieldProps('customerCity', 'City:')"
+                        @blur="validateForm"
                     />
 
                     <form-input
@@ -48,6 +53,7 @@
                         class="col"
                         type="tel"
                         v-bind="getFieldProps('customerPhone', 'Phone Number:')"
+                        @blur="validateForm"
                     />
                 </div>
 
