diff --git a/assets/js/components/checkout/form-input.vue b/assets/js/components/checkout/form-input.vue
index 3dd97d79..ceac325b 100644
--- a/assets/js/components/checkout/form-input.vue
+++ b/assets/js/components/checkout/form-input.vue
@@ -15,6 +15,7 @@
             }"
             :value="value"
             @input="$emit('input', $event.target.value)"
+            @blur="$emit('blur', $event)"
         >
         <span
             v-show="!isValid"
diff --git a/assets/js/components/checkout/index.vue b/assets/js/components/checkout/index.vue
index 52b05e3b..0517ce8f 100644
--- a/assets/js/components/checkout/index.vue
+++ b/assets/js/components/checkout/index.vue
@@ -12,31 +12,37 @@
                 <form-input
                     v-model="form.customerName"
                     v-bind="getFormFields('customerName', 'Name:')"
+                    @blur="validateForm"
                 />
 
                 <form-input
                     v-model="form.customerEmail"
                     v-bind="getFormFields('customerEmail', 'Email:', 'email')"
+                    @blur="validateForm"
                 />
 
                 <form-input
                     v-model="form.customerAddress"
                     v-bind="getFormFields('customerAddress', 'Address:')"
+                    @blur="validateForm"
                 />
 
                 <form-input
                     v-model="form.customerZip"
                     v-bind="getFormFields('customerZip', 'Zip Code:')"
+                    @blur="validateForm"
                 />
 
                 <form-input
                     v-model="form.customerCity"
                     v-bind="getFormFields('customerCity', 'City:')"
+                    @blur="validateForm"
                 />
 
                 <form-input
                     v-model="form.customerPhone"
                     v-bind="getFormFields('customerPhone', 'Phone Number:', 'tel')"
+                    @blur="validateForm"
                 />
 
                 <div class="row p-3 justify-content-end align-items-center">
