diff --git a/assets/js/components/sidebar.vue b/assets/js/components/sidebar.vue
index 9d7cd1619..818a244ea 100644
--- a/assets/js/components/sidebar.vue
+++ b/assets/js/components/sidebar.vue
@@ -28,6 +28,16 @@
                 </a>
             </li>
         </ul>
+
+        <hr>
+
+        <div class="d-flex justify-content-end">
+            <button
+                class="btn btn-secondary btn-sm"
+            >
+                {{ collapsed ? '>>' : '<< Collapse' }}
+            </button>
+        </div>
     </div>
 </template>
 
