diff --git a/assets/js/pages/products.vue b/assets/js/pages/products.vue
index 60ac57ff..1ae23aa4 100644
--- a/assets/js/pages/products.vue
+++ b/assets/js/pages/products.vue
@@ -1,14 +1,14 @@
 <template>
     <div class="container-fluid">
         <div class="row">
-            <aside class="col-xs-12 col-3">
+            <aside :class="asideClass">
                 <sidebar
                     :collapsed="sidebarCollapsed"
                     @toggle-collapsed="toggleSidebarCollapsed"
                 />
             </aside>
 
-            <div class="col-xs-12 col-9">
+            <div :class="contentClass">
                 <catalog />
             </div>
         </div>
@@ -30,6 +30,14 @@ export default {
             sidebarCollapsed: false,
         };
     },
+    computed: {
+        asideClass() {
+            return this.sidebarCollapsed ? 'aside-collapsed' : 'col-xs-12 col-3';
+        },
+        contentClass() {
+            return this.sidebarCollapsed ? 'col-xs-12 col-11' : 'col-xs-12 col-9';
+        },
+    },
     methods: {
         toggleSidebarCollapsed() {
             this.sidebarCollapsed = !this.sidebarCollapsed;
diff --git a/assets/scss/app.scss b/assets/scss/app.scss
index 819a5524..79b4e2f7 100644
--- a/assets/scss/app.scss
+++ b/assets/scss/app.scss
@@ -53,3 +53,7 @@ body {
     flex-direction: column;
     width: 900px;
 }
+
+.aside-collapsed {
+    padding: 0 15px;
+}
