diff --git a/assets/js/products.vue b/assets/js/products.vue
index 2920941..0f8cbf7 100644
--- a/assets/js/products.vue
+++ b/assets/js/products.vue
@@ -1,14 +1,14 @@
 <template>
     <div class="container-fluid">
         <div class="row row-no-wrap">
-            <aside class="col-xs-12 col-lg-3">
+            <aside :class="asideClass">
                 <sidebar-component
                     :collapsed="sidebarCollapsed"
                     @sidebar-collapsed="toggleSidebarCollapsed"
                 />
             </aside>
 
-            <div class="col-xs-12 col-lg-9">
+            <div :class="contentClass">
                 <catalog-component />
             </div>
         </div>
@@ -28,6 +28,14 @@ export default {
     data: () => ({
         sidebarCollapsed: false,
     }),
+    computed: {
+        asideClass() {
+            return this.sidebarCollapsed ? 'aside-collapsed' : 'col-xs-12 col-lg-3';
+        },
+        contentClass() {
+            return this.sidebarCollapsed ? 'col-xs-12 col-lg-11' : 'col-xs-12 col-lg-9';
+        },
+    },
     methods: {
         /**
          * Toggles the sidebarCollapsed value
diff --git a/assets/scss/app.scss b/assets/scss/app.scss
index 21e2e7a..7c847e7 100644
--- a/assets/scss/app.scss
+++ b/assets/scss/app.scss
@@ -1,6 +1,7 @@
 @import '~styles/components/black-component';
 @import '~styles/components/light-component';
 @import '~styles/variables/colors';
+@import '~styles/collapsible';
 @import '~bootstrap';
 
 .header {
diff --git a/assets/scss/collapsible.scss b/assets/scss/collapsible.scss
new file mode 100644
index 0000000..8a060a4
--- /dev/null
+++ b/assets/scss/collapsible.scss
@@ -0,0 +1,7 @@
+.aside-collapsed {
+  padding: 0 15px;
+}
+
+.row-no-wrap {
+  flex-wrap: nowrap !important;
+}
