diff --git a/assets/js/pages/products.vue b/assets/js/pages/products.vue
index 419509f4..d3dbea5e 100644
--- a/assets/js/pages/products.vue
+++ b/assets/js/pages/products.vue
@@ -2,7 +2,7 @@
     <div class="container-fluid">
         <div class="row">
             <aside class="col-xs-12 col-3">
-                <sidebar />
+                <sidebar :collapsed="sidebarCollapsed" />
             </aside>
 
             <div class="col-xs-12 col-9">
@@ -22,5 +22,10 @@ export default {
         Catalog,
         Sidebar,
     },
+    data() {
+        return {
+            sidebarCollapsed: false,
+        };
+    },
 };
 </script>
