diff --git a/assets/js/components/sidebar.vue b/assets/js/components/sidebar.vue
index 134b882a..1e1c38d4 100644
--- a/assets/js/components/sidebar.vue
+++ b/assets/js/components/sidebar.vue
@@ -43,8 +43,13 @@
 <script>
 export default {
     name: 'Sidebar',
+    props: {
+        collapsed: {
+            type: Boolean,
+            required: true,
+        },
+    },
     data: () => ({
-        collapsed: false,
         categories: [
             {
                 name: 'Category A',
