diff --git a/assets/js/components/sidebar.vue b/assets/js/components/sidebar.vue
index 935fe3e..b101b79 100644
--- a/assets/js/components/sidebar.vue
+++ b/assets/js/components/sidebar.vue
@@ -28,6 +28,13 @@
                 </a>
             </li>
         </ul>
+
+        <div :class="$style.buttons">
+            <button
+                class="btn btn-secondary btn-sm"
+                v-text="collapsed ? '>>' : '<< Collapse'"
+            />
+        </div>
     </div>
 </template>
 
@@ -70,5 +77,11 @@ export default {
             background: $blue-component-link-hover;
         }
     }
+
+    .buttons {
+        display: flex;
+        flex-direction: row;
+        justify-content: flex-end;
+    }
 }
 </style>
