diff --git a/assets/js/components/sidebar.vue b/assets/js/components/sidebar.vue
index d293499f..7db80dd8 100644
--- a/assets/js/components/sidebar.vue
+++ b/assets/js/components/sidebar.vue
@@ -1,5 +1,5 @@
 <template>
-    <div :class="componentClass">
+    <div :class="[$style.component, 'p-3', 'mb-5']">
         <div v-show="!collapsed">
             <h5 class="text-center">
                 Categories
@@ -61,22 +61,6 @@ export default {
             },
         ],
     }),
-    computed: {
-        /**
-         * Computes the component classes depending on collapsed state
-         *
-         * @return string[]
-         */
-        componentClass() {
-            const classes = [this.$style.component, 'p-3', 'mb-5'];
-
-            if (this.collapsed) {
-                classes.push(this.$style.collapsed);
-            }
-
-            return classes;
-        },
-    },
 };
 </script>
 
@@ -86,10 +70,6 @@ export default {
 .component {
     @include light-component;
 
-    &.collapsed {
-        width: 70px;
-    }
-
     ul {
         li a:hover {
             background: $blue-component-link-hover;
