diff --git a/assets/js/components/sidebar.vue b/assets/js/components/sidebar.vue
index baadf4a..244b1d4 100644
--- a/assets/js/components/sidebar.vue
+++ b/assets/js/components/sidebar.vue
@@ -3,7 +3,7 @@
         :class="[$style.component, 'p-3', 'mt-5', 'mb-5']"
         :style="{ width: collapsed ? '70px' : 'auto' }"
     >
-        <div>
+        <div  v-if="!collapsed">
             <h5 class="text-center">
                 Categories
             </h5>
