diff --git a/assets/js/components/sidebar.vue b/assets/js/components/sidebar.vue
index 5b8ce31a..5cfcdd58 100644
--- a/assets/js/components/sidebar.vue
+++ b/assets/js/components/sidebar.vue
@@ -34,6 +34,7 @@
         <div class="d-flex justify-content-end">
             <button
                 class="btn btn-secondary btn-sm"
+                v-on:click="toggleCollapsed"
                 v-text="collapsed ? '>>' : '<< Collapse'"
             />
         </div>
@@ -58,6 +59,11 @@ export default {
             ],
         };
     },
+    methods: {
+        toggleCollapsed() {
+            console.log('CLICKED!');
+        },
+    },
 };
 </script>
 
