diff --git a/assets/js/pages/products.vue b/assets/js/pages/products.vue
index d3dbea5e..586c4ffb 100644
--- a/assets/js/pages/products.vue
+++ b/assets/js/pages/products.vue
@@ -2,7 +2,10 @@
     <div class="container-fluid">
         <div class="row">
             <aside class="col-xs-12 col-3">
-                <sidebar :collapsed="sidebarCollapsed" />
+                <sidebar
+                    :collapsed="sidebarCollapsed"
+                    @toggle-collapsed="toggleSidebarCollapsed"
+                />
             </aside>
 
             <div class="col-xs-12 col-9">
@@ -27,5 +30,13 @@ export default {
             sidebarCollapsed: false,
         };
     },
+    methods: {
+        /**
+         * Toggles the sidebarCollapsed value
+         */
+        toggleSidebarCollapsed() {
+            this.sidebarCollapsed = !this.sidebarCollapsed;
+        },
+    },
 };
 </script>
