diff --git a/assets/js/pages/products.vue b/assets/js/pages/products.vue
index dcab241b..60ac57ff 100644
--- a/assets/js/pages/products.vue
+++ b/assets/js/pages/products.vue
@@ -4,7 +4,7 @@
             <aside class="col-xs-12 col-3">
                 <sidebar
                     :collapsed="sidebarCollapsed"
-                    v-on:toggle-collapsed="toggleSidebarCollapsed"
+                    @toggle-collapsed="toggleSidebarCollapsed"
                 />
             </aside>
 
