diff --git a/assets/js/components/sidebar.vue b/assets/js/components/sidebar.vue
index a4a70a1c..ae8d113b 100644
--- a/assets/js/components/sidebar.vue
+++ b/assets/js/components/sidebar.vue
@@ -3,33 +3,35 @@
         :class="[$style.component, 'p-3', 'mb-5']"
         :style="{ width: collapsed ? '70px' : 'auto' }"
     >
-        <h5 class="text-center">
-            Categories
-        </h5>
+        <div v-show="!collapsed">
+            <h5 class="text-center">
+                Categories
+            </h5>
 
-        <ul class="nav flex-column mb4">
-            <li class="nav-item">
-                <a
-                    class="nav-link"
-                    href="/"
-                >All Products</a>
-            </li>
+            <ul class="nav flex-column mb4">
+                <li class="nav-item">
+                    <a
+                        class="nav-link"
+                        href="/"
+                    >All Products</a>
+                </li>
 
-            <li
-                v-for="(category, index) in categories"
-                :key="index"
-                class="nav-item"
-            >
-                <a
-                    :href="category.link"
-                    class="nav-link"
+                <li
+                    v-for="(category, index) in categories"
+                    :key="index"
+                    class="nav-item"
                 >
-                    {{ category.name }}
-                </a>
-            </li>
-        </ul>
+                    <a
+                        :href="category.link"
+                        class="nav-link"
+                    >
+                        {{ category.name }}
+                    </a>
+                </li>
+            </ul>
 
-        <hr>
+            <hr>
+        </div>
 
         <div class="d-flex justify-content-end">
             <button
