diff --git a/assets/js/components/sidebar.vue b/assets/js/components/sidebar.vue
index eb822b6c..117c5225 100644
--- a/assets/js/components/sidebar.vue
+++ b/assets/js/components/sidebar.vue
@@ -28,6 +28,15 @@
                 </a>
             </li>
         </ul>
+
+        <hr>
+
+        <div class="d-flex justify-content-end">
+            <button
+                class="btn btn-secondary btn-sm"
+                v-text="collapsed ? '>>' : '<< Collapse'"
+            />
+        </div>
     </div>
 </template>
 
