diff --git a/assets/components/checkout/form-input.vue b/assets/components/checkout/form-input.vue
index c5e3632f8..22f78c99a 100644
--- a/assets/components/checkout/form-input.vue
+++ b/assets/components/checkout/form-input.vue
@@ -8,8 +8,8 @@
         </label>
         <input
             :id="id"
+            :type="type"
             :value="value"
-            type="text"
             :class="{
                 'is-invalid': !isValid,
                 'form-control': true,
@@ -41,6 +41,10 @@ export default {
             type: String,
             default: '',
         },
+        type: {
+            type: String,
+            default: 'text',
+        },
         value: {
             type: String,
             default: '',
