diff --git a/assets/js/components/checkout/form-input.vue b/assets/js/components/checkout/form-input.vue
index 10e506e1..3dd97d79 100644
--- a/assets/js/components/checkout/form-input.vue
+++ b/assets/js/components/checkout/form-input.vue
@@ -8,7 +8,7 @@
         </label>
         <input
             :id="id"
-            type="text"
+            :type="type"
             :class="{
                 'is-invalid': !isValid,
                 'form-control': true,
@@ -42,6 +42,11 @@ export default {
             required: false,
             default: '',
         },
+        type: {
+            type: String,
+            required: false,
+            default: 'text',
+        },
         value: {
             type: String,
             required: false,
