diff --git a/assets/components/checkout/index.vue b/assets/components/checkout/index.vue
index dec024d51..9b8d17411 100644
--- a/assets/components/checkout/index.vue
+++ b/assets/components/checkout/index.vue
@@ -2,15 +2,25 @@
     <div class="row p-3">
         <div class="col-12">
             <form>
-
+                <form-input
+                    id="customerName"
+                    :value="form.customerName"
+                    label="Name:"
+                    :error-message="validationErrors.customerName"
+                />
             </form>
         </div>
     </div>
 </template>
 
 <script>
+import FormInput from '@/components/checkout/form-input';
+
 export default {
     name: 'CheckoutForm',
+    components: {
+        FormInput,
+    },
     props: {
         cart: {
             type: Object,
