diff --git a/assets/components/checkout/index.vue b/assets/components/checkout/index.vue
index 60f8b4dd..3ea1e316 100644
--- a/assets/components/checkout/index.vue
+++ b/assets/components/checkout/index.vue
@@ -2,7 +2,12 @@
     <div class="row p-3">
         <div class="col-12">
             <form>
-                <form-input />
+                <form-input
+                    id="customerName"
+                    :value="form.customerName"
+                    label="Name:"
+                    :error-message="validationErrors.customerName"
+                />
             </form>
         </div>
     </div>
