diff --git a/assets/components/checkout/index.vue b/assets/components/checkout/index.vue
index cbf971587..94dfafd32 100644
--- a/assets/components/checkout/index.vue
+++ b/assets/components/checkout/index.vue
@@ -9,6 +9,7 @@
 
                 <form-input
                     v-model="form.customerEmail"
+                    type="email"
                     v-bind="getFieldProps('customerEmail', 'Email:')"
                 />
 
@@ -29,6 +30,7 @@
 
                 <form-input
                     v-model="form.customerPhone"
+                    type="tel"
                     v-bind="getFieldProps('customerPhone', 'Phone Number:')"
                 />
             </form>
