diff --git a/assets/components/checkout/form-input.vue b/assets/components/checkout/form-input.vue
index 052e4f661..848835cdd 100644
--- a/assets/components/checkout/form-input.vue
+++ b/assets/components/checkout/form-input.vue
@@ -14,6 +14,7 @@
                 'is-invalid': !isValid,
                 'form-control': true,
             }"
+            @input="$emit('input', $event.target.value)"
         >
         <span
             v-show="!isValid"
diff --git a/assets/components/checkout/index.vue b/assets/components/checkout/index.vue
index bf73c0154..af7d7b725 100644
--- a/assets/components/checkout/index.vue
+++ b/assets/components/checkout/index.vue
@@ -4,7 +4,7 @@
             <form>
                 <form-input
                     id="customerName"
-                    :value="form.customerName"
+                    v-model="form.customerName"
                     label="Name:"
                     :error-message="validationErrors.customerName"
                 />
