diff --git a/assets/components/checkout/form-input.vue b/assets/components/checkout/form-input.vue
index 0008e8c24..177555c99 100644
--- a/assets/components/checkout/form-input.vue
+++ b/assets/components/checkout/form-input.vue
@@ -14,6 +14,7 @@
                 'form-control': true,
             }"
             :value="value"
+            @input="$emit('input', $event.target.value)"
         >
         <span
             v-show="!isValid"
diff --git a/assets/components/checkout/index.vue b/assets/components/checkout/index.vue
index 9b8d17411..613711a35 100644
--- a/assets/components/checkout/index.vue
+++ b/assets/components/checkout/index.vue
@@ -4,7 +4,7 @@
             <form>
                 <form-input
                     id="customerName"
-                    :value="form.customerName"
+                    v-model="form.customerName"
                     label="Name:"
                     :error-message="validationErrors.customerName"
                 />
