diff --git a/assets/js/components/checkout/form-input.vue b/assets/js/components/checkout/form-input.vue
index 52715db0..3f753fa4 100644
--- a/assets/js/components/checkout/form-input.vue
+++ b/assets/js/components/checkout/form-input.vue
@@ -26,6 +26,21 @@
 <script>
 export default {
     name: 'FormInput',
+    props: {
+        id: {
+            type: String,
+            required: true,
+        },
+        label: {
+            type: String,
+            required: true,
+        },
+        errorMessage: {
+            type: String,
+            required: false,
+            default: '',
+        },
+    },
     methods: {
         isFieldValid(fieldName) {
             return true;
