diff --git a/assets/components/checkout/index.vue b/assets/components/checkout/index.vue
index af7d7b725..7e4f74797 100644
--- a/assets/components/checkout/index.vue
+++ b/assets/components/checkout/index.vue
@@ -8,6 +8,41 @@
                     label="Name:"
                     :error-message="validationErrors.customerName"
                 />
+
+                <form-input
+                    id="customerName"
+                    v-model="form.customerName"
+                    label="Name:"
+                    :error-message="validationErrors.customerName"
+                />
+
+                <form-input
+                    id="customerName"
+                    v-model="form.customerName"
+                    label="Name:"
+                    :error-message="validationErrors.customerName"
+                />
+
+                <form-input
+                    id="customerName"
+                    v-model="form.customerName"
+                    label="Name:"
+                    :error-message="validationErrors.customerName"
+                />
+
+                <form-input
+                    id="customerName"
+                    v-model="form.customerName"
+                    label="Name:"
+                    :error-message="validationErrors.customerName"
+                />
+
+                <form-input
+                    id="customerName"
+                    v-model="form.customerName"
+                    label="Name:"
+                    :error-message="validationErrors.customerName"
+                />
             </form>
         </div>
     </div>
