diff --git a/assets/components/checkout/index.vue b/assets/components/checkout/index.vue
index 8c789c82..53df3287 100644
--- a/assets/components/checkout/index.vue
+++ b/assets/components/checkout/index.vue
@@ -2,35 +2,44 @@
     <div class="row p-3">
         <div class="col-12">
             <form>
-                <form-input
-                    v-model="form.customerName"
-                    v-bind="getFieldProps('customerName', 'Name:')"
-                />
+                <div class="form-row">
+                    <form-input
+                        v-model="form.customerName"
+                        class="col"
+                        v-bind="getFieldProps('customerName', 'Name:')"
+                    />
 
-                <form-input
-                    v-model="form.customerEmail"
-                    v-bind="getFieldProps('customerEmail', 'Email:')"
-                />
+                    <form-input
+                        v-model="form.customerEmail"
+                        class="col"
+                        v-bind="getFieldProps('customerEmail', 'Email:')"
+                    />
+                </div>
 
                 <form-input
                     v-model="form.customerAddress"
                     v-bind="getFieldProps('customerAddress', 'Address:')"
                 />
 
-                <form-input
-                    v-model="form.customerZip"
-                    v-bind="getFieldProps('customerZip', 'Zip Code:')"
-                />
+                <div class="form-row">
+                    <form-input
+                        v-model="form.customerZip"
+                        class="col"
+                        v-bind="getFieldProps('customerZip', 'Zip Code:')"
+                    />
 
-                <form-input
-                    v-model="form.customerCity"
-                    v-bind="getFieldProps('customerCity', 'City:')"
-                />
+                    <form-input
+                        v-model="form.customerCity"
+                        class="col"
+                        v-bind="getFieldProps('customerCity', 'City:')"
+                    />
 
-                <form-input
-                    v-model="form.customerPhone"
-                    v-bind="getFieldProps('customerPhone', 'Phone Number:')"
-                />
+                    <form-input
+                        v-model="form.customerPhone"
+                        class="col"
+                        v-bind="getFieldProps('customerPhone', 'Phone Number:')"
+                    />
+                </div>
             </form>
         </div>
     </div>
