diff --git a/assets/js/components/checkout/form-input.vue b/assets/js/components/checkout/form-input.vue
index 51ab909e..78a918d2 100644
--- a/assets/js/components/checkout/form-input.vue
+++ b/assets/js/components/checkout/form-input.vue
@@ -13,6 +13,7 @@
                 'is-invalid': !isValid,
                 'form-control': true,
             }"
+            :value="value"
         >
         <span
             v-show="!isValid"
@@ -40,6 +41,11 @@ export default {
             required: false,
             default: '',
         },
+        value: {
+            type: String,
+            required: false,
+            default: '',
+        },
     },
     computed: {
         isValid() {
