diff --git a/assets/components/checkout/form-input.vue b/assets/components/checkout/form-input.vue
new file mode 100644
index 000000000..53c477cc1
--- /dev/null
+++ b/assets/components/checkout/form-input.vue
@@ -0,0 +1,36 @@
+<template>
+    <div class="p-3">
+        <label
+            for="customerName"
+            class="col-form-label"
+        >
+            Name:
+        </label>
+        <input
+            id="customerName"
+            v-model.trim="form.customerName"
+            type="text"
+            :class="{
+                'is-invalid': !isFieldValid('customerName'),
+                'form-control': true,
+            }"
+        >
+        <span
+            v-show="!isFieldValid('customerName')"
+            class="invalid-feedback"
+        >
+            {{ validationErrors.customerName }}
+        </span>
+    </div>
+</template>
+
+<script>
+export default {
+    name: 'FormInput',
+    methods: {
+        isFieldValid(fieldName) {
+            return !(fieldName in this.validationErrors);
+        },
+    },
+};
+</script>
diff --git a/assets/components/checkout/index.vue b/assets/components/checkout/index.vue
index e0230a97b..dec024d51 100644
--- a/assets/components/checkout/index.vue
+++ b/assets/components/checkout/index.vue
@@ -2,29 +2,7 @@
     <div class="row p-3">
         <div class="col-12">
             <form>
-                <div class="p-3">
-                    <label
-                        for="customerName"
-                        class="col-form-label"
-                    >
-                        Name:
-                    </label>
-                    <input
-                        id="customerName"
-                        v-model.trim="form.customerName"
-                        type="text"
-                        :class="{
-                            'is-invalid': !isFieldValid('customerName'),
-                            'form-control': true,
-                        }"
-                    >
-                    <span
-                        v-show="!isFieldValid('customerName')"
-                        class="invalid-feedback"
-                    >
-                        {{ validationErrors.customerName }}
-                    </span>
-                </div>
+
             </form>
         </div>
     </div>
@@ -53,10 +31,5 @@ export default {
             validationErrors: {},
         };
     },
-    methods: {
-        isFieldValid(fieldName) {
-            return !(fieldName in this.validationErrors);
-        },
-    },
 };
 </script>
