diff --git a/assets/components/checkout/index.vue b/assets/components/checkout/index.vue
index 53df3287f..393c9b89c 100644
--- a/assets/components/checkout/index.vue
+++ b/assets/components/checkout/index.vue
@@ -12,6 +12,7 @@
                     <form-input
                         v-model="form.customerEmail"
                         class="col"
+                        type="email"
                         v-bind="getFieldProps('customerEmail', 'Email:')"
                     />
                 </div>
@@ -37,6 +38,7 @@
                     <form-input
                         v-model="form.customerPhone"
                         class="col"
+                        type="tel"
                         v-bind="getFieldProps('customerPhone', 'Phone Number:')"
                     />
                 </div>
