diff --git a/assets/js/pages/products.vue b/assets/js/pages/products.vue
index 836385a6..e0b9ba7b 100644
--- a/assets/js/pages/products.vue
+++ b/assets/js/pages/products.vue
@@ -36,9 +36,9 @@ export default {
     },
     data() {
         return {
-            sidebarCollapsed: false,
             categories: [],
             currentCategoryId: getCurrentCategoryId(),
+            sidebarCollapsed: false,
         };
     },
     computed: {
