diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index 15f02e93..ef455b4d 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -28,8 +28,11 @@
                             </div>
                         </div>
 
-                        <div v-show="!loading && items.length">
-                            <div class="row p-3">
+                        <div
+                            v-show="!loading && items.length"
+                            class="row col-12"
+                        >
+                            <div class="row">
                                 <span class="col-3">Item Name</span>
                                 <span class="col-3">Quantity</span>
                                 <span class="col-3">Price</span>
@@ -40,6 +43,7 @@
                                 v-for="item in items"
                                 :key="item['@id']"
                                 :product="item"
+                                class="row"
                             />
 
                             <div class="row p-3">
