diff --git a/assets/js/components/catalog.vue b/assets/js/components/catalog.vue
index 9219509..f8e13db 100644
--- a/assets/js/components/catalog.vue
+++ b/assets/js/components/catalog.vue
@@ -24,7 +24,7 @@
 </template>
 
 <script>
-import axios from 'axios';
+import productsService from '@/services/products';
 import LegendComponent from '@/components/legend';
 import ProductList from '@/components/product-list';
 import SearchBar from '@/components/search-bar';
@@ -55,7 +55,7 @@ export default {
         legend: 'Shipping takes 10-12 weeks, and products probably won\'t work',
     }),
     created() {
-        this.fetchProducts('');
+        this.getProducts('');
     },
     methods: {
         /**
@@ -64,7 +64,7 @@ export default {
          * @param {string} term
          */
         onSearchProducts({ term }) {
-            this.fetchProducts(term);
+            this.getProducts(term);
         },
 
         /**
@@ -72,16 +72,13 @@ export default {
          *
          * @param {string} searchTerm
          */
-        async fetchProducts(searchTerm) {
+        async getProducts(searchTerm) {
             this.products = [];
             this.loading = true;
 
             try {
-                const response = await axios({
-                    method: 'get',
-                    url: '/api/products',
-                    params: this.generateListingParams(searchTerm),
-                });
+                const response = await productsService
+                    .fetchProducts(this.currentCategoryId, searchTerm);
 
                 this.loading = false;
                 this.products = response.data['hydra:member'];
