diff --git a/assets/js/helpers/generate-listing-url.js b/assets/js/helpers/generate-listing-url.js
new file mode 100644
index 0000000..949b6ee
--- /dev/null
+++ b/assets/js/helpers/generate-listing-url.js
@@ -0,0 +1,11 @@
+/**
+ * Generates the URL to call for a list of products by category or search term
+ *
+ * @param {number|null} category
+ * @param {string} searchTerm
+ * @return {string}
+ */
+const generateListingUrl = (category, searchTerm) => {
+};
+
+export default generateListingUrl;
