diff --git a/assets/js/helpers/generate-listing-params.js b/assets/js/helpers/generate-listing-params.js
index e6208b2..9c1bb94 100644
--- a/assets/js/helpers/generate-listing-params.js
+++ b/assets/js/helpers/generate-listing-params.js
@@ -7,6 +7,17 @@
  * @return {object}
  */
 const generateListingParams = (category, searchTerm) => {
+    const params = {};
+
+    if (this.currentCategoryId) {
+        params.category = this.currentCategoryId;
+    }
+
+    if (searchTerm !== '') {
+        params.name = searchTerm;
+    }
+
+    return params;
 };
 
 export default generateListingParams;
