diff --git a/assets/js/components/catalog.vue b/assets/js/components/catalog.vue
index 5fc8c9e3..d7601ba3 100644
--- a/assets/js/components/catalog.vue
+++ b/assets/js/components/catalog.vue
@@ -78,7 +78,7 @@ export default {
 
             let response;
             try {
-                response = await fetchProducts(this.generateListingParams(searchTerm));
+                response = await fetchProducts(this.currentCategoryId, searchTerm);
 
                 this.loading = false;
             } catch (e) {
