diff --git a/assets/js/helpers/generate-listing-params.js b/assets/js/helpers/generate-listing-params.js
index fecf89be..f920760d 100644
--- a/assets/js/helpers/generate-listing-params.js
+++ b/assets/js/helpers/generate-listing-params.js
@@ -7,4 +7,15 @@
  * @return {object}
  */
 export default (category, searchTerm) => {
+    const params = {};
+
+    if (this.currentCategoryId) {
+        params.category = this.currentCategoryId;
+    }
+
+    if (searchTerm !== '') {
+        params.name = searchTerm;
+    }
+
+    return params;
 };
