diff --git a/assets/js/helpers/generate-listing-params.js b/assets/js/helpers/generate-listing-params.js
index 9c1bb94..7cffebe 100644
--- a/assets/js/helpers/generate-listing-params.js
+++ b/assets/js/helpers/generate-listing-params.js
@@ -9,8 +9,8 @@
 const generateListingParams = (category, searchTerm) => {
     const params = {};
 
-    if (this.currentCategoryId) {
-        params.category = this.currentCategoryId;
+    if (category) {
+        params.category = category;
     }
 
     if (searchTerm !== '') {
