diff --git a/assets/js/helpers/generate-listing-params.js b/assets/js/helpers/generate-listing-params.js
new file mode 100644
index 00000000..e6208b20
--- /dev/null
+++ b/assets/js/helpers/generate-listing-params.js
@@ -0,0 +1,12 @@
+/**
+ * Generates the params for the URL to call for a list of products
+ * by category or search term
+ *
+ * @param {number|null} category
+ * @param {string} searchTerm
+ * @return {object}
+ */
+const generateListingParams = (category, searchTerm) => {
+};
+
+export default generateListingParams;
