diff --git a/assets/js/components/catalog.vue b/assets/js/components/catalog.vue
index a54e8c60..8904943b 100644
--- a/assets/js/components/catalog.vue
+++ b/assets/js/components/catalog.vue
@@ -85,27 +85,6 @@ export default {
                 this.loading = false;
             }
         },
-
-        /**
-         * Generates the params for the URL to call for a list of products
-         * by category or search term
-         *
-         * @param {string} searchTerm
-         * @return {object}
-         */
-        generateListingParams(searchTerm) {
-            const params = {};
-
-            if (this.currentCategoryId) {
-                params.category = this.currentCategoryId;
-            }
-
-            if (searchTerm !== '') {
-                params.name = searchTerm;
-            }
-
-            return params;
-        },
     },
 };
 </script>
