diff --git a/assets/js/services/products-service.js b/assets/js/services/products-service.js
index 62681e63..84b32c49 100644
--- a/assets/js/services/products-service.js
+++ b/assets/js/services/products-service.js
@@ -1,4 +1,5 @@
 import axios from 'axios';
+import generateListingParams from '@/helpers/generate-listing-params';
 
 /**
  * @param {string|null} categoryIri
@@ -6,16 +7,7 @@ import axios from 'axios';
  * @returns {Promise}
  */
 export function fetchProducts(categoryIri, searchTerm) {
-    const params = {};
-    if (categoryIri) {
-        params.category = categoryIri;
-    }
-
-    if (searchTerm) {
-        params.name = searchTerm;
-    }
-
     return axios.get('/api/products', {
-        params,
+        params: generateListingParams(categoryIri, searchTerm),
     });
 }
