diff --git a/assets/js/components/catalog.vue b/assets/js/components/catalog.vue
index d7601ba3..6be89825 100644
--- a/assets/js/components/catalog.vue
+++ b/assets/js/components/catalog.vue
@@ -56,7 +56,7 @@ export default {
         };
     },
     created() {
-        this.fetchProducts('');
+        this.getProducts('');
     },
     methods: {
         /**
@@ -65,15 +65,15 @@ export default {
          * @param {string} term
          */
         onSearchProducts({ term }) {
-            this.fetchProducts(term);
+            this.getProducts(term);
         },
 
         /**
-         * Fetches products from the API according to current category and search term
+         * Fetches products from the API
          *
          * @param {string} searchTerm
          */
-        async fetchProducts(searchTerm) {
+        async getProducts(searchTerm) {
             this.loading = true;
 
             let response;
