diff --git a/assets/js/services/products-service.js b/assets/js/services/products-service.js
index 72e78fd5..ea2e1a9e 100644
--- a/assets/js/services/products-service.js
+++ b/assets/js/services/products-service.js
@@ -1,16 +1,15 @@
 import axios from 'axios';
+import generateListingParams from '@/helpers/generate-listing-params';
 
 /**
+ * Fetches products from the API according to current category and search term
+ *
  * @param {string|null} categoryIri
+ * @param {string} searchTerm
  * @returns {Promise}
  */
-export function fetchProducts(categoryIri) {
-    const params = {};
-    if (categoryIri) {
-        params.category = categoryIri;
-    }
-
+export function fetchProducts(categoryIri, searchTerm) {
     return axios.get('/api/products', {
-        params,
+        params: generateListingParams(categoryIri, searchTerm),
     });
 }
