diff --git a/assets/js/components/search-bar.vue b/assets/js/components/search-bar.vue
new file mode 100644
index 00000000..af10e4b0
--- /dev/null
+++ b/assets/js/components/search-bar.vue
@@ -0,0 +1,15 @@
+<template>
+    <div>
+        <input
+            class="form-control"
+            placeholder="Search products..."
+            type="search"
+        >
+    </div>
+</template>
+
+<script>
+export default {
+    name: 'SearchBar',
+};
+</script>
