diff --git a/assets/js/components/catalog.vue b/assets/js/components/catalog.vue
index 2d7df7a2..e8821455 100644
--- a/assets/js/components/catalog.vue
+++ b/assets/js/components/catalog.vue
@@ -1,12 +1,15 @@
 <template>
     <div>
         <div class="row">
-            <div class="col-12">
+            <div class="col-3">
                 <title-component
                     :current-category-id="currentCategoryId"
                     :categories="categories"
                 />
             </div>
+            <div class="col-9">
+                <search-bar />
+            </div>
         </div>
 
         <product-list
@@ -24,6 +27,7 @@
 import axios from 'axios';
 import LegendComponent from '@/components/legend';
 import ProductList from '@/components/product-list';
+import SearchBar from '@/components/search-bar';
 import TitleComponent from '@/components/title';
 
 export default {
@@ -31,6 +35,7 @@ export default {
     components: {
         LegendComponent,
         ProductList,
+        SearchBar,
         TitleComponent,
     },
     props: {
