diff --git a/webpack.config.js b/webpack.config.js
index 6a91c70..63ea46e 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -67,6 +67,12 @@ Encore
     // enables Sass/SCSS support
     .enableSassLoader()
 
+    .configureCssLoader(function(config) {
+        if (!Encore.isProduction()) {
+            config.localIdentName = '[name]_[local]';
+        }
+    });
+
     // uncomment if you use TypeScript
     //.enableTypeScriptLoader()
 
