diff --git a/assets/js/components/catalog.vue b/assets/js/components/catalog.vue
index eed2ed5a..4b001d2c 100644
--- a/assets/js/components/catalog.vue
+++ b/assets/js/components/catalog.vue
@@ -8,7 +8,7 @@
                 />
             </div>
             <div class="col-9">
-                <search-bar />
+                <search-bar @search-products="onSearchProducts" />
             </div>
         </div>
 
@@ -74,5 +74,10 @@ export default {
             this.loading = false;
         }
     },
+    methods: {
+        onSearchProducts(event) {
+            this.searchTerm = event.term;
+        },
+    },
 };
 </script>
