diff --git a/assets/js/pages/products.vue b/assets/js/pages/products.vue
index 607dc438..770a780b 100644
--- a/assets/js/pages/products.vue
+++ b/assets/js/pages/products.vue
@@ -2,7 +2,7 @@
     <div class="container-fluid">
         <div class="row">
             <aside class="col-xs-12 col-3">
-                <div class="p-3 mb-5">
+                <div class="sidebar p-3 mb-5">
                     <h5 class="text-center">
                         Categories
                     </h5>
@@ -65,3 +65,11 @@ export default {
     },
 };
 </script>
+
+<style>
+.sidebar {
+    border: 1px solid #efefee;
+    box-shadow: 0px 0px 7px 4px #efefee;
+    border-radius: 5px;
+}
+</style>
