diff --git a/assets/js/pages/products.vue b/assets/js/pages/products.vue
index 770a780b..b875335c 100644
--- a/assets/js/pages/products.vue
+++ b/assets/js/pages/products.vue
@@ -66,10 +66,10 @@ export default {
 };
 </script>
 
-<style>
+<style lang="scss">
+@import '../../scss/components/light-component';
+
 .sidebar {
-    border: 1px solid #efefee;
-    box-shadow: 0px 0px 7px 4px #efefee;
-    border-radius: 5px;
+    @include light-component;
 }
 </style>
