diff --git a/assets/js/components/search-bar.vue b/assets/js/components/search-bar.vue
index 0c9b8164..752fb21b 100644
--- a/assets/js/components/search-bar.vue
+++ b/assets/js/components/search-bar.vue
@@ -1,6 +1,7 @@
 <template>
     <div :class="$style.component">
         <input
+            v-model="searchTerm"
             class="form-control"
             placeholder="Search products..."
             type="text"
@@ -11,6 +12,9 @@
 <script>
 export default {
     name: 'SearchBar',
+    data: () => ({
+        searchTerm: '',
+    }),
 };
 </script>
 
