diff --git a/assets/js/components/catalog.vue b/assets/js/components/catalog.vue
index 0650680..43e9ac4 100644
--- a/assets/js/components/catalog.vue
+++ b/assets/js/components/catalog.vue
@@ -26,12 +26,16 @@
 import axios from 'axios';
 import LegendComponent from '@/components/legend';
 import ProductList from '@/components/product-list';
+import SearchBar from '@/components/search-bar';
+import TitleComponent from '@/components/title';
 
 export default {
     name: 'Catalog',
     components: {
         LegendComponent,
         ProductList,
+        SearchBar,
+        TitleComponent,
     },
     props: {
         currentCategoryId: {
