diff --git a/assets/js/components/search-bar.vue b/assets/js/components/search-bar.vue
index 4c7d2e24..1e29847c 100644
--- a/assets/js/components/search-bar.vue
+++ b/assets/js/components/search-bar.vue
@@ -17,5 +17,10 @@ export default {
             searchTerm: '',
         };
     },
+    watch: {
+        searchTerm(newTerm) {
+            this.$emit('search-products', { term: newTerm });
+        },
+    },
 };
 </script>
