diff --git a/assets/js/components/catalog.vue b/assets/js/components/catalog.vue
index de1d75f4..51388c4d 100644
--- a/assets/js/components/catalog.vue
+++ b/assets/js/components/catalog.vue
@@ -13,7 +13,7 @@
         </div>
 
         <product-list
-            :products="products"
+            :products="filteredProducts"
             :loading="loading"
         />
 
@@ -56,6 +56,17 @@ export default {
             legend: 'Shipping takes 10-13 weeks, and products probably won\'t work',
         };
     },
+    computed: {
+        filteredProducts() {
+            if (!this.searchTerm) {
+                return this.products;
+            }
+
+            return this.products.filter((product) => (
+                product.name.toLowerCase().includes(this.searchTerm.toLowerCase())
+            ));
+        },
+    },
     async created() {
         this.loading = true;
 
