diff --git a/assets/js/components/search-bar.vue b/assets/js/components/search-bar.vue
index 4c7d2e24..845aab58 100644
--- a/assets/js/components/search-bar.vue
+++ b/assets/js/components/search-bar.vue
@@ -5,6 +5,7 @@
             class="form-control"
             placeholder="Search products..."
             type="search"
+            @input="onInput"
         >
     </div>
 </template>
@@ -17,5 +18,10 @@ export default {
             searchTerm: '',
         };
     },
+    methods: {
+        onInput() {
+            this.$emit('search-products', { term: this.searchTerm });
+        },
+    },
 };
 </script>
