diff --git a/assets/js/products.vue b/assets/js/products.vue
index 828ffd9..59a8b81 100644
--- a/assets/js/products.vue
+++ b/assets/js/products.vue
@@ -11,13 +11,11 @@
             </aside>
 
             <div :class="contentClass">
-                <title-component
+                <catalog-component
                     :current-category-id="currentCategoryId"
                     :categories="categories"
                     :class="$style.title"
                 />
-
-                <catalog-component :current-category-id="currentCategoryId" />
             </div>
         </div>
     </div>
@@ -26,14 +24,12 @@
 <script>
 import CatalogComponent from '@/components/catalog';
 import SidebarComponent from '@/components/sidebar';
-import TitleComponent from '@/components/title';
 
 export default {
     name: 'Products',
     components: {
         CatalogComponent,
         SidebarComponent,
-        TitleComponent,
     },
     props: {
         currentCategoryId: {
