diff --git a/assets/js/components/catalog.vue b/assets/js/components/catalog.vue
index a915bc1..17beb50 100644
--- a/assets/js/components/catalog.vue
+++ b/assets/js/components/catalog.vue
@@ -1,9 +1,21 @@
 <template>
     <div>
-        <product-list
-            :products="products"
-            :loading="loading"
-        />
+        <div class="row">
+            <div :class="$style['top-bar']">
+                <title-component
+                    :current-category-id="currentCategoryId"
+                    :categories="categories"
+                />
+
+                <search-bar @search-products="onSearchProducts" />
+            </div>
+
+            <product-list
+                :products="products"
+                :loading="loading"
+            />
+        </div>
+
 
         <div class="row">
             <legend-component :title="legend" />
