diff --git a/assets/js/components/search-bar.vue b/assets/js/components/search-bar.vue
new file mode 100644
index 0000000..0c9b816
--- /dev/null
+++ b/assets/js/components/search-bar.vue
@@ -0,0 +1,21 @@
+<template>
+    <div :class="$style.component">
+        <input
+            class="form-control"
+            placeholder="Search products..."
+            type="text"
+        >
+    </div>
+</template>
+
+<script>
+export default {
+    name: 'SearchBar',
+};
+</script>
+
+<style lang="scss" module>
+.component {
+    width: 400px;
+}
+</style>
