diff --git a/assets/js/products.vue b/assets/js/products.vue
index e52614f..d84824c 100644
--- a/assets/js/products.vue
+++ b/assets/js/products.vue
@@ -2,7 +2,7 @@
     <div class="container-fluid">
         <div class="row row-no-wrap">
             <aside class="col-xs-12 col-lg-3">
-                <div class="p-3 mt-5 mb-5">
+                <div :class="[$style.aside, 'p-3', 'mt-5', 'mb-5']">
                     <h5 class="text-center">
                         Categories
                     </h5>
@@ -40,3 +40,30 @@ export default {
     name: 'Products',
 };
 </script>
+
+<style lang="scss" module>
+@import '../scss/components/light-component';
+
+.aside {
+    @include light-component;
+    margin-top: 65px;
+
+    ul {
+        border-bottom: 1px solid $light-component-border;
+        margin-bottom: 20px;
+        padding-bottom: 10px;
+
+        li a {
+            color: #000;
+        }
+
+        li a:hover {
+            background: $blue-component-link-hover;
+        }
+
+        li a.selected {
+            background: $light-component-border;
+        }
+    }
+}
+</style>
