diff --git a/webpack.config.js b/webpack.config.js
index 09b7d5b..47b8f4e 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -65,6 +65,12 @@ Encore
     // enables Sass/SCSS support
     .enableSassLoader()
 
+    .configureCssLoader(function(config) {
+        if (!Encore.isProduction()) {
+            config.localIdentName = '[name]_[local]';
+        }
+    });
+
     // uncomment if you use TypeScript
     //.enableTypeScriptLoader()
 
