diff --git a/assets/js/components/catalog.vue b/assets/js/components/catalog.vue
index a4bfc46..20a705a 100644
--- a/assets/js/components/catalog.vue
+++ b/assets/js/components/catalog.vue
@@ -49,6 +49,7 @@ export default {
     },
     data: () => ({
         products: [],
+        searchTerm: '',
         loading: true,
         legend: 'Shipping takes 10-12 weeks, and products probably won\'t work',
     }),
