diff --git a/assets/js/components/catalog.vue b/assets/js/components/catalog.vue
index da1a54e..e16fe5c 100644
--- a/assets/js/components/catalog.vue
+++ b/assets/js/components/catalog.vue
@@ -74,6 +74,11 @@ export default {
             this.loading = false;
         }
     },
+    methods: {
+        onSearchProducts(event) {
+            this.searchTerm = event.term;
+        },
+    },
 };
 </script>
 
