diff --git a/assets/modules/shopping-cart.js b/assets/modules/shopping-cart.js
index b2c78ee6..0bf2b94b 100644
--- a/assets/modules/shopping-cart.js
+++ b/assets/modules/shopping-cart.js
@@ -62,6 +62,7 @@ export default function useShoppingCart() {
     return {
         addToCartLoading,
         addToCartSuccess,
+        cartLoading,
         cart,
         loadCart,
         addProductToCart,
