diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index 422c1df7..0e420d0d 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -141,12 +141,6 @@ export default {
         switchState() {
             this.currentState = this.currentState === 'cart' ? 'checkout' : 'cart';
         },
-        async loadProducts() {
-            const productIds = this.cart.items.map((item) => item.product);
-
-            const productsResponse = await fetchProductsById(productIds);
-            this.products = productsResponse.data['hydra:member'];
-        },
         updateQuantity({ productId, colorId, quantity }) {
             this.updateProductQuantity(productId, colorId, quantity);
         },
@@ -173,6 +167,13 @@ export default {
             updateProductQuantity,
         } = useShoppingCart();
 
+        const loadProducts = async () => {
+            const productIds = this.cart.items.map((item) => item.product);
+
+            const productsResponse = await fetchProductsById(productIds);
+            this.products = productsResponse.data['hydra:member'];
+        };
+
         loadCart();
 
         return {
