diff --git a/assets/modules/shopping-cart.js b/assets/modules/shopping-cart.js
index f60a7188..b2c78ee6 100644
--- a/assets/modules/shopping-cart.js
+++ b/assets/modules/shopping-cart.js
@@ -18,7 +18,9 @@ const cart = reactive({
 
 export default function useShoppingCart() {
     const loadCart = async () => {
+        cartLoading.value = true;
         cart.items = (await fetchCart()).items;
+        cartLoading.value = false;
     };
 
     const updateCartHeaderTotal = () => {
