diff --git a/assets/js/services/page-context.js b/assets/js/services/page-context.js
index bc18cdc3..0f57ae96 100644
--- a/assets/js/services/page-context.js
+++ b/assets/js/services/page-context.js
@@ -6,3 +6,12 @@
 export function getCurrentCategoryId() {
     return window.currentCategoryId;
 }
+
+/**
+ * Returns the current product id that's set by the server.
+ *
+ * @returns {string|null}
+ */
+export function getCurrentProductId() {
+    return window.currentProductId;
+}
