diff --git a/assets/pages/products.vue b/assets/pages/products.vue
index dd9d84ad7..b5e400ff2 100644
--- a/assets/pages/products.vue
+++ b/assets/pages/products.vue
@@ -11,9 +11,10 @@
             </aside>
 
             <div :class="contentClass">
-                <catalog
-                    :current-category-id="currentCategoryId"
+                <component
+                    :is="currentComponent"
                     :categories="categories"
+                    :current-category-id="currentCategoryId"
                 />
             </div>
         </div>
