diff --git a/assets/pages/products.vue b/assets/pages/products.vue
index 00a83fd7..6d5b3cc1 100644
--- a/assets/pages/products.vue
+++ b/assets/pages/products.vue
@@ -13,9 +13,7 @@
             <div :class="contentClass">
                 <component
                     :is="currentComponent"
-                    :current-category-id="currentCategoryId"
-                    :categories="categories"
-                    :product-id="currentProductId"
+                    v-bind="currentProps"
                 />
             </div>
         </div>
@@ -56,6 +54,14 @@ export default {
         currentComponent() {
             return this.currentProductId !== null ? ProductShow : Catalog;
         },
+        currentProps() {
+            return this.currentComponent === ProductShow ? {
+                productId: this.currentProductId,
+            } : {
+                currentCategoryId: this.currentCategoryId,
+                categories: this.categories,
+            };
+        },
     },
     async created() {
         const response = await fetchCategories();
