diff --git a/assets/pages/products.vue b/assets/pages/products.vue
index c11efb455..9b1ab3fa0 100644
--- a/assets/pages/products.vue
+++ b/assets/pages/products.vue
@@ -22,6 +22,7 @@
 
 <script>
 import Catalog from '@/components/catalog';
+import Product from '@/components/product';
 import Sidebar from '@/components/sidebar';
 import { getCurrentCategoryId, getCurrentProductId } from '@/services/page-context';
 import { fetchCategories } from '@/services/categories-service';
@@ -30,6 +31,7 @@ export default {
     name: 'Products',
     components: {
         Catalog,
+        Product,
         Sidebar,
     },
     data() {
