diff --git a/assets/js/pages/products.vue b/assets/js/pages/products.vue
index b5e400ff..391599f0 100644
--- a/assets/js/pages/products.vue
+++ b/assets/js/pages/products.vue
@@ -13,8 +13,7 @@
             <div :class="contentClass">
                 <component
                     :is="currentComponent"
-                    :categories="categories"
-                    :current-category-id="currentCategoryId"
+                    v-bind="currentProps"
                 />
             </div>
         </div>
@@ -55,6 +54,16 @@ export default {
         currentComponent() {
             return this.currentProductId !== null ? Product : Catalog;
         },
+        currentProps() {
+            return this.currentProductId !== null
+                ? {
+                    currentProductId: this.currentProductId,
+                }
+                : {
+                    currentCategoryId: this.currentCategoryId,
+                    categories: this.categories,
+                };
+        },
     },
     async created() {
         const response = await fetchCategories();
