diff --git a/assets/js/pages/products.vue b/assets/js/pages/products.vue
index 9b1ab3fa..dd9d84ad 100644
--- a/assets/js/pages/products.vue
+++ b/assets/js/pages/products.vue
@@ -51,6 +51,9 @@ export default {
         currentProductId() {
             return getCurrentProductId();
         },
+        currentComponent() {
+            return this.currentProductId !== null ? Product : Catalog;
+        },
     },
     async created() {
         const response = await fetchCategories();
