diff --git a/assets/pages/products.vue b/assets/pages/products.vue
index e3e02195d..97df2fa4b 100644
--- a/assets/pages/products.vue
+++ b/assets/pages/products.vue
@@ -11,10 +11,8 @@
             </aside>
 
             <div :class="contentClass">
-                Product: {{ currentProductId }}
-                <product-show />
-
-                <catalog
+                <component
+                    :is="currentComponent"
                     :current-category-id="currentCategoryId"
                     :categories="categories"
                 />
