diff --git a/assets/js/components/product-list/product-card.vue b/assets/js/components/product-list/product-card.vue
index 138118ae..ac68a526 100644
--- a/assets/js/components/product-list/product-card.vue
+++ b/assets/js/components/product-list/product-card.vue
@@ -15,7 +15,7 @@
 
             <div class="p-2 my-3 d-md-flex justify-content-between">
                 <p class="p-0 d-inline">
-                    <strong>${{ formatPrice(product.price) }}</strong>
+                    <strong>${{ price }}</strong>
                 </p>
 
                 <button
@@ -49,8 +49,7 @@ export default {
          * @returns {string}
          */
         price() {
-            return (this.product.price / 100)
-                .toLocaleString('en-US', { minimumFractionDigits: 2 });
+            return formatPrice(this.product.price);
         },
     },
 };
