diff --git a/assets/js/components/product-list/product-card.vue b/assets/js/components/product-list/product-card.vue
index 4af48029..138118ae 100644
--- a/assets/js/components/product-list/product-card.vue
+++ b/assets/js/components/product-list/product-card.vue
@@ -15,7 +15,7 @@
 
             <div class="p-2 my-3 d-md-flex justify-content-between">
                 <p class="p-0 d-inline">
-                    <strong>${{ price }}</strong>
+                    <strong>${{ formatPrice(product.price) }}</strong>
                 </p>
 
                 <button
@@ -33,6 +33,8 @@
 </template>
 
 <script>
+import formatPrice from '@/helpers/format-price';
+
 export default {
     name: 'ProductCard',
     props: {
