diff --git a/assets/components/product-show/index.vue b/assets/components/product-show/index.vue
index 375ea9fa..dbcb01df 100644
--- a/assets/components/product-show/index.vue
+++ b/assets/components/product-show/index.vue
@@ -58,6 +58,7 @@ import { fetchOneProduct } from '@/services/products-service';
 import Loading from '@/components/loading';
 import TitleComponent from '@/components/title';
 import useShoppingCart from '@/modules/shopping-cart';
+import usePageContext from '@/modules/page-context';
 import CartAddControls from './cart-add-controls';
 
 export default {
@@ -67,12 +68,6 @@ export default {
         TitleComponent,
         CartAddControls,
     },
-    props: {
-        productId: {
-            type: String,
-            required: true,
-        },
-    },
     data() {
         return {
             product: null,
@@ -90,7 +85,7 @@ export default {
     },
     async created() {
         try {
-            this.product = (await fetchOneProduct(this.productId)).data;
+            this.product = (await fetchOneProduct(this.currentProductId)).data;
         } finally {
             this.loading = false;
         }
@@ -108,10 +103,13 @@ export default {
             addProductToCart,
         } = useShoppingCart();
 
+        const { currentProductId } = usePageContext();
+
         return {
             addToCartLoading,
             addToCartSuccess,
             cart,
+            currentProductId,
             addProductToCart,
         };
     },
