diff --git a/assets/modules/page-context.js b/assets/modules/page-context.js
index 6caf3dd0..299ca3e4 100644
--- a/assets/modules/page-context.js
+++ b/assets/modules/page-context.js
@@ -3,30 +3,14 @@ import Product from '@/components/product-show';
 import Catalog from '@/components/catalog';
 import { getCurrentCategoryId, getCurrentProductId } from '@/services/page-context';
 import { fetchCategories } from '@/services/categories-service';
-import useShoppingCart from '@/modules/shopping-cart';
 
 const categories = ref([]);
 
 export default function usePageContext() {
-    const { cart } = useShoppingCart();
-
     const currentCategoryId = computed(() => (getCurrentCategoryId()));
     const currentProductId = computed(() => (getCurrentProductId()));
-
     const currentComponent = computed(() => (currentProductId.value !== null ? Product : Catalog));
 
-    const currentProps = computed(() => (
-        currentProductId.value !== null
-            ? {
-                productId: currentProductId.value,
-                cart,
-            }
-            : {
-                currentCategoryId: currentCategoryId.value,
-                categories,
-            }
-    ));
-
     const loadCategories = async () => {
         const response = await fetchCategories();
 
@@ -38,7 +22,6 @@ export default function usePageContext() {
         currentCategoryId,
         currentProductId,
         currentComponent,
-        currentProps,
         loadCategories,
     };
 }
diff --git a/assets/pages/products.vue b/assets/pages/products.vue
index 83fd4de4..90071a18 100644
--- a/assets/pages/products.vue
+++ b/assets/pages/products.vue
@@ -6,10 +6,7 @@
             </aside>
 
             <div :class="contentClass">
-                <component
-                    :is="currentComponent"
-                    v-bind="currentProps"
-                />
+                <component :is="currentComponent" />
             </div>
         </div>
     </div>
@@ -26,21 +23,13 @@ export default {
         Sidebar,
     },
     setup() {
+        const { currentComponent } = usePageContext();
         const { asideClass, contentClass } = useCollapsibleSidebar();
 
-        const {
-            loadCategories,
-            currentComponent,
-            currentProps,
-        } = usePageContext();
-
-        loadCategories();
-
         return {
             asideClass,
             contentClass,
             currentComponent,
-            currentProps,
         };
     },
 };
