diff --git a/assets/modules/collapsible-sidebar.js b/assets/modules/collapsible-sidebar.js
index 36542766..f271e13c 100644
--- a/assets/modules/collapsible-sidebar.js
+++ b/assets/modules/collapsible-sidebar.js
@@ -1,8 +1,8 @@
 import { computed, ref } from 'vue';
 
-export default function useCollapsibleSidebar() {
-    const sidebarCollapsed = ref(false);
+const sidebarCollapsed = ref(false);
 
+export default function useCollapsibleSidebar() {
     const asideClass = computed(
         () => (sidebarCollapsed.value ? 'aside-collapsed' : 'col-12 col-md-3'),
     );
