diff --git a/assets/pages/products.vue b/assets/pages/products.vue
index e0d8810b..989a8bc8 100644
--- a/assets/pages/products.vue
+++ b/assets/pages/products.vue
@@ -3,10 +3,8 @@
         <div class="row">
             <aside :class="asideClass">
                 <sidebar
-                    :collapsed="sidebarCollapsed"
                     :current-category-id="currentCategoryId"
                     :categories="categories"
-                    @toggle-collapsed="toggleSidebarCollapsed"
                 />
             </aside>
 
@@ -31,12 +29,15 @@ export default {
         Sidebar,
     },
     setup() {
+        const { asideClass, contentClass } = useCollapsibleSidebar();
+
         const pageContext = usePageContext();
 
         pageContext.loadCategories();
 
         return {
-            ...useCollapsibleSidebar(),
+            asideClass,
+            contentClass,
             ...pageContext,
         };
     },
