diff --git a/assets/pages/products.vue b/assets/pages/products.vue
index 9b5579dc..83fd4de4 100644
--- a/assets/pages/products.vue
+++ b/assets/pages/products.vue
@@ -2,10 +2,7 @@
     <div class="container-fluid">
         <div class="row">
             <aside :class="asideClass">
-                <sidebar
-                    :current-category-id="currentCategoryId"
-                    :categories="categories"
-                />
+                <sidebar />
             </aside>
 
             <div :class="contentClass">
@@ -33,8 +30,6 @@ export default {
 
         const {
             loadCategories,
-            categories,
-            currentCategoryId,
             currentComponent,
             currentProps,
         } = usePageContext();
@@ -44,8 +39,6 @@ export default {
         return {
             asideClass,
             contentClass,
-            categories,
-            currentCategoryId,
             currentComponent,
             currentProps,
         };
