diff --git a/assets/js/pages/products.vue b/assets/js/pages/products.vue
index ddf87537..f2684b73 100644
--- a/assets/js/pages/products.vue
+++ b/assets/js/pages/products.vue
@@ -22,12 +22,14 @@
 
 <script>
 import Catalog from '@/components/catalog';
+import Product from '@/components/product';
 import Sidebar from '@/components/sidebar';
 
 export default {
     name: 'Products',
     components: {
         Catalog,
+        Product,
         Sidebar,
     },
     props: {
