diff --git a/assets/js/pages/products.vue b/assets/js/pages/products.vue
index 67813da0..0fb305df 100644
--- a/assets/js/pages/products.vue
+++ b/assets/js/pages/products.vue
@@ -22,6 +22,7 @@
 
 <script>
 import Catalog from '@/components/catalog';
+import Product from '@/components/product';
 import Sidebar from '@/components/sidebar';
 import { getCurrentCategoryId, getCategories, getCurrentProductId } from '@/services/page-context';
 
@@ -29,6 +30,7 @@ export default {
     name: 'Products',
     components: {
         Catalog,
+        Product,
         Sidebar,
     },
     data() {
