diff --git a/assets/js/pages/products.vue b/assets/js/pages/products.vue
index c4c0fb34..6c425631 100644
--- a/assets/js/pages/products.vue
+++ b/assets/js/pages/products.vue
@@ -50,6 +50,9 @@ export default {
         currentProductId() {
             return getCurrentProductId();
         },
+        currentComponent() {
+            return this.currentProductId !== null ? Product : Catalog;
+        },
     },
     methods: {
         toggleSidebarCollapsed() {
