diff --git a/assets/js/components/product-list/product-card.vue b/assets/js/components/product-list/product-card.vue
index 6cb701c..2161496 100644
--- a/assets/js/components/product-list/product-card.vue
+++ b/assets/js/components/product-list/product-card.vue
@@ -65,6 +65,14 @@ export default {
             return `/product/${this.item.id}`;
         },
     },
+    methods: {
+        /**
+         * Goes to the URL of the product
+         */
+        goToProduct() {
+            window.location = this.productUrl;
+        },
+    },
 };
 </script>
 
