diff --git a/assets/js/products.vue b/assets/js/products.vue
index e23370d..36c6562 100644
--- a/assets/js/products.vue
+++ b/assets/js/products.vue
@@ -22,12 +22,14 @@
 
 <script>
 import CatalogComponent from '@/components/catalog';
+import ProductComponent from '@/components/product';
 import SidebarComponent from '@/components/sidebar';
 
 export default {
     name: 'Products',
     components: {
         CatalogComponent,
+        ProductComponent,
         SidebarComponent,
     },
     props: {
