diff --git a/assets/js/pages/products.vue b/assets/js/pages/products.vue
index 2b323bd1..c7f1cdd6 100644
--- a/assets/js/pages/products.vue
+++ b/assets/js/pages/products.vue
@@ -23,7 +23,7 @@
 <script>
 import Catalog from '@/components/catalog';
 import Sidebar from '@/components/sidebar';
-import { getCurrentCategoryId } from '@/services/page-context';
+import { getCurrentCategoryId, getCurrentProductId } from '@/services/page-context';
 import { fetchCategories } from '@/services/categories-service';
 
 export default {
@@ -48,6 +48,9 @@ export default {
         currentCategoryId() {
             return getCurrentCategoryId();
         },
+        currentProductId() {
+            return getCurrentProductId();
+        },
     },
     async created() {
         const response = await fetchCategories();
