diff --git a/assets/components/product-list/product-card.vue b/assets/components/product-list/product-card.vue
index 9f236bfcd..918bbb0d5 100644
--- a/assets/components/product-list/product-card.vue
+++ b/assets/components/product-list/product-card.vue
@@ -66,6 +66,14 @@ export default {
             return `/product/${this.product.id}`;
         },
     },
+    methods: {
+        /**
+         * Goes to the URL of the product
+         */
+        goToProduct() {
+            window.location = this.productUrl;
+        },
+    },
 };
 </script>
 
