diff --git a/assets/js/components/product-list/product-card.vue b/assets/js/components/product-list/product-card.vue
index 02a4e302..744083c9 100644
--- a/assets/js/components/product-list/product-card.vue
+++ b/assets/js/components/product-list/product-card.vue
@@ -56,6 +56,14 @@ export default {
             return (this.product.price / 100)
                 .toLocaleString('en-US', { minimumFractionDigits: 2 });
         },
+
+        /**
+         * Returns the URL for the product
+         * @returns {string}
+         */
+        productUrl() {
+            return `/product/${this.item.id}`;
+        },
     },
 };
 </script>
