diff --git a/assets/js/pages/products.vue b/assets/js/pages/products.vue
index dd9d84ad..8955d1a2 100644
--- a/assets/js/pages/products.vue
+++ b/assets/js/pages/products.vue
@@ -11,9 +11,11 @@
             </aside>
 
             <div :class="contentClass">
-                <catalog
-                    :current-category-id="currentCategoryId"
+                <component
+                    :is="currentComponent"
                     :categories="categories"
+                    :current-category-id="currentCategoryId"
+                    :current-product-id="currentProductId"
                 />
             </div>
         </div>
