diff --git a/assets/js/pages/products.vue b/assets/js/pages/products.vue
index 0fb305df..852b9f60 100644
--- a/assets/js/pages/products.vue
+++ b/assets/js/pages/products.vue
@@ -54,6 +54,9 @@ export default {
         currentProductId() {
             return getCurrentProductId();
         },
+        currentComponent() {
+            return this.currentProductId !== null ? Product : Catalog;
+        },
     },
     methods: {
         toggleSidebarCollapsed() {
