diff --git a/assets/js/components/product-list/product-card.vue b/assets/js/components/product-list/product-card.vue
index f52e0395..9f236bfc 100644
--- a/assets/js/components/product-list/product-card.vue
+++ b/assets/js/components/product-list/product-card.vue
@@ -57,6 +57,14 @@ export default {
         price() {
             return formatPrice(this.product.price);
         },
+
+        /**
+         * Returns the URL for the product
+         * @returns {string}
+         */
+        productUrl() {
+            return `/product/${this.product.id}`;
+        },
     },
 };
 </script>
