diff --git a/assets/js/products.vue b/assets/js/products.vue
index 59a8b81..bd36351 100644
--- a/assets/js/products.vue
+++ b/assets/js/products.vue
@@ -40,6 +40,10 @@ export default {
             type: Array,
             default: () => ([]),
         },
+        currentProductId: {
+            type: Number,
+            default: null,
+        },
     },
     data: () => ({
         sidebarCollapsed: false,
