diff --git a/src/Controller/ProductController.php b/src/Controller/ProductController.php
index 0264359..28451d4 100644
--- a/src/Controller/ProductController.php
+++ b/src/Controller/ProductController.php
@@ -34,8 +34,11 @@ class ProductController extends AbstractController
     /**
      * @Route("/product/{id}", name="app_product")
      */
-    public function showProduct(int $id): Response
+    public function showProduct(int $id, CategoryRepository $categoryRepository): Response
     {
-        return $this->render('product/index.html.twig');
+        return $this->render('product/index.html.twig', [
+            'currentProductId' => $id,
+            'categoriesJson' => $categoryRepository->findAll(),
+        ]);
     }
 }
