diff --git a/assets/js/components/product.vue b/assets/js/components/product.vue
index d58c33e3..bc657108 100644
--- a/assets/js/components/product.vue
+++ b/assets/js/components/product.vue
@@ -7,5 +7,15 @@
 <script>
 export default {
     name: 'Product',
+    props: {
+        currentProductId: {
+            type: String,
+            default: null,
+        },
+        categories: {
+            type: Array,
+            default: () => ([]),
+        },
+    },
 };
 </script>
