diff --git a/src/Controller/ProductController.php b/src/Controller/ProductController.php
index 4c235fd31..7a392ae92 100644
--- a/src/Controller/ProductController.php
+++ b/src/Controller/ProductController.php
@@ -36,10 +36,11 @@ class ProductController extends AbstractController
     /**
      * @Route("/product/{id}", name="app_product")
      */
-    public function showProduct(Product $product, CategoryRepository $categoryRepository): Response
+    public function showProduct(Product $product, CategoryRepository $categoryRepository, IriConverterInterface $iriConverter): Response
     {
         return $this->render('product/index.html.twig', [
             'categories' => $categoryRepository->findAll(),
+            'currentProductId' => $iriConverter->getIriFromItem($product),
         ]);
     }
 }
