diff --git a/templates/product/index.html.twig b/templates/product/index.html.twig
index 9fd746b6..47b78ad0 100644
--- a/templates/product/index.html.twig
+++ b/templates/product/index.html.twig
@@ -19,6 +19,13 @@
         {% else %}
             window.currentCategoryId = null;
         {% endif %}
+
+        {% if currentProductId is defined %}
+            window.currentProductId = '{{ currentProductId|e('js') }}';
+        {% else %}
+            window.currentProductId = null;
+        {% endif %}
+
         window.categories = {{ categories|jsonld }};
     </script>
 
