diff --git a/assets/js/pages/products.vue b/assets/js/pages/products.vue
index 930a6394..c11efb45 100644
--- a/assets/js/pages/products.vue
+++ b/assets/js/pages/products.vue
@@ -23,7 +23,7 @@
 <script>
 import Catalog from '@/components/catalog';
 import Sidebar from '@/components/sidebar';
-import { getCurrentCategoryId } from '@/services/page-context';
+import { getCurrentCategoryId, getCurrentProductId } from '@/services/page-context';
 import { fetchCategories } from '@/services/categories-service';
 
 export default {
@@ -46,6 +46,9 @@ export default {
         contentClass() {
             return this.sidebarCollapsed ? 'col-xs-12 col-11' : 'col-xs-12 col-9';
         },
+        currentProductId() {
+            return getCurrentProductId();
+        },
     },
     async created() {
         const response = await fetchCategories();
