diff --git a/assets/js/services/page-context.js b/assets/js/services/page-context.js
index cc3592b3..6c2fd71e 100644
--- a/assets/js/services/page-context.js
+++ b/assets/js/services/page-context.js
@@ -15,3 +15,12 @@ export function getCurrentCategoryId() {
 export function getCategories() {
     return window.categories;
 }
+
+/**
+ * Returns the current product id that's set by the server.
+ *
+ * @returns {string|null}
+ */
+export function getCurrentProductId() {
+    return window.currentProductId;
+}
