diff --git a/templates/product/index.html.twig b/templates/product/index.html.twig
index 878c9bcb..7d5d5997 100644
--- a/templates/product/index.html.twig
+++ b/templates/product/index.html.twig
@@ -16,6 +16,7 @@
     <script>
         window.currentCategoryId = {{ currentCategoryId|default('null') }};
         window.categories = {{ categories|jsonld }};
+        window.currentProductId = {{ currentProductId|default('null') }};
     </script>
 
     {{ encore_entry_script_tags('products') }}
