diff --git a/assets/js/components/product-list/product-card.vue b/assets/js/components/product-list/product-card.vue
index 4af48029..246f2ee1 100644
--- a/assets/js/components/product-list/product-card.vue
+++ b/assets/js/components/product-list/product-card.vue
@@ -2,14 +2,19 @@
     <div class="col-xs-12 col-6 mb-2 pb-2">
         <div :class="$style['product-box']">
             <div :class="$style.image">
-                <img
-                    :alt="product.name"
-                    :src="product.image"
-                    class="d-block mb-2"
-                >
+                <a :href="productUrl">
+                    <img
+                        :alt="product.name"
+                        :src="product.image"
+                        class="d-block mb-2"
+                    >
+                </a>
 
                 <h3 class="font-weight-bold mb-2 px-2">
-                    {{ product.name }}
+                    <a
+                        :href="productUrl"
+                        v-text="product.name"
+                    />
                 </h3>
             </div>
 
