diff --git a/assets/js/products.vue b/assets/js/products.vue
index 32e196a..f4fb467 100644
--- a/assets/js/products.vue
+++ b/assets/js/products.vue
@@ -57,6 +57,9 @@ export default {
         contentClass() {
             return this.sidebarCollapsed ? 'col-xs-12 col-lg-11' : 'col-xs-12 col-lg-9';
         },
+        currentComponent() {
+            return this.currentProductId !== null ? 'ProductComponent' : 'CatalogComponent';
+        },
     },
     methods: {
         /**
