diff --git a/templates/product/index.html.twig b/templates/product/index.html.twig
index 9fd746b6..ed1c098e 100644
--- a/templates/product/index.html.twig
+++ b/templates/product/index.html.twig
@@ -20,6 +20,7 @@
             window.currentCategoryId = null;
         {% endif %}
         window.categories = {{ categories|jsonld }};
+        window.currentProductId = {{ currentProductId|default('null') }};
     </script>
 
     {{ encore_entry_script_tags('products') }}
