diff --git a/assets/js/pages/products.vue b/assets/js/pages/products.vue
index 6c425631..950e1628 100644
--- a/assets/js/pages/products.vue
+++ b/assets/js/pages/products.vue
@@ -10,9 +10,11 @@
             </aside>
 
             <div :class="contentClass">
-                <catalog
-                    :current-category-id="currentCategoryId"
+                <component
+                    :is="currentComponent"
                     :categories="categories"
+                    :current-category-id="currentCategoryId"
+                    :current-product-id="currentProductId"
                 />
             </div>
         </div>
