diff --git a/assets/js/pages/products.vue b/assets/js/pages/products.vue
index 2736d6dc..b0ce4858 100644
--- a/assets/js/pages/products.vue
+++ b/assets/js/pages/products.vue
@@ -13,9 +13,7 @@
             <div :class="contentClass">
                 <component
                     :is="currentComponent"
-                    :categories="categories"
-                    :current-category-id="currentCategoryId"
-                    :current-product-id="currentProductId"
+                    v-bind="currentProps"
                 />
             </div>
         </div>
@@ -59,6 +57,17 @@ export default {
         currentComponent() {
             return this.currentProductId !== null ? Product : Catalog;
         },
+        currentProps() {
+            return this.currentProductId !== null
+                ? {
+                    currentProductId: this.currentProductId,
+                    categories: this.categories,
+                }
+                : {
+                    currentCategoryId: this.currentCategoryId,
+                    categories: this.categories,
+                };
+        },
     },
     methods: {
         toggleSidebarCollapsed() {
