diff --git a/assets/js/products.vue b/assets/js/products.vue
index 15c4909..e23370d 100644
--- a/assets/js/products.vue
+++ b/assets/js/products.vue
@@ -39,6 +39,10 @@ export default {
             type: Array,
             default: () => ([]),
         },
+        currentProductId: {
+            type: Number,
+            default: null,
+        },
     },
     data: () => ({
         sidebarCollapsed: false,
