diff --git a/assets/js/components/loading.vue b/assets/js/components/loading.vue
new file mode 100644
index 0000000..601dc66
--- /dev/null
+++ b/assets/js/components/loading.vue
@@ -0,0 +1,18 @@
+<template>
+    <h4 :class="$style.component">
+        Loading...
+    </h4>
+</template>
+
+<script>
+export default {
+    name: 'Loading',
+};
+</script>
+
+<style lang="scss" module>
+.component {
+    background: url('../../images/loading.gif') no-repeat left center;
+    padding: 0 0 4px 50px;
+}
+</style>
