diff --git a/assets/js/components/sidebar.vue b/assets/js/components/sidebar.vue
index e134d33..fdea93f 100644
--- a/assets/js/components/sidebar.vue
+++ b/assets/js/components/sidebar.vue
@@ -39,6 +39,8 @@
 </template>
 
 <script>
+import axios from 'axios';
+
 export default {
     name: 'Sidebar',
     props: {
@@ -51,6 +53,16 @@ export default {
     data: () => ({
         categories: [],
     }),
+    async created() {
+        this.products = [];
+
+        const response = await axios({
+            method: 'get',
+            url: '/api/products',
+        });
+
+        this.products = response.data['hydra:member'];
+    },
 };
 </script>
 
