diff --git a/assets/js/components/catalog.vue b/assets/js/components/catalog.vue
index 92414ae5..2108d218 100644
--- a/assets/js/components/catalog.vue
+++ b/assets/js/components/catalog.vue
@@ -32,7 +32,7 @@ export default {
         products: [],
         legend: 'Shipping takes 10-12 weeks, and products probably won\'t work',
     }),
-    async mounted() {
+    async created() {
         this.products = [];
 
         const response = await axios.get('/api/products');
