diff --git a/assets/js/components/sidebar.vue b/assets/js/components/sidebar.vue
index b286db6d..ca8e221c 100644
--- a/assets/js/components/sidebar.vue
+++ b/assets/js/components/sidebar.vue
@@ -41,6 +41,8 @@
 </template>
 
 <script>
+import axios from 'axios';
+
 export default {
     name: 'Sidebar',
     props: {
@@ -52,6 +54,13 @@ export default {
     data: () => ({
         categories: [],
     }),
+    async created() {
+        this.products = [];
+
+        const response = await axios.get('/api/products');
+
+        this.products = response.data['hydra:member'];
+    },
 };
 </script>
 
