diff --git a/assets/js/components/sidebar.vue b/assets/js/components/sidebar.vue
index 4fba123a..1b11d93f 100644
--- a/assets/js/components/sidebar.vue
+++ b/assets/js/components/sidebar.vue
@@ -51,16 +51,7 @@ export default {
     },
     data() {
         return {
-            categories: [
-                {
-                    name: 'Dot Matrix Printers',
-                    link: '#',
-                },
-                {
-                    name: 'Iomega Zip Drives',
-                    link: '#',
-                },
-            ],
+            categories: [],
         };
     },
 };
