diff --git a/assets/js/components/product-list/index.vue b/assets/js/components/product-list/index.vue
index 91b5e79..ca60eb6 100644
--- a/assets/js/components/product-list/index.vue
+++ b/assets/js/components/product-list/index.vue
@@ -9,11 +9,13 @@
 </template>
 
 <script>
+import Loading from '@/components/loading';
 import ProductCard from '@/components/product-list/product-card';
 
 export default {
     name: 'ProductList',
     components: {
+        Loading,
         ProductCard,
     },
     props: {
