diff --git a/assets/js/components/product-list/index.vue b/assets/js/components/product-list/index.vue
index 440f25a..f357a0d 100644
--- a/assets/js/components/product-list/index.vue
+++ b/assets/js/components/product-list/index.vue
@@ -1,7 +1,10 @@
 <template>
     <div :class="$style.component">
+        <loading v-show="products.length === 0" />
+
         <product-card
             v-for="product in products"
+            v-show="products.length > 0"
             :key="product['@id']"
             :item="product"
         />
