diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index 8f568c83..6dfd1f55 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -67,7 +67,7 @@
 </template>
 
 <script>
-import { reactive } from 'vue';
+import { reactive, watch } from 'vue';
 import { fetchColors } from '@/services/colors-service';
 import { fetchFeaturedProducts, fetchProductsById } from '@/services/products-service';
 import CheckoutForm from '@/components/checkout';
@@ -129,11 +129,6 @@ export default {
                 : '<< Back';
         },
     },
-    watch: {
-        'cart.items.length': function watchCartItemsLength() {
-            this.loadProducts();
-        },
-    },
     async created() {
         this.loadFeaturedProduct();
         this.colors = (await fetchColors()).data['hydra:member'];
@@ -169,7 +164,7 @@ export default {
 
         const products = reactive([]);
 
-        const loadProducts = async () => {
+        watch(cart.items.length, async () => {
             const productIds = cart.items.map((item) => item.product);
 
             const productsResponse = await fetchProductsById(productIds);
@@ -179,7 +174,7 @@ export default {
             productsResponse.data['hydra:member'].forEach((product) => {
                 products.push(product);
             });
-        };
+        });
 
         return {
             addToCartLoading,
@@ -188,7 +183,6 @@ export default {
             cart,
             products,
             addProductToCart,
-            loadProducts,
             removeProductFromCart,
             updateProductQuantity,
         };
