diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index 486ae401..f6e89336 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -67,7 +67,7 @@
 </template>
 
 <script>
-import { reactive, watch } from 'vue';
+import { reactive, watchEffect } from 'vue';
 import { fetchColors } from '@/services/colors-service';
 import { fetchFeaturedProducts, fetchProductsById } from '@/services/products-service';
 import CheckoutForm from '@/components/checkout';
@@ -165,7 +165,7 @@ export default {
 
         const products = reactive([]);
 
-        watch(cart.items, async () => {
+        watchEffect(async () => {
             const productIds = cart.items.map((item) => item.product);
 
             const productsResponse = await fetchProductsById(productIds);
@@ -175,6 +175,9 @@ export default {
             productsResponse.data['hydra:member'].forEach((product) => {
                 products.push(product);
             });
+        }, {
+            onTrack: (data) => { console.log('tracking', data); },
+            onTrigger: (data) => { console.log('triggering', data); },
         });
 
         loadCart();
