diff --git a/assets/js/modules/shopping-cart.js b/assets/js/modules/shopping-cart.js
index 834dc5c9..12b575c2 100644
--- a/assets/js/modules/shopping-cart.js
+++ b/assets/js/modules/shopping-cart.js
@@ -11,6 +11,8 @@ const cart = reactive({
     items: [],
 });
 
+const fullCartItems = reactive([]);
+
 export default function useShoppingCart() {
     const updateShoppingCartHeader = () => {
         document.getElementById('js-shopping-cart-items')
@@ -61,6 +63,7 @@ export default function useShoppingCart() {
 
     return {
         cart,
+        fullCartItems,
         loadShoppingCart,
         updateQuantity,
         removeFromCart,
diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index 7624d525..a35a8e37 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -87,7 +87,6 @@ export default {
     data() {
         return {
             currentState: 'cart',
-            items: [],
             loading: false,
         };
     },
@@ -128,6 +127,7 @@ export default {
             removeFromCart,
             updateQuantity,
             cart,
+            fullCartItems,
         } = useShoppingCart();
 
         onMounted(() => {
@@ -136,6 +136,7 @@ export default {
 
         return {
             cart,
+            items: fullCartItems,
             removeFromCart,
             updateQuantity,
         };
