diff --git a/assets/js/factory/shopping-cart.js b/assets/js/factory/shopping-cart.js
index ded4c954..e1555979 100644
--- a/assets/js/factory/shopping-cart.js
+++ b/assets/js/factory/shopping-cart.js
@@ -8,13 +8,20 @@ const cart = reactive({
 
 export default function useShoppingCart() {
     onMounted(async () => {
+        cart.items.length = 0;
+
+        let newCart;
         try {
-            this.cart = await getCart();
+            newCart = await getCart();
         } catch (e) {
             return;
         }
 
-        this.updateShoppingCartHeader();
+        newCart.items.forEach((item) => {
+            cart.items.push(item);
+        });
+
+        // this.updateShoppingCartHeader();
     });
 
     return {
