diff --git a/assets/components/product-show/index.vue b/assets/components/product-show/index.vue
index 972701db..0d443c50 100644
--- a/assets/components/product-show/index.vue
+++ b/assets/components/product-show/index.vue
@@ -57,7 +57,7 @@ import formatPrice from '@/helpers/format-price';
 import { fetchOneProduct } from '@/services/products-service';
 import Loading from '@/components/loading';
 import TitleComponent from '@/components/title';
-import ShoppingCartMixin from '@/mixins/get-shopping-cart';
+import useShoppingCart from '@/modules/shopping-cart';
 import CartAddControls from './cart-add-controls';
 
 export default {
@@ -67,7 +67,6 @@ export default {
         TitleComponent,
         CartAddControls,
     },
-    mixins: [ShoppingCartMixin],
     props: {
         productId: {
             type: String,
@@ -101,6 +100,11 @@ export default {
             this.addProductToCart(this.product, selectedColorId, quantity);
         },
     },
+    setup() {
+        return {
+            ...useShoppingCart(),
+        };
+    },
 };
 </script>
 
diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index ea3d9054..c27fc640 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -69,12 +69,12 @@
 <script>
 import { fetchColors } from '@/services/colors-service';
 import { fetchFeaturedProducts, fetchProductsById } from '@/services/products-service';
-import ShoppingCartMixin from '@/mixins/get-shopping-cart';
 import CheckoutForm from '@/components/checkout';
 import TitleComponent from '@/components/title';
 import ShoppingCartList from '@/components/shopping-cart';
 import Loading from '@/components/loading';
 import CartSidebar from '@/components/shopping-cart/cart-sidebar';
+import useShoppingCart from '@/modules/shopping-cart';
 
 export default {
     name: 'ShoppingCart',
@@ -85,7 +85,6 @@ export default {
         TitleComponent,
         CartSidebar,
     },
-    mixins: [ShoppingCartMixin],
     data() {
         return {
             currentState: 'cart',
@@ -162,6 +161,11 @@ export default {
             [this.featuredProduct] = featuredProducts;
         },
     },
+    setup() {
+        return {
+            ...useShoppingCart(),
+        };
+    },
 };
 </script>
 
