diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index 95febdfd..46ec5cb4 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -68,7 +68,7 @@
 </template>
 
 <script>
-import { onMounted, ref } from 'vue';
+import { onMounted, ref, watch } from 'vue';
 import { getFullShoppingCart } from '@/services/cart-service';
 import Loading from '@/components/loading';
 import CheckoutForm from '@/components/checkout';
@@ -112,25 +112,29 @@ export default {
             fullCartItems,
         } = useShoppingCart();
 
-        watch: {
-            'cart.items.length': async function watchCartItemsLength() {
-                this.loading = true;
-                this.items = [];
+        watch(cart.items.length, async () => {
+            loading.value = true;
+            fullCartItems.length = 0;
 
-                if (!this.cart.items.length) {
-                    this.loading = false;
-                    return;
-                }
+            if (!cart.items.length) {
+                loading.value = false;
+                return;
+            }
 
-                try {
-                    this.items = await getFullShoppingCart(this.cart);
-                } catch (e) {
-                    this.loading = false;
-                }
+            let newItems;
+            try {
+                newItems = await getFullShoppingCart(cart);
+            } catch (e) {
+                loading.value = false;
+                return;
+            }
 
-                this.loading = false;
-            },
-        },
+            newItems.forEach((item) => {
+                fullCartItems.push(item);
+            });
+
+            loading.value = false;
+        });
 
         onMounted(() => {
             loadShoppingCart();
