diff --git a/assets/js/modules/page-context.js b/assets/js/modules/page-context.js
index 136371ff..8ddc19b1 100644
--- a/assets/js/modules/page-context.js
+++ b/assets/js/modules/page-context.js
@@ -1,4 +1,4 @@
-import { computed, ref, onCreated } from 'vue';
+import { computed, ref } from 'vue';
 import Product from '@/components/product';
 import Catalog from '@/components/catalog';
 import { getCurrentCategoryId, getCurrentProductId } from '@/services/page-context';
@@ -23,11 +23,11 @@ export default function usePageContext() {
             }
     ));
 
-    onCreated(async () => {
+    const initCategories = async () => {
         const response = await fetchCategories();
 
         categories.value = response.data['hydra:member'];
-    });
+    };
 
     return {
         categories,
@@ -35,5 +35,6 @@ export default function usePageContext() {
         currentProductId,
         currentComponent,
         currentProps,
+        initCategories,
     };
 }
diff --git a/assets/js/pages/products.vue b/assets/js/pages/products.vue
index 36f4df76..880caf43 100644
--- a/assets/js/pages/products.vue
+++ b/assets/js/pages/products.vue
@@ -22,6 +22,7 @@
 
 <script>
 import Sidebar from '@/components/sidebar';
+import usePageContext from '@/modules/page-context';
 import useShoppingCart from '@/modules/shopping-cart';
 import useCollapsibleSidebar from '@/modules/collapsible-sidebar';
 
@@ -31,9 +32,14 @@ export default {
         Sidebar,
     },
     setup() {
+        const pageContext = usePageContext();
+
+        pageContext.initCategories();
+
         return {
             ...useShoppingCart(),
             ...useCollapsibleSidebar(),
+            ...pageContext,
         };
     },
 };
