diff --git a/assets/js/modules/shopping-cart.js b/assets/js/modules/shopping-cart.js
index 2b43d6e4..3b59fe01 100644
--- a/assets/js/modules/shopping-cart.js
+++ b/assets/js/modules/shopping-cart.js
@@ -1,5 +1,10 @@
 import { reactive } from 'vue';
-import { getCart, getCartTotalItems, removeItemFromCart, updateCartItemQuantity } from '@/services/cart-service';
+import {
+    getCart,
+    getCartTotalItems,
+    removeItemFromCart,
+    updateCartItemQuantity,
+} from '@/services/cart-service';
 
 /** @type CartCollection */
 const cart = reactive({
@@ -31,11 +36,11 @@ export default function useShoppingCart() {
      * @param {string|null} color
      * @param {number} quantity
      */
-    async updateQuantity({ product, color, quantity }) {
-        await updateCartItemQuantity(this.cart, product, color, quantity);
+    const updateQuantity = async ({ product, color, quantity }) => {
+        await updateCartItemQuantity(cart, product, color, quantity);
 
-        this.updateShoppingCartHeader();
-    },
+        updateShoppingCartHeader();
+    };
 
     /**
      * Removes a product from the cart, then refreshes the page
@@ -43,15 +48,17 @@ export default function useShoppingCart() {
      * @param {string} product
      * @param {string|null} color
      */
-    async removeFromCart({ product, color }) {
-        await removeItemFromCart(this.cart, product, color);
+    const removeFromCart = async ({ product, color }) => {
+        await removeItemFromCart(cart, product, color);
 
-        this.updateShoppingCartHeader();
-    },
+        updateShoppingCartHeader();
+    };
 
     return {
         cart,
         loadShoppingCart,
         updateShoppingCartHeader,
+        updateQuantity,
+        removeFromCart,
     };
 }
diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index cc8bb5ec..87591f6b 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -69,7 +69,7 @@
 
 <script>
 import { onMounted } from 'vue';
-import { getFullShoppingCart, updateCartItemQuantity, removeItemFromCart } from '@/services/cart-service';
+import { getFullShoppingCart } from '@/services/cart-service';
 import Loading from '@/components/loading';
 import CheckoutForm from '@/components/checkout';
 import TitleComponent from '@/components/title';
