diff --git a/assets/js/factory/get-page-shopping-cart.js b/assets/js/factory/get-page-shopping-cart.js
new file mode 100644
index 00000000..f231172c
--- /dev/null
+++ b/assets/js/factory/get-page-shopping-cart.js
@@ -0,0 +1,11 @@
+import { ref, reactive } from 'vue';
+
+export default function usePageShoppingCart() {
+    const items = reactive([]);
+    const loading = ref(false);
+
+    return {
+        items,
+        loading,
+    };
+}
diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index 6f89cc8c..a640ed55 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -74,6 +74,7 @@ import CheckoutForm from '@/components/checkout';
 import TitleComponent from '@/components/title';
 import ShoppingCartList from '@/components/shopping-cart';
 import useShoppingCart from '@/factory/get-shopping-cart';
+import usePageShoppingCart from '@/factory/get-page-shopping-cart';
 
 export default {
     name: 'ShoppingCart',
@@ -86,8 +87,6 @@ export default {
     data() {
         return {
             currentState: 'cart',
-            items: [],
-            loading: false,
         };
     },
     computed: {
@@ -124,6 +123,7 @@ export default {
     setup() {
         return {
             ...useShoppingCart(),
+            ...usePageShoppingCart(),
         };
     },
 };
