diff --git a/assets/js/modules/shopping-cart.js b/assets/js/modules/shopping-cart.js
index c848c02e..14533204 100644
--- a/assets/js/modules/shopping-cart.js
+++ b/assets/js/modules/shopping-cart.js
@@ -1,27 +1,29 @@
 import { getCart, getCartTotalItems } from '@/services/cart-service';
 
-export default {
-    data() {
-        return {
-            /** @type CartCollection */
-            cart: {
-                items: [],
-            },
-        };
-    },
-    async mounted() {
-        try {
-            this.cart = await getCart();
-        } catch (e) {
-            return;
-        }
+export default function useShoppingCart() {
+    return {
+        data() {
+            return {
+                /** @type CartCollection */
+                cart: {
+                    items: [],
+                },
+            };
+        },
+        async mounted() {
+            try {
+                this.cart = await getCart();
+            } catch (e) {
+                return;
+            }
 
-        this.updateShoppingCartHeader();
-    },
-    methods: {
-        updateShoppingCartHeader() {
-            document.getElementById('js-shopping-cart-items')
-                .innerHTML = getCartTotalItems(this.cart).toString();
+            this.updateShoppingCartHeader();
+        },
+        methods: {
+            updateShoppingCartHeader() {
+                document.getElementById('js-shopping-cart-items')
+                    .innerHTML = getCartTotalItems(this.cart).toString();
+            },
         },
-    },
-};
+    };
+}
