diff --git a/assets/js/modules/shopping-cart.js b/assets/js/modules/shopping-cart.js
index ded4c954..275d9ce2 100644
--- a/assets/js/modules/shopping-cart.js
+++ b/assets/js/modules/shopping-cart.js
@@ -9,12 +9,12 @@ const cart = reactive({
 export default function useShoppingCart() {
     onMounted(async () => {
         try {
-            this.cart = await getCart();
+            cart.items = (await getCart()).items;
         } catch (e) {
             return;
         }
 
-        this.updateShoppingCartHeader();
+        // this.updateShoppingCartHeader();
     });
 
     return {
