diff --git a/assets/js/modules/shopping-cart.js b/assets/js/modules/shopping-cart.js
index 3a0db4ff..919a069a 100644
--- a/assets/js/modules/shopping-cart.js
+++ b/assets/js/modules/shopping-cart.js
@@ -2,6 +2,7 @@ import { reactive } from 'vue';
 import {
     getCart,
     getCartTotalItems,
+    getFullShoppingCart,
     removeItemFromCart,
     updateCartItemQuantity,
 } from '@/services/cart-service';
@@ -58,6 +59,22 @@ export default function useShoppingCart() {
         updateShoppingCartHeader();
     };
 
+    const loadFullShoppingCart = async () => {
+        fullCartItems.items.length = 0;
+
+        if (!cart.items.length) {
+            loading.value = false;
+            return;
+        }
+
+        try {
+            fullCartItems.items = await getFullShoppingCart(cart);
+        } catch (e) {
+            loading.value = false;
+            return;
+        }
+    };
+
     return {
         cart,
         fullCartItems,
diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index 8c2ef7a3..d2e29378 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -69,7 +69,6 @@
 
 <script>
 import { onMounted, ref, watch } from 'vue';
-import { getFullShoppingCart } from '@/services/cart-service';
 import Loading from '@/components/loading';
 import CheckoutForm from '@/components/checkout';
 import TitleComponent from '@/components/title';
@@ -114,19 +113,6 @@ export default {
 
         watch(cart, async () => {
             loading.value = true;
-            fullCartItems.items.length = 0;
-
-            if (!cart.items.length) {
-                loading.value = false;
-                return;
-            }
-
-            try {
-                fullCartItems.items = await getFullShoppingCart(cart);
-            } catch (e) {
-                loading.value = false;
-                return;
-            }
 
             loading.value = false;
         });
