diff --git a/assets/modules/shopping-cart.js b/assets/modules/shopping-cart.js
index 78890e54..8d57f4c9 100644
--- a/assets/modules/shopping-cart.js
+++ b/assets/modules/shopping-cart.js
@@ -1,4 +1,4 @@
-import { ref } from 'vue';
+import { reactive, ref } from 'vue';
 import {
     addItemToCart,
     fetchCart,
@@ -10,14 +10,15 @@ import {
 const addToCartLoading = ref(false);
 const addToCartSuccess = ref(false);
 
+/** @type CartCollection|null */
+const cart = reactive(null);
+
 export default function useShoppingCart() {
     return {
-        data() {
-            return {
-                cart: null,
-            };
-        },
-        async created() {
+        addToCartLoading,
+        addToCartSuccess,
+        cart,
+        /* async created() {
             this.cart = await fetchCart();
         },
         methods: {
@@ -56,6 +57,6 @@ export default function useShoppingCart() {
                 document.getElementById('js-shopping-cart-items')
                     .innerHTML = getCartTotalItems(this.cart).toString();
             },
-        },
+        },*/
     };
 }
