diff --git a/assets/js/factory/get-page-shopping-cart.js b/assets/js/factory/get-page-shopping-cart.js
index 0a579b13..91c3e21c 100644
--- a/assets/js/factory/get-page-shopping-cart.js
+++ b/assets/js/factory/get-page-shopping-cart.js
@@ -1,4 +1,4 @@
-import { ref, reactive, watch } from 'vue';
+import { ref, reactive, watchEffect } from 'vue';
 import { getFullShoppingCart } from '@/services/cart-service';
 
 /**
@@ -9,7 +9,7 @@ export default function usePageShoppingCart(cart) {
     const items = reactive([]);
     const loading = ref(false);
 
-    watch(cart, async () => {
+    watchEffect(async () => {
         loading.value = true;
         items.length = 0;
 
@@ -29,6 +29,9 @@ export default function usePageShoppingCart(cart) {
         }
 
         loading.value = false;
+    }, {
+        onTrack: (data) => { console.log('tracking', data); },
+        onTrigger: (data) => { console.log('triggering', data); },
     });
 
     return {
