diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index eae2b268..bf36d58a 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -101,8 +101,18 @@ export default {
         },
     },
     setup() {
+        const {
+            cart,
+            updateShoppingCartHeader,
+            updateQuantity,
+            removeFromCart,
+        } = useShoppingCart();
+
         return {
-            ...useShoppingCart(),
+            cart,
+            updateShoppingCartHeader,
+            updateQuantity,
+            removeFromCart,
             ...usePageShoppingCart(),
         };
     },
