diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index 50ff5ed9..b1f49468 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -112,7 +112,7 @@ export default {
             fullCartItems,
         } = useShoppingCart();
 
-        watch(cart.items.length, async () => {
+        watch(cart.items, async () => {
             loading.value = true;
             fullCartItems.items.length = 0;
 
