diff --git a/assets/js/modules/shopping-cart.js b/assets/js/modules/shopping-cart.js
index 3b59fe01..5d247e62 100644
--- a/assets/js/modules/shopping-cart.js
+++ b/assets/js/modules/shopping-cart.js
@@ -57,7 +57,6 @@ export default function useShoppingCart() {
     return {
         cart,
         loadShoppingCart,
-        updateShoppingCartHeader,
         updateQuantity,
         removeFromCart,
     };
diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index 87591f6b..7624d525 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -123,7 +123,12 @@ export default {
         },
     },
     setup() {
-        const { loadShoppingCart, cart, updateShoppingCartHeader } = useShoppingCart();
+        const {
+            loadShoppingCart,
+            removeFromCart,
+            updateQuantity,
+            cart,
+        } = useShoppingCart();
 
         onMounted(() => {
             loadShoppingCart();
@@ -131,7 +136,8 @@ export default {
 
         return {
             cart,
-            updateShoppingCartHeader,
+            removeFromCart,
+            updateQuantity,
         };
     },
 };
