diff --git a/assets/js/modules/shopping-cart.js b/assets/js/modules/shopping-cart.js
index 9abf0ecc..457e0ec9 100644
--- a/assets/js/modules/shopping-cart.js
+++ b/assets/js/modules/shopping-cart.js
@@ -1,5 +1,5 @@
 import { reactive } from 'vue';
-import { getCart, getCartTotalItems } from '@/services/cart-service';
+import { getCart, getCartTotalItems, removeItemFromCart, updateCartItemQuantity } from '@/services/cart-service';
 
 /** @type CartCollection */
 const cart = reactive({
@@ -29,6 +29,31 @@ export default function useShoppingCart() {
         updateShoppingCartHeader();
     };
 
+    /**
+     * Updates the product quantity in the cart, then refreshes the page
+     *
+     * @param {string} product
+     * @param {string|null} color
+     * @param {number} quantity
+     */
+    async updateQuantity({ product, color, quantity }) {
+        await updateCartItemQuantity(this.cart, product, color, quantity);
+
+        this.updateShoppingCartHeader();
+    },
+
+    /**
+     * Removes a product from the cart, then refreshes the page
+     *
+     * @param {string} product
+     * @param {string|null} color
+     */
+    async removeFromCart({ product, color }) {
+        await removeItemFromCart(this.cart, product, color);
+
+        this.updateShoppingCartHeader();
+    },
+
     return {
         cart,
         loadShoppingCart,
diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index eff675d1..cc8bb5ec 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -121,31 +121,6 @@ export default {
         switchState() {
             this.currentState = this.currentState === 'cart' ? 'checkout' : 'cart';
         },
-
-        /**
-         * Updates the product quantity in the cart, then refreshes the page
-         *
-         * @param {string} product
-         * @param {string|null} color
-         * @param {number} quantity
-         */
-        async updateQuantity({ product, color, quantity }) {
-            await updateCartItemQuantity(this.cart, product, color, quantity);
-
-            this.updateShoppingCartHeader();
-        },
-
-        /**
-         * Removes a product from the cart, then refreshes the page
-         *
-         * @param {string} product
-         * @param {string|null} color
-         */
-        async removeFromCart({ product, color }) {
-            await removeItemFromCart(this.cart, product, color);
-
-            this.updateShoppingCartHeader();
-        },
     },
     setup() {
         const { loadShoppingCart, cart, updateShoppingCartHeader } = useShoppingCart();
