diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index 9a049412..bc483758 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -37,7 +37,7 @@
                     >
                         <shopping-cart-list
                             v-if="!loading && currentState === 'cart'"
-                            :items="items"
+                            :items="fullCartItems.items"
                             @updateQuantity="updateQuantity"
                             @removeFromCart="removeFromCart"
                         />
@@ -49,7 +49,7 @@
                     </transition>
 
                     <div
-                        v-show="!loading && items.length"
+                        v-show="!loading && fullCartItems.items.length"
                         class="row p-3 justify-content-end"
                     >
                         <div class="col-2">
@@ -100,7 +100,7 @@ export default {
     watch: {
         'cart.items.length': async function watchCartItemsLength() {
             this.loading = true;
-            this.items = [];
+            this.fullCartItems.items = [];
 
             if (!this.cart.items.length) {
                 this.loading = false;
@@ -108,7 +108,7 @@ export default {
             }
 
             try {
-                this.items = await getFullShoppingCart(this.cart);
+                this.fullCartItems.items = await getFullShoppingCart(this.cart);
             } catch (e) {
                 this.loading = false;
             }
