diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index 471b8234..20d483b1 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -68,7 +68,7 @@
 </template>
 
 <script>
-import { onMounted, ref, watch } from 'vue';
+import { onMounted, ref, watchEffect } from 'vue';
 import Loading from '@/components/loading';
 import CheckoutForm from '@/components/checkout';
 import TitleComponent from '@/components/title';
@@ -112,12 +112,15 @@ export default {
             fullCartItems,
         } = useShoppingCart();
 
-        watch(cart, async () => {
+        watchEffect(async () => {
             loading.value = true;
 
             await loadFullShoppingCart();
 
             loading.value = false;
+        }, {
+            onTrack: (data) => { console.log('tracking', data); },
+            onTrigger: (data) => { console.log('triggering', data); },
         });
 
         onMounted(() => {
