diff --git a/assets/js/modules/page-context.js b/assets/js/modules/page-context.js
index 8ddc19b1..2bd4e7a6 100644
--- a/assets/js/modules/page-context.js
+++ b/assets/js/modules/page-context.js
@@ -3,8 +3,11 @@ import Product from '@/components/product';
 import Catalog from '@/components/catalog';
 import { getCurrentCategoryId, getCurrentProductId } from '@/services/page-context';
 import { fetchCategories } from '@/services/categories-service';
+import useShoppingCart from '@/modules/shopping-cart';
 
 export default function usePageContext() {
+    const { cart } = useShoppingCart();
+
     const categories = ref([]);
     const currentCategoryId = computed(() => (getCurrentCategoryId()));
     const currentProductId = computed(() => (getCurrentProductId()));
