diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index f6e89336..c22ccad5 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -175,9 +175,6 @@ export default {
             productsResponse.data['hydra:member'].forEach((product) => {
                 products.push(product);
             });
-        }, {
-            onTrack: (data) => { console.log('tracking', data); },
-            onTrigger: (data) => { console.log('triggering', data); },
         });
 
         loadCart();
