diff --git a/assets/js/modules/get-page-shopping-cart.js b/assets/js/modules/get-page-shopping-cart.js
index fbc98707..17910c7e 100644
--- a/assets/js/modules/get-page-shopping-cart.js
+++ b/assets/js/modules/get-page-shopping-cart.js
@@ -1,7 +1,11 @@
 import { ref, reactive, watch } from 'vue';
 import { getFullShoppingCart } from '@/services/cart-service';
 
-export default function usePageShoppingCart() {
+/**
+ * @param {CartCollection} cart
+ * @returns {Object}
+ */
+export default function usePageShoppingCart(cart) {
     const items = reactive([]);
     const loading = ref(false);
 
diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index ea2130f7..e526d8b9 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -117,7 +117,7 @@ export default {
             cart,
             updateQuantity,
             removeFromCart,
-            ...usePageShoppingCart(),
+            ...usePageShoppingCart(cart),
         };
     },
 };
