diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index b22cda74..ea2130f7 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -102,7 +102,12 @@ export default {
         },
     },
     setup() {
-        const { loadShoppingCart, cart } = useShoppingCart();
+        const {
+            loadShoppingCart,
+            cart,
+            updateQuantity,
+            removeFromCart,
+        } = useShoppingCart();
 
         onMounted(() => {
             loadShoppingCart();
@@ -110,6 +115,8 @@ export default {
 
         return {
             cart,
+            updateQuantity,
+            removeFromCart,
             ...usePageShoppingCart(),
         };
     },
