diff --git a/assets/js/factory/shopping-cart.js b/assets/js/factory/shopping-cart.js
index 9fdb5d47..ded4c954 100644
--- a/assets/js/factory/shopping-cart.js
+++ b/assets/js/factory/shopping-cart.js
@@ -1,4 +1,4 @@
-import { reactive } from 'vue';
+import { onMounted, reactive } from 'vue';
 import { getCart, getCartTotalItems } from '@/services/cart-service';
 
 /** @type CartCollection */
@@ -7,19 +7,21 @@ const cart = reactive({
 });
 
 export default function useShoppingCart() {
+    onMounted(async () => {
+        try {
+            this.cart = await getCart();
+        } catch (e) {
+            return;
+        }
+
+        this.updateShoppingCartHeader();
+    });
+
     return {
         cart,
     };
-    /*
-        async mounted() {
-            try {
-                this.cart = await getCart();
-            } catch (e) {
-                return;
-            }
 
-            this.updateShoppingCartHeader();
-        },
+    /*
         methods: {
             updateShoppingCartHeader() {
                 document.getElementById('js-shopping-cart-items')
