diff --git a/assets/js/factory/get-page-shopping-cart.js b/assets/js/factory/get-page-shopping-cart.js
index 17910c7e..0a579b13 100644
--- a/assets/js/factory/get-page-shopping-cart.js
+++ b/assets/js/factory/get-page-shopping-cart.js
@@ -9,7 +9,7 @@ export default function usePageShoppingCart(cart) {
     const items = reactive([]);
     const loading = ref(false);
 
-    watch(cart.items.length, async () => {
+    watch(cart, async () => {
         loading.value = true;
         items.length = 0;
 
