diff --git a/assets/js/modules/get-page-shopping-cart.js b/assets/js/modules/get-page-shopping-cart.js
new file mode 100644
index 00000000..f231172c
--- /dev/null
+++ b/assets/js/modules/get-page-shopping-cart.js
@@ -0,0 +1,11 @@
+import { ref, reactive } from 'vue';
+
+export default function usePageShoppingCart() {
+    const items = reactive([]);
+    const loading = ref(false);
+
+    return {
+        items,
+        loading,
+    };
+}
diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index 87591f6b..22c77272 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -75,6 +75,7 @@ import CheckoutForm from '@/components/checkout';
 import TitleComponent from '@/components/title';
 import ShoppingCartList from '@/components/shopping-cart';
 import useShoppingCart from '@/modules/shopping-cart';
+import usePageShoppingCart from '@/modules/get-page-shopping-cart';
 
 export default {
     name: 'ShoppingCart',
@@ -87,8 +88,6 @@ export default {
     data() {
         return {
             currentState: 'cart',
-            items: [],
-            loading: false,
         };
     },
     computed: {
@@ -123,7 +122,7 @@ export default {
         },
     },
     setup() {
-        const { loadShoppingCart, cart, updateShoppingCartHeader } = useShoppingCart();
+        const { loadShoppingCart, cart } = useShoppingCart();
 
         onMounted(() => {
             loadShoppingCart();
@@ -131,7 +130,7 @@ export default {
 
         return {
             cart,
-            updateShoppingCartHeader,
+            ...usePageShoppingCart(),
         };
     },
 };
