diff --git a/assets/js/modules/shopping-cart.js b/assets/js/modules/shopping-cart.js
index 275d9ce2..597c4ed7 100644
--- a/assets/js/modules/shopping-cart.js
+++ b/assets/js/modules/shopping-cart.js
@@ -7,6 +7,11 @@ const cart = reactive({
 });
 
 export default function useShoppingCart() {
+    const updateShoppingCartHeader = () => {
+        document.getElementById('js-shopping-cart-items')
+            .innerHTML = getCartTotalItems(cart).toString();
+    };
+
     onMounted(async () => {
         try {
             cart.items = (await getCart()).items;
@@ -14,19 +19,11 @@ export default function useShoppingCart() {
             return;
         }
 
-        // this.updateShoppingCartHeader();
+        updateShoppingCartHeader();
     });
 
     return {
         cart,
+        updateShoppingCartHeader,
     };
-
-    /*
-        methods: {
-            updateShoppingCartHeader() {
-                document.getElementById('js-shopping-cart-items')
-                    .innerHTML = getCartTotalItems(this.cart).toString();
-            },
-        },
-    */
 }
