diff --git a/assets/js/factory/get-shopping-cart.js b/assets/js/factory/get-shopping-cart.js
index 87a2dd5d..c09efe09 100644
--- a/assets/js/factory/get-shopping-cart.js
+++ b/assets/js/factory/get-shopping-cart.js
@@ -7,6 +7,8 @@ const cart = reactive({
 });
 
 export default function useShoppingCart() {
+    const items = reactive([]);
+
     const updateShoppingCartHeader = () => {
         document.getElementById('js-shopping-cart-items')
             .innerHTML = getCartTotalItems(cart).toString();
@@ -31,6 +33,7 @@ export default function useShoppingCart() {
 
     return {
         cart,
+        items,
         updateShoppingCartHeader,
     };
 }
diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index 2f703531..c74860ac 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -86,7 +86,6 @@ export default {
     data() {
         return {
             currentState: 'cart',
-            items: [],
             loading: false,
         };
     },
