diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index b24ca4da..486ae401 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -165,7 +165,7 @@ export default {
 
         const products = reactive([]);
 
-        watch(cart.items.length, async () => {
+        watch(cart.items, async () => {
             const productIds = cart.items.map((item) => item.product);
 
             const productsResponse = await fetchProductsById(productIds);
