diff --git a/assets/js/pages/products.vue b/assets/js/pages/products.vue
index a4f1d6b8..f074bc1f 100644
--- a/assets/js/pages/products.vue
+++ b/assets/js/pages/products.vue
@@ -46,18 +46,7 @@ export default {
             default: null,
         },
     },
-    data() {
-        return {
-            sidebarCollapsed: false,
-        };
-    },
     computed: {
-        asideClass() {
-            return this.sidebarCollapsed ? 'aside-collapsed' : 'col-xs-12 col-3';
-        },
-        contentClass() {
-            return this.sidebarCollapsed ? 'col-xs-12 col-11' : 'col-xs-12 col-9';
-        },
         currentComponent() {
             return this.currentProductId !== null ? Product : Catalog;
         },
@@ -73,13 +62,5 @@ export default {
                 };
         },
     },
-    methods: {
-        /**
-         * Toggles the sidebarCollapsed value
-         */
-        toggleSidebarCollapsed() {
-            this.sidebarCollapsed = !this.sidebarCollapsed;
-        },
-    },
 };
 </script>
