diff --git a/assets/components/product-show.vue b/assets/components/product-show.vue
index 4ff036336..7d8348faa 100644
--- a/assets/components/product-show.vue
+++ b/assets/components/product-show.vue
@@ -120,6 +120,10 @@ export default {
         }
     },
     methods: {
+        addToCart() {
+            this.addProductToCart(this.product, this.selectedColorId, this.quantity);
+        },
+
         updateSelectedColor(iri) {
             this.selectedColorId = iri;
         },
diff --git a/assets/mixins/get-shopping-cart.js b/assets/mixins/get-shopping-cart.js
index 2dd5a57c0..68f11e6f1 100644
--- a/assets/mixins/get-shopping-cart.js
+++ b/assets/mixins/get-shopping-cart.js
@@ -12,7 +12,7 @@ export default {
         this.cart = await fetchCart();
     },
     methods: {
-        async addToCart(product, selectedColorId, quantity) {
+        async addProductToCart(product, selectedColorId, quantity) {
             if (product.colors.length && selectedColorId === null) {
                 alert('Please select a color first!');
                 return;
