diff --git a/assets/js/pages/products.vue b/assets/js/pages/products.vue
index c7266c6e..c1e5f1ac 100644
--- a/assets/js/pages/products.vue
+++ b/assets/js/pages/products.vue
@@ -26,6 +26,7 @@ import Product from '@/components/product';
 import Sidebar from '@/components/sidebar';
 import { getCurrentCategoryId, getCurrentProductId } from '@/services/page-context';
 import { fetchCategories } from '@/services/categories-service';
+import collapsible from '@/mixins/collapsible';
 
 export default {
     name: 'Products',
@@ -34,6 +35,7 @@ export default {
         Product,
         Sidebar,
     },
+    mixins: [collapsible],
     data() {
         return {
             categories: [],
