diff --git a/assets/components/product-show.vue b/assets/components/product-show.vue
index ad24cbc3..325d3597 100644
--- a/assets/components/product-show.vue
+++ b/assets/components/product-show.vue
@@ -74,7 +74,6 @@
 </template>
 
 <script>
-import { addItemToCart, getCartTotalItems } from '@/services/cart-service.js';
 import formatPrice from '@/helpers/format-price';
 import { fetchOneProduct } from '@/services/products-service';
 import ColorSelector from '@/components/color-selector';
@@ -123,26 +122,6 @@ export default {
         }
     },
     methods: {
-        async addToCart() {
-            if (this.product.colors.length && this.selectedColorId === null) {
-                alert('Please select a color first!');
-                return;
-            }
-
-            this.addToCartLoading = true;
-            this.addToCartSuccess = false;
-            await addItemToCart(this.cart, {
-                product: this.product['@id'],
-                color: this.selectedColorId,
-                quantity: this.quantity,
-            });
-            this.addToCartLoading = false;
-            this.addToCartSuccess = true;
-
-            document.getElementById('js-shopping-cart-items')
-                .innerHTML = getCartTotalItems(this.cart).toString();
-        },
-
         updateSelectedColor(iri) {
             this.selectedColorId = iri;
         },
diff --git a/assets/mixins/get-shopping-cart.js b/assets/mixins/get-shopping-cart.js
index ede946de..deda9b99 100644
--- a/assets/mixins/get-shopping-cart.js
+++ b/assets/mixins/get-shopping-cart.js
@@ -1,4 +1,4 @@
-import { fetchCart } from '@/services/cart-service';
+import { addItemToCart, fetchCart, getCartTotalItems } from '@/services/cart-service';
 
 export default {
     data() {
@@ -9,4 +9,25 @@ export default {
     async created() {
         this.cart = await fetchCart();
     },
+    methods: {
+        async addToCart() {
+            if (this.product.colors.length && this.selectedColorId === null) {
+                alert('Please select a color first!');
+                return;
+            }
+
+            this.addToCartLoading = true;
+            this.addToCartSuccess = false;
+            await addItemToCart(this.cart, {
+                product: this.product['@id'],
+                color: this.selectedColorId,
+                quantity: this.quantity,
+            });
+            this.addToCartLoading = false;
+            this.addToCartSuccess = true;
+
+            document.getElementById('js-shopping-cart-items')
+                .innerHTML = getCartTotalItems(this.cart).toString();
+        },
+    },
 };
