diff --git a/assets/js/pages/products.vue b/assets/js/pages/products.vue
index dbcc9ee6..7e4f1cd1 100644
--- a/assets/js/pages/products.vue
+++ b/assets/js/pages/products.vue
@@ -24,6 +24,7 @@ import Catalog from '@/components/catalog';
 import Product from '@/components/product';
 import Sidebar from '@/components/sidebar';
 import { getCurrentCategoryId, getCurrentProductId } from '@/services/page-context';
+import collapsible from '@/mixins/collapsible';
 
 export default {
     name: 'Products',
@@ -32,6 +33,7 @@ export default {
         Product,
         Sidebar,
     },
+    mixins: [collapsible],
     computed: {
         currentCategoryId() {
             return getCurrentCategoryId();
