diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index 447b03e6b..e9ef10372 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -15,6 +15,7 @@
 </template>
 
 <script>
+import ShoppingCartMixin from '@/mixins/get-shopping-cart';
 import TitleComponent from '@/components/title';
 
 export default {
@@ -22,6 +23,7 @@ export default {
     components: {
         TitleComponent,
     },
+    mixins: [ShoppingCartMixin],
 };
 </script>
 
