diff --git a/assets/js/products.vue b/assets/js/products.vue
index d6df22b..9af3761 100644
--- a/assets/js/products.vue
+++ b/assets/js/products.vue
@@ -27,6 +27,7 @@
 import CatalogComponent from '@/components/catalog';
 import ProductComponent from '@/components/product';
 import SidebarComponent from '@/components/sidebar';
+import collapsible from '@/mixins/collapsible';
 
 export default {
     name: 'Products',
@@ -35,6 +36,7 @@ export default {
         ProductComponent,
         SidebarComponent,
     },
+    mixins: [collapsible],
     props: {
         currentCategoryId: {
             type: Number,
