diff --git a/assets/js/pages/products.vue b/assets/js/pages/products.vue
index 0db27565..3ffe62d7 100644
--- a/assets/js/pages/products.vue
+++ b/assets/js/pages/products.vue
@@ -25,6 +25,7 @@ import Catalog from '@/components/catalog';
 import Product from '@/components/product';
 import Sidebar from '@/components/sidebar';
 import { getCurrentCategoryId, getCategories, getCurrentProductId } from '@/services/page-context';
+import collapsible from '@/mixins/collapsible';
 
 export default {
     name: 'Products',
@@ -33,6 +34,7 @@ export default {
         Product,
         Sidebar,
     },
+    mixins: [collapsible],
     computed: {
         currentCategoryId() {
             return getCurrentCategoryId();
