diff --git a/assets/js/pages/products.vue b/assets/js/pages/products.vue
index 836385a6..c7266c6e 100644
--- a/assets/js/pages/products.vue
+++ b/assets/js/pages/products.vue
@@ -36,18 +36,11 @@ export default {
     },
     data() {
         return {
-            sidebarCollapsed: false,
             categories: [],
             currentCategoryId: getCurrentCategoryId(),
         };
     },
     computed: {
-        asideClass() {
-            return this.sidebarCollapsed ? 'aside-collapsed' : 'col-xs-12 col-3';
-        },
-        contentClass() {
-            return this.sidebarCollapsed ? 'col-xs-12 col-11' : 'col-xs-12 col-9';
-        },
         currentProductId() {
             return getCurrentProductId();
         },
@@ -71,10 +64,5 @@ export default {
 
         this.categories = response.data['hydra:member'];
     },
-    methods: {
-        toggleSidebarCollapsed() {
-            this.sidebarCollapsed = !this.sidebarCollapsed;
-        },
-    },
 };
 </script>
