diff --git a/assets/js/pages/products.vue b/assets/js/pages/products.vue
index b0ce4858..0db27565 100644
--- a/assets/js/pages/products.vue
+++ b/assets/js/pages/products.vue
@@ -33,18 +33,7 @@ export default {
         Product,
         Sidebar,
     },
-    data() {
-        return {
-            sidebarCollapsed: false,
-        };
-    },
     computed: {
-        asideClass() {
-            return this.sidebarCollapsed ? 'aside-collapsed' : 'col-xs-12 col-3';
-        },
-        contentClass() {
-            return this.sidebarCollapsed ? 'col-xs-12 col-11' : 'col-xs-12 col-9';
-        },
         currentCategoryId() {
             return getCurrentCategoryId();
         },
@@ -69,10 +58,5 @@ export default {
                 };
         },
     },
-    methods: {
-        toggleSidebarCollapsed() {
-            this.sidebarCollapsed = !this.sidebarCollapsed;
-        },
-    },
 };
 </script>
