diff --git a/assets/components/product-show.vue b/assets/components/product-show.vue
index 325d35970..4ff036336 100644
--- a/assets/components/product-show.vue
+++ b/assets/components/product-show.vue
@@ -99,8 +99,6 @@ export default {
         return {
             quantity: 1,
             selectedColorId: null,
-            addToCartLoading: false,
-            addToCartSuccess: false,
             product: null,
             loading: true,
         };
diff --git a/assets/mixins/get-shopping-cart.js b/assets/mixins/get-shopping-cart.js
index deda9b991..aa7eff62a 100644
--- a/assets/mixins/get-shopping-cart.js
+++ b/assets/mixins/get-shopping-cart.js
@@ -4,6 +4,8 @@ export default {
     data() {
         return {
             cart: null,
+            addToCartLoading: false,
+            addToCartSuccess: false,
         };
     },
     async created() {
