diff --git a/assets/js/products.vue b/assets/js/products.vue
index 892bad6..91471eb 100644
--- a/assets/js/products.vue
+++ b/assets/js/products.vue
@@ -24,6 +24,7 @@
 import CatalogComponent from '@/components/catalog';
 import ProductComponent from '@/components/product';
 import SidebarComponent from '@/components/sidebar';
+import collapsible from '@/mixins/collapsible';
 
 export default {
     name: 'Products',
@@ -32,6 +33,7 @@ export default {
         ProductComponent,
         SidebarComponent,
     },
+    mixins: [collapsible],
     props: {
         currentCategoryId: {
             type: Number,
