diff --git a/assets/mixins/get-shopping-cart.js b/assets/mixins/get-shopping-cart.js
index af917eaf5..ede946de1 100644
--- a/assets/mixins/get-shopping-cart.js
+++ b/assets/mixins/get-shopping-cart.js
@@ -7,8 +7,6 @@ export default {
         };
     },
     async created() {
-        fetchCart().then((cart) => {
-            this.cart = cart;
-        });
+        this.cart = await fetchCart();
     },
 };
