diff --git a/assets/js/pages/products.vue b/assets/js/pages/products.vue
index 61b3057f..917a123d 100644
--- a/assets/js/pages/products.vue
+++ b/assets/js/pages/products.vue
@@ -36,17 +36,10 @@ export default {
     },
     data() {
         return {
-            sidebarCollapsed: false,
             categories: [],
         };
     },
     computed: {
-        asideClass() {
-            return this.sidebarCollapsed ? 'aside-collapsed' : 'col-xs-12 col-3';
-        },
-        contentClass() {
-            return this.sidebarCollapsed ? 'col-xs-12 col-11' : 'col-xs-12 col-9';
-        },
         currentCategoryId() {
             return getCurrentCategoryId();
         },
@@ -73,10 +66,5 @@ export default {
 
         this.categories = response.data['hydra:member'];
     },
-    methods: {
-        toggleSidebarCollapsed() {
-            this.sidebarCollapsed = !this.sidebarCollapsed;
-        },
-    },
 };
 </script>
