diff --git a/assets/components/product-show.vue b/assets/components/product-show.vue
index c0ed11611..ef896d41d 100644
--- a/assets/components/product-show.vue
+++ b/assets/components/product-show.vue
@@ -78,12 +78,13 @@
 </template>
 
 <script>
-import { fetchCart, addItemToCart, getCartTotalItems } from '@/services/cart-service.js';
+import { addItemToCart, getCartTotalItems } from '@/services/cart-service.js';
 import formatPrice from '@/helpers/format-price';
 import { fetchOneProduct } from '@/services/products-service';
 import ColorSelector from '@/components/color-selector';
 import Loading from '@/components/loading';
 import TitleComponent from '@/components/title';
+import ShoppingCartMixin from '@/mixins/get-shopping-cart';
 
 export default {
     name: 'ProductShow',
@@ -92,6 +93,7 @@ export default {
         Loading,
         TitleComponent,
     },
+    mixins: [ShoppingCartMixin],
     props: {
         productId: {
             type: String,
@@ -100,7 +102,6 @@ export default {
     },
     data() {
         return {
-            cart: null,
             quantity: 1,
             selectedColorId: null,
             addToCartLoading: false,
@@ -119,10 +120,6 @@ export default {
         },
     },
     async created() {
-        fetchCart().then((cart) => {
-            this.cart = cart;
-        });
-
         try {
             this.product = (await fetchOneProduct(this.productId)).data;
         } finally {
