diff --git a/assets/js/products.vue b/assets/js/products.vue
index a597044..d6df22b 100644
--- a/assets/js/products.vue
+++ b/assets/js/products.vue
@@ -49,28 +49,11 @@ export default {
             default: null,
         },
     },
-    data: () => ({
-        sidebarCollapsed: false,
-    }),
     computed: {
-        asideClass() {
-            return this.sidebarCollapsed ? 'aside-collapsed' : 'col-xs-12 col-lg-3';
-        },
-        contentClass() {
-            return this.sidebarCollapsed ? 'col-xs-12 col-lg-11' : 'col-xs-12 col-lg-9';
-        },
         currentComponent() {
             return this.currentProductId !== null ? 'ProductComponent' : 'CatalogComponent';
         },
     },
-    methods: {
-        /**
-         * Toggles the sidebarCollapsed value
-         */
-        toggleSidebarCollapsed() {
-            this.sidebarCollapsed = !this.sidebarCollapsed;
-        },
-    },
 };
 </script>
 
