diff --git a/assets/js/mixins/collapsible.js b/assets/js/mixins/collapsible.js
new file mode 100644
index 00000000..0d25b0c3
--- /dev/null
+++ b/assets/js/mixins/collapsible.js
@@ -0,0 +1,22 @@
+const collapsible = {
+    data() {
+        return {
+            sidebarCollapsed: false,
+        };
+    },
+    computed: {
+        asideClass() {
+            return this.sidebarCollapsed ? 'aside-collapsed' : 'col-xs-12 col-3';
+        },
+        contentClass() {
+            return this.sidebarCollapsed ? 'col-xs-12 col-11' : 'col-xs-12 col-9';
+        },
+    },
+    methods: {
+        toggleSidebarCollapsed() {
+            this.sidebarCollapsed = !this.sidebarCollapsed;
+        },
+    },
+};
+
+export default collapsible;
