diff --git a/assets/js/components/shopping-cart/cart-item.vue b/assets/js/components/shopping-cart/cart-item.vue
index 5191553a..772f8779 100644
--- a/assets/js/components/shopping-cart/cart-item.vue
+++ b/assets/js/components/shopping-cart/cart-item.vue
@@ -23,6 +23,7 @@
                 class="form-control"
                 type="number"
                 min="1"
+                @input="updateQty(product['@id'], product.colorId, product.qty)"
             >
         </div>
 
@@ -39,11 +40,16 @@
 </template>
 
 <script>
+import { updateCartItemQuantity } from '@/services/cart-service';
 import formatPrice from '@/helpers/format-price';
 
 export default {
     name: 'ShoppingCartProduct',
     props: {
+        cart: {
+            type: Object,
+            required: true,
+        },
         product: {
             type: Object,
             required: true,
@@ -54,6 +60,19 @@ export default {
             return formatPrice(this.product.price * this.product.qty);
         },
     },
+    methods: {
+        /**
+         * Updates the product quantity in the cart, then refreshes the page
+         *
+         * @param {string} productId
+         * @param {string|null} colorId
+         * @param {number} qty
+         */
+        async updateQty(productId, colorId, qty) {
+            await updateCartItemQuantity(this.cart, productId, colorId, qty);
+            window.location.reload();
+        },
+    },
 };
 </script>
 
diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index 99c3a5fc..a684b773 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -46,6 +46,7 @@
 
                             <shopping-cart-product
                                 v-for="item in items"
+                                :cart="cart"
                                 :key="item['@id']"
                                 :product="item"
                             />
