diff --git a/assets/components/shopping-cart/cart-item.vue b/assets/components/shopping-cart/cart-item.vue
index 08d6a061..894ae725 100644
--- a/assets/components/shopping-cart/cart-item.vue
+++ b/assets/components/shopping-cart/cart-item.vue
@@ -28,7 +28,10 @@
         </div>
 
         <div class="col-3">
-            <button class="btn btn-info btn-sm">
+            <button
+                class="btn btn-info btn-sm"
+                @click="$emit('removeFromCart')"
+            >
                 Remove
             </button>
         </div>
diff --git a/assets/components/shopping-cart/index.vue b/assets/components/shopping-cart/index.vue
index 5ac82d83..c1b24513 100644
--- a/assets/components/shopping-cart/index.vue
+++ b/assets/components/shopping-cart/index.vue
@@ -27,6 +27,10 @@
                     colorId: item.color ? item.color['@id'] : null,
                     quantity: $event.quantity,
                 })"
+                @removeFromCart="$emit('removeFromCart', {
+                    productId: item.product['@id'],
+                    colorId: item.color ? item.color['@id'] : null,
+                })"
             />
 
             <div class="p-3">
