diff --git a/assets/components/shopping-cart/index.vue b/assets/components/shopping-cart/index.vue
index 2f9c697f..99a4721d 100644
--- a/assets/components/shopping-cart/index.vue
+++ b/assets/components/shopping-cart/index.vue
@@ -22,6 +22,7 @@
                 v-for="item in items"
                 :key="item.id"
                 :item="item"
+                @updateQuantity="$emit('updateQuantity', $event)"
             />
 
             <div class="p-3">
