diff --git a/assets/js/components/shopping-cart/cart-item.vue b/assets/js/components/shopping-cart/cart-item.vue
index fd18ca00..47aabc20 100644
--- a/assets/js/components/shopping-cart/cart-item.vue
+++ b/assets/js/components/shopping-cart/cart-item.vue
@@ -31,7 +31,14 @@ export default {
 <style lang="scss" module>
 @import '~styles/variables/colors.scss';
 
-.component {
+.component :global {
     border-bottom: 1px solid $light-component-border;
+
+    span.color-square {
+        display: inline-block;
+        width: 25px;
+        height: 25px;
+        border-radius: 4px;
+    }
 }
 </style>
