diff --git a/assets/js/components/shopping-cart/cart-item.vue b/assets/js/components/shopping-cart/cart-item.vue
index e1492be1..6e58fa20 100644
--- a/assets/js/components/shopping-cart/cart-item.vue
+++ b/assets/js/components/shopping-cart/cart-item.vue
@@ -103,5 +103,11 @@ export default {
     .product-name {
         flex-grow: 1;
     }
+
+    .price {
+        font-weight: bold;
+        width: 100px;
+        text-align: right;
+    }
 }
 </style>
