diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index 3f6a8fe86..76326ecfb 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -42,8 +42,12 @@ export default {
         };
     },
     watch: {
-        async cart() {
-            this.completeCart = await fetchFullCart(this.cart);
+        cart: {
+            deep: true,
+
+            async handler() {
+                this.completeCart = await fetchFullCart(this.cart);
+            },
         },
     },
     methods: {
