diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index 19058bb0..7143672c 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -78,6 +78,8 @@ export default {
          */
         async updateQuantity({ product, color, quantity }) {
             await updateCartItemQuantity(this.cart, product, color, quantity);
+
+            this.updateShoppingCartHeader();
         },
 
         /**
@@ -88,6 +90,8 @@ export default {
          */
         async removeFromCart({ product, color }) {
             await removeItemFromCart(this.cart, product, color);
+
+            this.updateShoppingCartHeader();
         },
     },
 };
