diff --git a/assets/js/components/shopping-cart/cart-item.vue b/assets/js/components/shopping-cart/cart-item.vue
index a8118ae4..84f30a0e 100644
--- a/assets/js/components/shopping-cart/cart-item.vue
+++ b/assets/js/components/shopping-cart/cart-item.vue
@@ -1,5 +1,5 @@
 <template>
-    <div :class="[$style.component, 'row', 'p-3']">
+    <div :class="[$style.component, 'row', 'p-3', 'align-items-center']">
         <div class="col-3">
             <div class="row">
                 <span class="col">
@@ -16,10 +16,30 @@
                 </span>
             </div>
         </div>
+
+        <div class="col-3">
+            <input
+                class="form-control"
+                type="number"
+                min="1"
+            >
+        </div>
+
+        <div class="col-3">
+            ${{ totalPrice }}
+        </div>
+
+        <div class="col-3">
+            <button class="btn btn-info btn-sm">
+                Remove
+            </button>
+        </div>
     </div>
 </template>
 
 <script>
+import formatPrice from '@/helpers/format-price';
+
 export default {
     name: 'ShoppingCartItem',
     props: {
@@ -28,6 +48,11 @@ export default {
             required: true,
         },
     },
+    computed: {
+        totalPrice() {
+            return formatPrice(this.cartItem.price * this.cartItem.qty);
+        },
+    },
 };
 </script>
 
@@ -43,5 +68,9 @@ export default {
         height: 25px;
         border-radius: 4px;
     }
+
+    input {
+        width: 60px;
+    }
 }
 </style>
