diff --git a/assets/js/components/shopping-cart/cart-item.vue b/assets/js/components/shopping-cart/cart-item.vue
index 0accf3fa..afe589f5 100644
--- a/assets/js/components/shopping-cart/cart-item.vue
+++ b/assets/js/components/shopping-cart/cart-item.vue
@@ -21,6 +21,7 @@
                 class="form-control"
                 type="number"
                 min="1"
+                @input="updateQty(product['@id'], product.colorId, product.qty)"
             >
         </span>
 
@@ -37,6 +38,7 @@
 </template>
 
 <script>
+import cartService from '@/services/cart-service';
 import formatPrice from '@/helpers/format-price';
 
 export default {
@@ -52,6 +54,18 @@ export default {
             return formatPrice(this.product.price * this.product.qty);
         },
     },
+    methods: {
+        /**
+         * Updates the product quantity in the cart, then refreshes the page
+         *
+         * @param {string} productId
+         * @param {string} colorId
+         * @param {number} qty
+         */
+        updateQty(productId, colorId, qty) {
+            cartService.updateQty(productId, colorId, qty);
+        },
+    },
 };
 </script>
 
