diff --git a/assets/js/components/shopping-cart/cart-item.vue b/assets/js/components/shopping-cart/cart-item.vue
index 7e8018dc..6564b4c1 100644
--- a/assets/js/components/shopping-cart/cart-item.vue
+++ b/assets/js/components/shopping-cart/cart-item.vue
@@ -1,6 +1,16 @@
 <template>
     <div :class="[$style.component, 'row', 'p-3']">
-        <span class="col-12">{{ product.name }}</span>
+        <span class="col-3 row">
+            <span class="col">
+                {{ product.name }}
+            </span>
+
+            <span class="col-2">
+                <span
+                    :class="$style['color-square']"
+                />
+            </span>
+        </span>
     </div>
 </template>
 
diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index ef455b4d..15f02e93 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -28,11 +28,8 @@
                             </div>
                         </div>
 
-                        <div
-                            v-show="!loading && items.length"
-                            class="row col-12"
-                        >
-                            <div class="row">
+                        <div v-show="!loading && items.length">
+                            <div class="row p-3">
                                 <span class="col-3">Item Name</span>
                                 <span class="col-3">Quantity</span>
                                 <span class="col-3">Price</span>
@@ -43,7 +40,6 @@
                                 v-for="item in items"
                                 :key="item['@id']"
                                 :product="item"
-                                class="row"
                             />
 
                             <div class="row p-3">
