diff --git a/assets/js/components/shopping-cart/cart-item.vue b/assets/js/components/shopping-cart/cart-item.vue
index 8fcacd7e..05db3f8f 100644
--- a/assets/js/components/shopping-cart/cart-item.vue
+++ b/assets/js/components/shopping-cart/cart-item.vue
@@ -16,6 +16,7 @@
             class="form-control"
             type="number"
             min="1"
+            @input="updateQty(product['@id'], product.colorId, product.qty)"
         >
 
         <span>{{ totalPrice }}</span>
@@ -29,6 +30,7 @@
 </template>
 
 <script>
+import cartService from '@/services/cart';
 import formatPrice from '@/helpers/format-price';
 
 export default {
@@ -44,6 +46,18 @@ export default {
             return formatPrice(this.product.price * this.product.qty);
         },
     },
+    methods: {
+        /**
+         * Updates the product quantity in the cart, then refreshes the page
+         *
+         * @param {string} productId
+         * @param {string} colorId
+         * @param {number} qty
+         */
+        updateQty(productId, colorId, qty) {
+            cartService.updateQty(productId, colorId, qty);
+        },
+    },
 };
 </script>
 
