diff --git a/assets/js/components/shopping-cart/product.vue b/assets/js/components/shopping-cart/product.vue
index b711d14..0b9c8c9 100644
--- a/assets/js/components/shopping-cart/product.vue
+++ b/assets/js/components/shopping-cart/product.vue
@@ -20,6 +20,7 @@
 
         <button
             class="btn btn-info btn-sm"
+            @click="updateQty(product['@id'], product.colorId, product.qty)"
         >
             Update
         </button>
@@ -33,6 +34,8 @@
 </template>
 
 <script>
+import cartService from '@/services/cart';
+
 export default {
     name: 'ShoppingCartProduct',
     props: {
@@ -42,6 +45,20 @@ export default {
             default: () => ({}),
         },
     },
+    methods: {
+        /**
+         * Updates the product quantity in the cart, then refreshes the page
+         *
+         * @param {string} productId
+         * @param {string} colorId
+         * @param {number} qty
+         */
+        updateQty(productId, colorId, qty) {
+            cartService.updateQty(productId, colorId, qty);
+
+            window.location.reload();
+        },
+    },
 };
 </script>
 
