diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index cb69bded6..20afaeb02 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -63,8 +63,12 @@ export default {
         },
     },
     watch: {
-        async cart() {
-            this.loadColorsAndProducts();
+        cart: {
+            deep: true,
+
+            async handler() {
+                this.loadColorsAndProducts();
+            },
         },
     },
     methods: {
