diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index e978ccd16..0e63b56ea 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -56,6 +56,8 @@ export default {
          */
         async updateQuantity({ product, color, quantity }) {
             await updateCartItemQuantity(this.cart, product, color, quantity);
+
+            this.updateShoppingCartHeader();
         },
 
         /**
@@ -66,6 +68,8 @@ export default {
          */
         async removeFromCart({ product, color }) {
             await removeItemFromCart(this.cart, product, color);
+
+            this.updateShoppingCartHeader();
         },
     },
 };
