diff --git a/assets/js/components/shopping-cart/cart-item.vue b/assets/js/components/shopping-cart/cart-item.vue
index d195b86b..c1381717 100644
--- a/assets/js/components/shopping-cart/cart-item.vue
+++ b/assets/js/components/shopping-cart/cart-item.vue
@@ -1,6 +1,6 @@
 <template>
-    <div :class="[$style.component, 'row', 'p-3']">
-        <span class="col-3 row">
+    <div :class="[$style.component, 'row', 'p-3', 'align-items-center']">
+        <span class="col-3 row align-items-center">
             <span class="col">
                 {{ product.name }}
             </span>
@@ -14,10 +14,30 @@
                 />
             </span>
         </span>
+
+        <span class="col-3">
+            <input
+                class="form-control"
+                type="number"
+                min="1"
+            >
+        </span>
+
+        <span class="col-3">
+            ${{ totalPrice }}
+        </span>
+
+        <span class="col-3">
+            <button class="btn btn-info btn-sm">
+                Remove
+            </button>
+        </span>
     </div>
 </template>
 
 <script>
+import formatPrice from '@/helpers/format-price';
+
 export default {
     name: 'ShoppingCartProduct',
     props: {
@@ -26,6 +46,11 @@ export default {
             required: true,
         },
     },
+    computed: {
+        totalPrice() {
+            return formatPrice(this.product.price * this.product.qty);
+        },
+    },
 };
 </script>
 
@@ -41,5 +66,9 @@ export default {
         height: 25px;
         border-radius: 4px;
     }
+
+    input {
+        width: 60px;
+    }
 }
 </style>
