diff --git a/assets/js/components/shopping-cart/product.vue b/assets/js/components/shopping-cart/product.vue
index 66a0d93..127190b 100644
--- a/assets/js/components/shopping-cart/product.vue
+++ b/assets/js/components/shopping-cart/product.vue
@@ -3,6 +3,10 @@
         :class="$style.component"
     >
         <span :class="$style['product-name']">{{ product.name }}</span>
+
+        <span
+            :class="$style['color-square']"
+        />
     </div>
 </template>
 
