diff --git a/assets/components/shopping-cart/cart-item.vue b/assets/components/shopping-cart/cart-item.vue
index 401692a51..f7789e854 100644
--- a/assets/components/shopping-cart/cart-item.vue
+++ b/assets/components/shopping-cart/cart-item.vue
@@ -23,6 +23,7 @@
                 class="form-control"
                 type="number"
                 min="1"
+                @input="updateQuantity"
             />
         </div>
 
@@ -57,6 +58,15 @@ export default {
             return formatPrice(this.item.product.price * this.item.quantity);
         },
     },
+    methods: {
+        updateQuantity(event) {
+            this.$emit('updateQuantity', {
+                productId: this.item.product['@id'],
+                colorId: this.item.color ? this.item.color['@id'] : null,
+                quantity: event.target.value,
+            });
+        },
+    },
 };
 </script>
 
