diff --git a/assets/js/components/shopping-cart/cart-item.vue b/assets/js/components/shopping-cart/cart-item.vue
index a149d93d..86841cb9 100644
--- a/assets/js/components/shopping-cart/cart-item.vue
+++ b/assets/js/components/shopping-cart/cart-item.vue
@@ -27,6 +27,7 @@
 
         <button
             class="btn btn-info btn-sm"
+            @click="removeFromCart(product['@id'], product.colorId)"
         >
             Remove
         </button>
@@ -57,6 +58,18 @@ export default {
 
             window.location.reload();
         },
+
+        /**
+         * Removes a product from the cart, then refreshes the page
+         *
+         * @param {string} productId
+         * @param {string|null} colorId
+         */
+        removeFromCart(productId, colorId) {
+            cartService.removeItem(productId, colorId);
+
+            window.location.reload();
+        },
     },
 };
 </script>
