diff --git a/assets/js/components/shopping-cart/cart-item.vue b/assets/js/components/shopping-cart/cart-item.vue
index 51cced5c..febb9a1c 100644
--- a/assets/js/components/shopping-cart/cart-item.vue
+++ b/assets/js/components/shopping-cart/cart-item.vue
@@ -23,6 +23,7 @@
                 class="form-control"
                 type="number"
                 min="1"
+                @input="updateQty(product['@id'], product.colorId, product.qty)"
             >
         </div>
 
@@ -39,11 +40,16 @@
 </template>
 
 <script>
+import { updateCartItemQuantity } from '@/services/cart-service';
 import formatPrice from '@/helpers/format-price';
 
 export default {
     name: 'ShoppingCartItem',
     props: {
+        cart: {
+            type: Object,
+            required: true,
+        },
         product: {
             type: Object,
             required: true,
@@ -54,6 +60,19 @@ export default {
             return formatPrice(this.product.price * this.product.qty);
         },
     },
+    methods: {
+        /**
+         * Updates the product quantity in the cart, then refreshes the page
+         *
+         * @param {string} productId
+         * @param {string|null} colorId
+         * @param {number} qty
+         */
+        async updateQty(productId, colorId, qty) {
+            await updateCartItemQuantity(this.cart, productId, colorId, qty);
+            window.location.reload();
+        },
+    },
 };
 </script>
 
diff --git a/assets/js/components/shopping-cart/index.vue b/assets/js/components/shopping-cart/index.vue
index c425f197..944b668f 100644
--- a/assets/js/components/shopping-cart/index.vue
+++ b/assets/js/components/shopping-cart/index.vue
@@ -26,6 +26,7 @@
             <shopping-cart-item
                 v-for="item in items"
                 :key="item['@id']"
+                :cart="cart"
                 :product="item"
             />
 
@@ -48,6 +49,10 @@ export default {
         ShoppingCartItem,
     },
     props: {
+        cart: {
+            type: Object,
+            required: true,
+        },
         items: {
             type: Array,
             required: true,
diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index 22237090..93958f9a 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -22,6 +22,7 @@
                         <shopping-cart-list
                             v-show="!loading"
                             :items="items"
+                            :cart="cart"
                         />
                     </div>
                 </div>
