diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index e39b5a3d2..18b81f4cc 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -12,6 +12,7 @@
                     <shopping-cart-list
                         v-if="completeCart"
                         :items="completeCart.items"
+                        @updateQuantity="updateQuantity"
                     />
                 </div>
             </div>
@@ -20,7 +21,7 @@
 </template>
 
 <script>
-import { fetchFullCart } from '@/services/cart-service';
+import { fetchFullCart, updateCartItemQuantity } from '@/services/cart-service';
 import ShoppingCartMixin from '@/mixins/get-shopping-cart';
 import TitleComponent from '@/components/title';
 import ShoppingCartList from '@/components/shopping-cart';
@@ -44,6 +45,11 @@ export default {
             this.completeCart = await fetchFullCart(this.cart);
         },
     },
+    methods: {
+        updateQuantity({ productId, colorId, quantity }) {
+            updateCartItemQuantity(this.cart, productId, colorId, quantity);
+        },
+    },
 };
 </script>
 
