diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index 34cc71a86..540da2149 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -11,6 +11,7 @@
                 <shopping-cart-list
                     v-show="!loading"
                     :items="items"
+                    @updateQuantity="updateQuantity"
                 />
             </div>
         </div>
@@ -18,7 +19,7 @@
 </template>
 
 <script>
-import { getFullShoppingCart } from '@/services/cart-service';
+import { getFullShoppingCart, updateCartItemQuantity } from '@/services/cart-service';
 import ShoppingCartMixin from '@/mixins/get-shopping-cart';
 import TitleComponent from '@/components/title';
 import ShoppingCartList from '@/components/shopping-cart';
@@ -57,6 +58,18 @@ export default {
             this.itemsLoading = false;
         },
     },
+    methods: {
+        /**
+         * Updates the product quantity in the cart, then refreshes the page
+         *
+         * @param {string} product
+         * @param {string|null} color
+         * @param {number} quantity
+         */
+        async updateQuantity({ product, color, quantity }) {
+            await updateCartItemQuantity(this.cart, product, color, quantity);
+        },
+    },
 };
 </script>
 
