diff --git a/assets/js/components/shopping-cart/cart-item.vue b/assets/js/components/shopping-cart/cart-item.vue
index ca1ce3d2..104973b6 100644
--- a/assets/js/components/shopping-cart/cart-item.vue
+++ b/assets/js/components/shopping-cart/cart-item.vue
@@ -3,6 +3,10 @@
         :class="$style.component"
     >
         <span>{{ product.name }}</span>
+
+        <span
+            :class="$style['color-square']"
+        />
     </div>
 </template>
 
