diff --git a/assets/components/shopping-cart/cart-item.vue b/assets/components/shopping-cart/cart-item.vue
index bc010bb64..fc9add6e0 100644
--- a/assets/components/shopping-cart/cart-item.vue
+++ b/assets/components/shopping-cart/cart-item.vue
@@ -19,7 +19,7 @@
 
         <div class="col-3">
             <input
-                v-model.number="item.quantity"
+                :value="item.quantity"
                 class="form-control"
                 type="number"
                 min="1"
