diff --git a/assets/components/shopping-cart/cart-item.vue b/assets/components/shopping-cart/cart-item.vue
index 7202bc397..1ccb75aae 100644
--- a/assets/components/shopping-cart/cart-item.vue
+++ b/assets/components/shopping-cart/cart-item.vue
@@ -19,7 +19,7 @@
 
         <div class="col-3">
             <input
-                v-model="item.quantity"
+                v-model.number="item.quantity"
                 class="form-control"
                 type="number"
                 min="1"
