diff --git a/assets/js/services/cart.js b/assets/js/services/cart.js
index 6788fec..b1d41f5 100644
--- a/assets/js/services/cart.js
+++ b/assets/js/services/cart.js
@@ -52,6 +52,27 @@ const cart = {
         );
     },
 
+    /**
+     * Upodates the qty property on a product in the cart
+     *
+     * @param {string} productId
+     * @param {string|null} colorId
+     * @param {number} qty
+     */
+    updateQty(productId, colorId, qty) {
+        const storage = cart.getStorage();
+
+        const cartItemIndex = storage.findIndex(
+            (item) => (item.productId === productId && item.colorId === colorId),
+        );
+
+        if (cartItemIndex > -1) {
+            storage[cartItemIndex].qty = qty;
+        }
+
+        cart.saveStorage(storage);
+    },
+
     /**
      * Clears the shopping cart
      */
