diff --git a/assets/js/mixins/get-shopping-cart.js b/assets/js/mixins/get-shopping-cart.js
index 1138565e..b8b7a81e 100644
--- a/assets/js/mixins/get-shopping-cart.js
+++ b/assets/js/mixins/get-shopping-cart.js
@@ -3,7 +3,9 @@ import { getCart, getCartTotalItems } from '@/services/cart-service';
 export default {
     data() {
         return {
-            cart: null,
+            cart: {
+                items: [],
+            },
         };
     },
     async mounted() {
diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index 654c5a02..19058bb0 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -50,7 +50,7 @@ export default {
         };
     },
     watch: {
-        async cart() {
+        'cart.items.length': async function watchCartItemsLength() {
             this.loading = true;
             this.items = [];
 
