diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index 20afaeb02..f17907658 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -63,12 +63,8 @@ export default {
         },
     },
     watch: {
-        cart: {
-            deep: true,
-
-            async handler() {
-                this.loadColorsAndProducts();
-            },
+        'cart.items.length': async function watchCartItemsLength() {
+            this.loadColorsAndProducts();
         },
     },
     methods: {
