diff --git a/assets/mixins/get-shopping-cart.js b/assets/mixins/get-shopping-cart.js
index 68f11e6f1..d0dadbda8 100644
--- a/assets/mixins/get-shopping-cart.js
+++ b/assets/mixins/get-shopping-cart.js
@@ -3,7 +3,9 @@ import { addItemToCart, fetchCart, getCartTotalItems } from '@/services/cart-ser
 export default {
     data() {
         return {
-            cart: null,
+            cart: {
+                items: [],
+            },
             addToCartLoading: false,
             addToCartSuccess: false,
         };
diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index 00c445fff..e978ccd16 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -42,7 +42,7 @@ export default {
         };
     },
     watch: {
-        async cart() {
+        'cart.items.length': async function watchCartItemsLength() {
             this.completeCart = await fetchFullCart(this.cart);
         },
     },
