diff --git a/assets/components/shopping-cart/cart-item.vue b/assets/components/shopping-cart/cart-item.vue
index f99238fe8..344b8f4d7 100644
--- a/assets/components/shopping-cart/cart-item.vue
+++ b/assets/components/shopping-cart/cart-item.vue
@@ -12,6 +12,15 @@
                 }"
             />
         </div>
+
+        <div class="col-3">
+            <input
+                v-model.number="item.quantity"
+                class="form-control"
+                type="number"
+                min="1"
+            />
+        </div>
     </div>
 </template>
 
@@ -44,5 +53,9 @@ export default {
         height: 25px;
         border-radius: 4px;
     }
+
+    input {
+        width: 60px;
+    }
 }
 </style>
