diff --git a/assets/js/components/shopping-cart/cart-item.vue b/assets/js/components/shopping-cart/cart-item.vue
index ad5ed92a..8fcacd7e 100644
--- a/assets/js/components/shopping-cart/cart-item.vue
+++ b/assets/js/components/shopping-cart/cart-item.vue
@@ -12,6 +12,7 @@
         />
 
         <input
+            v-model.number="product.qty"
             class="form-control"
             type="number"
             min="1"
