diff --git a/assets/mixins/get-shopping-cart.js b/assets/mixins/get-shopping-cart.js
index 1761d4fbd..eaf6cb097 100644
--- a/assets/mixins/get-shopping-cart.js
+++ b/assets/mixins/get-shopping-cart.js
@@ -2,6 +2,7 @@ import {
     addItemToCart,
     fetchCart,
     getCartTotalItems,
+    removeItemFromCart,
     updateCartItemQuantity,
 } from '@/services/cart-service';
 
@@ -42,6 +43,12 @@ export default {
             this.updateCartHeaderTotal();
         },
 
+        async removeProductFromCart(productId, colorId) {
+            await removeItemFromCart(this.cart, productId, colorId);
+
+            this.updateCartHeaderTotal();
+        },
+
         updateCartHeaderTotal() {
             document.getElementById('js-shopping-cart-items')
                 .innerHTML = getCartTotalItems(this.cart).toString();
diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index 1dbcb7b0c..cb69bded6 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -13,6 +13,7 @@
                         v-if="completeCart"
                         :items="completeCart.items"
                         @updateQuantity="updateQuantity"
+                        @removeFromCart="removeFromCart"
                     />
                 </div>
             </div>
@@ -87,6 +88,10 @@ export default {
         updateQuantity({ productId, colorId, quantity }) {
             this.updateProductQuantity(productId, colorId, quantity);
         },
+
+        removeFromCart({ productId, colorId }) {
+            this.removeProductFromCart(productId, colorId);
+        },
     },
 };
 </script>
