diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index 1a96a4800..f9a1e796e 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -12,6 +12,7 @@
                     <shopping-cart-list
                         v-if="completeCart"
                         :items="completeCart.items"
+                        @updateQuantity="updateQuantity"
                     />
                 </div>
             </div>
@@ -20,6 +21,7 @@
 </template>
 
 <script>
+import { updateCartItemQuantity } from '@/services/cart-service';
 import { fetchColors } from '@/services/colors-service';
 import { fetchProductsById } from '@/services/products-service';
 import ShoppingCartMixin from '@/mixins/get-shopping-cart';
@@ -77,6 +79,10 @@ export default {
             this.products = productsResponse.data['hydra:member'];
             this.colors = colorsResponse.data['hydra:member'];
         },
+
+        updateQuantity({ productId, colorId, quantity }) {
+            updateCartItemQuantity(this.cart, productId, colorId, quantity);
+        },
     },
 };
 </script>
