diff --git a/assets/js/components/shopping-cart/cart-item.vue b/assets/js/components/shopping-cart/cart-item.vue
index 716d9370..2945e782 100644
--- a/assets/js/components/shopping-cart/cart-item.vue
+++ b/assets/js/components/shopping-cart/cart-item.vue
@@ -10,7 +10,7 @@
                     <span
                         class="color-square"
                         :style="{
-                            backgroundColor: `#${product.hexColor}`
+                            backgroundColor: `#${cartItem.hexColor}`
                         }"
                     />
                 </span>
@@ -19,10 +19,11 @@
 
         <div class="col-3">
             <input
-                v-model.number="product.qty"
+                v-model.number="cartItem.qty"
                 class="form-control"
                 type="number"
                 min="1"
+                @input="updateQty(cartItem['@id'], cartItem.colorId, cartItem.qty)"
             >
         </div>
 
@@ -39,11 +40,16 @@
 </template>
 
 <script>
+import { updateCartItemQuantity } from '@/services/cart-service';
 import formatPrice from '@/helpers/format-price';
 
 export default {
     name: 'ShoppingCartItem',
     props: {
+        cart: {
+            type: Object,
+            required: true,
+        },
         cartItem: {
             type: Object,
             required: true,
@@ -54,6 +60,19 @@ export default {
             return formatPrice(this.product.price * this.product.qty);
         },
     },
+    methods: {
+        /**
+         * Updates the product quantity in the cart, then refreshes the page
+         *
+         * @param {string} productId
+         * @param {string|null} colorId
+         * @param {number} qty
+         */
+        async updateQty(productId, colorId, qty) {
+            await updateCartItemQuantity(this.cart, productId, colorId, qty);
+            window.location.reload();
+        },
+    },
 };
 </script>
 
diff --git a/assets/js/components/shopping-cart/index.vue b/assets/js/components/shopping-cart/index.vue
index 37939f3f..cbf785f2 100644
--- a/assets/js/components/shopping-cart/index.vue
+++ b/assets/js/components/shopping-cart/index.vue
@@ -24,6 +24,7 @@
             <shopping-cart-item
                 v-for="item in items"
                 :key="item['@id']"
+                :cart="cart"
                 :cart-item="item"
             />
 
@@ -44,6 +45,10 @@ export default {
         ShoppingCartItem,
     },
     props: {
+        cart: {
+            type: Object,
+            required: true,
+        },
         items: {
             type: Array,
             required: true,
diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index 9cb70843..5d2d1571 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -19,6 +19,7 @@
                     <shopping-cart-list
                         v-show="!loading"
                         :items="items"
+                        :cart="cart"
                     />
                 </div>
             </div>
