diff --git a/assets/mixins/get-shopping-cart.js b/assets/mixins/get-shopping-cart.js
index 93406f27a..9c0a9c61a 100644
--- a/assets/mixins/get-shopping-cart.js
+++ b/assets/mixins/get-shopping-cart.js
@@ -3,6 +3,7 @@ import { addItemToCart, getCart, getCartTotalItems } from '@/services/cart-servi
 export default {
     data() {
         return {
+            /** @type CartCollection */
             cart: {
                 items: [],
             },
@@ -30,6 +31,10 @@ export default {
             this.addToCartLoading = false;
             this.addToCartSuccess = true;
 
+            this.updateShoppingCartHeader();
+        },
+
+        updateShoppingCartHeader() {
             document.getElementById('js-shopping-cart-items')
                 .innerHTML = getCartTotalItems(this.cart).toString();
         },
