diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index 9cb70843..6d95ba1d 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -19,6 +19,7 @@
                     <shopping-cart-list
                         v-show="!loading"
                         :items="items"
+                        @updateQuantity="updateQuantity"
                     />
                 </div>
             </div>
@@ -27,7 +28,7 @@
 </template>
 
 <script>
-import { getFullShoppingCart } from '@/services/cart-service';
+import { getFullShoppingCart, updateCartItemQuantity } from '@/services/cart-service';
 import Loading from '@/components/loading';
 import TitleComponent from '@/components/title';
 import ShoppingCartList from '@/components/shopping-cart';
@@ -67,6 +68,18 @@ export default {
             this.loading = false;
         },
     },
+    methods: {
+        /**
+         * Updates the product quantity in the cart, then refreshes the page
+         *
+         * @param {string} product
+         * @param {string|null} color
+         * @param {number} quantity
+         */
+        async updateQuantity({ product, color, quantity }) {
+            await updateCartItemQuantity(this.cart, product, color, quantity);
+        },
+    },
 };
 </script>
 
