diff --git a/assets/components/shopping-cart/cart-item.vue b/assets/components/shopping-cart/cart-item.vue
index 35a8083a1..e0075323c 100644
--- a/assets/components/shopping-cart/cart-item.vue
+++ b/assets/components/shopping-cart/cart-item.vue
@@ -23,6 +23,7 @@
                 class="form-control"
                 type="number"
                 min="1"
+                @input="updateQuantity"
             >
         </div>
 
@@ -31,7 +32,13 @@
         </div>
 
         <div class="col-3">
-            <button class="btn btn-info btn-sm">
+            <button
+                class="btn btn-info btn-sm"
+                @click="$emit('removeFromCart', {
+                    productId: item.product['@id'],
+                    colorId: item.color ? item.color['@id'] : null
+                })"
+            >
                 Remove
             </button>
         </div>
@@ -57,6 +64,15 @@ export default {
             return formatPrice(this.item.product.price * this.item.quantity);
         },
     },
+    methods: {
+        updateQuantity(event) {
+            this.$emit('updateQuantity', {
+                productId: this.item.product['@id'],
+                colorId: this.item.color ? this.item.color['@id'] : null,
+                quantity: parseInt(event.target.value, 10),
+            });
+        },
+    },
 };
 </script>
 
