diff --git a/assets/components/shopping-cart/cart-item.vue b/assets/components/shopping-cart/cart-item.vue
index 7defcd57..3003b989 100644
--- a/assets/components/shopping-cart/cart-item.vue
+++ b/assets/components/shopping-cart/cart-item.vue
@@ -1,6 +1,14 @@
 <template>
     <div :class="[$style.component, 'row', 'p-3']">
-        {{ item.product.name }}
+        <div class="col-2">
+            {{ item.product.name }}
+        </div>
+
+        <div class="col-1">
+            <span
+                class="color-square"
+            />
+        </div>
     </div>
 </template>
 
