diff --git a/assets/js/components/shopping-cart/cart-item.vue b/assets/js/components/shopping-cart/cart-item.vue
index 14c16636..a0fcead0 100644
--- a/assets/js/components/shopping-cart/cart-item.vue
+++ b/assets/js/components/shopping-cart/cart-item.vue
@@ -17,6 +17,7 @@
 
         <span class="col-3">
             <input
+                v-model.number="product.qty"
                 class="form-control"
                 type="number"
                 min="1"
