diff --git a/assets/js/components/shopping-cart/product.vue b/assets/js/components/shopping-cart/product.vue
index 494f55fc..b711d14e 100644
--- a/assets/js/components/shopping-cart/product.vue
+++ b/assets/js/components/shopping-cart/product.vue
@@ -12,6 +12,7 @@
         />
 
         <input
+            v-model.number="product.qty"
             class="form-control"
             type="number"
             min="1"
