diff --git a/assets/js/components/shopping-cart/cart-item.vue b/assets/js/components/shopping-cart/cart-item.vue
index 0c647534..ad5ed92a 100644
--- a/assets/js/components/shopping-cart/cart-item.vue
+++ b/assets/js/components/shopping-cart/cart-item.vue
@@ -10,10 +10,26 @@
                 backgroundColor: `#${product.hexColor}`
             }"
         />
+
+        <input
+            class="form-control"
+            type="number"
+            min="1"
+        >
+
+        <span>{{ totalPrice }}</span>
+
+        <button
+            class="btn btn-info btn-sm"
+        >
+            Remove
+        </button>
     </div>
 </template>
 
 <script>
+import formatPrice from '@/helpers/format-price';
+
 export default {
     name: 'ShoppingCartProduct',
     props: {
@@ -22,6 +38,11 @@ export default {
             required: true,
         },
     },
+    computed: {
+        totalPrice() {
+            return formatPrice(this.product.price * this.product.qty);
+        },
+    },
 };
 </script>
 
@@ -47,6 +68,10 @@ export default {
         border-radius: 4px;
     }
 
+    input {
+        width: 60px;
+    }
+
     .product-name {
         flex-grow: 1;
     }
