diff --git a/assets/components/shopping-cart/cart-item.vue b/assets/components/shopping-cart/cart-item.vue
index 41ae3a3c0..5fd02317e 100644
--- a/assets/components/shopping-cart/cart-item.vue
+++ b/assets/components/shopping-cart/cart-item.vue
@@ -23,6 +23,11 @@
                 class="form-control"
                 type="number"
                 min="1"
+                @input="$emit('updateQuantity', {
+                    productId: item.product['@id'],
+                    colorId: item.color['@id'],
+                    quantity: item.quantity
+                })"
             >
         </div>
 
@@ -31,7 +36,13 @@
         </div>
 
         <div class="col-3">
-            <button class="btn btn-info btn-sm">
+            <button
+                class="btn btn-info btn-sm"
+                @click="$emit('removeFromCart', {
+                    productId: item.product['@id'],
+                    colorId: item.color['@id']
+                })"
+            >
                 Remove
             </button>
         </div>
