diff --git a/assets/js/components/shopping-cart/cart-item.vue b/assets/js/components/shopping-cart/cart-item.vue
index 05db3f8f..e1492be1 100644
--- a/assets/js/components/shopping-cart/cart-item.vue
+++ b/assets/js/components/shopping-cart/cart-item.vue
@@ -23,6 +23,7 @@
 
         <button
             class="btn btn-info btn-sm"
+            @click="removeFromCart(product['@id'], product.colorId)"
         >
             Remove
         </button>
@@ -57,6 +58,18 @@ export default {
         updateQty(productId, colorId, qty) {
             cartService.updateQty(productId, colorId, qty);
         },
+
+        /**
+         * Removes a product from the cart, then refreshes the page
+         *
+         * @param {string} productId
+         * @param {string|null} colorId
+         */
+        removeFromCart(productId, colorId) {
+            cartService.removeItem(productId, colorId);
+
+            window.location.reload();
+        },
     },
 };
 </script>
