diff --git a/assets/components/shopping-cart/cart-item.vue b/assets/components/shopping-cart/cart-item.vue
index 848c82f8..08d6a061 100644
--- a/assets/components/shopping-cart/cart-item.vue
+++ b/assets/components/shopping-cart/cart-item.vue
@@ -57,8 +57,6 @@ export default {
     methods: {
         updateQuantity(event) {
             this.$emit('updateQuantity', {
-                productId: this.item.product['@id'],
-                colorId: this.item.color ? this.item.color['@id'] : null,
                 quantity: parseFloat(event.target.value),
             });
         },
diff --git a/assets/components/shopping-cart/index.vue b/assets/components/shopping-cart/index.vue
index 99a4721d..5ac82d83 100644
--- a/assets/components/shopping-cart/index.vue
+++ b/assets/components/shopping-cart/index.vue
@@ -22,7 +22,11 @@
                 v-for="item in items"
                 :key="item.id"
                 :item="item"
-                @updateQuantity="$emit('updateQuantity', $event)"
+                @updateQuantity="$emit('updateQuantity', {
+                    productId: item.product['@id'],
+                    colorId: item.color ? item.color['@id'] : null,
+                    quantity: $event.quantity,
+                })"
             />
 
             <div class="p-3">
