diff --git a/assets/js/components/shopping-cart/cart-item.vue b/assets/js/components/shopping-cart/cart-item.vue
index f3e91f09..ec9090c9 100644
--- a/assets/js/components/shopping-cart/cart-item.vue
+++ b/assets/js/components/shopping-cart/cart-item.vue
@@ -30,7 +30,10 @@
         </span>
 
         <span class="col-3">
-            <button class="btn btn-info btn-sm">
+            <button
+                class="btn btn-info btn-sm"
+                @click="removeFromCart(product['@id'], product.colorId)"
+            >
                 Remove
             </button>
         </span>
@@ -65,6 +68,18 @@ export default {
         updateQty(productId, colorId, qty) {
             cartService.updateQty(productId, colorId, qty);
         },
+
+        /**
+         * Removes a product from the cart, then refreshes the page
+         *
+         * @param {string} productId
+         * @param {string|null} colorId
+         */
+        removeFromCart(productId, colorId) {
+            cartService.removeItem(productId, colorId);
+
+            window.location.reload();
+        },
     },
 };
 </script>
