diff --git a/assets/js/components/shopping-cart/product.vue b/assets/js/components/shopping-cart/product.vue
index 86005f4..c243697 100644
--- a/assets/js/components/shopping-cart/product.vue
+++ b/assets/js/components/shopping-cart/product.vue
@@ -31,11 +31,16 @@
         >
             Remove
         </button>
+
+        <span :class="$style['price']">
+            ${{ getProductPrice(product.price, product.qty) }}
+        </span>
     </div>
 </template>
 
 <script>
 import cartService from '@/services/cart';
+import formatPrice from '@/helpers/format-price';
 
 export default {
     name: 'ShoppingCartProduct',
@@ -47,6 +52,17 @@ export default {
         },
     },
     methods: {
+        /**
+         * Returns the formatted product price
+         *
+         * @param {number} price
+         * @param {number} qty
+         * @return {string}
+         */
+        getProductPrice(price, qty) {
+            return formatPrice(price * qty);
+        },
+
         /**
          * Updates the product quantity in the cart, then refreshes the page
          *
@@ -104,5 +120,11 @@ export default {
     .product-name {
         flex-grow: 1;
     }
+
+    .price {
+        font-weight: bold;
+        width: 100px;
+        text-align: right;
+    }
 }
 </style>
