diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index fadb1056e..3dd3c5ed2 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -12,6 +12,7 @@
                     v-show="!loading"
                     :items="items"
                     @updateQuantity="updateQuantity"
+                    @removeFromCart="removeFromCart"
                 />
             </div>
         </div>
@@ -19,7 +20,7 @@
 </template>
 
 <script>
-import { getFullShoppingCart, updateCartItemQuantity } from '@/services/cart-service';
+    import { getFullShoppingCart, updateCartItemQuantity, removeItemFromCart } from '@/services/cart-service';
 import ShoppingCartMixin from '@/mixins/get-shopping-cart';
 import TitleComponent from '@/components/title';
 import ShoppingCartList from '@/components/shopping-cart';
@@ -60,6 +61,16 @@ export default {
         async updateQuantity({ product, color, quantity }) {
             await updateCartItemQuantity(this.cart, product, color, quantity);
         },
+
+        /**
+         * Removes a product from the cart, then refreshes the page
+         *
+         * @param {string} product
+         * @param {string|null} color
+         */
+        async removeFromCart({ product, color }) {
+            await removeItemFromCart(this.cart, product, color);
+        },
     },
 };
 </script>
