diff --git a/assets/components/shopping-cart/cart-item.vue b/assets/components/shopping-cart/cart-item.vue
index 3c0669e05..35a8083a1 100644
--- a/assets/components/shopping-cart/cart-item.vue
+++ b/assets/components/shopping-cart/cart-item.vue
@@ -29,6 +29,12 @@
         <div class="col-3">
             ${{ totalPrice }}
         </div>
+
+        <div class="col-3">
+            <button class="btn btn-info btn-sm">
+                Remove
+            </button>
+        </div>
     </div>
 </template>
 
