diff --git a/assets/js/components/shopping-cart/cart-item.vue b/assets/js/components/shopping-cart/cart-item.vue
index 2945e782..2d8cfeef 100644
--- a/assets/js/components/shopping-cart/cart-item.vue
+++ b/assets/js/components/shopping-cart/cart-item.vue
@@ -32,7 +32,10 @@
         </div>
 
         <div class="col-3">
-            <button class="btn btn-info btn-sm">
+            <button
+                class="btn btn-info btn-sm"
+                @click="removeFromCart(product['@id'], product.colorId)"
+            >
                 Remove
             </button>
         </div>
@@ -40,7 +43,7 @@
 </template>
 
 <script>
-import { updateCartItemQuantity } from '@/services/cart-service';
+import { updateCartItemQuantity, removeItemFromCart } from '@/services/cart-service';
 import formatPrice from '@/helpers/format-price';
 
 export default {
@@ -72,6 +75,18 @@ export default {
             await updateCartItemQuantity(this.cart, productId, colorId, qty);
             window.location.reload();
         },
+
+        /**
+         * Removes a product from the cart, then refreshes the page
+         *
+         * @param {string} productId
+         * @param {string|null} colorId
+         */
+        removeFromCart(productId, colorId) {
+            removeItemFromCart(this.cart, productId, colorId);
+
+            window.location.reload();
+        },
     },
 };
 </script>
