diff --git a/assets/js/components/shopping-cart/cart-item.vue b/assets/js/components/shopping-cart/cart-item.vue
index 926dbc26..5191553a 100644
--- a/assets/js/components/shopping-cart/cart-item.vue
+++ b/assets/js/components/shopping-cart/cart-item.vue
@@ -19,6 +19,7 @@
 
         <div class="col-3">
             <input
+                v-model.number="product.qty"
                 class="form-control"
                 type="number"
                 min="1"
