diff --git a/assets/components/shopping-cart/cart-item.vue b/assets/components/shopping-cart/cart-item.vue
index e6efba24..f99238fe 100644
--- a/assets/components/shopping-cart/cart-item.vue
+++ b/assets/components/shopping-cart/cart-item.vue
@@ -7,6 +7,9 @@
         <div class="col-1">
             <span
                 class="color-square"
+                :style="{
+                    backgroundColor: `#${hexColor}`
+                }"
             />
         </div>
     </div>
@@ -21,6 +24,11 @@ export default {
             required: true,
         },
     },
+    computed: {
+        hexColor() {
+            return this.item.color ? this.item.color.hexColor : 'fff';
+        },
+    },
 };
 </script>
 
