diff --git a/assets/js/components/shopping-cart/cart-item.vue b/assets/js/components/shopping-cart/cart-item.vue
index 1612e8f2..a8118ae4 100644
--- a/assets/js/components/shopping-cart/cart-item.vue
+++ b/assets/js/components/shopping-cart/cart-item.vue
@@ -9,6 +9,9 @@
                 <span class="col-2">
                     <span
                         class="color-square"
+                        :style="{
+                            backgroundColor: `#${cartItem.hexColor}`
+                        }"
                     />
                 </span>
             </div>
