diff --git a/assets/js/components/shopping-cart/index.vue b/assets/js/components/shopping-cart/index.vue
index 37939f3f..baf2d3b3 100644
--- a/assets/js/components/shopping-cart/index.vue
+++ b/assets/js/components/shopping-cart/index.vue
@@ -25,6 +25,8 @@
                 v-for="item in items"
                 :key="item['@id']"
                 :cart-item="item"
+                @updateQuantity="(data) => $emit('updateQuantity', data)"
+                @removeFromCart="(data) => $emit('removeFromCart', data)"
             />
 
             <div class="p-3">
