diff --git a/assets/components/shopping-cart/cart-item.vue b/assets/components/shopping-cart/cart-item.vue
index 357170463..9068bd052 100644
--- a/assets/components/shopping-cart/cart-item.vue
+++ b/assets/components/shopping-cart/cart-item.vue
@@ -21,10 +21,16 @@
                 min="1"
             />
         </div>
+
+        <div class="col-3">
+            ${{ totalPrice }}
+        </div>
     </div>
 </template>
 
 <script>
+import formatPrice from '@/helpers/format-price';
+
 export default {
     name: 'ShoppingCartItem',
     props: {
@@ -37,6 +43,9 @@ export default {
         hexColor() {
             return this.item.color ? this.item.color.hexColor : 'fff';
         },
+        totalPrice() {
+            return formatPrice(this.item.product.price * this.item.quantity);
+        },
     },
 };
 </script>
