diff --git a/assets/components/shopping-cart/cart-item.vue b/assets/components/shopping-cart/cart-item.vue
index 67b9b4d9e..848c82f8a 100644
--- a/assets/components/shopping-cart/cart-item.vue
+++ b/assets/components/shopping-cart/cart-item.vue
@@ -59,7 +59,7 @@ export default {
             this.$emit('updateQuantity', {
                 productId: this.item.product['@id'],
                 colorId: this.item.color ? this.item.color['@id'] : null,
-                quantity: event.target.value,
+                quantity: parseFloat(event.target.value),
             });
         },
     },
