diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index 6d95ba1d..00b06d0f 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -20,6 +20,7 @@
                         v-show="!loading"
                         :items="items"
                         @updateQuantity="updateQuantity"
+                        @removeFromCart="removeFromCart"
                     />
                 </div>
             </div>
@@ -28,7 +29,7 @@
 </template>
 
 <script>
-import { getFullShoppingCart, updateCartItemQuantity } from '@/services/cart-service';
+import { getFullShoppingCart, updateCartItemQuantity, removeItemFromCart } from '@/services/cart-service';
 import Loading from '@/components/loading';
 import TitleComponent from '@/components/title';
 import ShoppingCartList from '@/components/shopping-cart';
@@ -79,6 +80,16 @@ export default {
         async updateQuantity({ product, color, quantity }) {
             await updateCartItemQuantity(this.cart, product, color, quantity);
         },
+
+        /**
+         * Removes a product from the cart, then refreshes the page
+         *
+         * @param {string} product
+         * @param {string|null} color
+         */
+        async removeFromCart({ product, color }) {
+            await removeItemFromCart(this.cart, product, color);
+        },
     },
 };
 </script>
