diff --git a/templates/cart/cart.html.twig b/templates/cart/cart.html.twig
index 261522de..47b78ad0 100644
--- a/templates/cart/cart.html.twig
+++ b/templates/cart/cart.html.twig
@@ -1,5 +1,33 @@
 {% extends 'base.html.twig' %}
 
 {% block body %}
-    <h1>TODO - put a cart here!</h1>
+    <div id="app"></div>
+{% endblock %}
+
+{% block stylesheets %}
+    {{ parent() }}
+
+    {{ encore_entry_link_tags('products') }}
+{% endblock %}
+
+{% block javascripts %}
+    {{ parent() }}
+
+    <script>
+        {% if currentCategoryId is defined %}
+            window.currentCategoryId = '{{ currentCategoryId|e('js') }}';
+        {% else %}
+            window.currentCategoryId = null;
+        {% endif %}
+
+        {% if currentProductId is defined %}
+            window.currentProductId = '{{ currentProductId|e('js') }}';
+        {% else %}
+            window.currentProductId = null;
+        {% endif %}
+
+        window.categories = {{ categories|jsonld }};
+    </script>
+
+    {{ encore_entry_script_tags('products') }}
 {% endblock %}
