diff --git a/templates/base.html.twig b/templates/base.html.twig
index 47dcd878..5bcf2135 100644
--- a/templates/base.html.twig
+++ b/templates/base.html.twig
@@ -33,7 +33,7 @@
                         {% endif %}
                     </li>
                     <li class="nav-item">
-                        <a class="nav-link" href="#">
+                        <a class="nav-link" href="{{ path('app_cart') }}">
                             Shopping Cart (<span id="js-shopping-cart-items">0</span>)
                         </a>
                     </li>
