diff --git a/webpack.config.js b/webpack.config.js
index 9e12e76..11f099e 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -32,6 +32,7 @@ Encore
      */
     .addEntry('app', './assets/js/app.js')
     .addEntry('products', './assets/js/products.js')
+    .addEntry('shopping-cart', './assets/js/shopping-cart.js')
     //.addEntry('page1', './assets/js/page1.js')
     //.addEntry('page2', './assets/js/page2.js')
 
