diff --git a/assets/js/components/sidebar.vue b/assets/js/components/sidebar.vue
index 82433bc..2d99f74 100644
--- a/assets/js/components/sidebar.vue
+++ b/assets/js/components/sidebar.vue
@@ -5,8 +5,6 @@
                 Categories
             </h5>
 
-            <loading-component v-show="loading" />
-
             <ul class="nav flex-column">
                 <li class="nav-item">
                     <a
@@ -42,13 +40,9 @@
 
 <script>
 import axios from 'axios';
-import LoadingComponent from '@/components/loading';
 
 export default {
     name: 'Sidebar',
-    components: {
-        LoadingComponent,
-    },
     props: {
         collapsed: {
             type: Boolean,
@@ -64,10 +58,6 @@ export default {
             default: () => ([]),
         },
     },
-    data: () => ({
-        loading: true,
-        categories: [],
-    }),
     async created() {
         this.loading = true;
         this.categories = [];
