diff --git a/assets/js/components/sidebar.vue b/assets/js/components/sidebar.vue
index 419e7a02..73500abb 100644
--- a/assets/js/components/sidebar.vue
+++ b/assets/js/components/sidebar.vue
@@ -66,6 +66,10 @@ export default {
             type: String,
             default: null,
         },
+        categories: {
+            type: Array,
+            required: true,
+        },
     },
     data() {
         return {
diff --git a/assets/js/pages/products.vue b/assets/js/pages/products.vue
index 5a37f1b0..6fdf72b5 100644
--- a/assets/js/pages/products.vue
+++ b/assets/js/pages/products.vue
@@ -19,7 +19,7 @@
 <script>
 import Catalog from '@/components/catalog';
 import Sidebar from '@/components/sidebar';
-import { getCurrentCategoryId } from '@/services/page-context';
+import { getCurrentCategoryId, getCategories } from '@/services/page-context';
 
 export default {
     name: 'Products',
@@ -42,6 +42,9 @@ export default {
         currentCategoryId() {
             return getCurrentCategoryId();
         },
+        categories() {
+            return getCategories();
+        },
     },
     methods: {
         toggleSidebarCollapsed() {
