diff --git a/templates/product/index.html.twig b/templates/product/index.html.twig
index b437c50b..0db96c1a 100644
--- a/templates/product/index.html.twig
+++ b/templates/product/index.html.twig
@@ -14,6 +14,7 @@
 
     <script>
         window.currentCategoryId = {{ currentCategoryId|default('null') }};
+        window.categories = {{ categories|jsonld }};
     </script>
 
     {{ encore_entry_script_tags('products') }}
