diff --git a/assets/js/pages/products.vue b/assets/js/pages/products.vue
index e24dfa6e..3994ef15 100644
--- a/assets/js/pages/products.vue
+++ b/assets/js/pages/products.vue
@@ -4,6 +4,8 @@
             <aside :class="asideClass">
                 <sidebar
                     :collapsed="sidebarCollapsed"
+                    :current-category-id="currentCategoryId"
+                    :categories="categories"
                     @toggle-collapsed="toggleSidebarCollapsed"
                 />
             </aside>
