diff --git a/src/Controller/ProductController.php b/src/Controller/ProductController.php
index c03afdc..e40d70d 100644
--- a/src/Controller/ProductController.php
+++ b/src/Controller/ProductController.php
@@ -7,33 +7,40 @@ use App\Entity\Category;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\Routing\Annotation\Route;
 use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Serializer\SerializerInterface;
 
 class ProductController extends AbstractController
 {
     /**
      * @Route("/", name="app_homepage")
      */
-    public function index(): Response
+    public function index(SerializerInterface $serializer): Response
     {
-        return $this->render('product/index.html.twig');
+        return $this->render('product/index.html.twig', [
+            'categoriesJson' => $serializer->serialize($this->getCategories(), 'json'),
+        ]);
     }
 
     /**
      * @Route("/category/{id}", name="app_category")
      */
-    public function showCategory(int $id): Response
+    public function showCategory(int $id, SerializerInterface $serializer): Response
     {
         return $this->render('product/index.html.twig', [
             'currentCategoryId' => $id,
+            'categoriesJson' => $serializer->serialize($this->getCategories(), 'json'),
         ]);
     }
 
     /**
      * @Route("/product/{id}", name="app_product")
      */
-    public function showProduct(int $id): Response
+    public function showProduct(int $id, SerializerInterface $serializer): Response
     {
-        return $this->render('product/index.html.twig');
+        return $this->render('product/index.html.twig', [
+            'currentProductId' => $id,
+            'categoriesJson' => $serializer->serialize($this->getCategories(), 'json'),
+        ]);
     }
 
     /**
