diff --git a/assets/js/services/categories-service.js b/assets/js/services/categories-service.js
index 87ec7767..b060f43d 100644
--- a/assets/js/services/categories-service.js
+++ b/assets/js/services/categories-service.js
@@ -1,8 +1,6 @@
-import axios from 'axios';
-
 /**
- * @return {Promise}
+ * @return Array
  */
 export function fetchCategories() {
-    return axios.get('/api/categories');
+    return window.categories;
 }
