diff --git a/assets/js/components/sidebar.vue b/assets/js/components/sidebar.vue
index cf53a895..fd9a3b23 100644
--- a/assets/js/components/sidebar.vue
+++ b/assets/js/components/sidebar.vue
@@ -78,9 +78,9 @@ export default {
         },
     },
     async created() {
-        this.categories = await fetchCategories();
+        const response = await fetchCategories();
 
-        //this.categories = response.data['hydra:member'];
+        this.categories = response.data['hydra:member'];
     },
 };
 </script>
diff --git a/assets/js/services/categories-service.js b/assets/js/services/categories-service.js
index d10d548c..c3bd36b2 100644
--- a/assets/js/services/categories-service.js
+++ b/assets/js/services/categories-service.js
@@ -1,6 +1,12 @@
 /**
- * @returns Array
+ * @returns Promise
  */
 export function fetchCategories() {
-    return window.categories;
+    return new Promise((resolve, reject) => {
+        resolve({
+            data: {
+                'hydra:member': window.categories,
+            },
+        });
+    });
 }
