diff --git a/assets/js/services/page-context.js b/assets/js/services/page-context.js
index bc18cdc3..cc3592b3 100644
--- a/assets/js/services/page-context.js
+++ b/assets/js/services/page-context.js
@@ -6,3 +6,12 @@
 export function getCurrentCategoryId() {
     return window.currentCategoryId;
 }
+
+/**
+ * Returns the array of categories set by the server.
+ *
+ * @returns {array}
+ */
+export function getCategories() {
+    return window.categories;
+}
