diff --git a/src/Controller/ProductController.php b/src/Controller/ProductController.php
index 1c4b453..0264359 100644
--- a/src/Controller/ProductController.php
+++ b/src/Controller/ProductController.php
@@ -3,6 +3,7 @@
 namespace App\Controller;
 
 
+use App\Repository\CategoryRepository;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\Routing\Annotation\Route;
 use Symfony\Component\HttpFoundation\Response;
@@ -12,18 +13,21 @@ class ProductController extends AbstractController
     /**
      * @Route("/", name="app_homepage")
      */
-    public function index(): Response
+    public function index(CategoryRepository $categoryRepository): Response
     {
-        return $this->render('product/index.html.twig');
+        return $this->render('product/index.html.twig', [
+            'categoriesJson' => $categoryRepository->findAll(),
+        ]);
     }
 
     /**
      * @Route("/category/{id}", name="app_category")
      */
-    public function showCategory(int $id): Response
+    public function showCategory(int $id, CategoryRepository $categoryRepository): Response
     {
         return $this->render('product/index.html.twig', [
             'currentCategoryId' => $id,
+            'categoriesJson' => $categoryRepository->findAll(),
         ]);
     }
 
