diff --git a/assets/js/components/sidebar.vue b/assets/js/components/sidebar.vue
index 73500abb..491873f1 100644
--- a/assets/js/components/sidebar.vue
+++ b/assets/js/components/sidebar.vue
@@ -5,8 +5,6 @@
                 Categories
             </h5>
 
-            <loading v-show="loading" />
-
             <ul class="nav flex-column mb4">
                 <li class="nav-item">
                     <a
@@ -50,13 +48,9 @@
 
 <script>
 import axios from 'axios';
-import Loading from '@/components/loading';
 
 export default {
     name: 'Sidebar',
-    components: {
-        Loading: Loading,
-    },
     props: {
         collapsed: {
             type: Boolean,
@@ -71,12 +65,6 @@ export default {
             required: true,
         },
     },
-    data() {
-        return {
-            loading: true,
-            categories: [],
-        };
-    },
     async created() {
         this.loading = true;
 
