diff --git a/assets/js/components/sidebar.vue b/assets/js/components/sidebar.vue
index 81bfcf4d..d826b162 100644
--- a/assets/js/components/sidebar.vue
+++ b/assets/js/components/sidebar.vue
@@ -76,10 +76,10 @@ export default {
     async created() {
         this.loading = true;
 
-        const response = await fetchCategories();
+        this.categories = await fetchCategories();
 
         this.loading = false;
-        this.categories = response.data['hydra:member'];
+        //this.categories = response.data['hydra:member'];
     },
 };
 </script>
