diff --git a/templates/product/index.html.twig b/templates/product/index.html.twig
index 3906b779..9fd746b6 100644
--- a/templates/product/index.html.twig
+++ b/templates/product/index.html.twig
@@ -19,6 +19,7 @@
         {% else %}
             window.currentCategoryId = null;
         {% endif %}
+        window.categories = {{ categories|jsonld }};
     </script>
 
     {{ encore_entry_script_tags('products') }}
