diff --git a/assets/js/components/sidebar.vue b/assets/js/components/sidebar.vue
index 6d6b56f6..6b8cc4ea 100644
--- a/assets/js/components/sidebar.vue
+++ b/assets/js/components/sidebar.vue
@@ -41,8 +41,6 @@
 </template>
 
 <script>
-import axios from 'axios';
-
 export default {
     name: 'Sidebar',
     props: {
@@ -59,19 +57,6 @@ export default {
             required: true,
         },
     },
-    async created() {
-        this.loading = true;
-        this.categories = [];
-
-        try {
-            const response = await axios.get('/api/categories');
-
-            this.loading = false;
-            this.categories = response.data['hydra:member'];
-        } catch (e) {
-            this.loading = false;
-        }
-    },
 };
 </script>
 
