diff --git a/src/Controller/ProductController.php b/src/Controller/ProductController.php
index e1f8cafe..a86ca51e 100644
--- a/src/Controller/ProductController.php
+++ b/src/Controller/ProductController.php
@@ -2,10 +2,10 @@
 
 namespace App\Controller;
 
-
 use ApiPlatform\Core\Api\IriConverterInterface;
 use App\Entity\Category;
 use App\Entity\Product;
+use App\Repository\CategoryRepository;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\Routing\Annotation\Route;
 use Symfony\Component\HttpFoundation\Response;
@@ -15,18 +15,21 @@ class ProductController extends AbstractController
     /**
      * @Route("/", name="app_homepage")
      */
-    public function index(): Response
+    public function index(CategoryRepository $categoryRepository): Response
     {
-        return $this->render('product/index.html.twig');
+        return $this->render('product/index.html.twig', [
+            'categories' => $categoryRepository->findAll(),
+        ]);
     }
 
     /**
      * @Route("/category/{id}", name="app_category")
      */
-    public function showCategory(Category $category, IriConverterInterface $iriConverter): Response
+    public function showCategory(Category $category, IriConverterInterface $iriConverter, CategoryRepository $categoryRepository): Response
     {
         return $this->render('product/index.html.twig', [
             'currentCategoryId' => $iriConverter->getIriFromItem($category),
+            'categories' => $categoryRepository->findAll(),
         ]);
     }
 
