diff --git a/src/Controller/CheckoutController.php b/src/Controller/CheckoutController.php
index cf8d97e87..9ee994f7f 100644
--- a/src/Controller/CheckoutController.php
+++ b/src/Controller/CheckoutController.php
@@ -23,6 +23,9 @@ class CheckoutController extends AbstractController
      */
     public function confirmation(Purchase $purchase): Response
     {
+        // TODO - you would have security here to prevent someone
+        // from viewing my order details!
+
         $totalPrice = 0;
         $purchaseItems = $purchase->getPurchaseItems();
 
