diff --git a/assets/interfaces/category.ts b/assets/interfaces/category.ts
deleted file mode 100644
index bb3cab54b..000000000
--- a/assets/interfaces/category.ts
+++ /dev/null
@@ -1,5 +0,0 @@
-export interface Category {
-  '@id'?: string;
-  name?: string;
-  id?: string;
-}
diff --git a/assets/interfaces/color.ts b/assets/interfaces/color.ts
deleted file mode 100644
index f47a20fd2..000000000
--- a/assets/interfaces/color.ts
+++ /dev/null
@@ -1,6 +0,0 @@
-export interface Color {
-  '@id'?: string;
-  name?: string;
-  hexColor?: string;
-  id?: string;
-}
diff --git a/assets/interfaces/product.ts b/assets/interfaces/product.ts
deleted file mode 100644
index 95b0128bd..000000000
--- a/assets/interfaces/product.ts
+++ /dev/null
@@ -1,12 +0,0 @@
-export interface Product {
-  '@id'?: string;
-  name?: string;
-  description?: string;
-  brand?: string;
-  weight?: number;
-  price?: number;
-  stockQuantity?: number;
-  category?: string;
-  colors?: string[];
-  id?: string;
-}
diff --git a/assets/interfaces/purchase.ts b/assets/interfaces/purchase.ts
deleted file mode 100644
index 6f94986a3..000000000
--- a/assets/interfaces/purchase.ts
+++ /dev/null
@@ -1,13 +0,0 @@
-export interface Purchase {
-  '@id'?: string;
-  customerName: string;
-  customerEmail: string;
-  customerAddress: string;
-  customerZip: string;
-  customerCity: string;
-  customerPhone: string;
-  purchaseItems?: string[];
-  readonly isShipped?: boolean;
-  readonly createdAt?: Date;
-  id?: string;
-}
diff --git a/assets/interfaces/purchaseitem.ts b/assets/interfaces/purchaseitem.ts
deleted file mode 100644
index 4057692ee..000000000
--- a/assets/interfaces/purchaseitem.ts
+++ /dev/null
@@ -1,8 +0,0 @@
-export interface PurchaseItem {
-  '@id'?: string;
-  purchase?: string;
-  product?: string;
-  color?: string;
-  quantity?: number;
-  id?: string;
-}
diff --git a/assets/interfaces/user.ts b/assets/interfaces/user.ts
deleted file mode 100644
index 0fb28e72d..000000000
--- a/assets/interfaces/user.ts
+++ /dev/null
@@ -1,5 +0,0 @@
-export interface User {
-  '@id'?: string;
-  email?: string;
-  id?: string;
-}
