diff --git a/src/Controller/ShoppingCartController.php b/src/Controller/CartController.php
similarity index 63%
rename from src/Controller/ShoppingCartController.php
rename to src/Controller/CartController.php
index 4af39da6d..e238e0003 100644
--- a/src/Controller/ShoppingCartController.php
+++ b/src/Controller/CartController.php
@@ -7,13 +7,13 @@ use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Routing\Annotation\Route;
 
-class ShoppingCartController extends AbstractController
+class CartController extends AbstractController
 {
     /**
-     * @Route("/shopping-cart", name="app_cart")
+     * @Route("/cart", name="app_cart")
      */
     public function shoppingCart(): Response
     {
-        return $this->render('cart/index.html.twig');
+        return $this->render('cart/cart.html.twig');
     }
 }
diff --git a/templates/cart/cart.html.twig b/templates/cart/cart.html.twig
new file mode 100644
index 000000000..261522de2
--- /dev/null
+++ b/templates/cart/cart.html.twig
@@ -0,0 +1,5 @@
+{% extends 'base.html.twig' %}
+
+{% block body %}
+    <h1>TODO - put a cart here!</h1>
+{% endblock %}
