diff --git a/src/DataFixtures/AppFixtures.php b/src/DataFixtures/AppFixtures.php
index 317c9cd13..39942791f 100644
--- a/src/DataFixtures/AppFixtures.php
+++ b/src/DataFixtures/AppFixtures.php
@@ -97,7 +97,8 @@ class AppFixtures extends Fixture
             'name' => 'Floppy disk',
             'description' => 'With 1.44 mb of storage space enjoy your favorite thumbnail photo on the go!',
             'image' => 'floppy-disc.png',
-            'category' => 'office_supplies'
+            'category' => 'office_supplies',
+            'with_colors' => true,
         ];
         yield [
             'name' => 'Blank CD\'s',
