diff --git a/assets/js/components/catalog.vue b/assets/js/components/catalog.vue
index 3761984d..af0aba68 100644
--- a/assets/js/components/catalog.vue
+++ b/assets/js/components/catalog.vue
@@ -30,12 +30,15 @@ export default {
         LegendComponent,
     },
     data: () => ({
+        products: [],
         legend: 'Shipping takes 10-12 weeks, and products probably won\'t work',
     }),
     async mounted() {
+        this.products = [];
+
         const response = await axios.get('/api/products');
 
-        console.log(response);
+        this.products = response.data['hydra:member'];
     },
 };
 </script>
