diff --git a/assets/js/components/product.vue b/assets/js/components/product.vue
index 64fa9dd0..2d893498 100644
--- a/assets/js/components/product.vue
+++ b/assets/js/components/product.vue
@@ -39,6 +39,13 @@
                             @color-selected="updateSelectedColor"
                         />
 
+                        <input
+                            v-model.number="qty"
+                            class="form-control"
+                            type="number"
+                            min="1"
+                        >
+
                         <button
                             class="btn btn-info btn-sm"
                             @click="addToCart"
@@ -82,6 +89,7 @@ export default {
                 name: '', image: '', price: 0, colors: [],
             },
             selectedColorId: null,
+            qty: 1,
             loading: true,
         };
     },
