diff --git a/assets/js/components/product-list/product-card.vue b/assets/js/components/product-list/product-card.vue
new file mode 100644
index 00000000..89928521
--- /dev/null
+++ b/assets/js/components/product-list/product-card.vue
@@ -0,0 +1,17 @@
+<template>
+    <div class="col-xs-12 col-6 mb-2 pb-2">
+        {{ product.name }}
+    </div>
+</template>
+
+<script>
+export default {
+    name: 'ProductCard',
+    props: {
+        product: {
+            type: Object,
+            required: true,
+        },
+    },
+};
+</script>
