diff --git a/assets/js/components/product.vue b/assets/js/components/product.vue
index 33e7d343..2a3166dc 100644
--- a/assets/js/components/product.vue
+++ b/assets/js/components/product.vue
@@ -57,6 +57,7 @@
 </template>
 
 <script>
+import ColorSelector from '@/components/color-selector';
 import { getProduct } from '@/services/products-service';
 import Loading from '@/components/loading';
 import TitleComponent from '@/components/title';
@@ -65,6 +66,7 @@ import formatPrice from '@/helpers/format-price';
 export default {
     name: 'Product',
     components: {
+        ColorSelector,
         Loading,
         TitleComponent,
     },
