diff --git a/assets/js/components/product.vue b/assets/js/components/product.vue
index 2b624fc8..191148bb 100644
--- a/assets/js/components/product.vue
+++ b/assets/js/components/product.vue
@@ -34,6 +34,10 @@
                     </div>
 
                     <div :class="$style['actions-wrapper']">
+                        <color-selector
+                            v-show="product.colors.length !== 0"
+                        />
+
                         <button
                             class="btn btn-info btn-sm"
                             @click="addToCart"
