diff --git a/assets/js/components/catalog.vue b/assets/js/components/catalog.vue
index d8176caf..3761984d 100644
--- a/assets/js/components/catalog.vue
+++ b/assets/js/components/catalog.vue
@@ -32,7 +32,7 @@ export default {
     data: () => ({
         legend: 'Shipping takes 10-12 weeks, and products probably won\'t work',
     }),
-    mounted() {
+    async mounted() {
         const response = await axios.get('/api/products');
 
         console.log(response);
