diff --git a/assets/js/components/product.vue b/assets/js/components/product.vue
index 74f43fff..16a2bea7 100644
--- a/assets/js/components/product.vue
+++ b/assets/js/components/product.vue
@@ -45,6 +45,10 @@
                     </div>
 
                     <div class="p-3 row align-items-center">
+                        <color-selector
+                            v-show="product.colors.length !== 0"
+                        />
+
                         <button
                             class="btn btn-info btn-sm"
                             @click="addToCart"
