diff --git a/assets/js/components/product.vue b/assets/js/components/product.vue
index 35c5bc73..64fa9dd0 100644
--- a/assets/js/components/product.vue
+++ b/assets/js/components/product.vue
@@ -36,6 +36,7 @@
                     <div :class="$style['actions-wrapper']">
                         <color-selector
                             v-show="product.colors.length !== 0"
+                            @color-selected="updateSelectedColor"
                         />
 
                         <button
@@ -80,6 +81,7 @@ export default {
             product: {
                 name: '', image: '', price: 0, colors: [],
             },
+            selectedColorId: null,
             loading: true,
         };
     },
@@ -111,6 +113,15 @@ export default {
         addToCart() {
             // TODO
         },
+
+        /**
+         * Update the selectedColorId index accordingly
+         *
+         * @params {string} colorId
+         */
+        updateSelectedColor(colorId) {
+            this.selectedColorId = colorId;
+        },
     },
 };
 </script>
