diff --git a/assets/js/components/catalog.vue b/assets/js/components/catalog.vue
index de90c1a1..fd843bda 100644
--- a/assets/js/components/catalog.vue
+++ b/assets/js/components/catalog.vue
@@ -8,15 +8,7 @@
             </div>
         </div>
 
-        <div class="row">
-            <div
-                v-for="product in products"
-                :key="product['@id']"
-                class="col-xs-12 col-6 mb-2 pb-2"
-            >
-                {{ product.name }}
-            </div>
-        </div>
+        <product-list :products="products" />
 
         <div class="row">
             <legend-component :title="legend" />
@@ -27,11 +19,13 @@
 <script>
 import axios from 'axios';
 import LegendComponent from '@/components/legend';
+import ProductList from '@/components/product-list';
 
 export default {
     name: 'Catalog',
     components: {
         LegendComponent,
+        ProductList,
     },
     data() {
         return {
