diff --git a/assets/js/components/catalog.vue b/assets/js/components/catalog.vue
index 5f667550..de90c1a1 100644
--- a/assets/js/components/catalog.vue
+++ b/assets/js/components/catalog.vue
@@ -35,6 +35,7 @@ export default {
     },
     data() {
         return {
+            /** @type {Product[]} */
             products: [],
             legend: 'Shipping takes 10-13 weeks, and products probably won\'t work',
         };
