diff --git a/assets/js/components/product.vue b/assets/js/components/product.vue
index ccae0cb..72d5538 100644
--- a/assets/js/components/product.vue
+++ b/assets/js/components/product.vue
@@ -50,12 +50,14 @@
 <script>
 import formatPrice from '@/helpers/format-price';
 import productsService from '@/services/products';
+import ColorSelector from '@/components/color-selector';
 import Loading from '@/components/loading';
 import TitleComponent from '@/components/title';
 
 export default {
     name: 'Product',
     components: {
+        ColorSelector,
         Loading,
         TitleComponent,
     },
