diff --git a/assets/js/components/product.vue b/assets/js/components/product.vue
index 2a3166dc..94f1694f 100644
--- a/assets/js/components/product.vue
+++ b/assets/js/components/product.vue
@@ -43,6 +43,10 @@
                     </div>
 
                     <div class="p-3">
+                        <color-selector
+                            v-show="product.colors.length !== 0"
+                        />
+
                         <button
                             class="btn btn-info btn-sm"
                             @click="addToCart"
