diff --git a/assets/js/services/colors-service.js b/assets/js/services/colors-service.js
new file mode 100644
index 00000000..d7bb4d00
--- /dev/null
+++ b/assets/js/services/colors-service.js
@@ -0,0 +1,13 @@
+import axios from 'axios';
+
+/**
+ * Gets the color information from our database
+ *
+ * @return {Promise}
+ */
+export function getColors() {
+    return axios({
+        method: 'get',
+        url: '/api/colors',
+    });
+}
