diff --git a/assets/js/components/catalog.vue b/assets/js/components/catalog.vue
index dad73109..307f84f4 100644
--- a/assets/js/components/catalog.vue
+++ b/assets/js/components/catalog.vue
@@ -35,9 +35,9 @@ export default {
         };
     },
     mounted() {
-        axios.get('/api/products').then((response) => {
-            console.log(response);
-        });
+        const response = await axios.get('/api/products');
+
+        console.log(response);
     },
 };
 </script>
