diff --git a/assets/js/components/color-selector.vue b/assets/js/components/color-selector.vue
new file mode 100644
index 00000000..dc107be8
--- /dev/null
+++ b/assets/js/components/color-selector.vue
@@ -0,0 +1,24 @@
+
+<script>
+import { getColors } from '@/services/colors-service';
+
+export default {
+    name: 'ColorSelector',
+    data() {
+        return {
+            colors: [],
+        };
+    },
+    async created() {
+        let response;
+
+        try {
+            response = await getColors();
+        } catch (e) {
+            return;
+        }
+
+        this.colors = response.data['hydra:member'];
+    },
+};
+</script>
