diff --git a/assets/js/components/product.vue b/assets/js/components/product.vue
index ed470ea0..7a3f8953 100644
--- a/assets/js/components/product.vue
+++ b/assets/js/components/product.vue
@@ -113,6 +113,15 @@ export default {
         addToCart() {
             // TODO
         },
+
+        /**
+         * Update the selectedColorId index accordingly
+         *
+         * @params {string} colorId
+         */
+        updateSelectedColor(colorId) {
+            this.selectedColorId = colorId;
+        },
     },
 };
 </script>
