diff --git a/assets/js/components/product-list/product-card.vue b/assets/js/components/product-list/product-card.vue
index b326dab..faa93dd 100644
--- a/assets/js/components/product-list/product-card.vue
+++ b/assets/js/components/product-list/product-card.vue
@@ -1,6 +1,35 @@
 <template>
     <div :class="[$style.component, 'col-xs-12', 'mb-2', 'ml-2', 'pb-2']">
-        {{ item.name }}
+        <div :class="$style.top">
+            <div :class="$style.image">
+                <img
+                    :alt="item.name"
+                    :src="item.image"
+                    class="mx-auto d-block my-2"
+                >
+
+                <h3
+                    class="mb-2"
+                    v-text="item.name"
+                />
+            </div>
+
+            <div class="pt-2 my-3 d-md-flex justify-content-between">
+                <p class="p-0 d-inline">
+                    <strong>$ {{ item.price }}</strong>
+                </p>
+
+                <button
+                    class="btn btn-info btn-sm"
+                >
+                    View Product
+                </button>
+            </div>
+        </div>
+
+        <div>
+            <small>brought to you by {{ item.brand }}</small>
+        </div>
     </div>
 </template>
 
