diff --git a/assets/js/components/product.vue b/assets/js/components/product.vue
index 94f1694f..758ac2cf 100644
--- a/assets/js/components/product.vue
+++ b/assets/js/components/product.vue
@@ -45,6 +45,7 @@
                     <div class="p-3">
                         <color-selector
                             v-show="product.colors.length !== 0"
+                            @color-selected="updateSelectedColor"
                         />
 
                         <button
@@ -89,6 +90,7 @@ export default {
             product: {
                 name: '', image: '', price: 0, colors: [],
             },
+            selectedColorId: null,
             loading: true,
         };
     },
@@ -120,6 +122,15 @@ export default {
         addToCart() {
             // TODO
         },
+
+        /**
+         * Update the selectedColorId index accordingly
+         *
+         * @params {string} colorId
+         */
+        updateSelectedColor(colorId) {
+            this.selectedColorId = colorId;
+        },
     },
 };
 </script>
