diff --git a/assets/js/components/product-list/index.vue b/assets/js/components/product-list/index.vue
new file mode 100644
index 00000000..cab856ba
--- /dev/null
+++ b/assets/js/components/product-list/index.vue
@@ -0,0 +1,17 @@
+<template>
+    <div class="row">
+
+    </div>
+</template>
+
+<script>
+export default {
+    name: 'ProductList',
+    props: {
+        products: {
+            type: Array,
+            required: true,
+        },
+    },
+};
+</script>
