diff --git a/assets/js/components/product.vue b/assets/js/components/product.vue
index 72d5538..d2fa28a 100644
--- a/assets/js/components/product.vue
+++ b/assets/js/components/product.vue
@@ -77,6 +77,8 @@ export default {
     },
     data: () => ({
         product: { name: '', price: 0, colors: [] },
+        selectedColorId: null,
+        qty: 1,
         loading: true,
     }),
     computed: {
