diff --git a/assets/js/components/product-list/index.vue b/assets/js/components/product-list/index.vue
new file mode 100644
index 0000000..2b8e1a3
--- /dev/null
+++ b/assets/js/components/product-list/index.vue
@@ -0,0 +1,26 @@
+<template>
+    <div :class="$style.component">
+
+    </div>
+</template>
+
+<script>
+export default {
+    name: 'ProductList',
+    props: {
+        products: {
+            type: Array,
+            required: true,
+        },
+    },
+};
+</script>
+
+<style lang="scss" module>
+.component {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    margin-top: 25px;
+}
+</style>
