diff --git a/assets/js/components/product-list/index.vue b/assets/js/components/product-list/index.vue
index 2b8e1a3..2727cce 100644
--- a/assets/js/components/product-list/index.vue
+++ b/assets/js/components/product-list/index.vue
@@ -1,6 +1,11 @@
 <template>
     <div :class="$style.component">
-
+        <div
+            v-for="product in products"
+            :key="product['@id']"
+        >
+            {{ product.name }}
+        </div>
     </div>
 </template>
 
