diff --git a/assets/js/components/product.vue b/assets/js/components/product.vue
index 5239cccc..fd00ca9a 100644
--- a/assets/js/components/product.vue
+++ b/assets/js/components/product.vue
@@ -39,6 +39,13 @@
                             @color-selected="updateSelectedColor"
                         />
 
+                        <input
+                            v-model.number="qty"
+                            class="form-control"
+                            type="number"
+                            min="1"
+                        >
+
                         <button
                             class="btn btn-info btn-sm"
                             @click="addToCart"
@@ -81,6 +88,7 @@ export default {
             name: '', image: '', price: 0, colors: [],
         },
         selectedColorId: null,
+        qty: 1,
         loading: true,
     }),
     computed: {
