diff --git a/assets/js/components/product.vue b/assets/js/components/product.vue
index 16a2bea7..36341544 100644
--- a/assets/js/components/product.vue
+++ b/assets/js/components/product.vue
@@ -88,7 +88,9 @@ export default {
     },
     data() {
         return {
-            product: { name: '', image: '', price: 0 },
+            product: {
+                name: '', image: '', price: 0, colors: [],
+            },
             loading: true,
         };
     },
