diff --git a/assets/js/components/catalog.vue b/assets/js/components/catalog.vue
index 769860e..12fe80d 100644
--- a/assets/js/components/catalog.vue
+++ b/assets/js/components/catalog.vue
@@ -6,6 +6,13 @@
             </h2>
         </div>
 
+        <div
+            v-for="product in products"
+            :key="product['@id']"
+        >
+            {{ product.name }}
+        </div>
+
         <div class="row">
             <legend-component :title="legend" />
         </div>
