diff --git a/assets/js/components/color-selector.vue b/assets/js/components/color-selector.vue
index a13f7226..97ed789c 100644
--- a/assets/js/components/color-selector.vue
+++ b/assets/js/components/color-selector.vue
@@ -3,6 +3,7 @@
         <span
             v-for="(color, index) in colors"
             :key="color['@id']"
+            :class="{ [$style.selected]: index === selectedIndex}"
             :title="color.name"
             :style="{ backgroundColor: `#${color.hexColor}` }"
             @click="selectColor(index)"
@@ -51,6 +52,10 @@ export default {
             width: 25px;
             height: 25px;
             margin-right: 10px;
+
+            &.selected {
+                border: 2px solid black
+            }
         }
     }
 </style>
