diff --git a/assets/js/components/product.vue b/assets/js/components/product.vue
index 36341544..62d63903 100644
--- a/assets/js/components/product.vue
+++ b/assets/js/components/product.vue
@@ -47,6 +47,7 @@
                     <div class="p-3 row align-items-center">
                         <color-selector
                             v-show="product.colors.length !== 0"
+                            @color-selected="updateSelectedColor"
                         />
 
                         <button
@@ -91,6 +92,7 @@ export default {
             product: {
                 name: '', image: '', price: 0, colors: [],
             },
+            selectedColorId: null,
             loading: true,
         };
     },
@@ -122,6 +124,15 @@ export default {
         addToCart() {
             // TODO
         },
+
+        /**
+         * Update the selectedColorId index accordingly
+         *
+         * @params {string} colorId
+         */
+        updateSelectedColor(colorId) {
+            this.selectedColorId = colorId;
+        },
     },
 };
 </script>
