diff --git a/assets/js/modules/collapsible-sidebar.js b/assets/js/modules/collapsible-sidebar.js
index 67028593..1a1a65a5 100644
--- a/assets/js/modules/collapsible-sidebar.js
+++ b/assets/js/modules/collapsible-sidebar.js
@@ -1,7 +1,8 @@
+import { ref } from 'vue';
+
 export default function useCollapsibleSidebar() {
+    const sidebarCollapsed = ref(false);
     /*
-    sidebarCollapsed: false,
-
     asideClass() {
         return this.sidebarCollapsed ? 'aside-collapsed' : 'col-xs-12 col-3';
     },
@@ -15,5 +16,7 @@ export default function useCollapsibleSidebar() {
         },
     },
    */
-    return {};
+    return {
+        sidebarCollapsed,
+    };
 }
diff --git a/assets/js/pages/products.vue b/assets/js/pages/products.vue
index 39a22d0a..914f8d12 100644
--- a/assets/js/pages/products.vue
+++ b/assets/js/pages/products.vue
@@ -38,7 +38,6 @@ export default {
     },
     data() {
         return {
-            sidebarCollapsed: false,
             categories: [],
             currentCategoryId: getCurrentCategoryId(),
         };
