diff --git a/assets/pages/products.vue b/assets/pages/products.vue
index 7656bf62..3dd7c2d9 100644
--- a/assets/pages/products.vue
+++ b/assets/pages/products.vue
@@ -26,6 +26,7 @@ import ProductShow from '@/components/product-show';
 import Sidebar from '@/components/sidebar';
 import { getCurrentCategoryId, getCurrentProductId } from '@/services/page-context';
 import { fetchCategories } from '@/services/categories-service';
+import useCollapsibleSidebar from '@/modules/collapsible-sidebar';
 
 export default {
     name: 'Products',
@@ -73,5 +74,10 @@ export default {
             this.sidebarCollapsed = !this.sidebarCollapsed;
         },
     },
+    setup() {
+        return {
+            ...useCollapsibleSidebar(),
+        };
+    },
 };
 </script>
