diff --git a/assets/modules/collapsible-sidebar.js b/assets/modules/collapsible-sidebar.js
index 771ecf85..b7179bca 100644
--- a/assets/modules/collapsible-sidebar.js
+++ b/assets/modules/collapsible-sidebar.js
@@ -1,7 +1,8 @@
+import { ref } from 'vue';
+
 export default function useCollapsibleSidebar() {
+    const sidebarCollapsed = ref(false);
     /*
-    sidebarCollapsed: false,
-
     asideClass() {
         return this.sidebarCollapsed ? 'aside-collapsed' : 'col-12 col-md-3';
     },
@@ -15,5 +16,7 @@ export default function useCollapsibleSidebar() {
         },
     },
    */
-    return {};
+    return {
+        sidebarCollapsed,
+    };
 }
diff --git a/assets/pages/products.vue b/assets/pages/products.vue
index 3dd7c2d9..32311a28 100644
--- a/assets/pages/products.vue
+++ b/assets/pages/products.vue
@@ -37,7 +37,6 @@ export default {
     },
     data() {
         return {
-            sidebarCollapsed: false,
             categories: [],
             currentCategoryId: getCurrentCategoryId(),
         };
