diff --git a/assets/modules/collapsible-sidebar.js b/assets/modules/collapsible-sidebar.js
index 4f82846e..36542766 100644
--- a/assets/modules/collapsible-sidebar.js
+++ b/assets/modules/collapsible-sidebar.js
@@ -10,16 +10,15 @@ export default function useCollapsibleSidebar() {
     const contentClass = computed(
         () => (sidebarCollapsed.value ? 'col-12 col-md-11' : 'col-12 col-md-9'),
     );
-    /*
-    methods: {
-        toggleSidebarCollapsed() {
-            this.sidebarCollapsed = !this.sidebarCollapsed;
-        },
-    },
-   */
+
+    const toggleSidebarCollapsed = () => {
+        sidebarCollapsed.value = !sidebarCollapsed.value;
+    };
+
     return {
         asideClass,
         contentClass,
         sidebarCollapsed,
+        toggleSidebarCollapsed,
     };
 }
diff --git a/assets/pages/products.vue b/assets/pages/products.vue
index 0e2c6899..a709c8bc 100644
--- a/assets/pages/products.vue
+++ b/assets/pages/products.vue
@@ -62,11 +62,6 @@ export default {
 
         this.categories = response.data['hydra:member'];
     },
-    methods: {
-        toggleSidebarCollapsed() {
-            this.sidebarCollapsed = !this.sidebarCollapsed;
-        },
-    },
     setup() {
         return {
             ...useCollapsibleSidebar(),
