diff --git a/assets/js/pages/products.vue b/assets/js/pages/products.vue
index 6899bf0a..39a22d0a 100644
--- a/assets/js/pages/products.vue
+++ b/assets/js/pages/products.vue
@@ -27,6 +27,7 @@ import Sidebar from '@/components/sidebar';
 import { getCurrentCategoryId, getCurrentProductId } from '@/services/page-context';
 import { fetchCategories } from '@/services/categories-service';
 import useShoppingCart from '@/modules/shopping-cart';
+import useCollapsibleSidebar from '@/modules/collapsible-sidebar';
 
 export default {
     name: 'Products',
@@ -80,6 +81,7 @@ export default {
     setup() {
         return {
             ...useShoppingCart(),
+            ...useCollapsibleSidebar(),
         };
     },
 };
