diff --git a/assets/js/modules/collapsible-sidebar.js b/assets/js/modules/collapsible-sidebar.js
index 2efd6c6e..ae157e9f 100644
--- a/assets/js/modules/collapsible-sidebar.js
+++ b/assets/js/modules/collapsible-sidebar.js
@@ -10,16 +10,15 @@ export default function useCollapsibleSidebar() {
     const contentClass = computed(
         () => (sidebarCollapsed.value ? 'col-xs-12 col-11' : 'col-xs-12 col-9'),
     );
-    /*
-    methods: {
-        toggleSidebarCollapsed() {
-            this.sidebarCollapsed = !this.sidebarCollapsed;
-        },
-    },
-   */
+
+    const toggleSidebarCollapsed = () => {
+        sidebarCollapsed.value = !sidebarCollapsed.value;
+    };
+
     return {
         asideClass,
         contentClass,
         sidebarCollapsed,
+        toggleSidebarCollapsed,
     };
 }
diff --git a/assets/js/pages/products.vue b/assets/js/pages/products.vue
index 43ab5950..6f317f5e 100644
--- a/assets/js/pages/products.vue
+++ b/assets/js/pages/products.vue
@@ -66,11 +66,6 @@ export default {
 
         this.categories = response.data['hydra:member'];
     },
-    methods: {
-        toggleSidebarCollapsed() {
-            this.sidebarCollapsed = !this.sidebarCollapsed;
-        },
-    },
     setup() {
         return {
             ...useShoppingCart(),
