diff --git a/assets/modules/collapsible-sidebar.js b/assets/modules/collapsible-sidebar.js
new file mode 100644
index 00000000..771ecf85
--- /dev/null
+++ b/assets/modules/collapsible-sidebar.js
@@ -0,0 +1,19 @@
+export default function useCollapsibleSidebar() {
+    /*
+    sidebarCollapsed: false,
+
+    asideClass() {
+        return this.sidebarCollapsed ? 'aside-collapsed' : 'col-12 col-md-3';
+    },
+    contentClass() {
+        return this.sidebarCollapsed ? 'col-12 col-md-11' : 'col-12 col-md-9';
+    },
+
+    methods: {
+        toggleSidebarCollapsed() {
+            this.sidebarCollapsed = !this.sidebarCollapsed;
+        },
+    },
+   */
+    return {};
+}
