diff --git a/assets/js/factory/get-sidebar-collapsed.js b/assets/js/factory/get-sidebar-collapsed.js
index dd417074..0de4658d 100644
--- a/assets/js/factory/get-sidebar-collapsed.js
+++ b/assets/js/factory/get-sidebar-collapsed.js
@@ -1,7 +1,8 @@
+import { ref } from 'vue';
+
 export default function useSidebarCollapsed() {
+    const sidebarCollapsed = ref(false);
     /*
-    sidebarCollapsed: false,
-
     asideClass() {
         return this.sidebarCollapsed ? 'aside-collapsed' : 'col-xs-12 col-3';
     },
@@ -15,5 +16,7 @@ export default function useSidebarCollapsed() {
         },
     },
    */
-    return {};
+    return {
+        sidebarCollapsed,
+    };
 }
diff --git a/assets/js/pages/products.vue b/assets/js/pages/products.vue
index 9249cda5..1accfb1c 100644
--- a/assets/js/pages/products.vue
+++ b/assets/js/pages/products.vue
@@ -38,7 +38,6 @@ export default {
     },
     data() {
         return {
-            sidebarCollapsed: false,
             categories: [],
             currentCategoryId: getCurrentCategoryId(),
         };
