diff --git a/assets/modules/collapsible-sidebar.js b/assets/modules/collapsible-sidebar.js
index 1a1a65a5..2efd6c6e 100644
--- a/assets/modules/collapsible-sidebar.js
+++ b/assets/modules/collapsible-sidebar.js
@@ -1,15 +1,16 @@
-import { ref } from 'vue';
+import { computed, ref } from 'vue';
 
 export default function useCollapsibleSidebar() {
     const sidebarCollapsed = ref(false);
-    /*
-    asideClass() {
-        return this.sidebarCollapsed ? 'aside-collapsed' : 'col-xs-12 col-3';
-    },
-    contentClass() {
-        return this.sidebarCollapsed ? 'col-xs-12 col-11' : 'col-xs-12 col-9';
-    },
 
+    const asideClass = computed(
+        () => (sidebarCollapsed.value ? 'aside-collapsed' : 'col-xs-12 col-3'),
+    );
+
+    const contentClass = computed(
+        () => (sidebarCollapsed.value ? 'col-xs-12 col-11' : 'col-xs-12 col-9'),
+    );
+    /*
     methods: {
         toggleSidebarCollapsed() {
             this.sidebarCollapsed = !this.sidebarCollapsed;
@@ -17,6 +18,8 @@ export default function useCollapsibleSidebar() {
     },
    */
     return {
+        asideClass,
+        contentClass,
         sidebarCollapsed,
     };
 }
diff --git a/assets/pages/products.vue b/assets/pages/products.vue
index 64009d39..0e2c6899 100644
--- a/assets/pages/products.vue
+++ b/assets/pages/products.vue
@@ -42,12 +42,6 @@ export default {
         };
     },
     computed: {
-        asideClass() {
-            return this.sidebarCollapsed ? 'aside-collapsed' : 'col-xs-12 col-3';
-        },
-        contentClass() {
-            return this.sidebarCollapsed ? 'col-xs-12 col-11' : 'col-xs-12 col-9';
-        },
         currentProductId() {
             return getCurrentProductId();
         },
