diff --git a/assets/js/pages/checkout.vue b/assets/js/pages/checkout.vue
index 8ea66afd..22798a25 100644
--- a/assets/js/pages/checkout.vue
+++ b/assets/js/pages/checkout.vue
@@ -81,6 +81,7 @@
 </template>
 
 <script>
+import { createOrder } from '@/services/checkout-service';
 import cartService from '@/services/cart-service';
 import FormInput from '@/components/checkout/form-input';
 import Loading from '@/components/loading';
@@ -124,6 +125,24 @@ export default {
             quantity: item.qty,
         }));
     },
+    methods: {
+        /**
+         * Sends the form data and process the response!
+         */
+        async onSubmit() {
+            this.loading = true;
+
+            try {
+                const response = await createOrder(this.form);
+
+                console.log(response.data);
+            } catch (e) {
+                console.log('error', e);
+            } finally {
+                this.loading = false;
+            }
+        },
+    },
 };
 </script>
 
