diff --git a/assets/js/pages/checkout.vue b/assets/js/pages/checkout.vue
index 0ff2af0a..23c44e4c 100644
--- a/assets/js/pages/checkout.vue
+++ b/assets/js/pages/checkout.vue
@@ -109,6 +109,7 @@ export default {
                 customerPhone: null,
             },
             loading: false,
+            serverError: false,
         };
     },
     async created() {
@@ -129,13 +130,18 @@ export default {
         async onSubmit(event) {
             event.preventDefault();
             this.loading = true;
+            this.serverError = false;
 
             try {
                 const response = await createOrder(this.form);
-
-                console.log(response.data);
-            } catch (e) {
-                console.log('error', e);
+            } catch (error) {
+                const { response } = error;
+
+                if (response.status !== 400) {
+                    this.serverError = true;
+                } else {
+                    console.log(response.data);
+                }
             } finally {
                 this.loading = false;
             }
