diff --git a/assets/js/pages/checkout.vue b/assets/js/pages/checkout.vue
index a3b1f45a..7ea69ae6 100644
--- a/assets/js/pages/checkout.vue
+++ b/assets/js/pages/checkout.vue
@@ -54,6 +54,17 @@
                                 type="tel"
                                 :error-message="validationErrors.customerPhone"
                             />
+
+                            <div :class="$style['form-action']">
+                                <loading v-show="loading" />
+
+                                <button
+                                    type="submit"
+                                    class="btn btn-info btn-sm"
+                                >
+                                    Checkout!
+                                </button>
+                            </div>
                         </form>
                     </div>
                 </div>
@@ -65,12 +76,14 @@
 <script>
 import cartService from '@/services/cart';
 import FormInput from '@/components/checkout/form-input';
+import Loading from '@/components/loading';
 import TitleComponent from '@/components/title';
 
 export default {
     name: 'Checkout',
     components: {
         FormInput,
+        Loading,
         TitleComponent,
     },
     data: () => ({
@@ -91,6 +104,7 @@ export default {
             customerCity: null,
             customerPhone: null,
         },
+        loading: false,
     }),
     async created() {
         const itemsInCart = cartService.getItems();
