diff --git a/assets/js/services/checkout.js b/assets/js/services/checkout.js
index 5ba4c9a..d34d8b5 100644
--- a/assets/js/services/checkout.js
+++ b/assets/js/services/checkout.js
@@ -10,7 +10,7 @@ const checkout = {
     createOrder(data) {
         return axios({
             method: 'post',
-            url: '/api/purchases',
+            url: '/api/slfjasfd',
             data,
         });
     },
