diff --git a/assets/js/pages/checkout.vue b/assets/js/pages/checkout.vue
index 7ea69ae6..a5077969 100644
--- a/assets/js/pages/checkout.vue
+++ b/assets/js/pages/checkout.vue
@@ -129,5 +129,16 @@ export default {
     display: flex;
     flex-direction: column;
     width: 900px;
+
+    .form-action {
+        margin-top: 20px;
+        display: flex;
+        flex-direction: row;
+        justify-content: flex-end;
+
+        button {
+            margin-left: 20px;
+        }
+    }
 }
 </style>
