diff --git a/assets/js/components/checkout/index.vue b/assets/js/components/checkout/index.vue
index 885217c2..83cc8e96 100644
--- a/assets/js/components/checkout/index.vue
+++ b/assets/js/components/checkout/index.vue
@@ -51,6 +51,7 @@
                         <button
                             type="submit"
                             class="btn btn-info btn-sm"
+                            :disabled="loading"
                         >
                             Checkout!
                         </button>
diff --git a/assets/js/services/checkout-service.js b/assets/js/services/checkout-service.js
index decb83e8..bca8d23a 100644
--- a/assets/js/services/checkout-service.js
+++ b/assets/js/services/checkout-service.js
@@ -9,7 +9,7 @@ import axios from 'axios';
 export function createOrder(data) {
     return axios({
         method: 'post',
-        url: '/api/slfjasfd',
+        url: '/api/purchases',
         data,
     });
 }
