diff --git a/assets/js/pages/checkout.vue b/assets/js/pages/checkout.vue
index 56827de0..8ea66afd 100644
--- a/assets/js/pages/checkout.vue
+++ b/assets/js/pages/checkout.vue
@@ -56,6 +56,22 @@
                                 type="tel"
                                 :error-message="validationErrors.customerPhone"
                             />
+
+                            <div class="row p-3 justify-content-end align-items-center">
+                                <loading
+                                    v-show="loading"
+                                    class="col-auto"
+                                />
+
+                                <div class="col-auto">
+                                    <button
+                                        type="submit"
+                                        class="btn btn-info btn-sm"
+                                    >
+                                        Checkout!
+                                    </button>
+                                </div>
+                            </div>
                         </form>
                     </div>
                 </div>
@@ -67,12 +83,14 @@
 <script>
 import cartService from '@/services/cart-service';
 import FormInput from '@/components/checkout/form-input';
+import Loading from '@/components/loading';
 import TitleComponent from '@/components/title';
 
 export default {
     name: 'Checkout',
     components: {
         FormInput,
+        Loading,
         TitleComponent,
     },
     data() {
@@ -94,6 +112,7 @@ export default {
                 customerCity: null,
                 customerPhone: null,
             },
+            loading: false,
         };
     },
     async created() {
