diff --git a/assets/js/pages/checkout.vue b/assets/js/pages/checkout.vue
index d824102b..ae44fcbd 100644
--- a/assets/js/pages/checkout.vue
+++ b/assets/js/pages/checkout.vue
@@ -13,6 +13,13 @@
                 <div class="row">
                     <div :class="[$style.content, 'col-11']">
                         <form @submit="onSubmit">
+                            <div
+                                v-show="serverError"
+                                class="alert alert-danger"
+                            >
+                                Oops, there's been an error sending your data! Please, try again!
+                            </div>
+
                             <form-input
                                 id="customerName"
                                 v-model="form.customerName"
