diff --git a/assets/components/checkout/index.vue b/assets/components/checkout/index.vue
index 404de01e4..ec33e5a18 100644
--- a/assets/components/checkout/index.vue
+++ b/assets/components/checkout/index.vue
@@ -73,6 +73,7 @@
 import FormInput from '@/components/checkout/form-input';
 import Loading from '@/components/loading';
 import { createOrder } from '@/services/checkout-service';
+import { clearCart } from '@/services/cart-service';
 
 export default {
     name: 'CheckoutForm',
@@ -127,7 +128,9 @@ export default {
                     purchaseItems: this.cart.items,
                 });
 
-                console.log(response.data);
+                await clearCart();
+
+                window.location = `/confirmation/${response.data.id}`;
             } catch (error) {
                 const { response } = error;
 
