diff --git a/assets/js/pages/checkout.vue b/assets/js/pages/checkout.vue
index 0390ae5a..d2c0572e 100644
--- a/assets/js/pages/checkout.vue
+++ b/assets/js/pages/checkout.vue
@@ -139,16 +139,20 @@ export default {
             event.preventDefault();
             this.loading = true;
             this.serverError = false;
+            this.resetValidationFields();
 
             try {
                 const response = await createOrder(this.form);
+                cartService.clear();
             } catch (error) {
                 const { response } = error;
 
                 if (response.status !== 400) {
                     this.serverError = true;
                 } else {
-                    console.log(response.data);
+                    response.data.violations.forEach((violation) => {
+                        this.validationErrors[violation.propertyPath] = violation.message;
+                    });
                 }
             } finally {
                 this.loading = false;
