diff --git a/assets/js/pages/checkout.vue b/assets/js/pages/checkout.vue
index 590561ea..91c18c44 100644
--- a/assets/js/pages/checkout.vue
+++ b/assets/js/pages/checkout.vue
@@ -150,6 +150,20 @@ export default {
                 this.loading = false;
             }
         },
+
+        /**
+         * Resets our validation fields back to null
+         */
+        resetValidationFields() {
+            this.validation = {
+                customerName: null,
+                customerEmail: null,
+                customerAddress: null,
+                customerZip: null,
+                customerCity: null,
+                customerPhone: null,
+            };
+        },
     },
 };
 </script>
