diff --git a/assets/js/components/checkout/index.vue b/assets/js/components/checkout/index.vue
index 2e518adc..8bb9fe51 100644
--- a/assets/js/components/checkout/index.vue
+++ b/assets/js/components/checkout/index.vue
@@ -82,6 +82,7 @@ export default {
             },
             validationErrors: {},
             loading: false,
+            serverError: false,
         };
     },
     created() {
@@ -107,13 +108,18 @@ export default {
         async onSubmit(event) {
             event.preventDefault();
             this.loading = true;
+            this.serverError = false;
 
             try {
                 const response = await createOrder(this.form);
+            } catch (error) {
+                const { response } = error;
 
-                console.log(response.data);
-            } catch (e) {
-                console.log('error', e);
+                if (response.status !== 400) {
+                    this.serverError = true;
+                } else {
+                    console.log(response.data);
+                }
             } finally {
                 this.loading = false;
             }
