diff --git a/assets/components/checkout/index.vue b/assets/components/checkout/index.vue
index 8e4b81fa0..d3166440f 100644
--- a/assets/components/checkout/index.vue
+++ b/assets/components/checkout/index.vue
@@ -66,6 +66,7 @@
 <script>
 import FormInput from '@/components/checkout/form-input';
 import Loading from '@/components/loading';
+import { createOrder } from '@/services/checkout-service';
 
 export default {
     name: 'CheckoutForm',
@@ -108,6 +109,22 @@ export default {
                 errorMessage: this.validationErrors[id],
             };
         },
+        async onSubmit() {
+            this.loading = true;
+
+            try {
+                const response = await createOrder({
+                    ...this.form,
+                    purchaseItems: this.cart.items,
+                });
+
+                console.log(response.data);
+            } catch (error) {
+                console.log('error', error);
+            } finally {
+                this.loading = false;
+            }
+        },
     },
 };
 </script>
