diff --git a/assets/js/pages/checkout.vue b/assets/js/pages/checkout.vue
index a5077969..9ca3df2c 100644
--- a/assets/js/pages/checkout.vue
+++ b/assets/js/pages/checkout.vue
@@ -75,6 +75,7 @@
 
 <script>
 import cartService from '@/services/cart';
+import checkoutService from '@/services/checkout';
 import FormInput from '@/components/checkout/form-input';
 import Loading from '@/components/loading';
 import TitleComponent from '@/components/title';
@@ -115,6 +116,24 @@ export default {
             quantity: item.qty,
         }));
     },
+    methods: {
+        /**
+         * Sends the form data and process the response!
+         */
+        async onSubmit() {
+            this.loading = true;
+
+            try {
+                const response = await checkoutService.createOrder(this.form);
+
+                console.log(response.data);
+            } catch (e) {
+                console.log('error', e);
+            } finally {
+                this.loading = false;
+            }
+        },
+    },
 };
 </script>
 
