diff --git a/assets/components/checkout/index.vue b/assets/components/checkout/index.vue
index c7668309f..1ea2724e2 100644
--- a/assets/components/checkout/index.vue
+++ b/assets/components/checkout/index.vue
@@ -100,7 +100,8 @@ export default {
                 errorMessage: this.validationErrors[id],
             };
         },
-        async onSubmit() {
+        async onSubmit(event) {
+            event.preventDefault();
             this.loading = true;
 
             try {
