diff --git a/assets/components/checkout/index.vue b/assets/components/checkout/index.vue
index 73ce55bac..1301ae2e7 100644
--- a/assets/components/checkout/index.vue
+++ b/assets/components/checkout/index.vue
@@ -1,7 +1,7 @@
 <template>
     <div class="row p-3">
         <div class="col-12">
-            <form @submit="onSubmit">
+            <form @submit.prevent="onSubmit">
                 <form-input
                     v-model="form.customerName"
                     v-bind="getFieldProps('customerName', 'Name:')"
@@ -101,8 +101,7 @@ export default {
                 errorMessage: this.validationErrors[id],
             };
         },
-        async onSubmit(event) {
-            event.preventDefault();
+        async onSubmit() {
             this.loading = true;
             this.form.purchaseItems = this.cart.items;
 
