diff --git a/assets/components/checkout/index.vue b/assets/components/checkout/index.vue
index 724a49f43..bbbb08b0c 100644
--- a/assets/components/checkout/index.vue
+++ b/assets/components/checkout/index.vue
@@ -57,6 +57,7 @@
 <script>
 import FormInput from '@/components/checkout/form-input';
 import Loading from '@/components/loading';
+import { createOrder } from '@/services/checkout-service';
 
 export default {
     name: 'CheckoutForm',
@@ -100,6 +101,20 @@ export default {
                 errorMessage: this.validationErrors[id],
             };
         },
+        async onSubmit() {
+            this.loading = true;
+            this.form.purchaseItems = this.cart.items;
+
+            try {
+                const response = await createOrder(this.form);
+
+                console.log(response.data);
+            } catch (error) {
+                console.log('error', error);
+            } finally {
+                this.loading = false;
+            }
+        },
     },
 };
 </script>
