diff --git a/assets/js/pages/checkout.vue b/assets/js/pages/checkout.vue
index fbf4b752..0ff2af0a 100644
--- a/assets/js/pages/checkout.vue
+++ b/assets/js/pages/checkout.vue
@@ -123,8 +123,11 @@ export default {
     methods: {
         /**
          * Sends the form data and process the response!
+         *
+         * @param {Event} event
          */
-        async onSubmit() {
+        async onSubmit(event) {
+            event.preventDefault();
             this.loading = true;
 
             try {
