diff --git a/assets/components/checkout/index.vue b/assets/components/checkout/index.vue
index 7c1dff204..5137d7d3b 100644
--- a/assets/components/checkout/index.vue
+++ b/assets/components/checkout/index.vue
@@ -112,6 +112,7 @@ export default {
         async onSubmit() {
             this.loading = true;
             this.serverError = false;
+            this.validationErrors = {};
 
             try {
                 const response = await createOrder({
@@ -126,7 +127,9 @@ export default {
                 if (response.status !== 400) {
                     this.serverError = true;
                 } else {
-                    console.log(response.data);
+                    response.data.violations.forEach((violation) => {
+                        this.validationErrors[violation.propertyPath] = violation.message;
+                    });
                 }
             } finally {
                 this.loading = false;
