diff --git a/assets/js/pages/checkout.vue b/assets/js/pages/checkout.vue
index 247012af..ce32a800 100644
--- a/assets/js/pages/checkout.vue
+++ b/assets/js/pages/checkout.vue
@@ -11,6 +11,13 @@
 
                     <div :class="$style.content">
                         <form @submit="onSubmit">
+                            <div
+                                v-show="serverError"
+                                class="alert alert-danger"
+                            >
+                                Oops, there's been an error sending your data! Please, try again!
+                            </div>
+
                             <form-input
                                 id="customerName"
                                 v-model="form.customerName"
