diff --git a/assets/js/components/checkout/index.vue b/assets/js/components/checkout/index.vue
index 4546214e..2e518adc 100644
--- a/assets/js/components/checkout/index.vue
+++ b/assets/js/components/checkout/index.vue
@@ -104,7 +104,8 @@ export default {
                 errorMessage: this.validationErrors[id],
             };
         },
-        async onSubmit() {
+        async onSubmit(event) {
+            event.preventDefault();
             this.loading = true;
 
             try {
