diff --git a/assets/js/pages/checkout.vue b/assets/js/pages/checkout.vue
index 22798a25..5682add9 100644
--- a/assets/js/pages/checkout.vue
+++ b/assets/js/pages/checkout.vue
@@ -12,7 +12,7 @@
 
                 <div class="row">
                     <div :class="[$style.content, 'col-11']">
-                        <form>
+                        <form @submit="onSubmit">
                             <form-input
                                 id="customerName"
                                 v-model="form.customerName"
