diff --git a/assets/components/checkout/index.vue b/assets/components/checkout/index.vue
index b93b27ccd..731a092c0 100644
--- a/assets/components/checkout/index.vue
+++ b/assets/components/checkout/index.vue
@@ -1,7 +1,7 @@
 <template>
     <div class="row p-3">
         <div class="col-12">
-            <form @submit="onSubmit">
+            <form @submit.prevent="onSubmit">
                 <div class="form-row">
                     <form-input
                         v-model="form.customerName"
@@ -109,8 +109,7 @@ export default {
                 errorMessage: this.validationErrors[id],
             };
         },
-        async onSubmit(event) {
-            event.preventDefault();
+        async onSubmit() {
             this.loading = true;
 
             try {
