diff --git a/assets/components/checkout/index.vue b/assets/components/checkout/index.vue
index 222504fe..0411b430 100644
--- a/assets/components/checkout/index.vue
+++ b/assets/components/checkout/index.vue
@@ -58,6 +58,7 @@
                         <button
                             type="submit"
                             class="btn btn-info btn-lg"
+                            :disabled="loading"
                         >
                             Order!
                         </button>
diff --git a/assets/services/checkout-service.js b/assets/services/checkout-service.js
index 5e5bb6a3..28e53074 100644
--- a/assets/services/checkout-service.js
+++ b/assets/services/checkout-service.js
@@ -7,5 +7,5 @@ import axios from 'axios';
  * @return {Promise}
  */
 export function createOrder(data) {
-    return axios.post('/api/slfjasfd', data);
+    return axios.post('/api/purchases', data);
 }
