diff --git a/assets/components/checkout/index.vue b/assets/components/checkout/index.vue
index 534a4b878..b93b27ccd 100644
--- a/assets/components/checkout/index.vue
+++ b/assets/components/checkout/index.vue
@@ -109,7 +109,8 @@ export default {
                 errorMessage: this.validationErrors[id],
             };
         },
-        async onSubmit() {
+        async onSubmit(event) {
+            event.preventDefault();
             this.loading = true;
 
             try {
