diff --git a/assets/services/checkout-service.js b/assets/services/checkout-service.js
new file mode 100644
index 000000000..bca8d23a7
--- /dev/null
+++ b/assets/services/checkout-service.js
@@ -0,0 +1,15 @@
+import axios from 'axios';
+
+/**
+ * Makes a POST call to create a purchase object after checkout
+ *
+ * @param {Object} data
+ * @return {Promise}
+ */
+export function createOrder(data) {
+    return axios({
+        method: 'post',
+        url: '/api/purchases',
+        data,
+    });
+}
