diff --git a/assets/js/components/checkout/index.vue b/assets/js/components/checkout/index.vue
index 83cc8e96..58618dfc 100644
--- a/assets/js/components/checkout/index.vue
+++ b/assets/js/components/checkout/index.vue
@@ -114,6 +114,7 @@ export default {
             this.loading = true;
             this.form.purchaseItems = this.cart.items;
             this.formError = false;
+            this.validationErrors = {};
 
             try {
                 const response = await createOrder(this.form);
@@ -123,7 +124,9 @@ export default {
                 if (response.status !== 400) {
                     this.formError = true;
                 } else {
-                    console.log(response.data);
+                    response.data.violations.forEach((violation) => {
+                        this.validationErrors[violation.propertyPath] = violation.message;
+                    });
                 }
             } finally {
                 this.loading = false;
