diff --git a/assets/js/pages/checkout.vue b/assets/js/pages/checkout.vue
index 214459c3..fbf4b752 100644
--- a/assets/js/pages/checkout.vue
+++ b/assets/js/pages/checkout.vue
@@ -12,7 +12,7 @@
 
                 <div class="row">
                     <div :class="$style.content">
-                        <form>
+                        <form @submit="onSubmit">
                             <form-input
                                 id="customerName"
                                 v-model="form.customerName"
