diff --git a/assets/services/checkout-service.js b/assets/services/checkout-service.js
index bca8d23a7..decb83e83 100644
--- a/assets/services/checkout-service.js
+++ b/assets/services/checkout-service.js
@@ -9,7 +9,7 @@ import axios from 'axios';
 export function createOrder(data) {
     return axios({
         method: 'post',
-        url: '/api/purchases',
+        url: '/api/slfjasfd',
         data,
     });
 }
