diff --git a/assets/js/pages/checkout.vue b/assets/js/pages/checkout.vue
index 7fd68246..be721d24 100644
--- a/assets/js/pages/checkout.vue
+++ b/assets/js/pages/checkout.vue
@@ -159,6 +159,20 @@ export default {
                 this.loading = false;
             }
         },
+
+        /**
+         * Resets our validation fields back to null
+         */
+        resetValidationFields() {
+            this.validationErrors = {
+                customerName: null,
+                customerEmail: null,
+                customerAddress: null,
+                customerZip: null,
+                customerCity: null,
+                customerPhone: null,
+            };
+        },
     },
 };
 </script>
