diff --git a/assets/components/checkout/index.vue b/assets/components/checkout/index.vue
index cc8480edb..372dc7579 100644
--- a/assets/components/checkout/index.vue
+++ b/assets/components/checkout/index.vue
@@ -83,6 +83,7 @@ export default {
             },
             validationErrors: {},
             loading: false,
+            serverError: false,
         };
     },
     methods: {
@@ -102,6 +103,7 @@ export default {
         },
         async onSubmit() {
             this.loading = true;
+            this.serverError = false;
 
             try {
                 const response = await createOrder({
@@ -111,7 +113,13 @@ export default {
 
                 console.log(response.data);
             } catch (error) {
-                console.log('error', error);
+                const { response } = error;
+
+                if (response.status !== 400) {
+                    this.serverError = true;
+                } else {
+                    console.log(response.data);
+                }
             } finally {
                 this.loading = false;
             }
