diff --git a/assets/js/pages/checkout.vue b/assets/js/pages/checkout.vue
index 08830d7..332a785 100644
--- a/assets/js/pages/checkout.vue
+++ b/assets/js/pages/checkout.vue
@@ -130,13 +130,18 @@ export default {
         async onSubmit(event) {
             event.preventDefault();
             this.loading = true;
+            this.formError = false;
 
             try {
                 const response = await checkoutService.createOrder(this.form);
-
-                console.log(response.data);
-            } catch (e) {
-                console.log('error', e);
+            } catch (error) {
+                const { response } = error;
+
+                if (response.status !== 400) {
+                    this.formError = true;
+                } else {
+                    console.log(response.data);
+                }
             } finally {
                 this.loading = false;
             }
