diff --git a/assets/components/checkout/index.vue b/assets/components/checkout/index.vue
index 23ff2a950..c7668309f 100644
--- a/assets/components/checkout/index.vue
+++ b/assets/components/checkout/index.vue
@@ -1,7 +1,7 @@
 <template>
     <div class="row p-3">
         <div class="col-12">
-            <form>
+            <form @submit="onSubmit">
                 <form-input
                     v-model="form.customerName"
                     v-bind="getFieldProps('customerName', 'Name:')"
