diff --git a/assets/services/checkout-service.js b/assets/services/checkout-service.js
index 28e530740..5e5bb6a3a 100644
--- a/assets/services/checkout-service.js
+++ b/assets/services/checkout-service.js
@@ -7,5 +7,5 @@ import axios from 'axios';
  * @return {Promise}
  */
 export function createOrder(data) {
-    return axios.post('/api/purchases', data);
+    return axios.post('/api/slfjasfd', data);
 }
