diff --git a/assets/js/pages/checkout.vue b/assets/js/pages/checkout.vue
index 4b2b0965..214459c3 100644
--- a/assets/js/pages/checkout.vue
+++ b/assets/js/pages/checkout.vue
@@ -77,6 +77,7 @@
 
 <script>
 import cartService from '@/services/cart';
+import { createOrder } from '@/services/checkout-service';
 import FormInput from '@/components/checkout/form-input';
 import Loading from '@/components/loading';
 import TitleComponent from '@/components/title';
@@ -119,6 +120,24 @@ export default {
             quantity: item.qty,
         }));
     },
+    methods: {
+        /**
+         * Sends the form data and process the response!
+         */
+        async onSubmit() {
+            this.loading = true;
+
+            try {
+                const response = await createOrder(this.form);
+
+                console.log(response.data);
+            } catch (e) {
+                console.log('error', e);
+            } finally {
+                this.loading = false;
+            }
+        },
+    },
 };
 </script>
 
