diff --git a/assets/components/checkout/index.vue b/assets/components/checkout/index.vue
index a6746e9b..b022700b 100644
--- a/assets/components/checkout/index.vue
+++ b/assets/components/checkout/index.vue
@@ -63,6 +63,7 @@
 <script>
 import FormInput from '@/components/checkout/form-input';
 import Loading from '@/components/loading';
+import { createOrder } from '@/services/checkout-service';
 
 export default {
     name: 'CheckoutForm',
@@ -99,6 +100,22 @@ export default {
                 errorMessage: this.validationErrors[id],
             };
         },
+        async onSubmit() {
+            this.loading = true;
+
+            try {
+                const response = await createOrder({
+                    ...this.form,
+                    purchaseItems: [],
+                });
+
+                console.log(response.data);
+            } catch (error) {
+                console.error(error.response);
+            } finally {
+                this.loading = false;
+            }
+        },
     },
 };
 </script>
