diff --git a/assets/js/components/checkout/index.vue b/assets/js/components/checkout/index.vue
index 3eb46b2c..d39b62cb 100644
--- a/assets/js/components/checkout/index.vue
+++ b/assets/js/components/checkout/index.vue
@@ -55,6 +55,7 @@
 <script>
 import FormInput from '@/components/checkout/form-input';
 import Loading from '@/components/loading';
+import { createOrder } from '@/services/checkout-service';
 
 export default {
     name: 'CheckoutForm',
@@ -103,6 +104,19 @@ export default {
                 errorMessage: this.validationErrors[id],
             };
         },
+        async onSubmit() {
+            this.loading = true;
+
+            try {
+                const response = await createOrder(this.form);
+
+                console.log(response.data);
+            } catch (e) {
+                console.log('error', e);
+            } finally {
+                this.loading = false;
+            }
+        },
     },
 };
 </script>
