diff --git a/assets/components/checkout/index.vue b/assets/components/checkout/index.vue
index 1301ae2e7..ececd6729 100644
--- a/assets/components/checkout/index.vue
+++ b/assets/components/checkout/index.vue
@@ -84,6 +84,7 @@ export default {
             },
             validationErrors: {},
             loading: false,
+            formError: false,
         };
     },
     methods: {
@@ -104,13 +105,18 @@ export default {
         async onSubmit() {
             this.loading = true;
             this.form.purchaseItems = this.cart.items;
+            this.formError = false;
 
             try {
                 const response = await createOrder(this.form);
-
-                console.log(response.data);
             } catch (error) {
-                console.log('error', error);
+                const { response } = error;
+
+                if (response.status !== 400) {
+                    this.formError = true;
+                } else {
+                    console.log(response.data);
+                }
             } finally {
                 this.loading = false;
             }
