diff --git a/assets/js/pages/checkout.vue b/assets/js/pages/checkout.vue
index e28fc84c..100dcbf9 100644
--- a/assets/js/pages/checkout.vue
+++ b/assets/js/pages/checkout.vue
@@ -126,8 +126,11 @@ export default {
     methods: {
         /**
          * Sends the form data and process the response!
+         *
+         * @param {Event} event
          */
-        async onSubmit() {
+        async onSubmit(event) {
+            event.preventDefault();
             this.loading = true;
 
             try {
@@ -135,7 +138,7 @@ export default {
 
                 console.log(response.data);
             } catch (e) {
-                // TODO Error handling
+                console.log('error', e);
             } finally {
                 this.loading = false;
             }
