diff --git a/assets/components/checkout/index.vue b/assets/components/checkout/index.vue
index 94dfafd32..58f47bec9 100644
--- a/assets/components/checkout/index.vue
+++ b/assets/components/checkout/index.vue
@@ -33,6 +33,22 @@
                     type="tel"
                     v-bind="getFieldProps('customerPhone', 'Phone Number:')"
                 />
+
+                <div class="row p-3 justify-content-end align-items-center">
+                    <loading
+                        v-show="loading"
+                        class="col-auto"
+                    />
+
+                    <div class="col-auto">
+                        <button
+                            type="submit"
+                            class="btn btn-info btn-lg"
+                        >
+                            Order!
+                        </button>
+                    </div>
+                </div>
             </form>
         </div>
     </div>
@@ -40,11 +56,13 @@
 
 <script>
 import FormInput from '@/components/checkout/form-input';
+import Loading from '@/components/loading';
 
 export default {
     name: 'CheckoutForm',
     components: {
         FormInput,
+        Loading,
     },
     props: {
         cart: {
@@ -63,6 +81,7 @@ export default {
                 customerPhone: '',
             },
             validationErrors: {},
+            loading: false,
         };
     },
     methods: {
