diff --git a/assets/js/pages/checkout.vue b/assets/js/pages/checkout.vue
index b1e66261..f6307916 100644
--- a/assets/js/pages/checkout.vue
+++ b/assets/js/pages/checkout.vue
@@ -70,6 +70,7 @@
                                 <button
                                     type="submit"
                                     class="btn btn-info btn-sm"
+                                    :disabled="loading"
                                 >
                                     Checkout!
                                 </button>
diff --git a/assets/js/services/checkout-service.js b/assets/js/services/checkout-service.js
index decb83e8..bca8d23a 100644
--- a/assets/js/services/checkout-service.js
+++ b/assets/js/services/checkout-service.js
@@ -9,7 +9,7 @@ import axios from 'axios';
 export function createOrder(data) {
     return axios({
         method: 'post',
-        url: '/api/slfjasfd',
+        url: '/api/purchases',
         data,
     });
 }
