diff --git a/assets/components/checkout/index.vue b/assets/components/checkout/index.vue
index 731a092c0..4a05e65c7 100644
--- a/assets/components/checkout/index.vue
+++ b/assets/components/checkout/index.vue
@@ -92,6 +92,7 @@ export default {
             },
             validationErrors: {},
             loading: false,
+            serverError: false,
         };
     },
     methods: {
@@ -111,6 +112,7 @@ export default {
         },
         async onSubmit() {
             this.loading = true;
+            this.serverError = false;
 
             try {
                 const response = await createOrder({
@@ -120,7 +122,13 @@ export default {
 
                 console.log(response.data);
             } catch (error) {
-                console.log('error', error);
+                const { response } = error;
+
+                if (response.status !== 400) {
+                    this.serverError = true;
+                } else {
+                    console.log(response.data);
+                }
             } finally {
                 this.loading = false;
             }
