diff --git a/assets/js/components/checkout/index.vue b/assets/js/components/checkout/index.vue
index 9702ccee..73ce55ba 100644
--- a/assets/js/components/checkout/index.vue
+++ b/assets/js/components/checkout/index.vue
@@ -101,7 +101,8 @@ export default {
                 errorMessage: this.validationErrors[id],
             };
         },
-        async onSubmit() {
+        async onSubmit(event) {
+            event.preventDefault();
             this.loading = true;
             this.form.purchaseItems = this.cart.items;
 
