diff --git a/assets/pages/products.vue b/assets/pages/products.vue
index 5f84c878..00a0e3cc 100644
--- a/assets/pages/products.vue
+++ b/assets/pages/products.vue
@@ -22,6 +22,7 @@
 </template>
 
 <script>
+import { ref } from 'vue';
 import Catalog from '@/components/catalog';
 import ProductShow from '@/components/product-show';
 import Sidebar from '@/components/sidebar';
@@ -75,27 +76,11 @@ export default {
         },
     },
     setup() {
-        class FirstName {
-            constructor(value) {
-                this._value = value;
-            }
-
-            get value() {
-                console.log('Value is being read!');
-                return this._value;
-            }
-
-            set value(newValue) {
-                console.log('Value is being set!');
-                this._value = newValue;
-            }
-        }
-
-        const firstName = new FirstName('Matias');
+        const firstName = ref('Matias');
         window.firstName = firstName;
 
         return {
-            firstName: firstName.value,
+            firstName,
         };
     },
 };
