diff --git a/assets/pages/products.vue b/assets/pages/products.vue
index a7c192b7..7656bf62 100644
--- a/assets/pages/products.vue
+++ b/assets/pages/products.vue
@@ -8,9 +8,6 @@
                     :categories="categories"
                     @toggle-collapsed="toggleSidebarCollapsed"
                 />
-                {{ person.firstName }}
-
-                {{ person.collapsedMessage }}
             </aside>
 
             <div :class="contentClass">
@@ -24,7 +21,6 @@
 </template>
 
 <script>
-import { reactive } from 'vue';
 import Catalog from '@/components/catalog';
 import ProductShow from '@/components/product-show';
 import Sidebar from '@/components/sidebar';
@@ -76,21 +72,6 @@ export default {
         toggleSidebarCollapsed() {
             this.sidebarCollapsed = !this.sidebarCollapsed;
         },
-        printFirstName() {
-            return this.person.firstName;
-        },
-    },
-    setup() {
-        const person = reactive({
-            firstName: 'Matias',
-            collapsedMessage: this.sidebarCollapsed ? 'Collapsed' : 'Expanded',
-        });
-
-        window.person = person;
-
-        return {
-            person,
-        };
     },
 };
 </script>
