diff --git a/assets/pages/products.vue b/assets/pages/products.vue
index b09263d1..c4fdddb8 100644
--- a/assets/pages/products.vue
+++ b/assets/pages/products.vue
@@ -10,7 +10,7 @@
                 />
                 {{ person.firstName }}
 
-                {{ printFirstname() }}
+                {{ person.collapsedMessage }}
             </aside>
 
             <div :class="contentClass">
@@ -83,6 +83,7 @@ export default {
     setup() {
         const person = reactive({
             firstName: 'Matias',
+            collapsedMessage: this.sidebarCollapsed ? 'Collapsed' : 'Expanded',
         });
 
         window.person = person;
