diff --git a/assets/pages/products.vue b/assets/pages/products.vue
index 7656bf62..562533e0 100644
--- a/assets/pages/products.vue
+++ b/assets/pages/products.vue
@@ -8,6 +8,7 @@
                     :categories="categories"
                     @toggle-collapsed="toggleSidebarCollapsed"
                 />
+                {{ firstName }}
             </aside>
 
             <div :class="contentClass">
@@ -73,5 +74,10 @@ export default {
             this.sidebarCollapsed = !this.sidebarCollapsed;
         },
     },
+    setup() {
+        return {
+            firstName: 'Matias',
+        };
+    },
 };
 </script>
