diff --git a/assets/js/pages/products.vue b/assets/js/pages/products.vue
index 65d5ff63..94e5c723 100644
--- a/assets/js/pages/products.vue
+++ b/assets/js/pages/products.vue
@@ -22,6 +22,7 @@
 </template>
 
 <script>
+import { ref } from 'vue';
 import Catalog from '@/components/catalog';
 import Product from '@/components/product';
 import Sidebar from '@/components/sidebar';
@@ -80,27 +81,11 @@ export default {
         },
     },
     setup() {
-        class FirstName {
-            constructor(value) {
-                this._value = value;
-            }
-
-            get value() {
-                console.log('Value is being read!');
-                return this._value;
-            }
-
-            set value(newValue) {
-                console.log('Value is being set!');
-                this._value = newValue;
-            }
-        }
-
-        const firstName = new FirstName('Ryan');
+        const firstName = ref('Ryan');
         window.firstName = firstName;
 
         return {
-            firstName: firstName.value,
+            firstName,
         };
     },
 };
