diff --git a/assets/pages/products.vue b/assets/pages/products.vue
index aaa22ec4..7e87cac1 100644
--- a/assets/pages/products.vue
+++ b/assets/pages/products.vue
@@ -75,8 +75,11 @@ export default {
         },
     },
     setup() {
+        const firstName = 'Matias';
+        window.firstName = firstName;
+
         return {
-            firstName: 'Matias',
+            firstName,
         };
     },
 };
