diff --git a/assets/js/pages/products.vue b/assets/js/pages/products.vue
index ca1dc591..7e2e8632 100644
--- a/assets/js/pages/products.vue
+++ b/assets/js/pages/products.vue
@@ -80,7 +80,23 @@ export default {
         },
     },
     setup() {
-        const firstName = 'Ryan';
+        class FirstName {
+            constructor(value) {
+                this._value = value;
+            }
+
+            get value() {
+                console.log('Value is being read!');
+                return this._value;
+            }
+
+            set value(newValue) {
+                console.log('Value is being set!');
+                this._value = newValue;
+            }
+        }
+
+        const firstName = new FirstName('Ryan');
         window.firstName = firstName;
 
         return {
