diff --git a/assets/pages/products.vue b/assets/pages/products.vue
index 2b6f77e2..f79c95d1 100644
--- a/assets/pages/products.vue
+++ b/assets/pages/products.vue
@@ -9,6 +9,8 @@
                     @toggle-collapsed="toggleSidebarCollapsed"
                 />
                 {{ person.firstName }}
+
+                {{ printFirstName() }}
             </aside>
 
             <div :class="contentClass">
@@ -74,6 +76,9 @@ export default {
         toggleSidebarCollapsed() {
             this.sidebarCollapsed = !this.sidebarCollapsed;
         },
+        printFirstName() {
+            return this.person.firstName;
+        },
     },
     setup() {
         const person = reactive({
