diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index 7729d6546..a879ef786 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -52,6 +52,7 @@ export default {
                 const color = this.colors.find((colorItem) => colorItem['@id'] === cartItem.color);
 
                 return {
+                    id: `${cartItem.product}_${cartItem.color ? cartItem.color : 'none'}`,
                     product,
                     color,
                     quantity: cartItem.quantity,
