diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index 0de4fb41a..bd751a0f5 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -15,7 +15,7 @@
                             v-for="(cartItem, index) in completeCartItems"
                             :key="index"
                         >
-                            {{ cartItem.name }}
+                            {{ cartItem.product.name }} ({{ cartItem.quantity }})
                         </div>
                     </div>
                 </div>
@@ -58,7 +58,6 @@ export default {
             const productIds = this.cart.items.map((item) => item.product);
 
             const productsResponse = await fetchProductsById(productIds);
-            this.completeCartItems = productsResponse.data['hydra:member'];
 
             const colorsResponse = await fetchColors();
             // Map all colors to our object dictionary by @id
@@ -66,6 +65,17 @@ export default {
                 this.colors[color['@id']] = color;
             });
 
+            // Create items with complete product, color & quantity info
+            this.completeCartItems = productsResponse.data['hydra:member'].map((product) => {
+                const cartItem = this.cart.items.find((item) => item.product === product['@id']);
+
+                return {
+                    product,
+                    color: cartItem.color ? this.colors[cartItem.color] : null,
+                    quantity: cartItem.quantity,
+                };
+            });
+
             this.itemsLoading = false;
         },
     },
