diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index b7af9b20..47d9ecf0 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -20,6 +20,27 @@
                                 Sorry! You haven't bought anything yet!
                             </div>
                         </div>
+
+                        <div
+                            v-show="!loading && items.length"
+                            :class="$style['product-list']"
+                        >
+                            <div :class="[$style['product-row'], $style['product-head']]">
+                                <span>Item Name</span>
+                                <span>Quantity</span>
+                                <span>Price</span>
+                            </div>
+
+                            <shopping-cart-item
+                                v-for="item in items"
+                                :key="item['@id']"
+                                :product="item"
+                            />
+
+                            <div :class="$style['product-row']">
+                                <span>Total: <strong>${{ totalPrice }}</strong></span>
+                            </div>
+                        </div>
                     </div>
                 </div>
             </div>
