diff --git a/assets/services/cart-service.js b/assets/services/cart-service.js
index af358f847..c43867ea1 100644
--- a/assets/services/cart-service.js
+++ b/assets/services/cart-service.js
@@ -144,8 +144,14 @@ export function getCartTotalItems(cart) {
     return cart.items.reduce((acc, item) => (acc + item.quantity), 0);
 }
 
+/**
+ * Gets the full info on the shopping cart
+ *
+ * @param {CartCollection} cart
+ * @returns {Promise}
+ */
 export async function fetchFullCart(cart) {
-    const productIds = this.cart.items.map((item) => item.product);
+    const productIds = cart.items.map((item) => item.product);
 
     const [colorsResponse, productsResponse] = await Promise.all([
         fetchColors(),
@@ -160,7 +166,7 @@ export async function fetchFullCart(cart) {
 
     const products = productsResponse.data['hydra:member'];
 
-    const completeItems = this.cart.items.map((cartItem) => (
+    const completeItems = cart.items.map((cartItem) => (
         {
             product: products.find((product) => product['@id'] === cartItem.product),
             color: cartItem.color ? mappedColors[cartItem.color] : null,
@@ -168,7 +174,7 @@ export async function fetchFullCart(cart) {
         }
     ));
 
-    this.completeCart = {
+    return {
         items: completeItems,
     };
 }
