diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index 3379768b..f0b341ba 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -55,6 +55,12 @@ export default {
     },
     async created() {
         const itemsInCart = cartService.getItems();
+
+        if (!itemsInCart.length) {
+            this.loading = false;
+            return;
+        }
+
         const productIds = itemsInCart.map((item) => item.productId);
 
         let colorsResponse = null;
