diff --git a/assets/services/cart-service.js b/assets/services/cart-service.js
index 45ae9441b..bf2b23abd 100644
--- a/assets/services/cart-service.js
+++ b/assets/services/cart-service.js
@@ -144,23 +144,31 @@ export function getCartTotalItems(cart) {
     return cart.items.reduce((acc, item) => (acc + item.quantity), 0);
 }
 
+/**
+ * Gets the full info on the shopping cart
+ *
+ * @param {CartCollection} cart
+ * @returns {Array}
+ */
 export async function getFullShoppingCart(cart) {
-    const productIds = this.cart.items.map((item) => item.product);
+    const productIds = cart.items.map((item) => item.product);
 
     const [colorsResponse, productsResponse] = await Promise.all([
         getColors(),
         fetchProductsById(productIds),
     ]);
 
+    const mappedColors = {};
+
     // Map all colors to our object dictionary by @id
     colorsResponse.data['hydra:member'].forEach((color) => {
-        this.colors[color['@id']] = color;
+        mappedColors[color['@id']] = color;
     });
 
     // Assign our returned products to our products array,
     // applying the proper colorId, hexColor and qty values
-    this.completeCartItems = productsResponse.data['hydra:member'].map((product) => {
-        const productInCart = this.cart.items.find(
+    return this.completeCartItems = productsResponse.data['hydra:member'].map((product) => {
+        const productInCart = cart.items.find(
             (item) => (item.product === product['@id']),
         );
 
@@ -168,7 +176,7 @@ export async function getFullShoppingCart(cart) {
             ...product,
             colorId: productInCart.color,
             hexColor: productInCart.color
-                ? this.colors[productInCart.color].hexColor
+                ? mappedColors[productInCart.color].hexColor
                 : 'fff',
             qty: productInCart.quantity,
         };
