diff --git a/assets/components/shopping-cart/index.vue b/assets/components/shopping-cart/index.vue
index 3556be6da..2f9c697f7 100644
--- a/assets/components/shopping-cart/index.vue
+++ b/assets/components/shopping-cart/index.vue
@@ -53,13 +53,9 @@ export default {
          * @return {string}
          */
         totalPrice() {
-            let total = 0;
-
-            this.items.forEach((item) => {
-                total += item.product.price * item.quantity;
-            });
-
-            return formatPrice(total);
+            return formatPrice(
+                this.items.reduce((acc, item) => (acc + (item.product.price * item.quantity)), 0),
+            );
         },
     },
 };
