diff --git a/assets/js/services/products-service.js b/assets/js/services/products-service.js
index 41780cfa..d5f9817f 100644
--- a/assets/js/services/products-service.js
+++ b/assets/js/services/products-service.js
@@ -29,3 +29,25 @@ export function fetchProducts(categoryIri, searchTerm) {
 export function getProduct(productIri) {
     return axios.get(productIri);
 }
+
+/**
+ * Retrieves a set of products identified by an array of IRIs
+ *
+ * @param {string[]} ids
+ * @return {Promise}
+ */
+export function getProductsById(ids) {
+    if (!ids.length) {
+        return Promise.resolve({
+            data: {
+                'hydra:member': [],
+            },
+        });
+    }
+
+    return axios({
+        method: 'get',
+        url: '/api/products',
+        params: { id: ids },
+    });
+}
