diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index a8e396df..4d360c8f 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -47,12 +47,14 @@ import cartService from '@/services/cart';
 import colorsService from '@/services/colors';
 import productsService from '@/services/products';
 import Loading from '@/components/loading';
+import ShoppingCartItem from '@/components/shopping-cart/cart-item';
 import TitleComponent from '@/components/title';
 
 export default {
     name: 'ShoppingCart',
     components: {
         Loading,
+        ShoppingCartItem,
         TitleComponent,
     },
     data: () => ({
