diff --git a/assets/js/components/shopping-cart/cart-item.vue b/assets/js/components/shopping-cart/cart-item.vue
index 04e800d6..e1706cc3 100644
--- a/assets/js/components/shopping-cart/cart-item.vue
+++ b/assets/js/components/shopping-cart/cart-item.vue
@@ -9,6 +9,12 @@
 <script>
 export default {
     name: 'ShoppingCartProduct',
+    props: {
+        product: {
+            type: Object,
+            required: true,
+        },
+    },
 };
 </script>
 
