diff --git a/assets/js/components/shopping-cart/index.vue b/assets/js/components/shopping-cart/index.vue
index 2f5bf8c2..c425f197 100644
--- a/assets/js/components/shopping-cart/index.vue
+++ b/assets/js/components/shopping-cart/index.vue
@@ -40,6 +40,7 @@
 
 <script>
 import ShoppingCartItem from '@/components/shopping-cart/cart-item';
+import formatPrice from '@/helpers/format-price';
 
 export default {
     name: 'ShoppingCartList',
@@ -52,5 +53,21 @@ export default {
             required: true,
         },
     },
+    computed: {
+        /**
+         * Returns the formatted total price of the list
+         *
+         * @return {string}
+         */
+        totalPrice() {
+            let total = 0;
+
+            this.items.forEach((item) => {
+                total += item.price * item.qty;
+            });
+
+            return formatPrice(total);
+        },
+    },
 };
 </script>
