diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index 4b03f123..c631aa78 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -56,6 +56,7 @@
 import cartService from '@/services/cart-service';
 import { getProductsById } from '@/services/products-service';
 import { getColors } from '@/services/colors-service';
+import formatPrice from '@/helpers/format-price';
 import Loading from '@/components/loading';
 import ShoppingCartItem from '@/components/shopping-cart/cart-item';
 import TitleComponent from '@/components/title';
@@ -74,6 +75,22 @@ export default {
             loading: true,
         };
     },
+    computed: {
+        /**
+         * Returns the formatted total price of the list
+         *
+         * @return {string}
+         */
+        totalPrice() {
+            let total = 0;
+
+            this.items.forEach((item) => {
+                total += item.price * item.qty;
+            });
+
+            return formatPrice(total);
+        },
+    },
     async created() {
         const itemsInCart = cartService.getItems();
 
