diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index 4aa9d9ba..7ddd92f4 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -55,12 +55,14 @@ import cartService from '@/services/cart';
 import { getProductsById } from '@/services/products-service';
 import { getColors } from '@/services/colors-service';
 import Loading from '@/components/loading';
+import ShoppingCartItem from '@/components/shopping-cart/cart-item';
 import TitleComponent from '@/components/title';
 
 export default {
     name: 'ShoppingCart',
     components: {
         Loading,
+        ShoppingCartItem,
         TitleComponent,
     },
     data() {
