diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index bd751a0f5..efcac4632 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -57,9 +57,11 @@ export default {
 
             const productIds = this.cart.items.map((item) => item.product);
 
-            const productsResponse = await fetchProductsById(productIds);
+            const [colorsResponse, productsResponse] = await Promise.all([
+                fetchColors(),
+                fetchProductsById(productIds),
+            ]);
 
-            const colorsResponse = await fetchColors();
             // Map all colors to our object dictionary by @id
             colorsResponse.data['hydra:member'].forEach((color) => {
                 this.colors[color['@id']] = color;
