diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index 6269cb8d..fba33cfe 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -26,6 +26,7 @@
 </template>
 
 <script>
+import { getProductsById } from '@/services/products-service';
 import { getColors } from '@/services/colors-service';
 import Loading from '@/components/loading';
 import TitleComponent from '@/components/title';
@@ -41,15 +42,20 @@ export default {
     data() {
         return {
             colors: {},
+            items: [],
             loading: true,
         };
     },
     async created() {
+        const productIds = this.cart.items.map((item) => item.product);
+
         let colorsResponse = null;
+        let productsResponse = null;
 
-        // Retrieve colors from the server
+        // Retrieve both colors and products from the server
         try {
             colorsResponse = await getColors();
+            productsResponse = await getProductsById(productIds);
         } catch (e) {
             this.loading = false;
 
@@ -61,6 +67,8 @@ export default {
             this.colors[color['@id']] = color;
         });
 
+        this.items = productsResponse.data['hydra:member'];
+
         this.loading = false;
     },
 };
