diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index e291e96a..cd0d5d52 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -16,18 +16,10 @@
                         <loading />
                     </div>
 
-                    <div
+                    <shopping-cart-list
                         v-show="!loading"
-                        class="p-3"
-                    >
-                        <div
-                            v-for="(cartItem, index) in items"
-                            :key="index"
-                            class="p-3"
-                        >
-                            {{ cartItem.name }}
-                        </div>
-                    </div>
+                        :items="items"
+                    />
                 </div>
             </div>
         </div>
@@ -38,6 +30,7 @@
 import { getFullShoppingCart } from '@/services/cart-service';
 import Loading from '@/components/loading';
 import TitleComponent from '@/components/title';
+import ShoppingCartList from '@/components/shopping-cart';
 import shoppingCartMixin from '@/mixins/get-shopping-cart';
 
 export default {
@@ -45,6 +38,7 @@ export default {
     components: {
         Loading,
         TitleComponent,
+        ShoppingCartList,
     },
     mixins: [shoppingCartMixin],
     data() {
