diff --git a/assets/js/components/shopping-cart/index.vue b/assets/js/components/shopping-cart/index.vue
new file mode 100644
index 00000000..f145b5e5
--- /dev/null
+++ b/assets/js/components/shopping-cart/index.vue
@@ -0,0 +1,22 @@
+<template>
+    <div>
+        <div
+            v-show="!items.length"
+            class="p-3"
+        >
+            Sorry! You haven't bought anything yet!
+        </div>
+    </div>
+</template>
+
+<script>
+export default {
+    name: 'ShoppingCartList',
+    props: {
+        items: {
+            type: Array,
+            required: true,
+        },
+    },
+};
+</script>
