diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index ecedb5f10..91cf30349 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -56,39 +56,10 @@ export default {
             this.completeCartItems = [];
 
             if (!this.cart.items.length) {
-                this.loading = false;
+                this.itemsLoading = false;
                 return;
             }
 
-            const productIds = this.cart.items.map((item) => item.product);
-
-            const [colorsResponse, productsResponse] = await Promise.all([
-                getColors(),
-                fetchProductsById(productIds),
-            ]);
-
-            // Map all colors to our object dictionary by @id
-            colorsResponse.data['hydra:member'].forEach((color) => {
-                this.colors[color['@id']] = color;
-            });
-
-            // Assign our returned products to our products array,
-            // applying the proper colorId, hexColor and qty values
-            this.completeCartItems = productsResponse.data['hydra:member'].map((product) => {
-                const productInCart = this.cart.items.find(
-                    (item) => (item.product === product['@id']),
-                );
-
-                return {
-                    ...product,
-                    colorId: productInCart.color,
-                    hexColor: productInCart.color
-                        ? this.colors[productInCart.color].hexColor
-                        : 'fff',
-                    qty: productInCart.quantity,
-                };
-            });
-
             this.itemsLoading = false;
         },
     },
diff --git a/assets/services/cart-service.js b/assets/services/cart-service.js
index 03fd8884e..45ae9441b 100644
--- a/assets/services/cart-service.js
+++ b/assets/services/cart-service.js
@@ -1,4 +1,6 @@
 import axios from 'axios';
+import { getColors } from '@/services/colors-service';
+import { fetchProductsById } from '@/services/products-service';
 
 /**
  * @typedef {Object} CartCollection
@@ -141,3 +143,34 @@ export function clearCart() {
 export function getCartTotalItems(cart) {
     return cart.items.reduce((acc, item) => (acc + item.quantity), 0);
 }
+
+export async function getFullShoppingCart(cart) {
+    const productIds = this.cart.items.map((item) => item.product);
+
+    const [colorsResponse, productsResponse] = await Promise.all([
+        getColors(),
+        fetchProductsById(productIds),
+    ]);
+
+    // Map all colors to our object dictionary by @id
+    colorsResponse.data['hydra:member'].forEach((color) => {
+        this.colors[color['@id']] = color;
+    });
+
+    // Assign our returned products to our products array,
+    // applying the proper colorId, hexColor and qty values
+    this.completeCartItems = productsResponse.data['hydra:member'].map((product) => {
+        const productInCart = this.cart.items.find(
+            (item) => (item.product === product['@id']),
+        );
+
+        return {
+            ...product,
+            colorId: productInCart.color,
+            hexColor: productInCart.color
+                ? this.colors[productInCart.color].hexColor
+                : 'fff',
+            qty: productInCart.quantity,
+        };
+    });
+}
