diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index 8ef5f226..6cd59826 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -57,6 +57,13 @@ export default {
     watch: {
         'cart.items.length': async function watchCartItems() {
             this.loading = true;
+
+            if (!this.cart.items.length) {
+                this.items = [];
+                this.loading = false;
+                return;
+            }
+
             const productIds = this.cart.items.map((item) => item.product);
 
             let colorsResponse = null;
