diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index e4998f658..7327564ac 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -52,32 +52,6 @@ export default {
                 this.itemsLoading = false;
                 return;
             }
-
-            const productIds = this.cart.items.map((item) => item.product);
-
-            const [colorsResponse, productsResponse] = await Promise.all([
-                fetchColors(),
-                fetchProductsById(productIds),
-            ]);
-
-            // Map all colors to our object dictionary by @id
-            colorsResponse.data['hydra:member'].forEach((color) => {
-                this.colors[color['@id']] = color;
-            });
-
-            const completeItems = productsResponse.data['hydra:member'].map((product) => {
-                const cartItem = this.cart.items.find((item) => item.product === product['@id']);
-
-                return {
-                    product,
-                    color: cartItem.color ? this.colors[cartItem.color] : null,
-                    quantity: cartItem.quantity,
-                };
-            });
-
-            this.completeCart = {
-                items: completeItems,
-            };
         },
     },
 };
diff --git a/assets/services/cart-service.js b/assets/services/cart-service.js
index a948b5937..f8bc2d7b4 100644
--- a/assets/services/cart-service.js
+++ b/assets/services/cart-service.js
@@ -1,4 +1,6 @@
 import axios from 'axios';
+import { fetchColors } from '@/services/colors-service';
+import { fetchProductsById } from '@/services/products-service';
 
 /**
  * @typedef {Object} CartCollection
@@ -141,3 +143,31 @@ export function clearCart() {
 export function getCartTotalItems(cart) {
     return cart.items.reduce((acc, item) => (acc + item.quantity), 0);
 }
+
+export async function getFullShoppingCart(cart) {
+    const productIds = this.cart.items.map((item) => item.product);
+
+    const [colorsResponse, productsResponse] = await Promise.all([
+        fetchColors(),
+        fetchProductsById(productIds),
+    ]);
+
+    // Map all colors to our object dictionary by @id
+    colorsResponse.data['hydra:member'].forEach((color) => {
+        this.colors[color['@id']] = color;
+    });
+
+    const completeItems = productsResponse.data['hydra:member'].map((product) => {
+        const cartItem = this.cart.items.find((item) => item.product === product['@id']);
+
+        return {
+            product,
+            color: cartItem.color ? this.colors[cartItem.color] : null,
+            quantity: cartItem.quantity,
+        };
+    });
+
+    this.completeCart = {
+        items: completeItems,
+    };
+}
