diff --git a/assets/js/components/shopping-cart/cart-item.vue b/assets/js/components/shopping-cart/cart-item.vue
index 67f60ab8..a926cba7 100644
--- a/assets/js/components/shopping-cart/cart-item.vue
+++ b/assets/js/components/shopping-cart/cart-item.vue
@@ -1,6 +1,6 @@
 <template>
     <div :class="[$style.component, 'row', 'p-3']">
-        <span class="col-12">{{ product.name }}</span>
+        <div class="col-12">{{ product.name }}</div>
     </div>
 </template>
 
diff --git a/assets/js/components/shopping-cart/index.vue b/assets/js/components/shopping-cart/index.vue
index 7d1a3ddc..2f5bf8c2 100644
--- a/assets/js/components/shopping-cart/index.vue
+++ b/assets/js/components/shopping-cart/index.vue
@@ -8,12 +8,44 @@
                 Sorry! You haven't bought anything yet!
             </div>
         </div>
+
+        <div v-show="items.length">
+            <div class="row p-3">
+                <div class="col-3">
+                    Item Name
+                </div>
+                <div class="col-3">
+                    Quantity
+                </div>
+                <div class="col-3">
+                    Price
+                </div>
+                <div class="col-3" />
+            </div>
+
+            <shopping-cart-item
+                v-for="item in items"
+                :key="item['@id']"
+                :product="item"
+            />
+
+            <div class="row p-3">
+                <span class="col-12">
+                    Total: <strong>${{ totalPrice }}</strong>
+                </span>
+            </div>
+        </div>
     </div>
 </template>
 
 <script>
+import ShoppingCartItem from '@/components/shopping-cart/cart-item';
+
 export default {
     name: 'ShoppingCartList',
+    components: {
+        ShoppingCartItem,
+    },
     props: {
         items: {
             type: Array,
