diff --git a/assets/services/cart-service.js b/assets/services/cart-service.js
index f8bc2d7b4..65224bd33 100644
--- a/assets/services/cart-service.js
+++ b/assets/services/cart-service.js
@@ -144,17 +144,25 @@ export function getCartTotalItems(cart) {
     return cart.items.reduce((acc, item) => (acc + item.quantity), 0);
 }
 
+/**
+ * Gets the full info on the shopping cart
+ *
+ * @param {CartCollection} cart
+ * @returns {Array}
+ */
 export async function getFullShoppingCart(cart) {
-    const productIds = this.cart.items.map((item) => item.product);
+    const productIds = cart.items.map((item) => item.product);
 
     const [colorsResponse, productsResponse] = await Promise.all([
         fetchColors(),
         fetchProductsById(productIds),
     ]);
 
+    const mappedColors = {};
+
     // Map all colors to our object dictionary by @id
     colorsResponse.data['hydra:member'].forEach((color) => {
-        this.colors[color['@id']] = color;
+        mappedColors[color['@id']] = color;
     });
 
     const completeItems = productsResponse.data['hydra:member'].map((product) => {
@@ -162,12 +170,12 @@ export async function getFullShoppingCart(cart) {
 
         return {
             product,
-            color: cartItem.color ? this.colors[cartItem.color] : null,
+            color: cartItem.color ? mappedColors[cartItem.color] : null,
             quantity: cartItem.quantity,
         };
     });
 
-    this.completeCart = {
+    return {
         items: completeItems,
     };
 }
