diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index f6e5bbe8..aa698ce4 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -158,6 +158,32 @@ export default {
             margin-bottom: 5px;
             border-bottom: 1px solid $light-component-border;
         }
+
+        .product-row {
+            display: flex;
+            flex-direction: row;
+            align-items: center;
+            padding-bottom: 5px;
+            margin-bottom: 5px;
+            border-bottom: 1px solid $light-component-border;
+            justify-content: flex-end;
+
+            span {
+                margin-right: 20px;
+            }
+
+            &.product-head {
+                justify-content: flex-start;
+
+                span {
+                    font-weight: bold;
+
+                    &:first-child {
+                        flex-grow: 1;
+                    }
+                }
+            }
+        }
     }
 }
 </style>
