diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index 91cf30349..c6875abb8 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -25,8 +25,7 @@
 </template>
 
 <script>
-import { getColors } from '@/services/colors-service';
-import { fetchProductsById } from '@/services/products-service';
+import { getFullShoppingCart } from '@/services/cart-service';
 import ShoppingCartMixin from '@/mixins/get-shopping-cart';
 import TitleComponent from '@/components/title';
 import Loading from '@/components/loading';
@@ -40,7 +39,6 @@ export default {
     mixins: [ShoppingCartMixin],
     data() {
         return {
-            colors: {},
             completeCartItems: [],
             itemsLoading: true,
         };
@@ -60,6 +58,7 @@ export default {
                 return;
             }
 
+            this.completeCartItems = await getFullShoppingCart(this.cart);
             this.itemsLoading = false;
         },
     },
