diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index 4d6fc4e9b..ecedb5f10 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -55,6 +55,11 @@ export default {
             this.itemsLoading = true;
             this.completeCartItems = [];
 
+            if (!this.cart.items.length) {
+                this.loading = false;
+                return;
+            }
+
             const productIds = this.cart.items.map((item) => item.product);
 
             const [colorsResponse, productsResponse] = await Promise.all([
