diff --git a/assets/services/cart-service.js b/assets/services/cart-service.js
index 1d765a6af..61be39257 100644
--- a/assets/services/cart-service.js
+++ b/assets/services/cart-service.js
@@ -144,8 +144,14 @@ export function getCartTotalItems(cart) {
     return cart.items.reduce((acc, item) => (acc + item.quantity), 0);
 }
 
+/**
+ * Gets the full info on the shopping cart
+ *
+ * @param {CartCollection} cart
+ * @returns {Promise}
+ */
 export async function fetchFullCart(cart) {
-    const productIds = this.cart.items.map((item) => item.product);
+    const productIds = cart.items.map((item) => item.product);
 
     const [colorsResponse, productsResponse] = await Promise.all([
         fetchColors(),
@@ -159,7 +165,7 @@ export async function fetchFullCart(cart) {
     });
 
     const completeItems = productsResponse.data['hydra:member'].map((product) => {
-        const cartItem = this.cart.items.find((item) => item.product === product['@id']);
+        const cartItem = cart.items.find((item) => item.product === product['@id']);
 
         return {
             product,
@@ -168,7 +174,7 @@ export async function fetchFullCart(cart) {
         };
     });
 
-    this.completeCart = {
+    return {
         items: completeItems,
     };
 }
