diff --git a/assets/js/components/shopping-cart/cart-item.vue b/assets/js/components/shopping-cart/cart-item.vue
index adb43503..ad318ec9 100644
--- a/assets/js/components/shopping-cart/cart-item.vue
+++ b/assets/js/components/shopping-cart/cart-item.vue
@@ -5,9 +5,9 @@
 </template>
 
 <script>
-    export default {
-        name: "product.vue"
-    }
+export default {
+    name: 'ShoppingCartItem',
+};
 </script>
 
 <style scoped>
