diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index c49084d3a..c32953bcc 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -25,6 +25,7 @@
 </template>
 
 <script>
+import { fetchProductsById } from '@/services/products-service';
 import ShoppingCartMixin from '@/mixins/get-shopping-cart';
 import TitleComponent from '@/components/title';
 import Loading from '@/components/loading';
@@ -36,6 +37,14 @@ export default {
         TitleComponent,
     },
     mixins: [ShoppingCartMixin],
+    watch: {
+        async cart() {
+            const productIds = this.cart.items.map((item) => item.product);
+
+            const productsResponse = await fetchProductsById(productIds);
+            console.log(productsResponse, this.cart);
+        },
+    },
 };
 </script>
 
