diff --git a/assets/components/shopping-cart/index.vue b/assets/components/shopping-cart/index.vue
index e90497b50..bfe0561f6 100644
--- a/assets/components/shopping-cart/index.vue
+++ b/assets/components/shopping-cart/index.vue
@@ -1,6 +1,6 @@
 <template>
     <div>
-        <div v-if="completeCart.items.length === 0">
+        <div v-if="items.length === 0">
             Your cart is empty! Get to shopping!
         </div>
     </div>
