diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index 7ddd92f4..c483f574 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -52,6 +52,7 @@
 
 <script>
 import cartService from '@/services/cart';
+import formatPrice from '@/helpers/format-price';
 import { getProductsById } from '@/services/products-service';
 import { getColors } from '@/services/colors-service';
 import Loading from '@/components/loading';
@@ -72,6 +73,22 @@ export default {
             items: [],
         };
     },
+    computed: {
+        /**
+         * Returns the formatted total price of the list
+         *
+         * @return {string}
+         */
+        totalPrice() {
+            let total = 0;
+
+            this.items.forEach((item) => {
+                total += item.price * item.qty;
+            });
+
+            return formatPrice(total);
+        },
+    },
     async created() {
         const itemsInCart = cartService.getItems();
 
