diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index d9bc124d..4153bbb1 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -36,6 +36,23 @@ export default {
         loading: true,
         items: [],
     }),
+    async created() {
+        let colorsResponse = null;
+
+        // Retrieve colors from the server
+        try {
+            colorsResponse = await colorsService.getColors();
+        } catch (e) {
+            this.loading = false;
+        }
+
+        // Map all colors to our object dictionary by @id
+        colorsResponse.data['hydra:member'].forEach((color) => {
+            this.colors[color['@id']] = color;
+        });
+
+        this.loading = false;
+    },
 };
 </script>
 
