diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index 36bd5a7e..22237090 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -19,13 +19,10 @@
                             <loading />
                         </div>
 
-                        <div
-                            v-for="item in items"
-                            class="p-3"
-                            :key="item['@id']"
-                        >
-                            {{ item.name }}
-                        </div>
+                        <shopping-cart-list
+                            v-show="!loading"
+                            :items="items"
+                        />
                     </div>
                 </div>
             </div>
@@ -38,6 +35,7 @@ import { getProductsById } from '@/services/products-service';
 import { getColors } from '@/services/colors-service';
 import Loading from '@/components/loading';
 import TitleComponent from '@/components/title';
+import ShoppingCartList from '@/components/shopping-cart';
 import shoppingCartMixin from '@/mixins/get-shopping-cart';
 
 export default {
@@ -45,6 +43,7 @@ export default {
     components: {
         Loading,
         TitleComponent,
+        ShoppingCartList,
     },
     mixins: [shoppingCartMixin],
     data() {
