diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index f89ffb346..b73b4dbf3 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -9,13 +9,13 @@
                 <div class="content p-3">
                     <loading v-if="cart === null" />
                     <div
-                        v-if="cart !== null"
+                        v-show="!loading"
                     >
                         <div
-                            v-for="(cartItem, index) in cart.items"
+                            v-for="(cartItem, index) in items"
                             :key="index"
                         >
-                            {{ cartItem.product }} ({{ cartItem.quantity }})
+                            {{ cartItem.name }}
                         </div>
                     </div>
                 </div>
@@ -25,6 +25,7 @@
 </template>
 
 <script>
+import { getProductsById } from '@/services/products-service';
 import ShoppingCartMixin from '@/mixins/get-shopping-cart';
 import TitleComponent from '@/components/title';
 import Loading from '@/components/loading';
@@ -36,6 +37,31 @@ export default {
         TitleComponent,
     },
     mixins: [ShoppingCartMixin],
+    data() {
+        return {
+            items: [],
+            loading: false,
+        };
+    },
+    watch: {
+        async cart() {
+            this.loading = true;
+            this.items = [];
+
+            const productIds = this.cart.items.map((item) => item.product);
+            let productsResponse = null;
+
+            try {
+                productsResponse = await getProductsById(productIds);
+            } catch (e) {
+                this.loading = false;
+                return;
+            }
+
+            this.items = productsResponse.data['hydra:member'];
+            this.loading = false;
+        },
+    },
 };
 </script>
 
