diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index 2993075e..dbb15516 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -68,15 +68,20 @@ export default {
     },
     watch: {
         async cart() {
+            this.loadProducts();
+        },
+    },
+    async created() {
+        this.colors = (await fetchColors()).data['hydra:member'];
+    },
+    methods: {
+        async loadProducts() {
             const productIds = this.cart.items.map((item) => item.product);
 
             const productsResponse = await fetchProductsById(productIds);
             this.products = productsResponse.data['hydra:member'];
         },
     },
-    async created() {
-        this.colors = (await fetchColors()).data['hydra:member'];
-    },
 };
 </script>
 
