diff --git a/assets/components/shopping-cart/index.vue b/assets/components/shopping-cart/index.vue
index 1b258b37e..e9a60510f 100644
--- a/assets/components/shopping-cart/index.vue
+++ b/assets/components/shopping-cart/index.vue
@@ -5,8 +5,8 @@
         </div>
 
         <shopping-cart-item
-            v-for="(item, index) in items"
-            :key="index"
+            v-for="item in items"
+            :key="item.id"
             :item="item"
         />
     </div>
