diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index 4bceb255..89a27d57 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -1,20 +1,28 @@
 <template>
     <div class="container-fluid">
-        <div class="row row-no-wrap">
+        <div class="row">
             <aside class="col-xs-12 col-lg-3" />
 
             <div class="col-xs-12 col-lg-9">
-                // TODO: List the products in the cart here
+                <div class="row">
+                    <div class="col-12">
+                        <title-component text="Shopping Cart" />
+                    </div>
+                </div>
             </div>
         </div>
     </div>
 </template>
 
 <script>
+import TitleComponent from '@/components/title';
 import shoppingCartMixin from '@/mixins/get-shopping-cart';
 
 export default {
     name: 'ShoppingCart',
+    components: {
+        TitleComponent,
+    },
     mixins: [shoppingCartMixin],
 };
 </script>
