diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index 402108de..16b572db 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -8,6 +8,10 @@
                     <div :class="$style['top-bar']">
                         <title-component text="Shopping Cart" />
                     </div>
+
+                    <div :class="$style.content">
+                        <loading v-show="loading" />
+                    </div>
                 </div>
             </div>
         </div>
@@ -15,16 +19,31 @@
 </template>
 
 <script>
+import Loading from '@/components/loading';
 import TitleComponent from '@/components/title';
 
 export default {
     name: 'ShoppingCart',
     components: {
+        Loading,
         TitleComponent,
-    }
+    },
+    data: () => ({
+        loading: true,
+    }),
 };
 </script>
 
 <style lang="scss" module>
 @import '~styles/top-bar.scss';
+@import '~styles/components/light-component.scss';
+
+.content {
+    @include light-component;
+    margin-top: 25px;
+    padding: 20px;
+    display: flex;
+    flex-direction: column;
+    width: 900px;
+}
 </style>
