diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index 36bd5a7e..a3f106dc 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -14,17 +14,20 @@
                     <div class="content col-11">
                         <div
                             v-show="loading"
-                            class="col-12 p-3"
+                            class="row"
                         >
-                            <loading />
+                            <div class="col-12 p-3">
+                                <loading />
+                            </div>
                         </div>
 
                         <div
-                            v-for="item in items"
-                            class="p-3"
-                            :key="item['@id']"
+                            v-show="!loading && !items.length"
+                            class="row"
                         >
-                            {{ item.name }}
+                            <div class="col-12 p-3">
+                                Sorry! You haven't bought anything yet!
+                            </div>
                         </div>
                     </div>
                 </div>
