diff --git a/assets/js/components/shopping-cart/cart-item.vue b/assets/js/components/shopping-cart/cart-item.vue
index 7f755948..5e1b9656 100644
--- a/assets/js/components/shopping-cart/cart-item.vue
+++ b/assets/js/components/shopping-cart/cart-item.vue
@@ -7,6 +7,12 @@
 <script>
 export default {
     name: 'ShoppingCartProduct',
+    props: {
+        product: {
+            type: Object,
+            required: true,
+        },
+    },
 };
 </script>
 
