diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index dd937594..4074c831 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -45,6 +45,7 @@
 <script>
 import cartService from '@/services/cart';
 import colorsService from '@/services/colors';
+import formatPrice from '@/helpers/format-price';
 import productsService from '@/services/products';
 import Loading from '@/components/loading';
 import ShoppingCartItem from '@/components/shopping-cart/cart-item';
@@ -62,6 +63,22 @@ export default {
         loading: true,
         items: [],
     }),
+    computed: {
+        /**
+         * Returns the formatted total price of the list
+         *
+         * @return {string}
+         */
+        totalPrice() {
+            let total = 0;
+
+            this.items.forEach((item) => {
+                total += item.price * item.qty;
+            });
+
+            return formatPrice(total);
+        },
+    },
     async created() {
         const itemsInCart = cartService.getItems();
 
