diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index a439f0b4d..6b5304821 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -14,6 +14,7 @@
                             :key="index"
                         >
                             {{ cartItem.product.name }} ({{ cartItem.quantity }})
+                            {{ cartItem.color ? cartItem.color.hexColor : '' }}
                         </div>
 
                         <div v-if="completeCart.items.length === 0">
@@ -58,14 +59,13 @@ export default {
             colorsResponse.data['hydra:member'].forEach((color) => {
                 mappedColors[color['@id']] = color;
             });
-            console.log(mappedColors);
 
             const completeItems = productsResponse.data['hydra:member'].map((product) => {
                 const cartItem = this.cart.items.find((item) => item.product === product['@id']);
 
                 return {
                     product,
-                    color: cartItem.color,
+                    color: cartItem.color ? mappedColors[cartItem.color] : null,
                     quantity: cartItem.quantity,
                 };
             });
