diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index 6b5304821..51f21e04a 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -50,9 +50,10 @@ export default {
         async cart() {
             const productIds = this.cart.items.map((item) => item.product);
 
-            const productsResponse = await fetchProductsById(productIds);
-
-            const colorsResponse = await fetchColors();
+            const [colorsResponse, productsResponse] = await Promise.all([
+                fetchColors(),
+                fetchProductsById(productIds),
+            ]);
 
             // Map all colors to our object dictionary by @id
             const mappedColors = {};
