diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index e19bb423f..44d03556f 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -7,15 +7,15 @@
                 <title-component text="Shopping Cart" />
 
                 <div class="content p-3">
-                    <loading v-show="cart === null" />
+                    <loading v-show="completeCart === null" />
                     <div
-                        v-if="cart !== null"
+                        v-if="completeCart !== null"
                     >
                         <div
-                            v-for="(cartItem, index) in cart.items"
+                            v-for="(cartItem, index) in completeCart.items"
                             :key="index"
                         >
-                            {{ cartItem.product }} ({{ cartItem.quantity }})
+                            {{ cartItem.product.name }} ({{ cartItem.quantity }})
                         </div>
                     </div>
                 </div>
@@ -37,6 +37,11 @@ export default {
         TitleComponent,
     },
     mixins: [ShoppingCartMixin],
+    data() {
+        return {
+            completeCart: null,
+        };
+    },
     watch: {
         async cart() {
             const productIds = this.cart.items.map((item) => item.product);
@@ -51,7 +56,10 @@ export default {
                     quantity: cartItem.quantity,
                 };
             });
-            console.log(completeItems);
+
+            this.completeCart = {
+                items: completeItems,
+            };
         },
     },
 };
