diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index 1a0bae8..2f00565 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -19,8 +19,18 @@
 </template>
 
 <script>
+import Loading from '@/components/loading';
+import TitleComponent from '@/components/title';
+
 export default {
     name: 'ShoppingCart',
+    components: {
+        Loading,
+        TitleComponent,
+    },
+    data: () => ({
+        loading: true,
+    }),
 };
 </script>
 
