diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index de76c5b2..36bd5a7e 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -51,12 +51,19 @@ export default {
         return {
             colors: {},
             items: [],
-            loading: true,
+            loading: false,
         };
     },
     watch: {
         'cart.items.length': async function watchCartItems() {
             this.loading = true;
+
+            if (!this.cart.items.length) {
+                this.items = [];
+                this.loading = false;
+                return;
+            }
+
             const productIds = this.cart.items.map((item) => item.product);
 
             let colorsResponse = null;
