diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index bd13fa64..880e15a3 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -64,8 +64,10 @@ export default {
 
         // Retrieve both colors and products from the server
         try {
-            colorsResponse = await getColors();
-            productsResponse = await getProductsById(productIds);
+            [colorsResponse, productsResponse] = await Promise.all([
+                getColors(),
+                getProductsById(productIds),
+            ]);
         } catch (e) {
             this.loading = false;
 
