diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index 4153bbb1..9191102b 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -21,7 +21,7 @@
 <script>
 import cartService from '@/services/cart';
 import colorsService from '@/services/colors';
-import productsService from '@/services/products';
+import { getProductsById } from '@/services/products-service';
 import Loading from '@/components/loading';
 import TitleComponent from '@/components/title';
 
@@ -37,13 +37,18 @@ export default {
         items: [],
     }),
     async created() {
+        const itemsInCart = cartService.getItems();
+        const productIds = itemsInCart.map((item) => (Number(item.productId.split('/').pop())));
         let colorsResponse = null;
+        let productsResponse = null;
 
-        // Retrieve colors from the server
+        // Retrieve both colors and products from the server
         try {
             colorsResponse = await colorsService.getColors();
+            productsResponse = await getProductsById(productIds);
         } catch (e) {
             this.loading = false;
+            return;
         }
 
         // Map all colors to our object dictionary by @id
@@ -51,6 +56,8 @@ export default {
             this.colors[color['@id']] = color;
         });
 
+        this.items = productsResponse.data['hydra:member'];
+
         this.loading = false;
     },
 };
