diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index 97cba85a..b2f8df12 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -18,6 +18,14 @@
                         >
                             <loading />
                         </div>
+
+                        <div
+                            v-for="item in items"
+                            class="p-3"
+                            :key="item['@id']"
+                        >
+                            {{ item.name }}
+                        </div>
                     </div>
                 </div>
             </div>
