diff --git a/assets/js/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
similarity index 100%
rename from assets/js/shopping-cart.vue
rename to assets/js/pages/shopping-cart.vue
diff --git a/assets/js/shopping-cart.js b/assets/js/shopping-cart.js
index 82cfcce9..c59a8fa7 100644
--- a/assets/js/shopping-cart.js
+++ b/assets/js/shopping-cart.js
@@ -1,5 +1,5 @@
 import Vue from 'vue';
-import App from '@/shopping-cart.vue';
+import App from '@/pages/shopping-cart.vue';
 
 new Vue({
     render: (h) => h(App),
