diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index 17092fb8b..5e8299f3a 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -27,6 +27,7 @@
 </template>
 
 <script>
+import { fetchColors } from '@/services/colors-service';
 import { fetchProductsById } from '@/services/products-service';
 import ShoppingCartMixin from '@/mixins/get-shopping-cart';
 import TitleComponent from '@/components/title';
@@ -42,6 +43,7 @@ export default {
     data() {
         return {
             products: null,
+            colors: null,
         };
     },
     computed: {
@@ -68,6 +70,14 @@ export default {
             const productIds = this.cart.items.map((item) => item.product);
 
             const productsResponse = await fetchProductsById(productIds);
+            const colorsResponse = await fetchColors();
+
+            // Map all colors to our object dictionary by @id
+            this.colors = {};
+            colorsResponse.data['hydra:member'].forEach((color) => {
+                this.colors[color['@id']] = color;
+            });
+
             this.products = productsResponse.data['hydra:member'];
         },
     },
