diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index 89a27d57..074ed867 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -9,23 +9,46 @@
                         <title-component text="Shopping Cart" />
                     </div>
                 </div>
+
+                <div class="row">
+                    <div :class="[$style.content, 'col-11']">
+                        <div
+                            v-show="loading"
+                            class="col-12 p-3"
+                        >
+                            <loading />
+                        </div>
+                    </div>
+                </div>
             </div>
         </div>
     </div>
 </template>
 
 <script>
+import Loading from '@/components/loading';
 import TitleComponent from '@/components/title';
 import shoppingCartMixin from '@/mixins/get-shopping-cart';
 
 export default {
     name: 'ShoppingCart',
     components: {
+        Loading,
         TitleComponent,
     },
     mixins: [shoppingCartMixin],
+    data() {
+        return {
+            loading: true,
+        };
+    },
 };
 </script>
 
 <style lang="scss" module>
+@import '~styles/components/light-component.scss';
+
+.content {
+    @include light-component;
+}
 </style>
