diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index cf719efd..41d85be3 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -61,7 +61,7 @@ import { getProductsById } from '@/services/products-service';
 import { getColors } from '@/services/colors-service';
 import Loading from '@/components/loading';
 import TitleComponent from '@/components/title';
-import ShoppingCartProduct from "@/components/shopping-cart/cart-item";
+import ShoppingCartProduct from '@/components/shopping-cart/cart-item';
 
 export default {
     name: 'ShoppingCart',
