diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index ee1b2dbb7..2863cfbce 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -28,8 +28,7 @@
 </template>
 
 <script>
-import { fetchColors } from '@/services/colors-service';
-import { fetchProductsById } from '@/services/products-service';
+import { fetchFullCart } from '@/services/cart-service';
 import ShoppingCartMixin from '@/mixins/get-shopping-cart';
 import TitleComponent from '@/components/title';
 import Loading from '@/components/loading';
@@ -48,6 +47,7 @@ export default {
     },
     watch: {
         async cart() {
+            this.completeCart = await fetchFullCart(this.cart);
         },
     },
 };
