diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index 2863cfbce..e39b5a3d2 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -8,19 +8,11 @@
 
                 <div class="content p-3">
                     <loading v-show="completeCart === null" />
-                    <div v-if="completeCart !== null">
-                        <div
-                            v-for="(cartItem, index) in completeCart.items"
-                            :key="index"
-                        >
-                            {{ cartItem.product.name }} ({{ cartItem.quantity }})
-                            {{ cartItem.color ? cartItem.color.hexColor : '' }}
-                        </div>
 
-                        <div v-if="completeCart.items.length === 0">
-                            Your cart is empty! Get to shopping!
-                        </div>
-                    </div>
+                    <shopping-cart-list
+                        v-if="completeCart"
+                        :items="completeCart.items"
+                    />
                 </div>
             </div>
         </div>
@@ -31,12 +23,14 @@
 import { fetchFullCart } from '@/services/cart-service';
 import ShoppingCartMixin from '@/mixins/get-shopping-cart';
 import TitleComponent from '@/components/title';
+import ShoppingCartList from '@/components/shopping-cart';
 import Loading from '@/components/loading';
 
 export default {
     name: 'ShoppingCart',
     components: {
         Loading,
+        ShoppingCartList,
         TitleComponent,
     },
     mixins: [ShoppingCartMixin],
