diff --git a/assets/components/shopping-cart/cart-item.vue b/assets/components/shopping-cart/cart-item.vue
index bf9f709f8..4e7e46455 100644
--- a/assets/components/shopping-cart/cart-item.vue
+++ b/assets/components/shopping-cart/cart-item.vue
@@ -7,9 +7,9 @@
 </template>
 
 <script>
-    export default {
-        name: "product.vue"
-    }
+export default {
+    name: 'ShoppingCartItem',
+};
 </script>
 
 <style scoped>
