diff --git a/assets/js/components/shopping-cart/product.vue b/assets/js/components/shopping-cart/product.vue
index 2cec2ed..b0273c3 100644
--- a/assets/js/components/shopping-cart/product.vue
+++ b/assets/js/components/shopping-cart/product.vue
@@ -1,5 +1,9 @@
 <template>
-    
+    <div
+        :class="$style.component"
+    >
+        <span :class="$style['product-name']">{{ product.name }}</span>
+    </div>
 </template>
 
 <script>
