diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index 0ec4b199..99c3a5fc 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -66,6 +66,7 @@
 <script>
 import { getProductsById } from '@/services/products-service';
 import { getColors } from '@/services/colors-service';
+import formatPrice from '@/helpers/format-price';
 import Loading from '@/components/loading';
 import TitleComponent from '@/components/title';
 import ShoppingCartProduct from '@/components/shopping-cart/cart-item';
@@ -86,6 +87,22 @@ export default {
             loading: false,
         };
     },
+    computed: {
+        /**
+         * Returns the formatted total price of the list
+         *
+         * @return {string}
+         */
+        totalPrice() {
+            let total = 0;
+
+            this.items.forEach((item) => {
+                total += item.price * item.qty;
+            });
+
+            return formatPrice(total);
+        },
+    },
     watch: {
         'cart.items.length': async function watchCartItems() {
             this.loading = true;
