diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index 6192d51fc..6aa39fcb9 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -55,8 +55,10 @@ export default {
             let productsResponse = null;
 
             try {
-                colorsResponse = await getColors();
-                productsResponse = await getProductsById(productIds);
+                [colorsResponse, productsResponse] = await Promise.all([
+                    getColors(),
+                    getProductsById(productIds),
+                ]);
             } catch (e) {
                 this.loading = false;
                 return;
