diff --git a/assets/components/shopping-cart/index.vue b/assets/components/shopping-cart/index.vue
index bfe0561f..1b258b37 100644
--- a/assets/components/shopping-cart/index.vue
+++ b/assets/components/shopping-cart/index.vue
@@ -3,12 +3,23 @@
         <div v-if="items.length === 0">
             Your cart is empty! Get to shopping!
         </div>
+
+        <shopping-cart-item
+            v-for="(item, index) in items"
+            :key="index"
+            :item="item"
+        />
     </div>
 </template>
 
 <script>
+import ShoppingCartItem from '@/components/shopping-cart/cart-item';
+
 export default {
     name: 'ShoppingCartList',
+    components: {
+        ShoppingCartItem,
+    },
     props: {
         items: {
             type: Array,
