diff --git a/assets/js/components/shopping-cart/product.vue b/assets/js/components/shopping-cart/product.vue
index 0cd3d4e..3134e7b 100644
--- a/assets/js/components/shopping-cart/product.vue
+++ b/assets/js/components/shopping-cart/product.vue
@@ -9,6 +9,13 @@
 <script>
 export default {
     name: 'ShoppingCartProduct',
+    props: {
+        product: {
+            type: Object,
+            require: true,
+            default: () => ({}),
+        },
+    },
 };
 </script>
 
