diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index 6b37980e..99431202 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -4,15 +4,24 @@
             <aside class="col-xs-12 col-lg-3" />
 
             <div class="col-xs-12 col-lg-9">
-                // TODO: List the products in the cart here
+                <div class="row">
+                    <div class="col-12">
+                        <title-component text="Shopping Cart" />
+                    </div>
+                </div>
             </div>
         </div>
     </div>
 </template>
 
 <script>
+import TitleComponent from '@/components/title';
+
 export default {
     name: 'ShoppingCart',
+    components: {
+        TitleComponent,
+    },
 };
 </script>
 
