diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index a1ad8683..2993075e 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -70,15 +70,13 @@ export default {
         async cart() {
             const productIds = this.cart.items.map((item) => item.product);
 
-            const [colorsResponse, productsResponse] = await Promise.all([
-                fetchColors(),
-                fetchProductsById(productIds),
-            ]);
-
+            const productsResponse = await fetchProductsById(productIds);
             this.products = productsResponse.data['hydra:member'];
-            this.colors = colorsResponse.data['hydra:member'];
         },
     },
+    async created() {
+        this.colors = (await fetchColors()).data['hydra:member'];
+    },
 };
 </script>
 
