diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index a061be77a..903476d25 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -48,9 +48,12 @@ export default {
             }
 
             const completeItems = this.cart.items.map((cartItem) => {
+                const product = this.products.find((product) => product['@id'] === cartItem.product);
+                const color = this.colors.find((color) => color['@id'] === cartItem.color);
+
                 return {
-                    product: this.products.find((product) => product['@id'] === cartItem.product),
-                    color: this.colors.find((color) => color['@id'] === cartItem.color),
+                    product,
+                    color,
                     quantity: cartItem.quantity,
                 };
             });
