diff --git a/assets/js/services/products.js b/assets/js/services/products.js
index 2b936e1f..96a65781 100644
--- a/assets/js/services/products.js
+++ b/assets/js/services/products.js
@@ -29,6 +29,24 @@ const products = {
             url: `/api/products/${id}`,
         });
     },
+
+    /**
+     * Retrieves a set of products identified by an array of IRIs
+     *
+     * @param {string[]} ids
+     * @return {Promise}
+     */
+    getProductsById(ids) {
+        if (!ids.length) {
+            return Promise.resolve({ 'hydra:member': [] });
+        }
+
+        return axios({
+            method: 'get',
+            url: '/api/products',
+            params: { id: ids },
+        });
+    },
 };
 
 export default products;
