diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index 880e15a3..c06da299 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -57,6 +57,12 @@ export default {
     },
     async created() {
         const itemsInCart = cartService.getItems();
+
+        if (!itemsInCart.length) {
+            this.loading = false;
+            return;
+        }
+
         const productIds = itemsInCart.map((item) => item.productId);
 
         let colorsResponse = null;
