diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index 89a27d57..cccec2b3 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -1,5 +1,5 @@
 <template>
-    <div class="container-fluid">
+    <div :class="[$style.component, 'container-fluid']">
         <div class="row">
             <aside class="col-xs-12 col-lg-3" />
 
@@ -9,23 +9,48 @@
                         <title-component text="Shopping Cart" />
                     </div>
                 </div>
+
+                <div class="row">
+                    <div class="content col-11">
+                        <div
+                            v-show="loading"
+                            class="col-12 p-3"
+                        >
+                            <loading />
+                        </div>
+                    </div>
+                </div>
             </div>
         </div>
     </div>
 </template>
 
 <script>
+import Loading from '@/components/loading';
 import TitleComponent from '@/components/title';
 import shoppingCartMixin from '@/mixins/get-shopping-cart';
 
 export default {
     name: 'ShoppingCart',
     components: {
+        Loading,
         TitleComponent,
     },
     mixins: [shoppingCartMixin],
+    data() {
+        return {
+            loading: true,
+        };
+    },
 };
 </script>
 
 <style lang="scss" module>
+@import '~styles/components/light-component.scss';
+
+.component :global {
+    .content {
+        @include light-component;
+    }
+}
 </style>
