diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index 326775499..42d34e829 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -14,6 +14,10 @@
                         >
                             {{ cartItem.product }} ({{ cartItem.quantity }})
                         </div>
+
+                        <div v-if="cart.items.length === 0">
+                            Your cart is empty! Get to shopping!
+                        </div>
                     </div>
                 </div>
             </div>
