diff --git a/assets/js/services/products.js b/assets/js/services/products.js
index ed9369e..9c788f4 100644
--- a/assets/js/services/products.js
+++ b/assets/js/services/products.js
@@ -28,6 +28,35 @@ const products = {
             url: `/api/products/${id}`,
         });
     },
+
+    /**
+     * Retrieves a set of products identified by an array of ids
+     *
+     * @param {number[]} ids
+     * @return {Promise}
+     */
+    getProductsById(ids) {
+        if (ids.length) {
+            let idString = '';
+
+            ids.forEach((id) => {
+                if (idString.length) {
+                    idString += '&';
+                }
+
+                idString += `id[]=${id}`;
+            });
+
+            return axios({
+                method: 'get',
+                url: `/api/products?${idString}`,
+            });
+        }
+
+        return new Promise((resolve) => {
+            resolve([]);
+        });
+    },
 };
 
 export default products;
