diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index e6cf78ce7..6192d51fc 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -62,13 +62,28 @@ export default {
                 return;
             }
 
-            this.items = productsResponse.data['hydra:member'];
-
             // Map all colors to our object dictionary by @id
             colorsResponse.data['hydra:member'].forEach((color) => {
                 this.colors[color['@id']] = color;
             });
 
+            // Assign our returned products to our products array,
+            // applying the proper colorId, hexColor and qty values
+            this.items = productsResponse.data['hydra:member'].map((product) => {
+                const productInCart = this.cart.items.find(
+                    (item) => (item.product === product['@id']),
+                );
+
+                return {
+                    ...product,
+                    colorId: productInCart.color,
+                    hexColor: productInCart.color
+                        ? this.colors[productInCart.color].hexColor
+                        : 'fff',
+                    qty: productInCart.quantity,
+                };
+            });
+
             this.loading = false;
         },
     },
