diff --git a/assets/js/services/products-service.js b/assets/js/services/products-service.js
index 1f741049..de210e1c 100644
--- a/assets/js/services/products-service.js
+++ b/assets/js/services/products-service.js
@@ -26,3 +26,21 @@ export function getProduct(id) {
         url: `/api/products/${id}`,
     });
 }
+
+/**
+ * Retrieves a set of products identified by an array of IRIs
+ *
+ * @param {string[]} ids
+ * @return {Promise}
+ */
+export function getProductsById(ids) {
+    if (!ids.length) {
+        return Promise.resolve({ 'hydra:member': [] });
+    }
+
+    return axios({
+        method: 'get',
+        url: '/api/products',
+        params: { id: ids },
+    });
+}
