diff --git a/assets/js/pages/products.js b/assets/js/pages/products.js
index 3b156c3..250d62a 100644
--- a/assets/js/pages/products.js
+++ b/assets/js/pages/products.js
@@ -1,5 +1,5 @@
 import Vue from 'vue';
-import App from '@/products.vue';
+import App from '@/pages/products.vue';
 
 new Vue({
     render: (h) => h(App, {
diff --git a/assets/js/pages/shopping-cart.js b/assets/js/pages/shopping-cart.js
index 82cfcce..c59a8fa 100644
--- a/assets/js/pages/shopping-cart.js
+++ b/assets/js/pages/shopping-cart.js
@@ -1,5 +1,5 @@
 import Vue from 'vue';
-import App from '@/shopping-cart.vue';
+import App from '@/pages/shopping-cart.vue';
 
 new Vue({
     render: (h) => h(App),
diff --git a/webpack.config.js b/webpack.config.js
index 11f099e..edc2d63 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -31,8 +31,8 @@ Encore
      * and one CSS file (e.g. app.css) if your JavaScript imports CSS.
      */
     .addEntry('app', './assets/js/app.js')
-    .addEntry('products', './assets/js/products.js')
-    .addEntry('shopping-cart', './assets/js/shopping-cart.js')
+    .addEntry('products', './assets/js/pages/products.js')
+    .addEntry('shopping-cart', './assets/js/pages/shopping-cart.js')
     //.addEntry('page1', './assets/js/page1.js')
     //.addEntry('page2', './assets/js/page2.js')
 
