diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index 799c13bc..7e7f8c33 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -39,6 +39,25 @@ export default {
             loading: true,
         };
     },
+    async created() {
+        let colorsResponse = null;
+
+        // Retrieve colors from the server
+        try {
+            colorsResponse = await getColors();
+        } catch (e) {
+            this.loading = false;
+
+            return;
+        }
+
+        // Map all colors to our object dictionary by @id
+        colorsResponse.data['hydra:member'].forEach((color) => {
+            this.colors[color['@id']] = color;
+        });
+
+        this.loading = false;
+    },
 };
 </script>
 
