diff --git a/assets/js/components/shopping-cart/index.vue b/assets/js/components/shopping-cart/index.vue
new file mode 100644
index 00000000..7d1a3ddc
--- /dev/null
+++ b/assets/js/components/shopping-cart/index.vue
@@ -0,0 +1,24 @@
+<template>
+    <div class="col-11">
+        <div
+            v-show="!items.length"
+            class="row"
+        >
+            <div class="col-12 p-3">
+                Sorry! You haven't bought anything yet!
+            </div>
+        </div>
+    </div>
+</template>
+
+<script>
+export default {
+    name: 'ShoppingCartList',
+    props: {
+        items: {
+            type: Array,
+            required: true,
+        },
+    },
+};
+</script>
