diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index dcb72f0c..46f77ba4 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -60,6 +60,11 @@ export default {
             this.loading = true;
             this.items = [];
 
+            if (!this.cart.items.length) {
+                this.loading = false;
+                return;
+            }
+
             const productIds = this.cart.items.map((item) => item.product);
             let colorsResponse = null;
             let productsResponse = null;
