diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index b506a27b..07962308 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -69,7 +69,22 @@ export default {
                 this.colors[color['@id']] = color;
             });
 
-            this.items = productsResponse.data['hydra:member'];
+            // Assign our returned products to our products array,
+            // applying the proper colorId, hexColor and qty values
+            this.items = productsResponse.data['hydra:member'].map((product) => {
+                const productInCart = this.cart.items.find(
+                    (item) => (item.product === product['@id']),
+                );
+
+                return {
+                    ...product,
+                    colorId: productInCart.color,
+                    hexColor: productInCart.color
+                        ? this.colors[productInCart.color].hexColor
+                        : 'fff',
+                    qty: productInCart.quantity,
+                };
+            });
 
             this.loading = false;
         },
