diff --git a/assets/services/products-service.js b/assets/services/products-service.js
index 63078a73f..2196da9a7 100644
--- a/assets/services/products-service.js
+++ b/assets/services/products-service.js
@@ -29,3 +29,22 @@ export function fetchProducts(categoryIri, searchTerm) {
 export function fetchOneProduct(iri) {
     return axios.get(iri);
 }
+
+/**
+ * Retrieves a set of products identified by an array of IRIs
+ *
+ * @param {string[]} ids
+ * @return {Promise}
+ */
+export function fetchProductsById(ids) {
+    if (!ids.length) {
+        return Promise.resolve({ 'hydra:member': [] });
+    }
+
+    return axios.get(
+        '/api/products',
+        {
+            params: { id: ids },
+        },
+    );
+}
