diff --git a/assets/js/components/shopping-cart/cart-item.vue b/assets/js/components/shopping-cart/cart-item.vue
index 4e7e4645..4831516c 100644
--- a/assets/js/components/shopping-cart/cart-item.vue
+++ b/assets/js/components/shopping-cart/cart-item.vue
@@ -9,6 +9,12 @@
 <script>
 export default {
     name: 'ShoppingCartItem',
+    props: {
+        cartItem: {
+            type: Object,
+            required: true,
+        },
+    },
 };
 </script>
 
