diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index 94b29ec78..709f503b5 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -60,29 +60,6 @@ export default {
                 return;
             }
 
-            const productIds = this.cart.items.map((item) => item.product);
-
-            const [colorsResponse, productsResponse] = await Promise.all([
-                fetchColors(),
-                fetchProductsById(productIds),
-            ]);
-
-            // Map all colors to our object dictionary by @id
-            colorsResponse.data['hydra:member'].forEach((color) => {
-                this.colors[color['@id']] = color;
-            });
-
-            // Create items with complete product, color & quantity info
-            this.completeCartItems = productsResponse.data['hydra:member'].map((product) => {
-                const cartItem = this.cart.items.find((item) => item.product === product['@id']);
-
-                return {
-                    product,
-                    color: cartItem.color ? this.colors[cartItem.color] : null,
-                    quantity: cartItem.quantity,
-                };
-            });
-
             this.itemsLoading = false;
         },
     },
diff --git a/assets/services/cart-service.js b/assets/services/cart-service.js
index a948b5937..26f5528a1 100644
--- a/assets/services/cart-service.js
+++ b/assets/services/cart-service.js
@@ -1,4 +1,6 @@
 import axios from 'axios';
+import { fetchColors } from '@/services/colors-service';
+import { fetchProductsById } from '@/services/products-service';
 
 /**
  * @typedef {Object} CartCollection
@@ -141,3 +143,28 @@ export function clearCart() {
 export function getCartTotalItems(cart) {
     return cart.items.reduce((acc, item) => (acc + item.quantity), 0);
 }
+
+export async function getFullShoppingCart(cart) {
+    const productIds = this.cart.items.map((item) => item.product);
+
+    const [colorsResponse, productsResponse] = await Promise.all([
+        fetchColors(),
+        fetchProductsById(productIds),
+    ]);
+
+    // Map all colors to our object dictionary by @id
+    colorsResponse.data['hydra:member'].forEach((color) => {
+        this.colors[color['@id']] = color;
+    });
+
+    // Create items with complete product, color & quantity info
+    this.completeCartItems = productsResponse.data['hydra:member'].map((product) => {
+        const cartItem = this.cart.items.find((item) => item.product === product['@id']);
+
+        return {
+            product,
+            color: cartItem.color ? this.colors[cartItem.color] : null,
+            quantity: cartItem.quantity,
+        };
+    });
+}
