diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index 172ec3814..e4998f658 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -47,6 +47,12 @@ export default {
     },
     watch: {
         async cart() {
+            if (!this.cart.items.length) {
+                // TODO - this var no longer exists
+                this.itemsLoading = false;
+                return;
+            }
+
             const productIds = this.cart.items.map((item) => item.product);
 
             const [colorsResponse, productsResponse] = await Promise.all([
