diff --git a/assets/components/shopping-cart/index.vue b/assets/components/shopping-cart/index.vue
index 28cf97f6..3556be6d 100644
--- a/assets/components/shopping-cart/index.vue
+++ b/assets/components/shopping-cart/index.vue
@@ -25,7 +25,7 @@
             />
 
             <div class="p-3">
-                Total: <strong>$ // TODO</strong>
+                Total: <strong>${{ totalPrice }}</strong>
             </div>
         </div>
     </div>
@@ -33,6 +33,7 @@
 
 <script>
 import ShoppingCartItem from '@/components/shopping-cart/cart-item';
+import formatPrice from '@/helpers/format-price';
 
 export default {
     name: 'ShoppingCartList',
@@ -45,5 +46,21 @@ export default {
             required: true,
         },
     },
+    computed: {
+        /**
+         * Returns the formatted total price of the list
+         *
+         * @return {string}
+         */
+        totalPrice() {
+            let total = 0;
+
+            this.items.forEach((item) => {
+                total += item.product.price * item.quantity;
+            });
+
+            return formatPrice(total);
+        },
+    },
 };
 </script>
