diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index ba3b5ba4..99f7b99b 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -13,6 +13,13 @@
                 <div class="row">
                     <div :class="$style.content">
                         <loading v-show="loading" />
+
+                        <div
+                            v-for="item in items"
+                            :key="item['@id']"
+                        >
+                            {{ item.name }}
+                        </div>
                     </div>
                 </div>
             </div>
