diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index 44a6fdd32..7b3c3a670 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -27,6 +27,7 @@
 </template>
 
 <script>
+import { fetchColors } from '@/services/colors-service';
 import { fetchProductsById } from '@/services/products-service';
 import ShoppingCartMixin from '@/mixins/get-shopping-cart';
 import TitleComponent from '@/components/title';
@@ -49,6 +50,15 @@ export default {
             const productIds = this.cart.items.map((item) => item.product);
 
             const productsResponse = await fetchProductsById(productIds);
+            const colorsResponse = await fetchColors();
+
+            // Map all colors to our object dictionary by @id
+            const mappedColors = {};
+            colorsResponse.data['hydra:member'].forEach((color) => {
+                mappedColors[color['@id']] = color;
+            });
+            console.log(mappedColors);
+
             const products = productsResponse.data['hydra:member'];
 
             const completeItems = this.cart.items.map((cartItem) => (
