diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index b82ce9218..651c7786a 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -25,6 +25,7 @@
 </template>
 
 <script>
+import { getColors } from '@/services/colors-service';
 import { fetchProductsById } from '@/services/products-service';
 import ShoppingCartMixin from '@/mixins/get-shopping-cart';
 import TitleComponent from '@/components/title';
@@ -39,6 +40,7 @@ export default {
     mixins: [ShoppingCartMixin],
     data() {
         return {
+            colors: {},
             completeCartItems: [],
             itemsLoading: true,
         };
@@ -57,6 +59,13 @@ export default {
 
             const productsResponse = await fetchProductsById(productIds);
             this.completeCartItems = productsResponse.data['hydra:member'];
+
+            const colorsResponse = await getColors();
+            // Map all colors to our object dictionary by @id
+            colorsResponse.data['hydra:member'].forEach((color) => {
+                this.colors[color['@id']] = color;
+            });
+
             this.itemsLoading = false;
         },
     },
