diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index 6cd3bb3..3b05f2f 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -37,11 +37,15 @@ export default {
         products: [],
     }),
     async created() {
+        const itemsInCart = cartService.getItems();
+        const productIds = itemsInCart.map((item) => (Number(item.productId.split('/').pop())));
         let colorsResponse = null;
+        let productsResponse = null;
 
-        // Retrieve colors from the server
+        // Retrieve both colors and products from the server
         try {
             colorsResponse = await colorsService.getColors();
+            productsResponse = await productsService.getProductsById(productIds);
         } catch (e) {
             this.loading = false;
         }
@@ -51,6 +55,8 @@ export default {
             this.colors[color['@id']] = color;
         });
 
+        this.products = productsResponse.data['hydra:member'];
+
         this.loading = false;
     },
 };
