diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index 2d37fb638..5f8dc8c63 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -25,8 +25,7 @@
 </template>
 
 <script>
-import { getColors } from '@/services/colors-service';
-import { getProductsById } from '@/services/products-service';
+import { getFullShoppingCart } from '@/services/cart-service';
 import ShoppingCartMixin from '@/mixins/get-shopping-cart';
 import TitleComponent from '@/components/title';
 import Loading from '@/components/loading';
@@ -40,7 +39,6 @@ export default {
     mixins: [ShoppingCartMixin],
     data() {
         return {
-            colors: {},
             items: [],
             loading: false,
         };
@@ -55,6 +53,12 @@ export default {
                 return;
             }
 
+            try {
+                this.items = await getFullShoppingCart(this.cart);
+            } catch (e) {
+                this.loading = false;
+            }
+
             this.loading = false;
         },
     },
