diff --git a/assets/js/components/sidebar.vue b/assets/js/components/sidebar.vue
index 6fda540f..c6b9fc7a 100644
--- a/assets/js/components/sidebar.vue
+++ b/assets/js/components/sidebar.vue
@@ -1,7 +1,11 @@
 <template>
     <div
-        :class="[$style.component, 'p-3', 'mb-5']"
-        :style="{ width: collapsed ? '70px' : 'auto' }"
+        :class="{
+            [$style.component]: true,
+            [$style.collapsed]: collapsed,
+            'p-3': true,
+            'mb-5': true
+        }"
     >
         <div v-show="!collapsed">
             <h5 class="text-center">
@@ -73,6 +77,10 @@ export default {
 .component {
     @include light-component;
 
+    &.collapsed {
+        width: 70px;
+    }
+
     ul {
         li a:hover {
             background: $blue-component-link-hover;
