diff --git a/assets/js/app.js b/assets/js/app.js
index 5a9d6cee..88da0017 100644
--- a/assets/js/app.js
+++ b/assets/js/app.js
@@ -6,4 +6,4 @@
  */
 
 // any CSS you import will output into a single css file (app.css in this case)
-import '../scss/app.scss';
+import 'styles/app.scss';
diff --git a/assets/js/components/catalog.vue b/assets/js/components/catalog.vue
index 81be3cf1..43d764ef 100644
--- a/assets/js/components/catalog.vue
+++ b/assets/js/components/catalog.vue
@@ -21,7 +21,7 @@
 </template>
 
 <script>
-import LegendComponent from './legend';
+import LegendComponent from '@/components/legend';
 
 export default {
     name: 'Catalog',
diff --git a/assets/js/components/sidebar.vue b/assets/js/components/sidebar.vue
index 3ef3f137..6ec47113 100644
--- a/assets/js/components/sidebar.vue
+++ b/assets/js/components/sidebar.vue
@@ -34,7 +34,7 @@ export default {
 </script>
 
 <style lang="scss" module>
-@import '../../scss/components/light-component';
+@import '~styles/components/light-component';
 
 .component {
     @include light-component;
diff --git a/assets/js/pages/products.vue b/assets/js/pages/products.vue
index f7c41932..419509f4 100644
--- a/assets/js/pages/products.vue
+++ b/assets/js/pages/products.vue
@@ -13,8 +13,8 @@
 </template>
 
 <script>
-import Catalog from '../components/catalog';
-import Sidebar from '../components/sidebar';
+import Catalog from '@/components/catalog';
+import Sidebar from '@/components/sidebar';
 
 export default {
     name: 'Products',
diff --git a/assets/js/products.js b/assets/js/products.js
index e82a9bda..790dbc03 100644
--- a/assets/js/products.js
+++ b/assets/js/products.js
@@ -1,5 +1,5 @@
 import Vue from 'vue';
-import App from './pages/products';
+import App from '@/pages/products';
 
 new Vue({
     render: (h) => h(App),
