diff --git a/assets/js/components/sidebar.vue b/assets/js/components/sidebar.vue
index 0c8bcbd..abbdda3 100644
--- a/assets/js/components/sidebar.vue
+++ b/assets/js/components/sidebar.vue
@@ -1,5 +1,5 @@
 <template>
-    <div :class="[$style.sidebar, 'p-3', 'mb-5']">
+    <div :class="[$style.component, 'p-3', 'mb-5']">
         <h5 class="text-center">
             Categories
         </h5>
@@ -36,7 +36,7 @@ export default {
 <style lang="scss" module>
 @import '../../scss/components/light-component';
 
-.sidebar {
+.component {
     @include light-component;
     margin-top: 65px;
 
