diff --git a/assets/js/components/sidebar.vue b/assets/js/components/sidebar.vue
index c6b9fc7a..134b882a 100644
--- a/assets/js/components/sidebar.vue
+++ b/assets/js/components/sidebar.vue
@@ -1,12 +1,5 @@
 <template>
-    <div
-        :class="{
-            [$style.component]: true,
-            [$style.collapsed]: collapsed,
-            'p-3': true,
-            'mb-5': true
-        }"
-    >
+    <div :class="componentClass">
         <div v-show="!collapsed">
             <h5 class="text-center">
                 Categories
@@ -63,6 +56,22 @@ export default {
             },
         ],
     }),
+    computed: {
+        /**
+         * Computes the component classes depending on collapsed state
+         *
+         * @return string[]
+         */
+        componentClass() {
+            const classes = [this.$style.component, 'p-3', 'mb-5'];
+
+            if (this.collapsed) {
+                classes.push(this.$style.collapsed);
+            }
+
+            return classes;
+        },
+    },
     methods: {
         toggleCollapsed() {
             this.collapsed = !this.collapsed;
