diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index d43a00151..7f06dabb9 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -87,12 +87,8 @@ export default {
         },
     },
     watch: {
-        cart: {
-            deep: true,
-
-            handler() {
-                this.loadProducts();
-            },
+        'cart.items.length': function watchCartItemsLength() {
+            this.loadProducts();
         },
     },
     async created() {
