diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index 027a616c6..d67aec569 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -77,7 +77,8 @@ export default {
             });
 
             return {
-                items: completeItems,
+                // filter out missing products: they may still be loading
+                items: completeItems.filter((item) => item.product),
             };
         },
     },
