diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index c4ddb22b1..83284aee3 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -76,12 +76,8 @@ export default {
         },
     },
     watch: {
-        cart: {
-            deep: true,
-
-            async handler() {
-                this.loadProducts();
-            },
+        'cart.items.length': async function watchCartItemsLength() {
+            this.loadProducts();
         },
     },
     async created() {
