diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index fe1dc9661..b7b763f22 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -80,12 +80,8 @@ export default {
         },
     },
     watch: {
-        cart: {
-            deep: true,
-
-            async handler() {
-                this.loadProducts();
-            },
+        'cart.items.length': async function watchCartItemsLength() {
+            this.loadProducts();
         },
     },
     async created() {
