diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index 057657dd6..151d9cbbb 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -81,7 +81,8 @@ export default {
             });
 
             return {
-                items: completeItems,
+                // filter out missing products: they may still be loading
+                items: completeItems.filter((item) => item.product),
             };
         },
     },
