diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index d67aec569..cb9a58194 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -83,12 +83,8 @@ export default {
         },
     },
     watch: {
-        cart: {
-            deep: true,
-
-            async handler() {
-                this.loadProducts();
-            },
+        'cart.items.length': async function watchCartItemsLength() {
+            this.loadProducts();
         },
     },
     async created() {
