diff --git a/assets/js/components/title.vue b/assets/js/components/title.vue
new file mode 100644
index 00000000..2cdcab40
--- /dev/null
+++ b/assets/js/components/title.vue
@@ -0,0 +1,21 @@
+<template>
+    <div :class="$style.component">
+        <h1>
+            Products
+        </h1>
+    </div>
+</template>
+
+<script>
+export default {
+    name: 'Title',
+};
+</script>
+
+<style lang="scss" module>
+.component {
+    h1 {
+        font-size: 1.7rem;
+    }
+}
+</style>
