diff --git a/assets/js/components/catalog.vue b/assets/js/components/catalog.vue
index 04661cf..198be8c 100644
--- a/assets/js/components/catalog.vue
+++ b/assets/js/components/catalog.vue
@@ -1,11 +1,5 @@
 <template>
     <div>
-        <div class="row">
-            <h2 class="p-3">
-                Products
-            </h2>
-        </div>
-
         <product-list
             :products="products"
             :loading="loading"
