diff --git a/assets/js/products.vue b/assets/js/products.vue
index aada0c7d..828ffd9e 100644
--- a/assets/js/products.vue
+++ b/assets/js/products.vue
@@ -14,6 +14,7 @@
                 <title-component
                     :current-category-id="currentCategoryId"
                     :categories="categories"
+                    :class="$style.title"
                 />
 
                 <catalog-component :current-category-id="currentCategoryId" />
@@ -65,3 +66,9 @@ export default {
     },
 };
 </script>
+
+<style lang="scss" module>
+.title {
+    margin-left: 10px;
+}
+</style>
