diff --git a/assets/js/products.vue b/assets/js/products.vue
index 4685a18..aada0c7 100644
--- a/assets/js/products.vue
+++ b/assets/js/products.vue
@@ -11,6 +11,11 @@
             </aside>
 
             <div :class="contentClass">
+                <title-component
+                    :current-category-id="currentCategoryId"
+                    :categories="categories"
+                />
+
                 <catalog-component :current-category-id="currentCategoryId" />
             </div>
         </div>
@@ -20,12 +25,14 @@
 <script>
 import CatalogComponent from '@/components/catalog';
 import SidebarComponent from '@/components/sidebar';
+import TitleComponent from '@/components/title';
 
 export default {
     name: 'Products',
     components: {
         CatalogComponent,
         SidebarComponent,
+        TitleComponent,
     },
     props: {
         currentCategoryId: {
