diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index 0d1cfbe1c..5f9425222 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -8,16 +8,18 @@
 
                 <loading v-show="loading" />
 
-                <shopping-cart-list
-                    v-if="!loading && currentState === 'cart'"
-                    :items="items"
-                    @updateQuantity="updateQuantity"
-                    @removeFromCart="removeFromCart"
-                />
+                <transition name="fade">
+                    <shopping-cart-list
+                        v-if="!loading && currentState === 'cart'"
+                        :items="items"
+                        @updateQuantity="updateQuantity"
+                        @removeFromCart="removeFromCart"
+                    />
 
-                <checkout-form
-                    v-else
-                />
+                    <checkout-form
+                        v-else
+                    />
+                </transition>
 
                 <div
                     v-show="!loading && items.length"
@@ -130,5 +132,13 @@ export default {
     .content {
         @include light-component;
     }
+
+    .fade-enter-active, .fade-leave-active {
+        transition: opacity .2s;
+    }
+
+    .fade-enter, .fade-leave-to {
+        opacity: 0;
+    }
 }
 </style>
