diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index 488fd9149..e91e1b14e 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -19,11 +19,13 @@
                 >
                     <title-component
                         v-if="currentState === 'cart'"
+                        key="1"
                         text="Shopping Cart"
                     />
 
                     <title-component
                         v-else
+                        key="2"
                         text="Checkout"
                     />
                 </transition>
