diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index a729575c..fa195c25 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -16,15 +16,17 @@
                         <loading />
                     </div>
 
-                    <shopping-cart-list
-                        v-show="!loading && currentState === 1"
-                        :items="items"
-                        :cart="cart"
-                    />
+                    <transition name="fade">
+                        <shopping-cart-list
+                            v-if="!loading && currentState === 1"
+                            :items="items"
+                            :cart="cart"
+                        />
 
-                    <checkout-form
-                        v-show="!loading && currentState === 2"
-                    />
+                        <checkout-form
+                            v-else
+                        />
+                    </transition>
 
                     <div
                         v-show="!loading && items.length"
@@ -116,5 +118,13 @@ export default {
     .content {
         @include light-component;
     }
+
+    .fade-enter-active, .fade-leave-active {
+        transition: opacity .2s;
+    }
+
+    .fade-enter, .fade-leave-to {
+        opacity: 0;
+    }
 }
 </style>
