diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index 959089b5..f97ce1e4 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -17,11 +17,15 @@
                     </div>
 
                     <shopping-cart-list
-                        v-show="!loading"
+                        v-show="!loading && currentState === 1"
                         :items="items"
                         :cart="cart"
                     />
 
+                    <checkout-form
+                        v-show="!loading && currentState === 2"
+                    />
+
                     <div
                         v-show="!loading && items.length"
                         class="row p-3 justify-content-end"
@@ -44,6 +48,7 @@
 <script>
 import { getProductsById } from '@/services/products-service';
 import { getColors } from '@/services/colors-service';
+import CheckoutForm from '@/components/checkout';
 import Loading from '@/components/loading';
 import TitleComponent from '@/components/title';
 import ShoppingCartList from '@/components/shopping-cart';
@@ -52,6 +57,7 @@ import shoppingCartMixin from '@/mixins/get-shopping-cart';
 export default {
     name: 'ShoppingCart',
     components: {
+        CheckoutForm,
         Loading,
         TitleComponent,
         ShoppingCartList,
