diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index 2bd9cad83..c930c476f 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -13,7 +13,20 @@
             </aside>
 
             <div class="col-xs-12 col-lg-9">
-                <title-component :text="pageTitle" />
+                <transition
+                    name="fade"
+                    mode="out-in"
+                >
+                    <title-component
+                        v-if="currentState === 'cart'"
+                        text="Shopping Cart"
+                    />
+
+                    <title-component
+                        v-else
+                        text="Checkout"
+                    />
+                </transition>
 
                 <div class="content p-3">
                     <loading v-show="completeCart === null" />
@@ -98,11 +111,6 @@ export default {
             };
         },
 
-        pageTitle() {
-            return this.currentState === 'cart'
-                ? 'Shopping Cart'
-                : 'Checkout';
-        },
         buttonText() {
             return this.currentState === 'cart'
                 ? 'Check Out >>'
