diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index f39aae43c..eee83bf3d 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -30,6 +30,7 @@
                         <div class="col-2">
                             <button
                                 class="btn btn-info btn-sm"
+                                @click="switchState"
                             >
                                 Checkout!
                             </button>
@@ -58,6 +59,7 @@ export default {
     mixins: [shoppingCartMixin],
     data() {
         return {
+            currentState: 'cart',
             items: [],
             loading: false,
         };
@@ -82,6 +84,10 @@ export default {
         },
     },
     methods: {
+        switchState() {
+            this.currentState = this.currentState === 'cart' ? 'checkout' : 'cart';
+        },
+
         /**
          * Updates the product quantity in the cart, then refreshes the page
          *
