diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index 6b80b8072..0eb76d880 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -4,7 +4,7 @@
             <aside class="col-xs-12 col-lg-3" />
 
             <div class="col-xs-12 col-lg-9">
-                <title-component text="Shopping Cart" />
+                <title-component :text="pageTitle" />
 
                 <div class="content p-3">
                     <loading v-show="completeCart === null" />
@@ -24,7 +24,7 @@
                             class="btn btn-info btn-sm"
                             @click="switchState"
                         >
-                            Checkout!
+                            {{ buttonText }}
                         </button>
                     </div>
                 </div>
@@ -54,6 +54,18 @@ export default {
             completeCart: null,
         };
     },
+    computed: {
+        pageTitle() {
+            return this.currentState === 'cart'
+                ? 'Shopping Cart'
+                : 'Checkout';
+        },
+        buttonText() {
+            return this.currentState === 'cart'
+                ? 'Checkout >>'
+                : '<< Back';
+        },
+    },
     watch: {
         'cart.items.length': async function watchCartItemsLength() {
             this.completeCart = await fetchFullCart(this.cart);
