diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index 4004a4224..3c0ffeb74 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -17,7 +17,12 @@
             </aside>
 
             <div class="col-xs-12 col-lg-9">
-                <title-component :text="pageTitle" />
+                <transition
+                    name="fade"
+                    mode="out-in"
+                >
+                    <title-component :text="pageTitle" />
+                </transition>
 
                 <div class="content p-3">
                     <loading v-show="completeCart === null" />
