diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index 96db931e6..b2264bca3 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -47,12 +47,14 @@
                         </button>
                     </div>
 
-                    <div
-                        v-show="currentState === 'cart'"
-                        class="transition-testing"
-                    >
-                        Testing transitions!
-                    </div>
+                    <transition>
+                        <div
+                            v-show="currentState === 'cart'"
+                            class="transition-testing"
+                        >
+                            Testing transitions!
+                        </div>
+                    </transition>
                 </div>
             </div>
         </div>
