diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index 9b65e8072..e6e86840a 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -20,7 +20,10 @@
                         v-if="completeCart && completeCart.items.length > 0"
                         class="text-right"
                     >
-                        <button class="btn btn-primary xl">
+                        <button
+                            class="btn btn-info btn-sm"
+                            @click="switchState"
+                        >
                             Checkout!
                         </button>
                     </div>
@@ -48,6 +51,7 @@ export default {
     mixins: [ShoppingCartMixin],
     data() {
         return {
+            currentState: 'cart',
             products: null,
             colors: null,
         };
@@ -77,6 +81,10 @@ export default {
         },
     },
     methods: {
+        switchState() {
+            this.currentState = this.currentState === 'cart' ? 'checkout' : 'cart';
+        },
+
         async loadColorsAndProducts() {
             const productIds = this.cart.items.map((item) => item.product);
 
