diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index 232ad4504..ce8a77cb0 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -13,7 +13,7 @@
             </aside>
 
             <div class="col-xs-12 col-lg-9">
-                <title-component text="Shopping Cart" />
+                <title-component :text="pageTitle" />
 
                 <div class="content p-3">
                     <loading v-show="completeCart === null" />
@@ -32,7 +32,7 @@
                             class="btn btn-primary"
                             @click="switchState"
                         >
-                            Check Out!
+                            {{ buttonText }}
                         </button>
                     </div>
                 </div>
@@ -90,6 +90,17 @@ export default {
                 items: completeItems.filter((item) => item.product),
             };
         },
+
+        pageTitle() {
+            return this.currentState === 'cart'
+                ? 'Shopping Cart'
+                : 'Checkout';
+        },
+        buttonText() {
+            return this.currentState === 'cart'
+                ? 'Check Out >>'
+                : '<< Back';
+        },
     },
     watch: {
         'cart.items.length': async function watchCartItemsLength() {
