diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index 30e37a5f..fd52b6a9 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -5,7 +5,7 @@
 
             <div class="col-xs-12 col-lg-9">
                 <div class="pb-3">
-                    <title-component text="Shopping Cart" />
+                    <title-component :text="pageTitle" />
                 </div>
 
                 <div class="content">
@@ -32,7 +32,7 @@
                                 class="btn btn-info btn-sm"
                                 @click="switchState"
                             >
-                                Checkout!
+                                {{ buttonText }}
                             </button>
                         </div>
                     </div>
@@ -65,6 +65,18 @@ export default {
             loading: false,
         };
     },
+    computed: {
+        pageTitle() {
+            return this.currentState === 1
+                ? 'Shopping Cart'
+                : 'Checkout';
+        },
+        buttonText() {
+            return this.currentState === 1
+                ? 'Checkout >>'
+                : '<< Back';
+        },
+    },
     watch: {
         'cart.items.length': async function watchCartItemsLength() {
             this.loading = true;
