diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index 227f26f4..d9fe00f2 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -29,6 +29,7 @@
                         <div class="col-2">
                             <button
                                 class="btn btn-info btn-sm"
+                                @click="switchState"
                             >
                                 Checkout!
                             </button>
@@ -58,6 +59,7 @@ export default {
     mixins: [shoppingCartMixin],
     data() {
         return {
+            currentState: 1,
             colors: {},
             items: [],
             loading: false,
@@ -114,6 +116,11 @@ export default {
             this.loading = false;
         },
     },
+    methods: {
+        switchState() {
+            this.currentState = 3 - this.currentState;
+        },
+    },
 };
 </script>
 
