diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index 8a6a8b475..6b80b8072 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -20,7 +20,10 @@
                         v-if="completeCart && completeCart.items.length > 0"
                         class="text-right"
                     >
-                        <button class="btn btn-primary xl">
+                        <button
+                            class="btn btn-info btn-sm"
+                            @click="switchState"
+                        >
                             Checkout!
                         </button>
                     </div>
@@ -47,6 +50,7 @@ export default {
     mixins: [ShoppingCartMixin],
     data() {
         return {
+            currentState: 'cart',
             completeCart: null,
         };
     },
@@ -56,6 +60,10 @@ export default {
         },
     },
     methods: {
+        switchState() {
+            this.currentState = this.currentState === 'cart' ? 'checkout' : 'cart';
+        },
+
         updateQuantity({ productId, colorId, quantity }) {
             this.updateProductQuantity(productId, colorId, quantity);
         },
