diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index d26710fe9..57a97f79a 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -14,6 +14,19 @@
                     @updateQuantity="updateQuantity"
                     @removeFromCart="removeFromCart"
                 />
+
+                <div
+                    v-show="!loading && items.length"
+                    class="row p-3 justify-content-end"
+                >
+                    <div class="col-2">
+                        <button
+                            class="btn btn-info btn-sm"
+                        >
+                            Checkout!
+                        </button>
+                    </div>
+                </div>
             </div>
         </div>
     </div>
