diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index aedf7109a..4e1673e19 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -28,7 +28,10 @@
                     <div
                         v-if="completeCart && completeCart.items.length > 0"
                     >
-                        <button class="btn btn-primary">
+                        <button
+                            class="btn btn-primary"
+                            @click="switchState"
+                        >
                             Check Out!
                         </button>
                     </div>
@@ -58,6 +61,7 @@ export default {
     mixins: [ShoppingCartMixin],
     data() {
         return {
+            currentState: 'cart',
             products: null,
             colors: null,
             featuredProduct: null,
@@ -92,6 +96,10 @@ export default {
         this.loadFeaturedProduct();
     },
     methods: {
+        switchState() {
+            this.currentState = this.currentState === 'cart' ? 'checkout' : 'cart';
+        },
+
         async loadColorsAndProducts() {
             const productIds = this.cart.items.map((item) => item.product);
 
