diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index 49d8bec4b..5820c84ca 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -4,7 +4,22 @@
             <aside class="col-xs-12 col-lg-3" />
 
             <div class="col-xs-12 col-lg-9">
-                <title-component :text="pageTitle" />
+                <transition
+                    name="fade"
+                    mode="out-in"
+                >
+                    <title-component
+                        v-if="currentState === 'cart'"
+                        key="1"
+                        text="Shopping Cart"
+                    />
+
+                    <title-component
+                        v-else
+                        key="2"
+                        text="Checkout"
+                    />
+                </transition>
 
                 <loading v-show="loading" />
 
@@ -66,11 +81,6 @@ export default {
         };
     },
     computed: {
-        pageTitle() {
-            return this.currentState === 'cart'
-                ? 'Shopping Cart'
-                : 'Checkout';
-        },
         buttonText() {
             return this.currentState === 'cart'
                 ? 'Checkout >>'
