diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index 27c0242f..30e37a5f 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -30,6 +30,7 @@
                         <div class="col-2">
                             <button
                                 class="btn btn-info btn-sm"
+                                @click="switchState"
                             >
                                 Checkout!
                             </button>
@@ -58,6 +59,7 @@ export default {
     mixins: [shoppingCartMixin],
     data() {
         return {
+            currentState: 1,
             colors: {},
             items: [],
             loading: false,
@@ -83,6 +85,10 @@ export default {
         },
     },
     methods: {
+        switchState() {
+            this.currentState = 3 - this.currentState;
+        },
+
         /**
          * Updates the product quantity in the cart, then refreshes the page
          *
