diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index 52ed70d9a..17b6a20e5 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -9,16 +9,18 @@
                 <div class="content p-3">
                     <loading v-show="completeCart === null" />
 
-                    <shopping-cart-list
-                        v-if="completeCart && currentState === 'cart'"
-                        :items="completeCart.items"
-                        @updateQuantity="updateQuantity"
-                        @removeFromCart="removeFromCart"
-                    />
+                    <transition name="fade">
+                        <shopping-cart-list
+                            v-if="completeCart && currentState === 'cart'"
+                            :items="completeCart.items"
+                            @updateQuantity="updateQuantity"
+                            @removeFromCart="removeFromCart"
+                        />
 
-                    <checkout-form
-                        v-if="completeCart && currentState === 'checkout'"
-                    />
+                        <checkout-form
+                            v-if="completeCart && currentState === 'checkout'"
+                        />
+                    </transition>
 
                     <div
                         v-if="completeCart && completeCart.items.length > 0"
@@ -100,5 +102,13 @@ export default {
     .content {
         @include light-component;
     }
+
+    .fade-enter-active, .fade-leave-active {
+        transition: opacity .2s;
+    }
+
+    .fade-enter, .fade-leave-to {
+        opacity: 0;
+    }
 }
 </style>
