diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index 41c85643f..49d8bec4b 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -8,7 +8,10 @@
 
                 <loading v-show="loading" />
 
-                <transition name="fade">
+                <transition
+                    name="fade"
+                    mode="out-in"
+                >
                     <shopping-cart-list
                         v-if="!loading && currentState === 'cart'"
                         :items="items"
