diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index 7f06dabb9..469564ab3 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -31,6 +31,14 @@
                             $event.colorId,
                         )"
                     />
+
+                    <div
+                        v-if="completeCart && completeCart.items.length > 0"
+                    >
+                        <button class="btn btn-primary">
+                            Check Out!
+                        </button>
+                    </div>
                 </div>
             </div>
         </div>
