diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index d3c2f1990..d6e9e7d0f 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -4,7 +4,7 @@
             <aside class="col-xs-12 col-lg-3" />
 
             <div class="col-xs-12 col-lg-9">
-                <title-component text="Shopping Cart" />
+                <title-component :text="pageTitle" />
 
                 <loading v-show="loading" />
 
@@ -24,7 +24,7 @@
                             class="btn btn-info btn-sm"
                             @click="switchState"
                         >
-                            Checkout!
+                            {{ buttonText }}
                         </button>
                     </div>
                 </div>
@@ -59,6 +59,16 @@ export default {
         isLoading() {
             return this.itemsLoading || this.cart === null;
         },
+        pageTitle() {
+            return this.currentState === 'cart'
+                ? 'Shopping Cart'
+                : 'Checkout';
+        },
+        buttonText() {
+            return this.currentState === 'cart'
+                ? 'Checkout >>'
+                : '<< Back';
+        },
     },
     watch: {
         'cart.items.length': async function watchCartItemsLength() {
