diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index 3c2759f1e..4004a4224 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -22,7 +22,10 @@
                 <div class="content p-3">
                     <loading v-show="completeCart === null" />
 
-                    <transition name="fade">
+                    <transition
+                        name="fade"
+                        mode="out-in"
+                    >
                         <shopping-cart-list
                             v-if="completeCart && currentState === 'cart'"
                             :items="completeCart.items"
