diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index 7916086dc..8e59faeb9 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -4,7 +4,13 @@
             <aside class="col-xs-12 col-lg-3" />
 
             <div class="col-xs-12 col-lg-9">
+<<<<<<< HEAD
+                <div class="pb-3">
+                    <title-component :text="pageTitle" />
+                </div>
+=======
                 <title-component text="Shopping Cart" />
+>>>>>>> _tuts/master/transitions-implement-step-switch
 
                 <loading v-show="loading" />
 
@@ -24,7 +30,7 @@
                             class="btn btn-info btn-sm"
                             @click="switchState"
                         >
-                            Checkout!
+                            {{ buttonText }}
                         </button>
                     </div>
                 </div>
@@ -55,6 +61,18 @@ export default {
             loading: false,
         };
     },
+    computed: {
+        pageTitle() {
+            return this.currentState === 'cart'
+                ? 'Shopping Cart'
+                : 'Checkout';
+        },
+        buttonText() {
+            return this.currentState === 'cart'
+                ? 'Checkout >>'
+                : '<< Back';
+        },
+    },
     watch: {
         'cart.items.length': async function watchCartItemsLength() {
             this.loading = true;
