diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index 7c9fcf2e..ba317d15 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -16,7 +16,10 @@
                         <loading />
                     </div>
 
-                    <transition name="fade">
+                    <transition
+                        name="fade"
+                        mode="out-in"
+                    >
                         <shopping-cart-list
                             v-if="!loading && currentState === 1"
                             :items="items"
