diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index 9e915eecd..41182ae69 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -29,7 +29,10 @@
                         v-if="completeCart && completeCart.items.length > 0"
                         class="text-right"
                     >
-                        <button class="btn btn-primary xl">
+                        <button
+                            class="btn btn-info btn-sm"
+                            @click="switchState"
+                        >
                             Checkout!
                         </button>
                     </div>
@@ -59,6 +62,7 @@ export default {
     mixins: [ShoppingCartMixin],
     data() {
         return {
+            currentState: 'cart',
             products: null,
             colors: null,
             featuredProduct: null,
@@ -93,6 +97,10 @@ export default {
         this.loadFeaturedProduct();
     },
     methods: {
+        switchState() {
+            this.currentState = this.currentState === 'cart' ? 'checkout' : 'cart';
+        },
+
         async loadColorsAndProducts() {
             const productIds = this.cart.items.map((item) => item.product);
 
