diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index 85e113cdc..1bec5ac91 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -16,16 +16,18 @@
                         <loading />
                     </div>
 
-                    <shopping-cart-list
-                        v-if="!loading && currentState === 'cart'"
-                        :items="items"
-                        @updateQuantity="updateQuantity"
-                        @removeFromCart="removeFromCart"
-                    />
-
-                    <checkout-form
-                        v-else
-                    />
+                    <transition name="fade">
+                        <shopping-cart-list
+                            v-if="!loading && currentState === 'cart'"
+                            :items="items"
+                            @updateQuantity="updateQuantity"
+                            @removeFromCart="removeFromCart"
+                        />
+
+                        <checkout-form
+                            v-else
+                        />
+                    </transition>
 
                     <div
                         v-show="!loading && items.length"
@@ -141,5 +143,13 @@ export default {
     .content {
         @include light-component;
     }
+
+    .fade-enter-active, .fade-leave-active {
+        transition: opacity .2s;
+    }
+
+    .fade-enter, .fade-leave-to {
+        opacity: 0;
+    }
 }
 </style>
