diff --git a/assets/js/components/checkout/index.vue b/assets/js/components/checkout/index.vue
new file mode 100644
index 00000000..8aa32543
--- /dev/null
+++ b/assets/js/components/checkout/index.vue
@@ -0,0 +1,13 @@
+<template>
+    <div class="row p-3">
+        <div class="col-12">
+            A cool checkout form will appear right here!
+        </div>
+    </div>
+</template>
+
+<script>
+export default {
+    name: 'CheckoutForm',
+};
+</script>
