diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index 83284aee3..a30640700 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -24,6 +24,14 @@
                         @updateQuantity="updateQuantity"
                         @removeFromCart="removeFromCart"
                     />
+
+                    <div
+                        v-if="completeCart && completeCart.items.length > 0"
+                    >
+                        <button class="btn btn-primary">
+                            Check Out!
+                        </button>
+                    </div>
                 </div>
             </div>
         </div>
