diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index cb9a58194..df720d8af 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -27,6 +27,14 @@
                             $event.colorId,
                         )"
                     />
+
+                    <div
+                        v-if="completeCart && completeCart.items.length > 0"
+                    >
+                        <button class="btn btn-primary">
+                            Check Out!
+                        </button>
+                    </div>
                 </div>
             </div>
         </div>
