diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index df720d8af..9faa949ce 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -31,7 +31,10 @@
                     <div
                         v-if="completeCart && completeCart.items.length > 0"
                     >
-                        <button class="btn btn-primary">
+                        <button
+                            class="btn btn-primary"
+                            @click="switchState"
+                        >
                             Check Out!
                         </button>
                     </div>
@@ -61,6 +64,7 @@ export default {
     mixins: [ShoppingCartMixin],
     data() {
         return {
+            currentState: 'cart',
             products: null,
             colors: null,
             featuredProduct: null,
@@ -100,6 +104,9 @@ export default {
         this.colors = (await fetchColors()).data['hydra:member'];
     },
     methods: {
+        switchState() {
+            this.currentState = this.currentState === 'cart' ? 'checkout' : 'cart';
+        },
         async loadProducts() {
             const productIds = this.cart.items.map((item) => item.product);
 
