diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index 24a0565b1..51d7c4c68 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -9,7 +9,7 @@
                 <div class="content p-3">
                     <loading v-show="completeCart === null" />
 
-                    <transition name="fade">
+                    <transition name="fade" mode="out-in">
                         <shopping-cart-list
                             v-if="completeCart && currentState === 'cart'"
                             :items="completeCart.items"
