diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index d6e9e7d0f..0d1cfbe1c 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -9,12 +9,16 @@
                 <loading v-show="loading" />
 
                 <shopping-cart-list
-                    v-show="!loading"
+                    v-if="!loading && currentState === 'cart'"
                     :items="items"
                     @updateQuantity="updateQuantity"
                     @removeFromCart="removeFromCart"
                 />
 
+                <checkout-form
+                    v-else
+                />
+
                 <div
                     v-show="!loading && items.length"
                     class="row p-3 justify-content-end"
@@ -36,6 +40,7 @@
 <script>
 import { getFullShoppingCart, updateCartItemQuantity, removeItemFromCart } from '@/services/cart-service';
 import ShoppingCartMixin from '@/mixins/get-shopping-cart';
+import CheckoutForm from '@/components/checkout';
 import TitleComponent from '@/components/title';
 import ShoppingCartList from '@/components/shopping-cart';
 import Loading from '@/components/loading';
@@ -43,6 +48,7 @@ import Loading from '@/components/loading';
 export default {
     name: 'ShoppingCart',
     components: {
+        CheckoutForm,
         Loading,
         ShoppingCartList,
         TitleComponent,
