diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index e2531b64b..8a6a8b475 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -15,6 +15,15 @@
                         @updateQuantity="updateQuantity"
                         @removeFromCart="removeFromCart"
                     />
+
+                    <div
+                        v-if="completeCart && completeCart.items.length > 0"
+                        class="text-right"
+                    >
+                        <button class="btn btn-primary xl">
+                            Checkout!
+                        </button>
+                    </div>
                 </div>
             </div>
         </div>
