diff --git a/assets/pages/shopping-cart.vue b/assets/pages/shopping-cart.vue
index 2a2c08ce0..3f5325563 100644
--- a/assets/pages/shopping-cart.vue
+++ b/assets/pages/shopping-cart.vue
@@ -19,12 +19,16 @@
                     <loading v-show="completeCart === null" />
 
                     <shopping-cart-list
-                        v-if="completeCart"
+                        v-if="completeCart && currentState === 'cart'"
                         :items="completeCart.items"
                         @updateQuantity="updateQuantity"
                         @removeFromCart="removeFromCart"
                     />
 
+                    <checkout-form
+                        v-if="completeCart && currentState === 'checkout'"
+                    />
+
                     <div
                         v-if="completeCart && completeCart.items.length > 0"
                         class="text-right"
@@ -46,6 +50,7 @@
 import { fetchColors } from '@/services/colors-service';
 import { fetchFeaturedProducts, fetchProductsById } from '@/services/products-service';
 import ShoppingCartMixin from '@/mixins/get-shopping-cart';
+import CheckoutForm from '@/components/checkout';
 import TitleComponent from '@/components/title';
 import ShoppingCartList from '@/components/shopping-cart';
 import Loading from '@/components/loading';
@@ -54,6 +59,7 @@ import CartSidebar from '@/components/shopping-cart/cart-sidebar';
 export default {
     name: 'ShoppingCart',
     components: {
+        CheckoutForm,
         Loading,
         ShoppingCartList,
         TitleComponent,
