diff --git a/assets/js/pages/shopping-cart.vue b/assets/js/pages/shopping-cart.vue
index 69d43508..227f26f4 100644
--- a/assets/js/pages/shopping-cart.vue
+++ b/assets/js/pages/shopping-cart.vue
@@ -21,6 +21,19 @@
                         :items="items"
                         :cart="cart"
                     />
+
+                    <div
+                        v-show="!loading && items.length"
+                        class="row p-3 justify-content-end"
+                    >
+                        <div class="col-2">
+                            <button
+                                class="btn btn-info btn-sm"
+                            >
+                                Checkout!
+                            </button>
+                        </div>
+                    </div>
                 </div>
             </div>
         </div>
