diff --git a/assets/components/checkout/form-input.vue b/assets/components/checkout/form-input.vue
index 40fbf317..36ecf88b 100644
--- a/assets/components/checkout/form-input.vue
+++ b/assets/components/checkout/form-input.vue
@@ -10,7 +10,7 @@
             :id="id"
             :name="id"
             :type="type"
-            :value="value"
+            :value="modelValue"
             :class="{
                 'is-invalid': !isValid,
                 'form-control': true,
@@ -47,7 +47,7 @@ export default {
             type: String,
             default: 'text',
         },
-        value: {
+        modelValue: {
             type: String,
             default: '',
         },
