diff --git a/assets/products.js b/assets/products.js
index 790dbc03..40a2b3d8 100644
--- a/assets/products.js
+++ b/assets/products.js
@@ -1,6 +1,4 @@
-import Vue from 'vue';
+import { createApp } from 'vue';
 import App from '@/pages/products';
 
-new Vue({
-    render: (h) => h(App),
-}).$mount('#app');
+createApp(App).mount('#app');
diff --git a/assets/shopping-cart.js b/assets/shopping-cart.js
index c59a8fa7..b526f8fe 100644
--- a/assets/shopping-cart.js
+++ b/assets/shopping-cart.js
@@ -1,6 +1,4 @@
-import Vue from 'vue';
+import { createApp } from 'vue';
 import App from '@/pages/shopping-cart.vue';
 
-new Vue({
-    render: (h) => h(App),
-}).$mount('#app');
+createApp(App).mount('#app');
