diff --git a/assets/components/checkout/form-input.vue b/assets/components/checkout/form-input.vue
index 36ecf88b..1098ed27 100644
--- a/assets/components/checkout/form-input.vue
+++ b/assets/components/checkout/form-input.vue
@@ -15,7 +15,7 @@
                 'is-invalid': !isValid,
                 'form-control': true,
             }"
-            @input="$emit('input', $event.target.value)"
+            @input="$emit('update:modelValue', $event.target.value)"
             @blur="$emit('blur', $event)"
         >
         <span
