diff --git a/assets/js/components/checkout/form-input.vue b/assets/js/components/checkout/form-input.vue
index 1f8d5f2d..f0753db7 100644
--- a/assets/js/components/checkout/form-input.vue
+++ b/assets/js/components/checkout/form-input.vue
@@ -14,7 +14,7 @@
                 'form-control': true,
             }"
             :value="modelValue"
-            @input="$emit('input', $event.target.value)"
+            @input="$emit('update:modelValue', $event.target.value)"
             @blur="$emit('blur', $event)"
         >
         <span
