diff --git a/assets/js/components/checkout/form-input.vue b/assets/js/components/checkout/form-input.vue
index f8b29d00..1f8d5f2d 100644
--- a/assets/js/components/checkout/form-input.vue
+++ b/assets/js/components/checkout/form-input.vue
@@ -13,7 +13,7 @@
                 'is-invalid': !isValid,
                 'form-control': true,
             }"
-            :value="value"
+            :value="modelValue"
             @input="$emit('input', $event.target.value)"
             @blur="$emit('blur', $event)"
         >
@@ -46,7 +46,7 @@ export default {
             type: String,
             default: 'text',
         },
-        value: {
+        modelValue: {
             type: String,
             default: '',
         },
