diff --git a/assets/js/components/search-bar.vue b/assets/js/components/search-bar.vue
index dde0a0c..17e36fd 100644
--- a/assets/js/components/search-bar.vue
+++ b/assets/js/components/search-bar.vue
@@ -44,5 +44,18 @@ export default {
 
 .component {
     width: 400px;
+
+    .erase {
+        color: $red-component-border;
+        cursor: pointer;
+        display: block;
+        font-weight: bold;
+        height: 20px;
+        left: 372px;
+        margin: -32px 0 0 0;
+        position: relative;
+        text-align: center;
+        width: 20px;
+    }
 }
 </style>
