diff --git a/assets/js/components/search-bar.vue b/assets/js/components/search-bar.vue
index e07e7352..b7376565 100644
--- a/assets/js/components/search-bar.vue
+++ b/assets/js/components/search-bar.vue
@@ -11,6 +11,7 @@
         <div
             class="input-group-append"
             v-show="searchTerm !== ''"
+            @click="eraseSearchTerm"
         >
             <button class="btn btn-outline-secondary">
                 X
@@ -40,5 +41,13 @@ export default {
             }, 200);
         },
     },
+    methods: {
+        /**
+         * Resets the search term
+         */
+        eraseSearchTerm() {
+            this.searchTerm = '';
+        },
+    },
 };
 </script>
