diff --git a/assets/js/components/search-bar.vue b/assets/js/components/search-bar.vue
index 97398e37..a3a4d264 100644
--- a/assets/js/components/search-bar.vue
+++ b/assets/js/components/search-bar.vue
@@ -12,6 +12,7 @@
         <div
             class="input-group-append"
             v-show="searchTerm !== ''"
+            @click="eraseSearchTerm"
         >
             <button class="btn btn-outline-secondary">
                 X
@@ -40,6 +41,12 @@ export default {
                 this.$emit('search-products', { term: this.searchTerm });
             }, 200);
         },
+        /**
+         * Resets the search term
+         */
+        eraseSearchTerm() {
+            this.searchTerm = '';
+        },
     },
 };
 </script>
