diff --git a/assets/js/components/search-bar.vue b/assets/js/components/search-bar.vue
index 70c478fe..97398e37 100644
--- a/assets/js/components/search-bar.vue
+++ b/assets/js/components/search-bar.vue
@@ -1,5 +1,6 @@
 <template>
-    <div>
+    <div class="input-group">
+
         <input
             v-model="searchTerm"
             @input="onInput"
@@ -7,6 +8,15 @@
             placeholder="Search products..."
             type="search"
         >
+
+        <div
+            class="input-group-append"
+            v-show="searchTerm !== ''"
+        >
+            <button class="btn btn-outline-secondary">
+                X
+            </button>
+        </div>
     </div>
 </template>
 
