diff --git a/assets/js/components/search-bar.vue b/assets/js/components/search-bar.vue
index ecc61c85..4c46db25 100644
--- a/assets/js/components/search-bar.vue
+++ b/assets/js/components/search-bar.vue
@@ -15,6 +15,7 @@
         >
             <button
                 class="btn btn-outline-secondary"
+                @click="eraseSearchTerm"
             >
                 X
             </button>
@@ -42,6 +43,9 @@ export default {
                 this.searchTimeout = null;
             }, 200);
         },
+        eraseSearchTerm() {
+            this.searchTerm = '';
+        },
     },
 };
 </script>
