diff --git a/assets/js/components/search-bar.vue b/assets/js/components/search-bar.vue
index 4c59a53c..cd915278 100644
--- a/assets/js/components/search-bar.vue
+++ b/assets/js/components/search-bar.vue
@@ -25,26 +25,28 @@ export default {
         searchTerm: '',
         searchTimeout: null,
     }),
-    watch: {
-        searchTerm(newTerm) {
+    methods: {
+        /**
+         * Resets the search term
+         */
+        eraseSearchTerm() {
+            this.searchTerm = '';
+        },
+
+        /**
+         * Handles input changes
+         */
+        onInput() {
             if (this.searchTimeout !== null) {
                 window.clearTimeout(this.searchTimeout);
                 this.searchTimeout = null;
             }
 
             this.searchTimeout = window.setTimeout(() => {
-                this.$emit('search-products', { term: newTerm });
+                this.$emit('search-products', { term: this.searchTerm });
             }, 200);
         },
     },
-    methods: {
-        /**
-         * Resets the search term
-         */
-        eraseSearchTerm() {
-            this.searchTerm = '';
-        },
-    },
 };
 </script>
 
