diff --git a/assets/js/components/search-bar.vue b/assets/js/components/search-bar.vue
index 69279864..cc36e2fa 100644
--- a/assets/js/components/search-bar.vue
+++ b/assets/js/components/search-bar.vue
@@ -36,6 +36,7 @@ export default {
          */
         eraseSearchTerm() {
             this.searchTerm = '';
+            this.$emit('search-products', { term: this.searchTerm });
         },
 
         /**
