diff --git a/assets/js/components/search-bar.vue b/assets/js/components/search-bar.vue
index c8d141c..8a59555 100644
--- a/assets/js/components/search-bar.vue
+++ b/assets/js/components/search-bar.vue
@@ -10,6 +10,7 @@
         <span
             v-show="searchTerm !== ''"
             :class="$style.erase"
+            @click="eraseSearchTerm"
         >
             X
         </span>
@@ -35,6 +36,14 @@ export default {
             }, 200);
         },
     },
+    methods: {
+        /**
+         * Resets the search term
+         */
+        eraseSearchTerm() {
+            this.searchTerm = '';
+        },
+    },
 };
 </script>
 
