diff --git a/assets/js/components/search-bar.vue b/assets/js/components/search-bar.vue
index bcb046b9..96bbc8fc 100644
--- a/assets/js/components/search-bar.vue
+++ b/assets/js/components/search-bar.vue
@@ -2,6 +2,7 @@
     <div>
         <input
             :value="searchTerm"
+            @input="searchTerm = $event.target.value"
             class="form-control"
             placeholder="Search products..."
             type="search"
