diff --git a/assets/js/components/search-bar.vue b/assets/js/components/search-bar.vue
index bcb046b9f..bd5bb05e6 100644
--- a/assets/js/components/search-bar.vue
+++ b/assets/js/components/search-bar.vue
@@ -5,6 +5,7 @@
             class="form-control"
             placeholder="Search products..."
             type="search"
+            @input="searchTerm = $event.target.value"
         >
     </div>
 </template>
