diff --git a/assets/js/components/search-bar.vue b/assets/js/components/search-bar.vue
index af10e4b0..bcb046b9 100644
--- a/assets/js/components/search-bar.vue
+++ b/assets/js/components/search-bar.vue
@@ -1,6 +1,7 @@
 <template>
     <div>
         <input
+            :value="searchTerm"
             class="form-control"
             placeholder="Search products..."
             type="search"
@@ -11,5 +12,10 @@
 <script>
 export default {
     name: 'SearchBar',
+    data() {
+        return {
+            searchTerm: '',
+        };
+    },
 };
 </script>
